--- distinfo.orig	2013-01-05 21:50:00.276809000 +0100
+++ distinfo	2013-01-05 21:38:52.435876000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_sat.tar.gz) = 4f23d2f583e0ad927a90a158e07f3553cc167fdde613ba8bda86b536a15e1c17
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_sat.tar.gz) = 1677255
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_sat.tar.gz) = 7d6e752b83025ba7fd3f6c5d1470b43f13967b4bfe52b48806f7e59ee78f7ca7
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_sat.tar.gz) = 1624568
