--- src/cjktexsty/cjktexsty.l.orig	2012-07-14 12:56:14.000000000 +0000
+++ src/cjktexsty/cjktexsty.l
@@ -1,4 +1,5 @@
 %{
+#define __FBSDID(s)	struct __hack
 /*-
  * Copyright (c) 2005, 2006 intron <intron@intron.ac>.  All rights reserved.
  * Copyright (c) 2005, 2006 The FreeBSD Simplified Chinese Project.
@@ -56,7 +57,7 @@ void
 transcode(char *ch)
 {
 	char *pchar,*pout,input[16],output[128];
-	const char *pin;
+	char *pin;
 	int c;
 	size_t lin,lout;
 	size_t outlen;
