--- bsd.cabal.mk.orig	2017-08-20 18:52:37 UTC
+++ bsd.cabal.mk
@@ -58,7 +58,7 @@ CABAL_ARCH=		x86_64
 .if ("${ARCH}" == "i386")
 CABAL_ARCH=		i386
 .endif
-CABAL_ARCHSUBDIR=	${CABAL_ARCH}-freebsd-ghc-${GHC_VERSION}
+CABAL_ARCHSUBDIR=	${CABAL_ARCH}-dragonfly-ghc-${GHC_VERSION}
 CABAL_LIBDIR_REL=	${CABAL_LIBDIR:S,^${PREFIX}/,,}
 
 CONFIGURE_ARGS+=	--libdir=${CABAL_LIBDIR} --libsubdir=${CABAL_LIBSUBDIR}
