--- bsd.cabal.mk.orig	2019-04-26 22:52:53 UTC
+++ bsd.cabal.mk
@@ -55,7 +55,7 @@ GHC_LIB_DOCSDIR_REL=	share/doc/ghc-${GHC
 CABAL_LIBDIR=		${PREFIX}/lib/cabal/ghc-${GHC_VERSION}
 CABAL_LIBSUBDIR=	${PACKAGE}
 CABAL_ARCH=		${ARCH:S/amd64/x86_64/:C/armv.*/arm/:S/powerpc64/ppc64/}
-CABAL_ARCHSUBDIR=	${CABAL_ARCH}-freebsd-ghc-${GHC_VERSION}
+CABAL_ARCHSUBDIR=	${CABAL_ARCH}-dragonfly-ghc-${GHC_VERSION}
 CABAL_LIBDIR_REL=	${CABAL_LIBDIR:S,^${PREFIX}/,,}
 
 CONFIGURE_ARGS+=	--libdir=${CABAL_LIBDIR} --libsubdir=${CABAL_LIBSUBDIR}
