--- distinfo.orig	2016-01-29 19:53:56 UTC
+++ distinfo
@@ -6,3 +6,5 @@ SHA256 (ghc-7.10.2-boot-i386-freebsd.tar
 SIZE (ghc-7.10.2-boot-i386-freebsd.tar.xz) = 39858716
 SHA256 (ghc-7.10.2-boot-amd64-freebsd.tar.xz) = 4af0ee0795537baa11a96fba23477f8390ccb5f45910cda119d0a16bdebaedc8
 SIZE (ghc-7.10.2-boot-amd64-freebsd.tar.xz) = 49833224
+SHA256 (ghc-8.0.2-boot-x86_64-dragonfly.tar.xz) = b20d7c0606b0fcff9036eae58d4ea3c17d89b38c2bdb1fdd19651537b432141a
+SIZE (ghc-8.0.2-boot-x86_64-dragonfly.tar.xz) = 73545008
