--- distinfo.orig	2022-11-04 00:05:56 UTC
+++ distinfo
@@ -17,3 +17,5 @@ SHA256 (ghc-8.10.7-boot-armv7-freebsd.ta
 SIZE (ghc-8.10.7-boot-armv7-freebsd.tar.xz) = 127945844
 SHA256 (ghc-8.10.7-boot-armv6-freebsd.tar.xz) = 84010e11f92e0a9fe106b80cbaf027ba342da055c3ad6d89eab0a389c0523e1b
 SIZE (ghc-8.10.7-boot-armv6-freebsd.tar.xz) = 129190568
+SHA256 (ghc-8.10.7-boot-x86_64-dragonfly.tar.xz) = 3970131a4c129135ffa2e16e2abe773cd63df2540c740278a87e4e943e110169
+SIZE (ghc-8.10.7-boot-x86_64-dragonfly.tar.xz) = 154596776
