--- distinfo.orig	2014-08-20 19:09:02.921900000 +0000
+++ distinfo
@@ -14,3 +14,7 @@ SHA256 (ghc-7.6.3-i386-freebsd10-boot.ta
 SIZE (ghc-7.6.3-i386-freebsd10-boot.tar.bz2) = 55512004
 SHA256 (ghc-7.6.3-amd64-freebsd10-boot.tar.bz2) = 8ce856cbd5b9fa230d17bc3d28b96206d84dce4aa3c1c9c7016262ff132271ad
 SIZE (ghc-7.6.3-amd64-freebsd10-boot.tar.bz2) = 55418435
+SHA256 (ghc-7.6.3-i386-dragonfly-boot.tar.bz2) = 711f21c976be246c06557bbb5e669dce35b524329b4c907508b4157a3ec5710a
+SIZE (ghc-7.6.3-i386-dragonfly-boot.tar.bz2) = 65258530
+SHA256 (ghc-7.6.3-x86_64-dragonfly-boot.tar.bz2) = ab6c6d899eb388f852ecc31ab6e5e1f993d5140dccbd820e70b615403132b7c5
+SIZE (ghc-7.6.3-x86_64-dragonfly-boot.tar.bz2) = 63406815
