--- distinfo.orig	2020-09-06 23:00:08 UTC
+++ distinfo
@@ -5,6 +5,8 @@ SHA256 (ghc-8.6.5-boot-amd64-freebsd.tar
 SIZE (ghc-8.6.5-boot-amd64-freebsd.tar.xz) = 70213260
 SHA256 (ghc-8.6.5-boot-i386-freebsd.tar.xz) = 47cc24cdcfa19601bb81a7f5660d1d72fec1110119e8ffe1daa0cddc7959206f
 SIZE (ghc-8.6.5-boot-i386-freebsd.tar.xz) = 67522080
+SHA256 (ghc-8.8.4-boot-x86_64-dragonfly.tar.xz) = 84e197185ca7d7150c2e5019814dc4b7a6894f8008164a256f4d8acb27ced3fe
+SIZE (ghc-8.8.4-boot-x86_64-dragonfly.tar.xz) = 73121164
 SHA256 (ghc-8.6.5-boot-amd64-freebsd11.tar.xz) = 07d4c54a70900128e24b5dd9b22d929ddcf530d911a8be93e694cd9a721f458a
 SIZE (ghc-8.6.5-boot-amd64-freebsd11.tar.xz) = 68965716
 SHA256 (ghc-8.6.5-boot-i386-freebsd11.tar.xz) = cd208fbb6b2a158ff8c5d6227e0babd8c224867ea301abf970dc446f04c9e95f
