--- Makefile.orig	2013-11-09 09:59:40.031025000 +0000
+++ Makefile
@@ -6,6 +6,7 @@ PORTVERSION=	7.6.3
 PORTREVISION=	1
 CATEGORIES=	lang haskell
 MASTER_SITES=	http://www.haskell.org/ghc/dist/${PORTVERSION}/:source \
+		http://leaf.dragonflybsd.org/~marino/dports-src/:dflyboot \
 		${MASTER_SITE_LOCAL:S/$/:boot/}
 MASTER_SITE_SUBDIR=	pgj/:boot
 DISTFILES=	ghc-${PORTVERSION}-src${EXTRACT_SUFX}:source
@@ -64,7 +65,7 @@ OPTIONS_DEFINE+=	BOOTH
 .endif
 
 NO_STAGE=	yes
-.include <bsd.port.options.mk>
+.include <bsd.port.pre.mk>
 
 .if ${PORT_OPTIONS:MDYNAMIC}
 LIB_DEPENDS+=	ffi.6:${PORTSDIR}/devel/libffi
@@ -72,7 +73,8 @@ LIB_DEPENDS+=	ffi.6:${PORTSDIR}/devel/li
 
 .if empty(PORT_OPTIONS:MBOOT)
 
-.if ${OSVERSION} >= 1000043
+.if ${OPSYS} == "FreeBSD"
+.if empty(ICONV_LIB)
 BOOT_GHC_VERSION=	7.6.3
 .else
 BOOT_GHC_VERSION=	7.4.1
@@ -92,7 +94,7 @@ DISTFILES+=	ghc-${BOOT_GHC_VERSION}-i386
 IGNORE=		unsupported
 .endif
 
-.elif ${ARCH} == "amd64"
+.elif ${ARCH:S///} == "amd64"
 
 .if ${OSVERSION} >= 1000043
 DISTFILES+=	ghc-${BOOT_GHC_VERSION}-amd64-freebsd10-boot${EXTRACT_SUFX}:boot
@@ -104,6 +106,26 @@ DISTFILES+=	ghc-${BOOT_GHC_VERSION}-amd6
 IGNORE=		unsupported
 .endif
 
+.endif		# FreeBSD platform list
+
+.elif ${OPSYS} == "DragonFly"
+
+.if empty(ICONV_LIB)
+BOOT_GHC_VERSION=	7.6.3
+.else
+.  if ${DFLYVERSION} > 300500
+BUILD_DEPENDS+=  ${LOCALBASE}/lib/compat/libm.so.3:${PORTSDIR}/misc/compat34x
+.  endif
+BOOT_GHC_VERSION=	7.4.2
+EXTRA_PATCHES=	${DFLY_PATCHDIR}/nopatch-ghc-${BOOT_GHC_VERSION}-boot_rts_package.conf.in
+.endif
+
+DISTFILES+=	ghc-${BOOT_GHC_VERSION}-${ARCH}-dragonfly-boot${EXTRACT_SUFX}:dflyboot
+CONFIGURE_TARGET=	${ARCH}-portbld-dragonfly
+CONFIGURE_ENV+=		BOOTDIR=${BOOT_DIR}
+
+.else
+IGNORE=		unsupported ${OPSYS} Operating System
 .endif
 
 .endif # MBOOT
@@ -122,7 +144,7 @@ CONFIGURE_ARGS=		--with-gcc=${CC} \
 			--with-gmp-includes=${LOCALBASE}/include \
 			--with-gmp-libraries=${LOCALBASE}/lib
 
-.if ${OSVERSION} < 1000043
+.if !empty(ICONV_LIB)
 CONFIGURE_ARGS+=	--with-iconv-includes=${LOCALBASE}/include \
 			--with-iconv-libraries=${LOCALBASE}/lib
 .endif
