--- Makefile.orig	2012-12-30 20:53:57.814369000 +0100
+++ Makefile	2012-12-31 11:20:52.537025000 +0100
@@ -69,6 +69,7 @@
 
 EXTRA_PATCHES=		${PATCHDIR}/nopatch-.._ghc-${BOOT_GHC_VERSION}-boot_rts_package.conf.in
 
+.if ${OPSYS} == "FreeBSD"
 .if ${ARCH} == "i386"
 
 .if ${OSVERSION} >= 900000
@@ -94,6 +95,9 @@
 .endif
 
 .endif
+.else  # FreeBSD
+IGNORE=		No DragonFly bootstrap compiler yet
+.endif # FreeBSD
 
 .endif # MBOOT
 
