--- Makefile.orig	2021-10-16 10:45:48 UTC
+++ Makefile
@@ -4,6 +4,7 @@ PORTNAME=	ghc
 PORTVERSION=	${GHC_VERSION}
 CATEGORIES=	lang haskell
 MASTER_SITES=	https://www.haskell.org/ghc/dist/${PORTVERSION}/:source \
+		http://leaf.dragonflybsd.org/~marino/dports-src/:dflyboot \
 		LOCAL/arrowd/:boot \
 		https://hackage.haskell.org/package/hscolour-${HSCOLOUR_VERSION}/:docs
 DISTFILES=	ghc-${PORTVERSION}-src${EXTRACT_SUFX}:source \
@@ -87,9 +88,11 @@ PORTDOCS=		*
 CONFIGURE_TARGET=	${GHC_ARCH}-portbld-${OPSYS:tl}
 
 # This version of ncurses is needed by bootstrap compiler
+.if ${OPSYS} == FreeBSD
 .if ${OSVERSION} > 1300078 && empty(PORT_OPTIONS:MBOOT) && ${ARCH} != powerpc64
 BUILD_DEPENDS+=	${LOCALBASE}/lib/compat/libncursesw.so.8:misc/compat12x
 .endif
+.endif
 
 .if ${ARCH} == powerpc64 || ${OSVERSION} >= 1400018
 CONFIGURE_ARGS+=	--enable-dtrace=0
@@ -129,7 +132,7 @@ _EXECUTABLES+=	${GHC_LIBDIR}/bin/ghc-ise
 _EXECUTABLES+=	${GHC_LIBDIR}/bin/ghc-iserv-dyn
 .endif
 
-.if empty(PORT_OPTIONS:MBOOT)
+.if empty(PORT_OPTIONS:MBOOT) && ${OPSYS} == "FreeBSD"
 BOOTSTRAPS_SUFFIX?=		${BOOTSTRAPS_SUFFIX_${ARCH}}
 BOOTSTRAPS_SUFFIX_powerpc64?=	-${PPC_ABI:tl}
 DISTFILES+=		ghc-${BOOT_GHC_VERSION}-boot-${ARCH}-freebsd${BOOTSTRAPS_SUFFIX}${EXTRACT_SUFX}:boot
@@ -170,11 +173,39 @@ SLAVE_ENV=		# empty
 CONFIGURE_ARGS+=	GHC=${LOCALBASE}/bin/ghc
 .endif # MBOOT
 
+.if empty(PORT_OPTIONS:MBOOT) && ${OPSYS} == "DragonFly"
+BUILD_DEPENDS+=		libtinfow-compat>0:misc/libtinfow-compat
+BOOT_GHC_VERSION=	8.8.4
+DISTFILES+=		ghc-${BOOT_GHC_VERSION}-boot-${ARCH}-dragonfly.tar.xz:dflyboot
+CONFIGURE_TARGET=	${ARCH}-portbld-dragonfly
+CONFIGURE_ENV+=		BOOTDIR=${BOOT_DIR} LBASE=${LOCALBASE}
+
+CONFIGURE_ARGS+=	--target=${CONFIGURE_TARGET}
+LLVM_VERSION:=	${LLVM_DEFAULT}
+# for bootstrap only disable all OPTIONS_DEFAULT except for GMP, previous boots used devel/libffi321
+.if 0
+CONFIGURE_ARGS+=	--with-intree-gmp
+DOCS_BUILD_DEPENDS:=
+PORT_OPTIONS:=
+.endif
+
+# avoid "sed: RE error: Illegal byte sequence" in bootstrap configure script, smth fishy
+# upstream configure does not support target x86_64-unknown-dragonfly5.9, resed solaris2
+BUILD_DEPENDS+=		gsed:textproc/gsed
+dfly-patch:
+	${REINPLACE_CMD} -e 's@solaris2\*@dragonfly*@g' -e 's@"solaris2"@"dragonfly"@g' \
+		${WRKSRC}/configure # this is fragile, but avoid patch-configure diffs...
+	${REINPLACE_CMD} -e 's@solaris2\*@dragonfly*@g' -e 's@"solaris2"@"dragonfly"@g' \
+		${BOOT_DIR}/configure
+#	${REINPLACE_CMD} -e 's@[[:<:]]sed[[:>:]]@gsed@g' \
+#		${BOOT_DIR}/configure
+.endif
+
 SLAVES_PREFIX=		${WRKDIR}/slaves_prefix
 SLAVES_WRKDIRPREFIX=	${WRKDIR}/slaves_wrkdirprefix
 
 post-patch:
-.if empty(PORT_OPTIONS:MBOOT)
+.if empty(PORT_OPTIONS:MBOOT) && ${OPSYS} == "FreeBSD"
 	@${REINPLACE_CMD} -e '/^mandir/d' ${BOOT_DIR}/mk/build.mk
 	@${REINPLACE_CMD} -e '/^infodir/d' ${BOOT_DIR}/mk/build.mk
 	@${REINPLACE_CMD} -e '/^docdir/d' ${BOOT_DIR}/mk/build.mk
