--- Makefile.orig	2013-04-02 21:43:06.445738000 +0000
+++ Makefile
@@ -6,6 +6,7 @@ PORTVERSION=	7.4.2
 PORTREVISION=	1
 CATEGORIES=	lang haskell
 MASTER_SITES=	http://www.haskell.org/ghc/dist/${PORTVERSION}/:source \
+		http://goetz-isenmann.de/dfly/:dflyboot \
 		${MASTER_SITE_LOCAL:S/$/:boot/}
 MASTER_SITE_SUBDIR=	pgj/:boot
 DISTFILES=	ghc-${PORTVERSION}-src${EXTRACT_SUFX}:source
@@ -16,7 +17,6 @@ COMMENT=	Compiler for the functional lan
 ONLY_FOR_ARCHS=	i386 amd64
 
 GHC_VERSION=		${PORTVERSION}
-BOOT_GHC_VERSION=	7.4.1
 HSCOLOUR_VERSION=	1.20.2
 
 DATADIR=	${PREFIX}/share/ghc-${GHC_VERSION}
@@ -75,8 +75,7 @@ LIB_DEPENDS+=	ffi.6:${PORTSDIR}/devel/li
 
 .if empty(PORT_OPTIONS:MBOOT)
 
-EXTRA_PATCHES=		${PATCHDIR}/nopatch-.._ghc-${BOOT_GHC_VERSION}-boot_rts_package.conf.in
-
+.if ${OPSYS} == "FreeBSD"
 .if ${ARCH} == "i386"
 
 .if ${OSVERSION} >= 900000
@@ -89,7 +88,7 @@ DISTFILES+=	ghc-${BOOT_GHC_VERSION}-i386
 IGNORE=		unsupported
 .endif
 
-.elif ${ARCH} == "amd64"
+.elif ${ARCH:S///} == "amd64"
 
 .if ${OSVERSION} >= 900000
 DISTFILES+=	ghc-${BOOT_GHC_VERSION}-amd64-freebsd9-boot${EXTRACT_SUFX}:boot
@@ -101,6 +100,24 @@ DISTFILES+=	ghc-${BOOT_GHC_VERSION}-amd6
 IGNORE=		unsupported
 .endif
 
+.endif		# FreeBSD platform list
+BOOT_GHC_VERSION=	7.4.1
+EXTRA_PATCHES=	${PATCHDIR}/nopatch-.._ghc-${BOOT_GHC_VERSION}-boot_rts_package.conf.in
+
+.elif ${OPSYS} == "DragonFly"
+
+.  if ${ARCH} == "i386"
+DISTFILES+=	ghc-${BOOT_GHC_VERSION}-i386-dragonfly-boot${EXTRACT_SUFX}:dflyboot
+.  elif ${ARCH} == "x86_64"
+DISTFILES+=	ghc-${BOOT_GHC_VERSION}-x86_64-dragonfly-boot${EXTRACT_SUFX}:dflyboot
+.  endif	# DragonFly platform list
+
+CONFIGURE_TARGET=	${ARCH}-portbld-dragonfly
+BOOT_GHC_VERSION=	7.4.2
+EXTRA_PATCHES=	${DFLY_PATCHDIR}/nopatch-ghc-${BOOT_GHC_VERSION}-boot_rts_package.conf.in
+
+.else
+IGNORE=		unsupported ${OPSYS} Operating System
 .endif
 
 .endif # MBOOT
