--- Makefile.orig	2015-08-22 16:49:52 UTC
+++ Makefile
@@ -5,6 +5,7 @@ PORTNAME=	ghc
 PORTVERSION=	7.10.2
 CATEGORIES=	lang haskell
 MASTER_SITES=	http://www.haskell.org/ghc/dist/${PORTVERSION}/:source \
+		http://leaf.dragonflybsd.org/~marino/dports-src/:dflyboot \
 		LOCAL/pgj/:boot
 DISTFILES=	ghc-${PORTVERSION}-src${EXTRACT_SUFX}:source
 
@@ -79,10 +80,23 @@ LIB_DEPENDS+=	libffi.so.6:${PORTSDIR}/de
 
 .if empty(PORT_OPTIONS:MBOOT)
 
+.if ${OPSYS} == "FreeBSD"
+
 BOOT_GHC_VERSION=	7.10.2
 DISTFILES+=		ghc-${BOOT_GHC_VERSION}-boot-${ARCH}-freebsd${EXTRACT_SUFX}:boot
 LIB_DEPENDS+=		libutil.so.9:${PORTSDIR}/misc/compat9x
 
+.elif ${OPSYS} == "DragonFly"
+
+BOOT_GHC_VERSION=	7.6.3
+DISTFILES+=		ghc-${BOOT_GHC_VERSION}-${ARCH}-dragonfly-boot.tar.bz2:dflyboot
+CONFIGURE_TARGET=	${ARCH}-portbld-dragonfly
+CONFIGURE_ENV+=		BOOTDIR=${BOOT_DIR}
+
+.else
+IGNORE=		unsupported ${OPSYS} Operating System
+.endif
+
 .endif # MBOOT
 
 .if ${PORT_OPTIONS:MPCLANG}
