--- Makefile.orig	2014-08-20 19:06:19.921686000 +0000
+++ Makefile
@@ -5,6 +5,7 @@ PORTNAME=	ghc
 PORTVERSION=	7.8.3
 CATEGORIES=	lang haskell
 MASTER_SITES=	http://www.haskell.org/ghc/dist/${PORTVERSION}/:source \
+		http://leaf.dragonflybsd.org/~marino/dports-src/:dflyboot \
 		${MASTER_SITE_LOCAL:S/$/:boot/}
 MASTER_SITE_SUBDIR=	pgj/:boot
 DISTFILES=	ghc-${PORTVERSION}-src${EXTRACT_SUFX}:source
@@ -64,7 +65,7 @@ OPTIONS_DEFINE+=	BOOT
 OPTIONS_DEFINE+=	BOOTH
 .endif
 
-.include <bsd.port.options.mk>
+.include <bsd.port.pre.mk>
 
 .if ${PORT_OPTIONS:MDYNAMIC}
 # The version number is needed as lang/gcc installs a different version
@@ -82,6 +83,7 @@ BUILD_DEPENDS+=		chrpath:${PORTSDIR}/dev
 
 .if empty(PORT_OPTIONS:MBOOT)
 
+.if ${OPSYS} == "FreeBSD"
 BOOT_GHC_VERSION=	7.6.3
 
 .if ${ARCH} == "i386"
@@ -96,7 +98,7 @@ DISTFILES+=	ghc-${BOOT_GHC_VERSION}-i386
 IGNORE=		unsupported
 .endif
 
-.elif ${ARCH} == "amd64"
+.elif ${ARCH:S///} == "amd64"
 
 .if ${OSVERSION} >= 1000043
 DISTFILES+=	ghc-${BOOT_GHC_VERSION}-amd64-freebsd10-boot${EXTRACT_SUFX}:boot
@@ -108,6 +110,17 @@ DISTFILES+=	ghc-${BOOT_GHC_VERSION}-amd6
 IGNORE=		unsupported
 .endif
 
+.endif		# FreeBSD platform list
+
+.elif ${OPSYS} == "DragonFly"
+
+BOOT_GHC_VERSION=	7.6.3
+DISTFILES+=		ghc-${BOOT_GHC_VERSION}-${ARCH}-dragonfly-boot${EXTRACT_SUFX}:dflyboot
+CONFIGURE_TARGET=	${ARCH}-portbld-dragonfly
+CONFIGURE_ENV+=		BOOTDIR=${BOOT_DIR}
+
+.else
+IGNORE=		unsupported ${OPSYS} Operating System
 .endif
 
 .endif # MBOOT
