--- Makefile.orig	2015-08-22 21:20:08 UTC
+++ Makefile
@@ -5,6 +5,7 @@ PORTNAME=	ghc
 PORTVERSION=	7.10.2
 CATEGORIES=	lang haskell
 MASTER_SITES=	http://www.haskell.org/ghc/dist/${PORTVERSION}/:source \
+		http://leaf.dragonflybsd.org/~marino/dports-src/:dflyboot \
 		LOCAL/pgj/:boot
 DISTFILES=	ghc-${PORTVERSION}-src${EXTRACT_SUFX}:source
 
@@ -61,6 +62,7 @@ LLVM_DESC=		Add support for generating c
 DOCS_DESC=		Install HTML documentation
 
 LOCALBASE?=	/usr/local
+PORTDOCS+=	html
 
 .if exists(${LOCALBASE}/bin/ghc)
 OPTIONS_DEFINE+=	BOOT
@@ -79,10 +81,23 @@ LIB_DEPENDS+=	libffi.so.6:${PORTSDIR}/de
 
 .if empty(PORT_OPTIONS:MBOOT)
 
+.if ${OPSYS} == "FreeBSD"
+
 BOOT_GHC_VERSION=	7.10.2
 DISTFILES+=		ghc-${BOOT_GHC_VERSION}-boot-${ARCH}-freebsd${EXTRACT_SUFX}:boot
 LIB_DEPENDS+=		libutil.so.9:${PORTSDIR}/misc/compat9x
 
+.elif ${OPSYS} == "DragonFly"
+
+BOOT_GHC_VERSION=	7.6.3
+DISTFILES+=		ghc-${BOOT_GHC_VERSION}-${ARCH}-dragonfly-boot.tar.bz2:dflyboot
+CONFIGURE_TARGET=	${ARCH}-portbld-dragonfly
+CONFIGURE_ENV+=		BOOTDIR=${BOOT_DIR}
+
+.else
+IGNORE=		unsupported ${OPSYS} Operating System
+.endif
+
 .endif # MBOOT
 
 .if ${PORT_OPTIONS:MPCLANG}
@@ -302,6 +317,7 @@ pre-configure:
 
 post-install:
 	@${RM} -f ${STAGEDIR}${PREFIX}/bin/haddock
+	${RM} ${GHC_LIBDIR}/package.conf.d/package.cache
 .if ${PORT_OPTIONS:MDOCS}
 	@${LN} -sf haddock-ghc-${GHC_VERSION} ${STAGEDIR}${PREFIX}/bin/haddock
 .endif
