--- Makefile.orig	2017-08-15 18:18:54 UTC
+++ Makefile
@@ -6,6 +6,7 @@ PORTVERSION=	${GHC_VERSION}
 PORTREVISION=	1
 CATEGORIES=	lang haskell
 MASTER_SITES=	http://www.haskell.org/ghc/dist/${PORTVERSION}/:source \
+		http://leaf.dragonflybsd.org/~marino/dports-src/:dflyboot \
 		LOCAL/pgj/:boot
 DISTFILES=	ghc-${PORTVERSION}-src${EXTRACT_SUFX}:source
 
@@ -89,8 +90,26 @@ LOCALBASE?=	/usr/local
 .include <bsd.port.options.mk>
 
 .if empty(PORT_OPTIONS:MBOOT)
+.  if ${OPSYS} == "FreeBSD"
+
 BOOT_GHC_VERSION=	7.10.2
 DISTFILES+=		ghc-${BOOT_GHC_VERSION}-boot-${ARCH}-freebsd${EXTRACT_SUFX}:boot
+
+.  elif ${OPSYS} == "DragonFly"
+
+BOOT_GHC_VERSION=	7.10.2
+.    if exists (/usr/lib/priv/libprivate_ncursesw.so)
+DISTFILES+=		ghc-${BOOT_GHC_VERSION}-boot-${ARCH}-dragonfly2.tar.xz:dflyboot
+.    else
+DISTFILES+=		ghc-${BOOT_GHC_VERSION}-boot-${ARCH}-dragonfly.tar.xz:dflyboot
+.    endif
+CONFIGURE_TARGET=	${ARCH}-portbld-dragonfly
+CONFIGURE_ENV+=		BOOTDIR=${BOOT_DIR} LBASE=${LOCALBASE}
+
+.  else
+IGNORE=		unsupported ${OPSYS} Operating System
+.  endif
+
 .endif # MBOOT
 
 # LLVM is still not properly supported, further it does not make sense to have
