--- Makefile.orig	2022-10-11 23:20:03 UTC
+++ Makefile
@@ -2,7 +2,8 @@ PORTNAME=	ghc
 PORTVERSION=	${GHC_VERSION}
 CATEGORIES=	lang haskell
 MASTER_SITES=	https://www.haskell.org/ghc/dist/${PORTVERSION}/:source \
-		LOCAL/arrowd/:boot
+		LOCAL/arrowd/:boot \
+		http://leaf.dragonflybsd.org/~marino/dports-src/:dflyboot
 DISTFILES=	ghc-${PORTVERSION}-src${EXTRACT_SUFX}:source
 
 MAINTAINER=	haskell@FreeBSD.org
@@ -106,11 +107,13 @@ BOOT_DIR=		${WRKDIR}/ghc-${BOOT_GHC_VERS
 BOOT_GHC=		${BOOT_DIR}/bin/ghc-${BOOT_GHC_VERSION}
 
 # This version of ncurses is needed by bootstrap compiler
+.if ${OPSYS} == FreeBSD
 .if ${OSVERSION} > 1300078 && empty(PORT_OPTIONS:MBOOT)
 BUILD_DEPENDS+=	${LOCALBASE}/lib/compat/libncursesw.so.8:misc/compat12x
 .endif
+.endif
 
-.if empty(PORT_OPTIONS:MBOOT)
+.if empty(PORT_OPTIONS:MBOOT) && ${OPSYS} == "FreeBSD"
 DISTFILES+=		ghc-${BOOT_GHC_VERSION}-boot-${ARCH}-freebsd${EXTRACT_SUFX}:boot
 .endif # MBOOT
 
@@ -129,6 +132,34 @@ BUILD_DEPENDS+=		llc${BOOT_LLVM_VERSION}
 .  endif
 .endif
 
+.if empty(PORT_OPTIONS:MBOOT) && ${OPSYS} == "DragonFly"
+BUILD_DEPENDS+=		libtinfow-compat>0:misc/libtinfow-compat
+BOOT_GHC_VERSION=	8.8.4
+DISTFILES+=		ghc-${BOOT_GHC_VERSION}-boot-${ARCH}-dragonfly.tar.xz:dflyboot
+CONFIGURE_TARGET=	${ARCH}-portbld-dragonfly
+CONFIGURE_ENV+=		BOOTDIR=${BOOT_DIR} LBASE=${LOCALBASE}
+
+CONFIGURE_ARGS+=	--target=${CONFIGURE_TARGET}
+LLVM_VERSION:=	${LLVM_DEFAULT}
+# for bootstrap only disable all OPTIONS_DEFAULT except for GMP, previous boots used devel/libffi321
+.if 0
+CONFIGURE_ARGS+=	--with-intree-gmp
+DOCS_BUILD_DEPENDS:=
+PORT_OPTIONS:=
+.endif
+
+# avoid "sed: RE error: Illegal byte sequence" in bootstrap configure script, smth fishy
+# upstream configure does not support target x86_64-unknown-dragonfly5.9, resed solaris2
+BUILD_DEPENDS+=		gsed:textproc/gsed
+dfly-patch:
+	${REINPLACE_CMD} -e 's@solaris2\*@dragonfly*@g' -e 's@"solaris2"@"dragonfly"@g' \
+		${WRKSRC}/configure # this is fragile, but avoid patch-configure diffs...
+	${REINPLACE_CMD} -e 's@solaris2\*@dragonfly*@g' -e 's@"solaris2"@"dragonfly"@g' \
+		${BOOT_DIR}/configure
+#	${REINPLACE_CMD} -e 's@[[:<:]]sed[[:>:]]@gsed@g' \
+#		${BOOT_DIR}/configure
+.endif
+
 post-patch:
 #	Generate the build.mk file
 	${RM} -f ${WRKSRC}/mk/build.mk
@@ -136,12 +167,14 @@ post-patch:
 	${ECHO_CMD} ${line} >> ${WRKSRC}/mk/build.mk
 .endfor
 
+.if ${OPSYS} == "FreeBSD"
 # TODO: remove this after rerolling all bootstraps
 post-patch-BOOT-off:
 	@${REINPLACE_CMD} -e '/^mandir/d' ${BOOT_DIR}/mk/build.mk
 	@${REINPLACE_CMD} -e '/^infodir/d' ${BOOT_DIR}/mk/build.mk
 	@${REINPLACE_CMD} -e '/^docdir/d' ${BOOT_DIR}/mk/build.mk
 	@${REINPLACE_CMD} -e '/^htmldir/d' ${BOOT_DIR}/mk/build.mk
+.endif
 
 pre-configure:
 	# Call the bootstrap script
