--- Makefile.orig	2019-12-06 19:20:08 UTC
+++ Makefile
@@ -6,6 +6,7 @@ PORTVERSION=	${GHC_VERSION}
 PORTREVISION=	3
 CATEGORIES=	lang haskell
 MASTER_SITES=	https://www.haskell.org/ghc/dist/${PORTVERSION}/:source \
+		http://leaf.dragonflybsd.org/~marino/dports-src/:dflyboot \
 		LOCAL/arrowd/:boot \
 		https://hackage.haskell.org/package/hscolour-${HSCOLOUR_VERSION}/:docs
 DISTFILES=	ghc-${PORTVERSION}-src${EXTRACT_SUFX}:source \
@@ -142,7 +143,7 @@ _EXECUTABLES+=	${GHC_LIBDIR}/bin/ghc-ise
 
 .include <bsd.port.options.mk>
 
-.if empty(PORT_OPTIONS:MBOOT)
+.if empty(PORT_OPTIONS:MBOOT) && ${OPSYS} == "FreeBSD"
 BOOTSTRAPS_SUFFIX?=		${BOOTSTRAPS_SUFFIX_${ARCH}}
 BOOTSTRAPS_SUFFIX_powerpc64?=	-${PPC_ABI:tl}
 DISTFILES+=		ghc-${BOOT_GHC_VERSION}-boot-${ARCH}-freebsd${BOOTSTRAPS_SUFFIX}${EXTRACT_SUFX}:boot
@@ -195,6 +196,19 @@ SLAVE_ENV=		# empty
 CONFIGURE_ARGS+=	--with-ghc=${LOCALBASE}/bin/ghc
 .endif # MBOOT
 
+.if empty(PORT_OPTIONS:MBOOT) && ${OPSYS} == "DragonFly"
+BOOT_GHC_VERSION=	8.4.4
+DISTFILES+=		ghc-${BOOT_GHC_VERSION}-boot-${ARCH}-dragonfly.tar.xz:dflyboot
+CONFIGURE_TARGET=	${ARCH}-portbld-dragonfly
+CONFIGURE_ENV+=		BOOTDIR=${BOOT_DIR} LBASE=${LOCALBASE}
+
+# avoid "sed: RE error: Illegal byte sequence" in bootstrap configure script, smth fishy
+BUILD_DEPENDS+=		gsed:textproc/gsed
+dfly-patch:
+	${REINPLACE_CMD} -e 's@sed@gsed@g' \
+		${BOOT_DIR}/configure
+.endif
+
 # override TMPDIR because /tmp often doesn't have enough space
 # to build some of the larger libraries.
 #TMPDIR=		${WRKSRC}/tmp
@@ -243,7 +257,7 @@ post-patch:
 	@${REINPLACE_CMD} -e 's/@SettingsLlcCommand@/llc${LLVM_VERSION}/' ${WRKSRC}/settings.in
 	@${REINPLACE_CMD} -e 's/@SettingsOptCommand@/opt${LLVM_VERSION}/' ${WRKSRC}/settings.in
 
-.if empty(PORT_OPTIONS:MBOOT)
+.if empty(PORT_OPTIONS:MBOOT) && ${OPSYS} == "FreeBSD"
 	@${REINPLACE_CMD} -e '/^mandir/d' ${BOOT_DIR}/mk/build.mk
 	@${REINPLACE_CMD} -e '/^infodir/d' ${BOOT_DIR}/mk/build.mk
 	@${REINPLACE_CMD} -e '/^docdir/d' ${BOOT_DIR}/mk/build.mk
