--- Makefile.orig	2013-09-14 11:46:10.240477000 +0000
+++ Makefile
@@ -5,6 +5,7 @@ PORTNAME=	ghc
 PORTVERSION=	7.6.3
 CATEGORIES=	lang haskell
 MASTER_SITES=	http://www.haskell.org/ghc/dist/${PORTVERSION}/:source \
+		http://goetz-isenmann.de/dfly/:dflyboot \
 		${MASTER_SITE_LOCAL:S/$/:boot/}
 MASTER_SITE_SUBDIR=	pgj/:boot
 DISTFILES=	ghc-${PORTVERSION}-src${EXTRACT_SUFX}:source
@@ -71,6 +72,7 @@ LIB_DEPENDS+=	ffi.6:${PORTSDIR}/devel/li
 
 .if empty(PORT_OPTIONS:MBOOT)
 
+.if ${OPSYS} == "FreeBSD"
 .if ${OSVERSION} >= 1000043
 BOOT_GHC_VERSION=	7.6.3
 .else
@@ -91,7 +93,7 @@ DISTFILES+=	ghc-${BOOT_GHC_VERSION}-i386
 IGNORE=		unsupported
 .endif
 
-.elif ${ARCH} == "amd64"
+.elif ${ARCH:S///} == "amd64"
 
 .if ${OSVERSION} >= 1000043
 DISTFILES+=	ghc-${BOOT_GHC_VERSION}-amd64-freebsd10-boot${EXTRACT_SUFX}:boot
@@ -103,6 +105,25 @@ DISTFILES+=	ghc-${BOOT_GHC_VERSION}-amd6
 IGNORE=		unsupported
 .endif
 
+.endif		# FreeBSD platform list
+
+.elif ${OPSYS} == "DragonFly"
+
+.  if ${DFLYVERSION} > 300500
+BUILD_DEPENDS+=  ${LOCALBASE}/lib/compat/libm.so.3:${PORTSDIR}/misc/compat34x
+.  endif
+.  if ${ARCH} == "i386"
+DISTFILES+=	ghc-${BOOT_GHC_VERSION}-i386-dragonfly-boot${EXTRACT_SUFX}:dflyboot
+.  elif ${ARCH} == "x86_64"
+DISTFILES+=	ghc-${BOOT_GHC_VERSION}-x86_64-dragonfly-boot${EXTRACT_SUFX}:dflyboot
+.  endif	# DragonFly platform list
+
+CONFIGURE_TARGET=	${ARCH}-portbld-dragonfly
+BOOT_GHC_VERSION=	7.4.2
+EXTRA_PATCHES=	${DFLY_PATCHDIR}/nopatch-ghc-${BOOT_GHC_VERSION}-boot_rts_package.conf.in
+
+.else
+IGNORE=		unsupported ${OPSYS} Operating System
 .endif
 
 .endif # MBOOT
