--- pkg-plist.orig	2020-07-28 22:57:53 UTC
+++ pkg-plist
@@ -70,8 +70,8 @@ include/kodi/xbmc_pvr_types.h
 %%WAYLAND%%lib/kodi/kodi-wayland
 %%X11%%lib/kodi/kodi-x11
 %%X11%%lib/kodi/kodi-xrandr
-%%SSE4_1%%lib/kodi/system/libsse4-%%ARCH%%-freebsd.so
-lib/kodi/system/players/VideoPlayer/libdvdnav-%%ARCH%%-freebsd.so
+%%SSE4_1%%lib/kodi/system/libsse4-%%ARCH%%-dragonfly.so
+lib/kodi/system/players/VideoPlayer/libdvdnav-%%ARCH%%-dragonfly.so
 man/man1/kodi.1.gz
 man/man1/kodi.bin.1.gz
 share/applications/kodi.desktop
@@ -1343,6 +1343,7 @@ share/icons/hicolor/64x64/apps/kodi.png
 %%DATADIR%%/system/settings/darwin.xml
 %%DATADIR%%/system/settings/darwin_ios.xml
 %%DATADIR%%/system/settings/darwin_osx.xml
+%%DATADIR%%/system/settings/dragonfly.xml
 %%DATADIR%%/system/settings/freebsd.xml
 %%DATADIR%%/system/settings/gbm.xml
 %%DATADIR%%/system/settings/linux.xml
