--- pkg-plist.orig	2015-06-10 06:14:08 UTC
+++ pkg-plist
@@ -7,15 +7,19 @@ include/canna/sysdep.h
 lib/libRKC.a
 lib/libRKC.so
 lib/libRKC.so.1
+lib/libRKC.so.1.2
 lib/libRKC16.a
 lib/libRKC16.so
 lib/libRKC16.so.1
+lib/libRKC16.so.1.2
 lib/libcanna.a
 lib/libcanna.so
 lib/libcanna.so.1
+lib/libcanna.so.1.2
 lib/libcanna16.a
 lib/libcanna16.so
 lib/libcanna16.so.1
+lib/libcanna16.so.1.2
 %%DATADIR%%/default.canna
 @(bin,bin,) %%DATADIR%%/dic/default.cbp
 @(bin,bin,) %%DATADIR%%/dic/english.cbp
