--- Makefile.orig	2017-10-30 17:35:17 UTC
+++ Makefile
@@ -16,8 +16,7 @@ OPTIONS_DEFINE=	BUNDLED_SSL DOCS NLS DTR
 OPTIONS_DEFAULT=DTRACE
 OPTIONS_SUB=	yes
 
-OPTIONS_DEFAULT_FreeBSD_10=	BUNDLED_SSL
-OPTIONS_DEFAULT+=		${OPTIONS_DEFAULT_${OPSYS}_${OSREL:R}}
+OPTIONS_DEFAULT+=		BUNDLED_SSL
 
 .if !exists(/usr/sbin/dtrace)
 OPTIONS_EXCLUDE+=	DTRACE
@@ -77,9 +76,8 @@ IGNORE=		cannot build node.js with Libre
 
 .include <bsd.port.pre.mk>
 
-.if ${ARCH} == armv6 || ${ARCH} == armv7
+# something is wrong with configure wrt to DF I think
 CONFIGURE_ARGS+=--openssl-no-asm
-.endif
 
 .if ${COMPILER_TYPE} == gcc
 # GCC does not expose std::snprintf() without this define
