--- Makefile.orig	2018-05-08 19:49:48 UTC
+++ Makefile
@@ -15,7 +15,7 @@ LICENSE_FILE=	${WRKSRC}/LICENSE
 ONLY_FOR_ARCHS=		aarch64 amd64 armv6 armv7 i386 powerpc64
 
 OPTIONS_DEFINE=		BUNDLED_SSL DOCS NLS DTRACE
-OPTIONS_DEFAULT=	BUNDLED_SSL DTRACE
+OPTIONS_DEFAULT=	BUNDLED_SSL
 OPTIONS_SUB=		yes
 
 .if !exists(/usr/sbin/dtrace)
@@ -76,9 +76,7 @@ IGNORE=		cannot build node.js with Libre
 
 .include <bsd.port.pre.mk>
 
-.if ${ARCH} != amd64
 CONFIGURE_ARGS+=--openssl-no-asm
-.endif
 
 .if ${COMPILER_TYPE} == gcc
 # GCC does not expose std::snprintf() without this define
