--- pkg-plist.ori	2021-10-22 17:09:38.781467000 +0200
+++ pkg-plist	2021-10-22 17:10:58.199694000 +0200
@@ -288,6 +288,11 @@
 %%BUNDLED_SSL%%include/node/openssl/archs/linux-x86_64/no-asm/crypto/include/internal/dso_conf.h
 %%BUNDLED_SSL%%include/node/openssl/archs/linux-x86_64/no-asm/include/openssl/opensslconf.h
 %%BUNDLED_SSL%%include/node/openssl/archs/linux-x86_64/no-asm/include/progs.h
+%%BUNDLED_SSL%%include/node/openssl/archs/linux64-riscv64/no-asm/crypto/buildinf.h
+%%BUNDLED_SSL%%include/node/openssl/archs/linux64-riscv64/no-asm/crypto/include/internal/bn_conf.h
+%%BUNDLED_SSL%%include/node/openssl/archs/linux64-riscv64/no-asm/crypto/include/internal/dso_conf.h
+%%BUNDLED_SSL%%include/node/openssl/archs/linux64-riscv64/no-asm/include/openssl/opensslconf.h
+%%BUNDLED_SSL%%include/node/openssl/archs/linux64-riscv64/no-asm/include/progs.h
 %%BUNDLED_SSL%%include/node/openssl/archs/linux32-s390x/asm/crypto/buildinf.h
 %%BUNDLED_SSL%%include/node/openssl/archs/linux32-s390x/asm/crypto/include/internal/bn_conf.h
 %%BUNDLED_SSL%%include/node/openssl/archs/linux32-s390x/asm/crypto/include/internal/dso_conf.h
