diff --git a/misc/mc/pkg-plist b/misc/mc/pkg-plist
index 64b3c353dfe..d2b58d98859 100644
--- a/misc/mc/pkg-plist
+++ b/misc/mc/pkg-plist
@@ -78,15 +78,15 @@ libexec/mc/mc-wrapper.csh
 libexec/mc/mc-wrapper.sh
 libexec/mc/mc.csh
 libexec/mc/mc.sh
-%%NLS%%man/es/man1/mc.1.gz
-%%NLS%%man/hu/man1/mc.1.gz
-%%NLS%%man/it/man1/mc.1.gz
-man/man1/mc.1.gz
-man/man1/mcedit.1.gz
-man/man1/mcview.1.gz
-%%NLS%%man/pl/man1/mc.1.gz
-%%NLS%%man/ru/man1/mc.1.gz
-%%NLS%%man/sr/man1/mc.1.gz
+%%NLS%%share/man/es/man1/mc.1.gz
+%%NLS%%share/man/hu/man1/mc.1.gz
+%%NLS%%share/man/it/man1/mc.1.gz
+share/man/man1/mc.1.gz
+share/man/man1/mcedit.1.gz
+share/man/man1/mcview.1.gz
+%%NLS%%share/man/pl/man1/mc.1.gz
+%%NLS%%share/man/ru/man1/mc.1.gz
+%%NLS%%share/man/sr/man1/mc.1.gz
 %%X11%%share/applications/mc.desktop
 %%NLS%%share/locale/af/LC_MESSAGES/mc.mo
 %%NLS%%share/locale/ar/LC_MESSAGES/mc.mo
