--- distinfo.orig	2013-01-05 21:50:00.636809000 +0100
+++ distinfo	2013-01-05 21:39:57.905967000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_sk.tar.gz) = 034d40c082174ed94312db053368106703315aa84e896109e7f70c5bf2cb9a9a
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_sk.tar.gz) = 3011771
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_sk.tar.gz) = b1eac4b651d8f44b91547a815bc81428cd0e1fda135a9859ab76817c51aefb70
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_sk.tar.gz) = 10498735
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_sk.tar.gz) = 8a53474dfbaf4a6865666352e9249b705b4ae20a002590a85c66e257162d7191
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_sk.tar.gz) = 2974343
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_sk.tar.gz) = 02b8dab93efaf62e251a38502b9ab9c0da475965fff4313a8778c08cda856b75
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_sk.tar.gz) = 10507044
