--- pkg-plist.orig	2018-11-28 17:26:19 UTC
+++ pkg-plist
@@ -19,7 +19,6 @@ bin/regpatch
 bin/regshell
 bin/regtree
 bin/rpcclient
-bin/samba-regedit
 %%AD_DC%%bin/samba-tool
 bin/sharesec
 bin/smbcacls
@@ -178,8 +177,6 @@ lib/samba4/libtevent-util.so
 lib/samba4/libtevent-util.so.0
 lib/samba4/libwbclient.so
 lib/samba4/libwbclient.so.0
-lib/nss_winbind.so.1
-lib/nss_wins.so.1
 lib/pam_winbind.so
 %%CUPS%%libexec/samba/smbspool_krb5_wrapper
 %%AD_DC%%lib/samba4/private/libdlz-bind9-for-torture-samba4.so
@@ -397,7 +394,6 @@ lib/shared-modules/vfs/unityed_media.so
 lib/shared-modules/vfs/virusfilter.so
 lib/shared-modules/vfs/worm.so
 lib/shared-modules/vfs/xattr_tdb.so
-lib/shared-modules/vfs/zfsacl.so
 @dir lib/shared-modules/vfs
 %%AD_DC%%@dir lib/shared-modules/service
 %%AD_DC%%@dir lib/shared-modules/process_model
@@ -1205,7 +1201,6 @@ man/man8/vfs_unityed_media.8.gz
 man/man8/vfs_virusfilter.8.gz
 man/man8/vfs_worm.8.gz
 man/man8/vfs_xattr_tdb.8.gz
-man/man8/vfs_zfsacl.8.gz
 man/man8/winbind_krb5_locator.8.gz
 man/man8/winbindd.8.gz
 @dir %%SAMBA4_RUNDIR%%
