--- pkg-plist.orig	2020-10-28 15:45:09 UTC
+++ pkg-plist
@@ -4,14 +4,12 @@ etc/xdg/autostart/org.gnome.SettingsDaem
 etc/xdg/autostart/org.gnome.SettingsDaemon.Datetime.desktop
 etc/xdg/autostart/org.gnome.SettingsDaemon.Housekeeping.desktop
 etc/xdg/autostart/org.gnome.SettingsDaemon.Keyboard.desktop
-etc/xdg/autostart/org.gnome.SettingsDaemon.MediaKeys.desktop
 etc/xdg/autostart/org.gnome.SettingsDaemon.Power.desktop
 etc/xdg/autostart/org.gnome.SettingsDaemon.PrintNotifications.desktop
 etc/xdg/autostart/org.gnome.SettingsDaemon.Rfkill.desktop
 etc/xdg/autostart/org.gnome.SettingsDaemon.ScreensaverProxy.desktop
 etc/xdg/autostart/org.gnome.SettingsDaemon.Sharing.desktop
 etc/xdg/autostart/org.gnome.SettingsDaemon.Smartcard.desktop
-etc/xdg/autostart/org.gnome.SettingsDaemon.Sound.desktop
 etc/xdg/autostart/org.gnome.SettingsDaemon.UsbProtection.desktop
 etc/xdg/autostart/org.gnome.SettingsDaemon.Wacom.desktop
 etc/xdg/autostart/org.gnome.SettingsDaemon.Wwan.desktop
@@ -24,14 +22,12 @@ libexec/gsd-color
 libexec/gsd-datetime
 libexec/gsd-housekeeping
 libexec/gsd-keyboard
-libexec/gsd-media-keys
 libexec/gsd-power
 libexec/gsd-print-notifications
 libexec/gsd-printer
 libexec/gsd-screensaver-proxy
 libexec/gsd-sharing
 libexec/gsd-smartcard
-libexec/gsd-sound
 libexec/gsd-usb-protection
 libexec/gsd-xsettings
 share/GConf/gsettings/gnome-settings-daemon.convert
