--- pkg-plist.orig	2016-08-19 15:09:24 UTC
+++ pkg-plist
@@ -24,8 +24,6 @@ lib/gnome-settings-daemon-3.0/libhouseke
 lib/gnome-settings-daemon-3.0/libhousekeeping.so
 lib/gnome-settings-daemon-3.0/libkeyboard.a
 lib/gnome-settings-daemon-3.0/libkeyboard.so
-lib/gnome-settings-daemon-3.0/libmedia-keys.a
-lib/gnome-settings-daemon-3.0/libmedia-keys.so
 lib/gnome-settings-daemon-3.0/libmouse.a
 lib/gnome-settings-daemon-3.0/libmouse.so
 lib/gnome-settings-daemon-3.0/liborientation.a
@@ -41,20 +39,16 @@ lib/gnome-settings-daemon-3.0/libsharing
 lib/gnome-settings-daemon-3.0/libsharing.so
 %%SMARTCARD%%lib/gnome-settings-daemon-3.0/libsmartcard.a
 %%SMARTCARD%%lib/gnome-settings-daemon-3.0/libsmartcard.so
-lib/gnome-settings-daemon-3.0/libsound.a
-lib/gnome-settings-daemon-3.0/libsound.so
 lib/gnome-settings-daemon-3.0/libxrandr.a
 lib/gnome-settings-daemon-3.0/libxrandr.so
 lib/gnome-settings-daemon-3.0/libxsettings.a
 lib/gnome-settings-daemon-3.0/libxsettings.so
-lib/gnome-settings-daemon-3.0/media-keys.gnome-settings-plugin
 lib/gnome-settings-daemon-3.0/mouse.gnome-settings-plugin
 lib/gnome-settings-daemon-3.0/power.gnome-settings-plugin
 %%CUPS%%lib/gnome-settings-daemon-3.0/print-notifications.gnome-settings-plugin
 lib/gnome-settings-daemon-3.0/screensaver-proxy.gnome-settings-plugin
 lib/gnome-settings-daemon-3.0/sharing.gnome-settings-plugin
 %%SMARTCARD%%lib/gnome-settings-daemon-3.0/smartcard.gnome-settings-plugin
-lib/gnome-settings-daemon-3.0/sound.gnome-settings-plugin
 lib/gnome-settings-daemon-3.0/xrandr.gnome-settings-plugin
 lib/gnome-settings-daemon-3.0/xsettings.gnome-settings-plugin
 libdata/pkgconfig/gnome-settings-daemon.pc
@@ -68,12 +62,10 @@ libexec/gsd-test-input-helper
 libexec/gsd-test-datetime
 libexec/gsd-test-housekeeping
 libexec/gsd-test-keyboard
-libexec/gsd-test-media-keys
 libexec/gsd-test-mouse
 libexec/gsd-test-orientation
 %%CUPS%%libexec/gsd-test-print-notifications
 %%SMARTCARD%%libexec/gsd-test-smartcard
-libexec/gsd-test-sound
 libexec/gsd-test-screensaver-proxy
 libexec/gsd-test-xrandr
 libexec/gsd-test-xsettings
