--- distinfo.orig	2013-01-05 22:07:27.188271000 +0100
+++ distinfo	2013-01-05 22:06:59.418232000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_pt-BR.tar.gz) = 5412a8e0d34d708568a7a105cd1a1e92772d0ded503a31ced195f0f8e145035b
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_pt-BR.tar.gz) = 2953314
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_pt-BR.tar.gz) = f3e36fec8167769fa4733e45d21ebdb6f10e725476d3bd0c1dbb144be5bf6a5e
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_pt-BR.tar.gz) = 10760632
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_pt-BR.tar.gz) = b76a58f9b1356013e8c57cbd0c0f557e8104e54f3e3982fa632783aa86912029
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_pt-BR.tar.gz) = 2902366
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_pt-BR.tar.gz) = 198f8af417c9db6f51591c3b3a18b5c6180f1e91a81e75881e01d496098878e4
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_pt-BR.tar.gz) = 10773788
