--- pkg-plist.orig	2021-08-09 16:17:36.697526000 +0200
+++ pkg-plist	2021-08-09 16:17:53.817141000 +0200
@@ -386,6 +386,8 @@
 %%PJSIP%%@dir %%DATADIR%%/third-party/pjproject
 @owner %%ASTERISK_USER%%
 @group %%ASTERISK_GROUP%%
+@dir %%VARDIR%%/lib
+@dir %%VARDIR%%/lib/asterisk
 @dir %%VARDIR%%/log/asterisk/cdr-csv
 @dir %%VARDIR%%/log/asterisk/cdr-custom
 @dir %%VARDIR%%/log/asterisk/cel-custom
@@ -398,7 +400,6 @@
 @dir %%VARDIR%%/spool/asterisk
 @dir %%VARDIR%%/spool/asterisk/system
 @dir %%VARDIR%%/spool/asterisk/tmp
-@dir %%VARDIR%%/db/asterisk
 @dir %%VARDIR%%/run/asterisk
 @owner
 @group
