--- Makefile.orig	2018-08-22 16:51:12 UTC
+++ Makefile
@@ -23,11 +23,7 @@ BROKEN_powerpc64=	fails to configure: ca
 
 GNU_CONFIGURE=	yes
 CONFIGURE_FLAGS=	--with-llvm=${LOCALBASE}
-CONFIGURE_ENV=	LLVM_CONFIG=${LOCALBASE}/bin/llvm-config60 \
-		CC=${LOCALBASE}/bin/clang60 \
-		CXX=${LOCALBASE}/bin/clang++60 \
-		CPP=${LOCALBASE}/bin/clang-cpp60
 
-USES=		autoreconf gmake libtool perl5
+USES=		gmake perl5
 
 .include <bsd.port.mk>
