--- pkg-plist.orig	2013-03-21 14:31:23.880558000 +0000
+++ pkg-plist
@@ -1,5 +1,4 @@
 %%ASTERDIR%%/LICENSE.TERMS
-%%ASTERDIR%%/asteru
 %%ASTERDIR%%/astest/adls102a.comm
 %%ASTERDIR%%/astest/adls102a.datg
 %%ASTERDIR%%/astest/adls102a.mail
@@ -11398,7 +11397,6 @@
 %%ASTERDIR%%/bibpyt/Validation/__init__.py
 %%ASTERDIR%%/bibpyt/Validation/__init__.pyc
 %%ASTERDIR%%/bibpyt/Validation/__init__.pyo
-%%ASTERDIR%%/cata_ele.pickled
 %%ASTERDIR%%/catalo/compelem/grandeur_simple__.cata
 %%ASTERDIR%%/catalo/compelem/phenomene_modelisation__.cata
 %%ASTERDIR%%/catalo/compelem/type_maille__.cata
@@ -12364,9 +12362,6 @@
 %%ASTERDIR%%/catapy/commun/c_suivi_ddl.capy
 %%ASTERDIR%%/catapy/commun/c_type_cham_into.capy
 %%ASTERDIR%%/catapy/entete/accas.capy
-%%ASTERDIR%%/commande/cata.py
-%%ASTERDIR%%/commande/cata.pyc
-%%ASTERDIR%%/commande/cata.pyo
 %%ASTERDIR%%/config.txt
 %%ASTERDIR%%/elements
 %%ASTERDIR%%/forma01a.export
@@ -12400,7 +12395,6 @@ aster/config.txt
 %%GRACE%%@unexec rm -f  %D/%%TOOLDIR%%/xmgrace
 @unexec rm -rf  %D/%%TOOLDIR%% 2>&1 >/dev/null || true
 @dirrm %%ASTERDIR%%/materiau
-@dirrm %%ASTERDIR%%/commande
 @dirrm %%ASTERDIR%%/catapy/entete
 @dirrm %%ASTERDIR%%/catapy/commun
 @dirrm %%ASTERDIR%%/catapy/commande
