--- Makefile.orig	2016-06-12 10:26:09 UTC
+++ Makefile
@@ -24,12 +24,10 @@ RUN_DEPENDS=	aclock>0:x11-clocks/aclock
 		cartotheque>0:deskutils/cartotheque \
 		cddb-bundle>0:audio/cddb-bundle \
 		gnustep-cdplayer>0:audio/gnustep-cdplayer \
-		cenon>0:graphics/cenon \
 		duncan>0:textproc/duncan \
 		easydiff>0:textproc/easydiff \
 		edenmath>0:math/edenmath \
 		expense>0:finance/expense \
-		fortunate>0:games/fortunate \
 		fortytwo-bdb>0:databases/fortytwo-bdb \
 		fortytwo-encore>0:devel/fortytwo-encore \
 		fortytwo>0:graphics/fortytwo \
