--- distinfo.orig	2013-11-11 13:16:33.843324000 +0000
+++ distinfo
@@ -2,3 +2,5 @@ SHA256 (xorg/driver/xf86-video-intel-2.7
 SIZE (xorg/driver/xf86-video-intel-2.7.1.tar.bz2) = 780625
 SHA256 (xorg/driver/xf86-video-intel-2.21.15.tar.bz2) = 7d5a140f82a72fd1cbc8a664d66c3d4eca47ee240ca4927b8a98d7af6f65d6fc
 SIZE (xorg/driver/xf86-video-intel-2.21.15.tar.bz2) = 1977431
+SHA256 (xorg/driver/xf86-video-intel-2.21.9.tar.bz2) = 1359cbc9e494a284faa52d1db83e7388cb8ab590b660e29e78e6e7f5ee7ff189
+SIZE (xorg/driver/xf86-video-intel-2.21.9.tar.bz2) = 1735037
