--- pkg-plist.orig	2013-10-26 22:26:05.102956000 +0000
+++ pkg-plist
@@ -43,9 +43,7 @@ share/aclocal/cmake.m4
 %%PORTDOCS%%%%DOCSDIR%%/cmcompress/Copyright.txt
 %%PORTDOCS%%%%DOCSDIR%%/cmcurl/COPYING
 %%PORTDOCS%%%%DOCSDIR%%/cmexpat/COPYING
-%%PORTDOCS%%%%DOCSDIR%%/cmlibarchive/COPYING
 %%PORTDOCS%%%%DOCSDIR%%/cmsys/Copyright.txt
-%%PORTDOCS%%%%DOCSDIR%%/cmzlib/Copyright.txt
 %%PORTDOCS%%%%DOCSDIR%%/cpack.docbook
 %%PORTDOCS%%%%DOCSDIR%%/cpack.html
 %%PORTDOCS%%%%DOCSDIR%%/cpack.txt
@@ -53,9 +51,7 @@ share/aclocal/cmake.m4
 %%PORTDOCS%%%%DOCSDIR%%/ctest.html
 %%PORTDOCS%%%%DOCSDIR%%/ctest.txt
 share/emacs/site-lisp/cmake-mode.el
-%%PORTDOCS%%@dirrm %%DOCSDIR%%/cmzlib
 %%PORTDOCS%%@dirrm %%DOCSDIR%%/cmsys
-%%PORTDOCS%%@dirrm %%DOCSDIR%%/cmlibarchive
 %%PORTDOCS%%@dirrm %%DOCSDIR%%/cmexpat
 %%PORTDOCS%%@dirrm %%DOCSDIR%%/cmcurl
 %%PORTDOCS%%@dirrm %%DOCSDIR%%/cmcompress
