--- distinfo.orig	2013-01-05 21:50:00.916809000 +0100
+++ distinfo	2013-01-05 21:40:32.926016000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_ca.tar.gz) = 0f9cac802a1d8d2a834e465376f1ed8714d9f5415c6444af9040e845705a1240
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_ca.tar.gz) = 2290307
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_ca.tar.gz) = 2e9190053f83abbf973bc89eb812c0aa0b140ba0a8fed769112638f01599f8e7
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_ca.tar.gz) = 10924365
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_ca.tar.gz) = 596f9fa35b5e288218024278ce13f01d6cfe3aa7fcb82d07e40efe6013742d04
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_ca.tar.gz) = 2240361
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_ca.tar.gz) = 927f95db71a633b68c9e1f087fb99a454a74d88ca169c6639ab5639cb0c34b82
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_ca.tar.gz) = 10915365
