--- pkg-plist.orig	2011-07-12 14:44:50.000000000 +0000
+++ pkg-plist
@@ -64,9 +64,11 @@ include/mysql/typelib.h
 lib/mysql/libmysqlclient.a
 lib/mysql/libmysqlclient.so
 lib/mysql/libmysqlclient.so.18
+lib/mysql/libmysqlclient.so.18.0.0
 lib/mysql/libmysqlclient_r.a
 lib/mysql/libmysqlclient_r.so
 lib/mysql/libmysqlclient_r.so.18
+lib/mysql/libmysqlclient_r.so.18.0.0
 lib/mysql/libmysqlservices.a
 share/aclocal/mysql.m4
 @dirrm include/mysql/mysql/psi
