--- distinfo.orig	2022-07-23 05:35:58 UTC
+++ distinfo
@@ -1,4 +1,6 @@
 TIMESTAMP = 1657195709
+SHA256 (go-dragonfly-amd64-go1.17.9.tar.xz) = 02416ddcb1154819ef849ba1de1904c660fbc175bc89b6021f14234db450e8da
+SIZE (go-dragonfly-amd64-go1.17.9.tar.xz) = 41653156
 SHA256 (go-freebsd-arm64-go1.17.9.tar.xz) = d9e9180bdc5ad0eec1654679a50084ebb31ecbe9bef24f0bd8ba917db8d1830c
 SIZE (go-freebsd-arm64-go1.17.9.tar.xz) = 38572776
 SHA256 (golang-go-g20220706-bac4eb53d64ee402af1d52ac18fb9f0ea76c74e2_GH0.tar.gz) = cc13b0d7101239a9f5a16e6cf2c9dce419ba503f669ce8636338834f078cc34a
