--- Makefile.inc.orig	2013-06-20 14:12:36.952839000 +0000
+++ Makefile.inc
@@ -4,7 +4,7 @@
 MASTER_SITES?=	${MASTER_SITE_GNU}
 MASTER_SITE_SUBDIR?=	aspell/dict/${DICT}
 DISTNAME?=	aspell${DISTVERSION}
-PORTVERSION?=	${DISTVERSION:L:C/cvs/0_/:C/pt_../p/:C/[0-9]?-?[a-z]+-//g:C/([0-9])([a-z])/\1.\2/g:C/:(.)/\1/g:C/[^a-z0-9+]+/./g}
+PORTVERSION?=	${DISTVERSION:tl:C/cvs/0_/:C/pt_../p/:C/[0-9]?-?[a-z]+-//g:C/([0-9])([a-z])/\1.\2/g:C/:(.)/\1/g:C/[^a-z0-9+]+/./g}
 
 LIB_DEPENDS=	aspell:${PORTSDIR}/textproc/aspell
 
