--- pkg-plist.orig	2024-01-25 18:26:04 UTC
+++ pkg-plist
@@ -203,7 +203,6 @@ llvm%%LLVM_SUFFIX%%/bin/dsymutil
 %%CLANG%%llvm%%LLVM_SUFFIX%%/bin/git-clang-format
 %%CLANG%%llvm%%LLVM_SUFFIX%%/bin/hmaptool
 %%CLANG%%llvm%%LLVM_SUFFIX%%/bin/intercept-build
-%%LLD%%llvm%%LLVM_SUFFIX%%/bin/ld
 %%LLD%%llvm%%LLVM_SUFFIX%%/bin/ld.lld
 %%LLD%%llvm%%LLVM_SUFFIX%%/bin/ld64.lld
 llvm%%LLVM_SUFFIX%%/bin/llc
@@ -258,7 +257,6 @@ llvm%%LLVM_SUFFIX%%/bin/llvm-mt
 llvm%%LLVM_SUFFIX%%/bin/llvm-nm
 llvm%%LLVM_SUFFIX%%/bin/llvm-objcopy
 llvm%%LLVM_SUFFIX%%/bin/llvm-objdump
-%%OPENMP%%llvm%%LLVM_SUFFIX%%/bin/llvm-omp-device-info
 %%OPENMP%%llvm%%LLVM_SUFFIX%%/bin/llvm-omp-kernel-replay
 llvm%%LLVM_SUFFIX%%/bin/llvm-opt-report
 llvm%%LLVM_SUFFIX%%/bin/llvm-otool
@@ -6390,8 +6388,6 @@ llvm%%LLVM_SUFFIX%%/lib/libRemarks.so.%%
 %%MLIR%%llvm%%LLVM_SUFFIX%%/lib/libmlir_runner_utils.so
 %%MLIR%%llvm%%LLVM_SUFFIX%%/lib/libmlir_runner_utils.so.%%LLVM_MAJOR%%
 %%OPENMP%%llvm%%LLVM_SUFFIX%%/lib/libomp.so
-%%OPENMP%%llvm%%LLVM_SUFFIX%%/lib/libomptarget.so
-%%OPENMP%%llvm%%LLVM_SUFFIX%%/lib/libomptarget.so.%%LLVM_MAJOR%%
 %%CLANG%%llvm%%LLVM_SUFFIX%%/lib/libscanbuild/__init__.py
 %%CLANG%%llvm%%LLVM_SUFFIX%%/lib/libscanbuild/analyze.py
 %%CLANG%%llvm%%LLVM_SUFFIX%%/lib/libscanbuild/arguments.py
