--- Makefile.orig	2019-10-26 10:14:15 UTC
+++ Makefile
@@ -21,13 +21,13 @@ RUN_DEPENDS=	${BU_PREFIX}-as:devel/${PKG
 
 USES=	gmake iconv libtool tar:xz makeinfo compiler:c++11-lang
 PLIST_SUB=	TARGETARCH=${PKGNAMEPREFIX:C/-//g} \
-		OPSYS=${OPSYS:tl} \
 		GCC_TARGET=${GCC_TARGET} \
 		GCC_VERSION=${PORTVERSION}
 
 TARGETARCH=	${PKGNAMEPREFIX:C/-//g}
 
-GCC_TARGET=	${PKGNAMEPREFIX:S/amd64/x86_64/}unknown-${OPSYS:tl}${OSREL}
+GCC_TARGET=	${PKGNAMEPREFIX:S/amd64/x86_64/}unknown-freebsd10
+PLIST_SUB+=	OPSYS=freebsd OSREL=10
 BU_PREFIX?=	${GCC_TARGET}
 
 GNU_CONFIGURE=	yes
