--- Makefile.orig
+++ Makefile
@@ -130,6 +130,7 @@
 CONFIGURE_ARGS+=	--with-toolchain-type=${COMPILER_TYPE}
 CONFIGURE_ARGS+=	--disable-warnings-as-errors
 
+.if ${OPSYS} == "FreeBSD"
 .if ${COMPILER_TYPE} == gcc
 USE_GCC=	yes
 CONFIGURE_ARGS+=	--with-extra-ldflags="-Wl,-rpath=${LOCALBASE}/lib/gcc${GCC_DEFAULT} -L${LOCALBASE}/lib/gcc${GCC_DEFAULT}" \
@@ -151,6 +152,7 @@
 ICONV_LDFLAGS=	-L${LOCALBASE}/lib
 ICONV_LIBS=	${ICONV_LIB}
 .endif
+.endif
 
 pre-configure:
 	@${FIND} ${WRKSRC} -name '*.orig' -delete
@@ -171,6 +173,9 @@
 
 do-install:
 	@${MKDIR} ${STAGEDIR}${INSTALLDIR}
+.if !defined(BUILD_JRE)
+	${CP} -rv ${JDK_IMAGEDIR}/include/freebsd ${JDK_IMAGEDIR}/include/dragonfly
+.endif
 	@cd ${JDK_IMAGEDIR} && ${COPYTREE_SHARE} . ${STAGEDIR}${INSTALLDIR}
 	@cd ${STAGEDIR}${INSTALLDIR} && \
 		${FIND} bin -type f -exec ${CHMOD} ${BINMODE} {} \;
