--- Makefile.common.orig	2015-12-28 12:07:41 UTC
+++ Makefile.common
@@ -60,6 +60,7 @@ SHEBANG_FILES+=	src/mapi/mapi_abi.py
 
 MASTERDIR=		${.CURDIR}/../../graphics/libGL
 PATCHDIR=		${MASTERDIR}/files
+DFLY_PATCHDIR=		${MASTERDIR}/dragonfly
 WRKSRC=			${WRKDIR}/mesa-${MESADISTVERSION}
 DESCR=			${.CURDIR}/pkg-descr
 PLIST=			${.CURDIR}/pkg-plist
@@ -175,5 +176,6 @@ CONFIGURE_ARGS+=--with-gallium-drivers="
 
 .if ${OPSYS} == DragonFly
 LIB_DEPENDS+= libelf.so:${PORTSDIR}/devel/libelf
+CFLAGS+= -I${LOCALBASE}/include/libelf
 .endif
 
