--- Makefile.common.orig	2015-10-17 16:01:48 +0200
+++ Makefile.common
@@ -60,6 +60,7 @@
 
 MASTERDIR=		${.CURDIR}/../../graphics/libGL
 PATCHDIR=		${MASTERDIR}/files
+DFLY_PATCHDIR=		${MASTERDIR}/dragonfly
 WRKSRC=			${WRKDIR}/mesa-${MESADISTVERSION}
 DESCR=			${.CURDIR}/pkg-descr
 PLIST=			${.CURDIR}/pkg-plist
@@ -75,8 +76,7 @@
 CONFIGURE_ARGS+=	--enable-gbm
 
 # libEGL, dri and clover need gallium enabled.
-# the third option in --with-egl-platforms is wayland.
-CONFIGURE_ARGS+=	--enable-egl --with-egl-platforms=x11,drm
+CONFIGURE_ARGS+=	--enable-egl --with-egl-platforms=x11,drm,wayland
 
 CONFIGURE_ARGS+=	--enable-gles2
 
@@ -129,16 +129,17 @@
 CONFIGURE_ARGS+=--enable-vdpau
 LIB_DEPENDS+=   libvdpau.so:${PORTSDIR}/multimedia/libvdpau
 PLIST_SUB+=     VDPAU=""
-.else
+# for now disable too .else
 CONFIGURE_ARGS+=--disable-vdpau
 PLIST_SUB+=     VDPAU="@comment "
 .endif
 
-# what to do with this one? Currently disabled in dri.
-.if ${COMPONENT:Mvdpau} == ""
+.if ${COMPONENT:Mvdpau} == "" && ${COMPONENT:Mdri} == ""
 CONFIGURE_ARGS+=--disable-vdpau
 .else
 CONFIGURE_ARGS+=--enable-vdpau
+LIB_DEPENDS+=   libvdpau.so:${PORTSDIR}/multimedia/libvdpau
+PLIST_SUB+=     VDPAU=""
 .endif
 
 ALL_DRI_DRIVERS=I915 I965 R200 RADEON SWRAST
