--- Makefile.common.orig	2015-11-22 16:21:02 UTC
+++ Makefile.common
@@ -36,7 +36,7 @@ USES+=		compiler:c++11-lib gettext-tools
 USE_LDCONFIG=	yes
 GNU_CONFIGURE=	yes
 CPPFLAGS+=	-isystem${LOCALBASE}/include
-LDFLAGS+=	-Wl,-Y${LOCALBASE}/lib
+LDFLAGS+=	-Wl,-Y/usr/lib:${LOCALBASE}/lib
 
 PKGINSTALL=	${.CURDIR}/pkg-install
 PKGDEINSTALL=	${.CURDIR}/pkg-deinstall
@@ -60,6 +60,7 @@ SHEBANG_FILES+=	src/mapi/mapi_abi.py
 
 MASTERDIR=		${.CURDIR}/../../graphics/libGL
 PATCHDIR=		${MASTERDIR}/files
+DFLY_PATCHDIR=		${MASTERDIR}/dragonfly
 WRKSRC=			${WRKDIR}/mesa-${MESADISTVERSION}
 DESCR=			${.CURDIR}/pkg-descr
 PLIST=			${.CURDIR}/pkg-plist
@@ -127,16 +128,17 @@ IGNORE= VDPAU option requires GALLIUM su
 CONFIGURE_ARGS+=--enable-vdpau
 LIB_DEPENDS+=   libvdpau.so:${PORTSDIR}/multimedia/libvdpau
 PLIST_SUB+=     VDPAU=""
-.else
+# for now disable too .else
 CONFIGURE_ARGS+=--disable-vdpau
 PLIST_SUB+=     VDPAU="@comment "
 .endif
 
-# what to do with this one? Currently disabled in dri.
-.if ${COMPONENT:Mvdpau} == ""
+.if ${COMPONENT:Mvdpau} == "" && ${COMPONENT:Mdri} == ""
 CONFIGURE_ARGS+=--disable-vdpau
 .else
 CONFIGURE_ARGS+=--enable-vdpau
+LIB_DEPENDS+=   libvdpau.so:${PORTSDIR}/multimedia/libvdpau
+PLIST_SUB+=     VDPAU=""
 .endif
 
 # VAAPI support?
