--- pkg-plist.orig	2020-04-16 11:48:44 UTC
+++ pkg-plist
@@ -1151,8 +1151,6 @@ lib/calibre/calibre/plugins/%%SODIR%%hyp
 lib/calibre/calibre/plugins/%%SODIR%%icu.so
 lib/calibre/calibre/plugins/%%SODIR%%imageops.so
 lib/calibre/calibre/plugins/%%SODIR%%libheadless.so
-lib/calibre/calibre/plugins/%%SODIR%%libmtp.so
-lib/calibre/calibre/plugins/%%SODIR%%libusb.so
 lib/calibre/calibre/plugins/%%SODIR%%lzma_binding.so
 lib/calibre/calibre/plugins/%%SODIR%%lzx.so
 lib/calibre/calibre/plugins/%%SODIR%%matcher.so
