--- pkg-plist.orig
+++ pkg-plist
@@ -15,6 +15,19 @@
 share/help/C/baobab/scan-folder.page
 share/help/C/baobab/scan-home.page
 share/help/C/baobab/scan-remote.page
+share/help/bg/baobab/index.page
+share/help/bg/baobab/introduction.page
+share/help/bg/baobab/legal.xml
+share/help/bg/baobab/media/org.gnome.baobab.svg
+share/help/bg/baobab/pref-view-chart.page
+share/help/bg/baobab/problem-permissions.page
+share/help/bg/baobab/problem-slow-scan.page
+share/help/bg/baobab/question-open-folder.page
+share/help/bg/baobab/question-trash.page
+share/help/bg/baobab/scan-file-system.page
+share/help/bg/baobab/scan-folder.page
+share/help/bg/baobab/scan-home.page
+share/help/bg/baobab/scan-remote.page
 share/help/ca/baobab/index.page
 share/help/ca/baobab/introduction.page
 share/help/ca/baobab/legal.xml
@@ -197,6 +210,19 @@
 share/help/it/baobab/scan-folder.page
 share/help/it/baobab/scan-home.page
 share/help/it/baobab/scan-remote.page
+share/help/kk/baobab/index.page
+share/help/kk/baobab/introduction.page
+share/help/kk/baobab/legal.xml
+share/help/kk/baobab/media/org.gnome.baobab.svg
+share/help/kk/baobab/pref-view-chart.page
+share/help/kk/baobab/problem-permissions.page
+share/help/kk/baobab/problem-slow-scan.page
+share/help/kk/baobab/question-open-folder.page
+share/help/kk/baobab/question-trash.page
+share/help/kk/baobab/scan-file-system.page
+share/help/kk/baobab/scan-folder.page
+share/help/kk/baobab/scan-home.page
+share/help/kk/baobab/scan-remote.page
 share/help/ko/baobab/index.page
 share/help/ko/baobab/introduction.page
 share/help/ko/baobab/legal.xml
@@ -389,6 +415,7 @@
 share/locale/hi/LC_MESSAGES/baobab.mo
 share/locale/hr/LC_MESSAGES/baobab.mo
 share/locale/hu/LC_MESSAGES/baobab.mo
+share/locale/ia/LC_MESSAGES/baobab.mo
 share/locale/id/LC_MESSAGES/baobab.mo
 share/locale/ie/LC_MESSAGES/baobab.mo
 share/locale/is/LC_MESSAGES/baobab.mo
@@ -401,6 +428,7 @@
 share/locale/kn/LC_MESSAGES/baobab.mo
 share/locale/ko/LC_MESSAGES/baobab.mo
 share/locale/ku/LC_MESSAGES/baobab.mo
+share/locale/kw/LC_MESSAGES/baobab.mo
 share/locale/lt/LC_MESSAGES/baobab.mo
 share/locale/lv/LC_MESSAGES/baobab.mo
 share/locale/mai/LC_MESSAGES/baobab.mo
@@ -440,10 +468,11 @@
 share/locale/tr/LC_MESSAGES/baobab.mo
 share/locale/ug/LC_MESSAGES/baobab.mo
 share/locale/uk/LC_MESSAGES/baobab.mo
+share/locale/uz/LC_MESSAGES/baobab.mo
 share/locale/vi/LC_MESSAGES/baobab.mo
 share/locale/wa/LC_MESSAGES/baobab.mo
 share/locale/xh/LC_MESSAGES/baobab.mo
 share/locale/zh_CN/LC_MESSAGES/baobab.mo
 share/locale/zh_HK/LC_MESSAGES/baobab.mo
 share/locale/zh_TW/LC_MESSAGES/baobab.mo
-share/metainfo/org.gnome.baobab.appdata.xml
+share/metainfo/org.gnome.baobab.metainfo.xml
