--- pkg-plist.orig	2023-02-19 21:54:39 UTC
+++ pkg-plist
@@ -19,7 +19,6 @@ bin/regpatch
 bin/regshell
 bin/regtree
 bin/rpcclient
-bin/samba-regedit
 bin/sharesec
 bin/smbcacls
 bin/smbclient
@@ -165,8 +164,6 @@ include/samba4/wbclient.h
 %%SAMBA4_LIBDIR%%/libtevent-util.so.0
 %%SAMBA4_LIBDIR%%/libwbclient.so
 %%SAMBA4_LIBDIR%%/libwbclient.so.0
-lib/nss_winbind.so.1
-lib/nss_wins.so.1
 lib/pam_winbind.so
 %%CUPS%%libexec/samba/smbspool_krb5_wrapper
 %%SAMBA4_LIBDIR%%/private/libCHARSET3-samba4.so
@@ -391,7 +388,6 @@ man/man8/vfs_virusfilter.8.gz
 man/man8/vfs_widelinks.8.gz
 man/man8/vfs_worm.8.gz
 man/man8/vfs_xattr_tdb.8.gz
-man/man8/vfs_zfsacl.8.gz
 man/man8/winbind_krb5_locator.8.gz
 man/man8/winbindd.8.gz
 @dir %%SAMBA4_RUNDIR%%
