--- pkg-plist.orig	2012-11-02 15:35:45.000000000 +0000
+++ pkg-plist
@@ -654,7 +654,6 @@
 %%WWWDIR%%/admin/editor/themes/advanced/skins/highcontrast/ui.css
 %%WWWDIR%%/admin/editor/themes/advanced/skins/cirkuit/content.css
 %%WWWDIR%%/admin/editor/themes/advanced/skins/cirkuit/dialog.css
-%%WWWDIR%%/admin/editor/themes/advanced/skins/cirkuit/img
 %%WWWDIR%%/admin/editor/themes/advanced/skins/cirkuit/img/butt2.png
 %%WWWDIR%%/admin/editor/themes/advanced/skins/cirkuit/img/button-bg.png
 %%WWWDIR%%/admin/editor/themes/advanced/skins/cirkuit/img/buttons.png
