--- pkg-plist.orig	2014-01-26 22:02:56.000000000 +0000
+++ pkg-plist
@@ -976,6 +976,8 @@ lib/lib%%PYTHON_VERSION%%.so.1
 %%PYTHON_LIBDIR%%/pickletools.py
 %%PYTHON_LIBDIR%%/pipes.py
 %%PYTHON_LIBDIR%%/pkgutil.py
+%%PYTHON_LIBDIR%%/plat-dragonfly3/IN.py
+%%PYTHON_LIBDIR%%/plat-dragonfly3/regen
 %%PYTHON_LIBDIR%%/plat-freebsd6/IN.py
 %%PYTHON_LIBDIR%%/plat-freebsd6/regen
 %%PYTHON_LIBDIR%%/plat-freebsd7/IN.py
@@ -2407,6 +2409,7 @@ man/man1/python2.7.1.gz
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd8
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd7
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd6
+@dirrm %%PYTHON_LIBDIR%%/plat-dragonfly3
 @dirrm %%PYTHON_LIBDIR%%/multiprocessing/dummy
 @dirrm %%PYTHON_LIBDIR%%/multiprocessing
 @dirrm %%PYTHON_LIBDIR%%/logging
