--- pkg-plist.orig	2024-06-22 11:53:08 UTC
+++ pkg-plist
@@ -2474,10 +2474,10 @@ lib/python2.7/pipes.pyo
 lib/python2.7/pkgutil.py
 lib/python2.7/pkgutil.pyc
 lib/python2.7/pkgutil.pyo
-lib/python2.7/plat-freebsd%%OSMAJOR%%/IN.py
-lib/python2.7/plat-freebsd%%OSMAJOR%%/IN.pyc
-lib/python2.7/plat-freebsd%%OSMAJOR%%/IN.pyo
-lib/python2.7/plat-freebsd%%OSMAJOR%%/regen
+lib/python2.7/plat-dragonfly%%OSMAJOR%%/IN.py
+lib/python2.7/plat-dragonfly%%OSMAJOR%%/IN.pyc
+lib/python2.7/plat-dragonfly%%OSMAJOR%%/IN.pyo
+lib/python2.7/plat-dragonfly%%OSMAJOR%%/regen
 lib/python2.7/platform.py
 lib/python2.7/platform.pyc
 lib/python2.7/platform.pyo
