--- pkg-plist.orig	2018-07-09 00:21:49 UTC
+++ pkg-plist
@@ -2973,6 +2973,9 @@ share/locale/de/LC_MESSAGES/mcs.mo
 share/locale/es/LC_MESSAGES/mcs.mo
 share/locale/ja/LC_MESSAGES/mcs.mo
 share/locale/pt_BR/LC_MESSAGES/mcs.mo
+lib/mono/4.5/Mono.Btls.Interface.dll
+lib/mono/gac/Mono.Btls.Interface/4.0.0.0__0738eb9f132ed756/Mono.Btls.Interface.dll
+lib/mono/gac/Mono.Btls.Interface/4.0.0.0__0738eb9f132ed756/Mono.Btls.Interface.pdb
 %%DATADIR%%-2.0/mono/cil/cil-opcodes.xml
 %%DATADIR%%-2.0/mono/profiler/mono-profiler-coverage.suppression
 %%DATADIR%%-2.0/mono/profiler/mono-profiler-log.suppression
