--- pkg-plist.orig	2021-06-20 11:23:54 UTC
+++ pkg-plist
@@ -2969,11 +2969,11 @@ man/man1/coqtop.byte.1.gz
 man/man1/coqtop.opt.1.gz
 man/man1/coqwc.1.gz
 man/man1/gallina.1.gz
-%%PORTDOCS%%%%DATADIR%%/coq-ssreflect.lang
-%%PORTDOCS%%%%DATADIR%%/coq.lang
-%%PORTDOCS%%%%DATADIR%%/coq.png
-%%PORTDOCS%%%%DATADIR%%/coq_style.xml
-%%PORTDOCS%%%%DOCSDIR%%/FAQ-CoqIde
+%%IDE%%%%PORTDOCS%%%%DATADIR%%/coq-ssreflect.lang
+%%IDE%%%%PORTDOCS%%%%DATADIR%%/coq.lang
+%%IDE%%%%PORTDOCS%%%%DATADIR%%/coq.png
+%%IDE%%%%PORTDOCS%%%%DATADIR%%/coq_style.xml
+%%IDE%%%%PORTDOCS%%%%DOCSDIR%%/FAQ-CoqIde
 %%EMACS_SITE_LISPDIR%%/coq/coq-font-lock.el
 %%EMACS_SITE_LISPDIR%%/coq/coq-inferior.el
 %%EMACS_SITE_LISPDIR%%/coq/gallina-db.el
