--- pkg-plist.orig	Tue Apr  1 22:13:52 2025
+++ pkg-plist	Tue Apr
@@ -20,10 +20,10 @@ bin/csdpcert
 bin/ocamllibdep
 bin/votour
 %%IDE%%@dir etc/xdg/coq
-%%PORTDOCS%%%%DATADIR%%/coq-ssreflect.lang
-%%PORTDOCS%%%%DATADIR%%/coq.lang
-%%PORTDOCS%%%%DATADIR%%/coq.png
-%%PORTDOCS%%%%DATADIR%%/coq_style.xml
+%%IDE%%%%PORTDOCS%%%%DATADIR%%/coq-ssreflect.lang
+%%IDE%%%%PORTDOCS%%%%DATADIR%%/coq.lang
+%%IDE%%%%PORTDOCS%%%%DATADIR%%/coq.png
+%%IDE%%%%PORTDOCS%%%%DATADIR%%/coq_style.xml
 @comment %%PORTDOCS%%%%DOCSDIR%%/FAQ-CoqIde
 @comment %%EMACS_SITE_LISPDIR%%/coq/coq-font-lock.el
 @comment %%EMACS_SITE_LISPDIR%%/coq/coq-inferior.el
