--- distinfo.orig	2017-07-11 12:42:20 UTC
+++ distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1498129310
-SHA256 (apache24/httpd-2.4.26.tar.bz2) = a07eb52fafc879e0149d31882f7da63173e72df4478db4dc69f7a775b663d387
-SIZE (apache24/httpd-2.4.26.tar.bz2) = 6524642
+TIMESTAMP = 1499776956
+SHA256 (apache24/httpd-2.4.27.tar.bz2) = 71fcc128238a690515bd8174d5330a5309161ef314a326ae45c7c15ed139c13a
+SIZE (apache24/httpd-2.4.27.tar.bz2) = 6527394
