--- Makefile.orig	2014-03-25 22:17:31.466740000 +0000
+++ Makefile
@@ -44,12 +44,21 @@ NLS_CONFIGURE_ENABLE=	nls
 XML_USE=		GNOME=libxml2
 XML_CONFIGURE_WITH=	libxml2
 
+USES+=	usb
+USBVER= usb1
+
 .include <bsd.port.options.mk>
 
 .if ${OPSYS} == FreeBSD && ${OSVERSION} < 1000040
 CONFIGURE_ENV+=	LIBUSB1_CFLAGS=" " LIBUSB1_LIBS="-lusb"
 .endif
 
+.if ${OPSYS} == DragonFly && ${DFLYVERSION} < 300703
+USBVER= usb
+.endif
+
+PLIST_SUB+= USB=${USBVER}
+
 post-patch:
 .for i in configure libgphoto2_port/configure
 	@${REINPLACE_CMD} -e \
