--- pkg-plist.orig	2014-09-13 14:07:51 UTC
+++ pkg-plist
@@ -92,7 +92,7 @@ lib/libgphoto2_port.so.10
 lib/libgphoto2_port.so.10.1.1
 lib/libgphoto2_port/%%VERSION_PORT%%/disk.so
 lib/libgphoto2_port/%%VERSION_PORT%%/ptpip.so
-lib/libgphoto2_port/%%VERSION_PORT%%/usb1.so
+lib/libgphoto2_port/%%VERSION_PORT%%/%%USB%%.so
 lib/libgphoto2_port/%%VERSION_PORT%%/usbdiskdirect.so
 lib/libgphoto2_port/%%VERSION_PORT%%/usbscsi.so
 libdata/pkgconfig/libgphoto2.pc
