--- pkg-plist.orig	2014-05-05 13:50:02.820934000 +0000
+++ pkg-plist
@@ -162,8 +162,8 @@ lib/libgphoto2_port/%%VERSION_PORT%%/ptp
 lib/libgphoto2_port/%%VERSION_PORT%%/ptpip.so
 lib/libgphoto2_port/%%VERSION_PORT%%/serial.la
 lib/libgphoto2_port/%%VERSION_PORT%%/serial.so
-lib/libgphoto2_port/%%VERSION_PORT%%/usb1.la
-lib/libgphoto2_port/%%VERSION_PORT%%/usb1.so
+lib/libgphoto2_port/%%VERSION_PORT%%/%%USB%%.la
+lib/libgphoto2_port/%%VERSION_PORT%%/%%USB%%.so
 lib/libgphoto2_port/%%VERSION_PORT%%/usbdiskdirect.la
 lib/libgphoto2_port/%%VERSION_PORT%%/usbdiskdirect.so
 lib/libgphoto2_port/%%VERSION_PORT%%/usbscsi.la
