--- distinfo.orig	2013-01-05 21:49:59.906808000 +0100
+++ distinfo	2013-01-05 21:37:55.365796000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_ca-XV.tar.gz) = 6094f916d8a50747671c5ad744c6b08df7a9c14b6de3079ab9b6c132ce7007e2
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_ca-XV.tar.gz) = 1639883
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_ca-XV.tar.gz) = 7477172019c65f065212aba81402e950553072205eca4ccf5662a64a2ad19a93
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_ca-XV.tar.gz) = 10929214
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_ca-XV.tar.gz) = bbdb4df7dd1b19dd01143fa09048142c7ffd2f216ee0c96441c1f3ba95093855
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_ca-XV.tar.gz) = 1590122
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_ca-XV.tar.gz) = c1843c6243edeb3f10c8775862aef1ed380a379bed9df0bfd664e4b9398b0bb3
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_ca-XV.tar.gz) = 10939865
