--- Makefile.orig	2013-09-09 16:35:48.557554000 +0000
+++ Makefile
@@ -22,6 +22,8 @@ LICENSE_COMB=	multi
 WRKSRC=		${WRKDIR}/${PORTNAME}-${DISTVERSION}
 CFLAGS+=	-fno-stack-protector
 
+CFLAGS+=	-D__FreeBSD__=9
+
 BUILD_DEPENDS=	${LOCALBASE}/bin/gperf:${PORTSDIR}/devel/gperf \
 		bash:${PORTSDIR}/shells/bash \
 		yasm:${PORTSDIR}/devel/yasm \
@@ -42,6 +44,8 @@ LIB_DEPENDS=	libexecinfo.so:${PORTSDIR}/
 		libgcrypt.so:${PORTSDIR}/security/libgcrypt \
 		libpci.so:${PORTSDIR}/devel/libpci
 
+LIB_DEPENDS+=	gssapi_krb5:${PORTSDIR}/security/krb5
+
 RUN_DEPENDS=	${LOCALBASE}/lib/alsa-lib/libasound_module_pcm_oss.so:${PORTSDIR}/audio/alsa-plugins \
 		${LOCALBASE}/lib/X11/fonts/Droid/fonts.dir:${PORTSDIR}/x11-fonts/droid-fonts-ttf
 
@@ -131,15 +135,11 @@ USE_GCC?=	yes
 GYP_DEFINES+=	gcc_version=${CXX:S/g++//}
 EXTRA_PATCHES+=	${FILESDIR}/extra-patch-gcc
 .else
-.if ${OSVERSION} < 902000
 BUILD_DEPENDS+=	${LOCALBASE}/bin/clang:${PORTSDIR}/lang/clang
 CC=		${LOCALBASE}/bin/clang
 CXX=		${LOCALBASE}/bin/clang++
-.else
-CC=		clang
-CXX=		clang++
-.endif
 GYP_DEFINES+=	clang=1
+CXXFLAGS+=	-D__float128=void -D_GLIBCXX_PERMIT_BACKWARD_HASH=1
 EXTRA_PATCHES+=	${FILESDIR}/extra-patch-clang
 .endif
 
@@ -180,6 +180,7 @@ CONFIGURE_ENV+= CFLAGS="${CFLAGS}" \
 		CXXFLAGS="${CXXFLAGS}" \
 		LDFLAGS="${LDFLAGS}"
 MAKE_ENV+=	BUILDTYPE=${BUILDTYPE} \
+		GYPFLAGS=-DOS=freebsd \
 		GPERF=${LOCALBASE}/bin/gperf
 
 .include <bsd.port.pre.mk>
@@ -196,7 +197,9 @@ pre-everything::
 
 post-patch:
 	@${REINPLACE_CMD} -e "s|/usr/local|${LOCALBASE}|" \
+		${WRKSRC}/base/base_paths_posix.cc \
 		${WRKSRC}/base/base.gypi \
+		${WRKSRC}/base/process_util_freebsd.cc \
 		${WRKSRC}/build/common.gypi \
 		${WRKSRC}/third_party/libvpx/libvpx.gyp \
 		${WRKSRC}/crypto/crypto.gyp \
@@ -206,7 +209,7 @@ post-patch:
 		${WRKSRC}/third_party/WebKit/Source/core/core.gypi
 	@${REINPLACE_CMD} -e "s|'../courgette/courgette.gyp:courgette_lib',||" \
 		${WRKSRC}/chrome/chrome_tests_unit.gypi
-	@${REINPLACE_CMD} -e "s|linux|freebsd|" \
+	@${REINPLACE_CMD} -e "s|linux|dragonfly|" \
 		${WRKSRC}/tools/gyp/pylib/gyp/generator/make.py
 	@${REINPLACE_CMD} -e "s|is_linux|(is_linux or is_bsd)|" \
 		${WRKSRC}/chrome/browser/browser_resources.grd \
@@ -219,6 +222,12 @@ post-patch:
 		${WRKSRC}/ui/base/strings/app_locale_settings_ko.xtb \
 		${WRKSRC}/ui/base/strings/app_locale_settings_ml.xtb \
 		${WRKSRC}/ui/base/strings/app_locale_settings_th.xtb
+	@${FIND} ${WRKSRC} -type f -name \*\.gyp\* | grep -v /v8/ | \
+		${XARGS} grep -l '"freebsd"' | \
+		${XARGS} ${REINPLACE_CMD} -e 's|"freebsd"|"dragonfly"|g'
+	@${FIND} ${WRKSRC} -type f -name \*\.py | grep -v /v8/ | \
+		${XARGS} grep -l "'freebsd'" | \
+		${XARGS} ${REINPLACE_CMD} -e "s|'freebsd'|'dragonfly'|g"
 
 do-configure:
 	cd ${WRKSRC} && ${SETENV} ${CONFIGURE_ENV} ${PYTHON_CMD} \
