--- Makefile.orig	2014-01-05 12:40:40.339259000 +0000
+++ Makefile
@@ -21,7 +21,8 @@ LICENSE=	BSD3CLAUSE LGPL21 MPL
 LICENSE_COMB=	multi
 
 WRKSRC=		${WRKDIR}/${PORTNAME}-${DISTVERSION}
-CFLAGS+=	-fno-stack-protector -Wno-unknown-warning-option
+CFLAGS+=	-fno-stack-protector
+CFLAGS+=	-D__FreeBSD__=9
 
 BUILD_DEPENDS=	${LOCALBASE}/bin/gperf:${PORTSDIR}/devel/gperf \
 		bash:${PORTSDIR}/shells/bash \
@@ -42,6 +43,8 @@ LIB_DEPENDS=	libcairo.so:${PORTSDIR}/gra
 		libgcrypt.so:${PORTSDIR}/security/libgcrypt \
 		libpci.so:${PORTSDIR}/devel/libpci
 
+LIB_DEPENDS+=	gssapi_krb5:${PORTSDIR}/security/krb5
+
 RUN_DEPENDS=	${LOCALBASE}/lib/alsa-lib/libasound_module_pcm_oss.so:${PORTSDIR}/audio/alsa-plugins \
 		${LOCALBASE}/lib/X11/fonts/Droid/fonts.dir:${PORTSDIR}/x11-fonts/droid-fonts-ttf
 
@@ -124,10 +127,10 @@ GYP_DEFINES+=	use_pulseaudio=0
 GYP_DEFINES+=	disable_sse2=1
 .endif
 
-.if ${OSVERSION} >= 1000052 && !exists(${LOCALBASE}/include/execinfo.h)
+.if ${OPSYS} == FreeBSD && ${OSVERSION} >= 1000052 && !exists(${LOCALBASE}/include/execinfo.h)
 EXTRA_PATCHES+=	${FILESDIR}/extra-patch-fixup-execinfo
 .endif
-.if ${OSVERSION} < 1000052
+.if ${OPSYS} == DragonFly || (${OPSYS} == FreeBSD && ${OSVERSION} < 1000052)
 LIB_DEPENDS+=	libexecinfo.so:${PORTSDIR}/devel/libexecinfo
 .endif
 
@@ -167,6 +170,7 @@ CONFIGURE_ENV+=	CFLAGS="${CFLAGS}" \
 		CXXFLAGS="${CXXFLAGS}" \
 		LDFLAGS="${LDFLAGS}"
 MAKE_ENV+=	BUILDTYPE=${BUILDTYPE} \
+		GYPFLAGS=-DOS=freebsd \
 		GPERF=${LOCALBASE}/bin/gperf
 
 .include <bsd.port.pre.mk>
@@ -176,6 +180,7 @@ GYP_DEFINES+=	gcc_version=${CXX:S/g++//}
 EXTRA_PATCHES+=	${FILESDIR}/extra-patch-gcc
 .else
 GYP_DEFINES+=	clang=1
+CXXFLAGS+=	-D__float128=void -D_GLIBCXX_PERMIT_BACKWARD_HASH=1
 EXTRA_PATCHES+=	${FILESDIR}/extra-patch-clang
 CONFIGURE_ENV+=	AR=/usr/bin/ar
 .endif
@@ -192,7 +197,9 @@ pre-everything::
 
 post-patch:
 	@${REINPLACE_CMD} -e "s|/usr/local|${LOCALBASE}|" \
+		${WRKSRC}/base/base_paths_posix.cc \
 		${WRKSRC}/base/base.gypi \
+		${WRKSRC}/base/process/process_handle_freebsd.cc \
 		${WRKSRC}/build/common.gypi \
 		${WRKSRC}/crypto/crypto.gyp \
 		${WRKSRC}/third_party/libvpx/libvpx.gyp \
@@ -205,7 +212,7 @@ post-patch:
 		${WRKSRC}/third_party/WebKit/Source/core/core.gypi
 	@${REINPLACE_CMD} -e "s|'../courgette/courgette.gyp:courgette_lib',||" \
 		${WRKSRC}/chrome/chrome_tests_unit.gypi
-	@${REINPLACE_CMD} -e "s|linux|freebsd|" \
+	@${REINPLACE_CMD} -e "s|linux|dragonfly|" \
 		${WRKSRC}/tools/gyp/pylib/gyp/generator/make.py
 	@${REINPLACE_CMD} -e "s|is_linux|(is_linux or is_bsd)|" \
 		${WRKSRC}/chrome/browser/browser_resources.grd \
@@ -218,6 +225,17 @@ post-patch:
 		${WRKSRC}/ui/base/strings/app_locale_settings_ko.xtb \
 		${WRKSRC}/ui/base/strings/app_locale_settings_ml.xtb \
 		${WRKSRC}/ui/base/strings/app_locale_settings_th.xtb
+	@${FIND} ${WRKSRC} -type f -name \*\.gyp\* | grep -v /v8/ | \
+		${XARGS} grep -l '"freebsd"' | \
+		${XARGS} ${REINPLACE_CMD} -e 's|"freebsd"|"dragonfly"|g'
+	@${FIND} ${WRKSRC} -type f -name \*\.py | grep -v /v8/ | \
+		${XARGS} grep -l "'freebsd'" | \
+		${XARGS} ${REINPLACE_CMD} -e "s|'freebsd'|'dragonfly'|g"
+	@${REINPLACE_CMD} -e 's|"freebsd"|"dragonfly"|g' \
+		${WRKSRC}/v8/build/standalone.gypi \
+		${WRKSRC}/v8/build/toolchain.gypi \
+		${WRKSRC}/v8/src/d8.gyp \
+		${WRKSRC}/v8/tools/gyp/v8.gyp
 
 do-configure:
 	cd ${WRKSRC} && ${SETENV} ${CONFIGURE_ENV} ${PYTHON_CMD} \
