--- Makefile.orig	2016-12-02 16:38:00.000000000 +0200
+++ Makefile
@@ -29,13 +29,10 @@ BUILD_DEPENDS=	${LOCALBASE}/bin/gperf:de
 
 LIB_DEPENDS=	libcairo.so:graphics/cairo \
 		libcups.so:print/cups \
-		libdbus-1.so:devel/dbus \
-		libdbus-glib-1.so:devel/dbus-glib \
 		libdrm.so:graphics/libdrm \
 		libexpat.so:textproc/expat2 \
 		libFLAC.so:audio/flac \
 		libfreetype.so:print/freetype2 \
-		libgnome-keyring.so:security/libgnome-keyring \
 		libharfbuzz.so:print/harfbuzz \
 		libexif.so:graphics/libexif \
 		libfontconfig.so:x11-fonts/fontconfig \
@@ -65,7 +62,7 @@ CPE_PRODUCT=	chrome
 USE_PERL5=	build
 USE_XORG=	scrnsaverproto x11 xcomposite xcursor xext xdamage xfixes xi \
 		xproto xrandr xrender xscrnsaver xtst
-USE_GNOME=	atk glib20 gtk20 dconf libxslt libxml2
+USE_GNOME=	atk glib20 gtk20 libxslt libxml2
 MAKE_ARGS=	-C out/${BUILDTYPE}
 SHEBANG_FILES=	chrome/tools/build/linux/chrome-wrapper
 ALL_TARGET=	chrome
@@ -95,6 +92,7 @@ GN_ARGS+=	\
 # use_system_icu use_system_harfbuzz use_system_minigbm
 GN_BOOTSTRAP_FLAGS=	--no-clean --no-rebuild
 
+# XXX: technically for us these are a no-no
 # FreeBSD Chromium Api Key
 # Set up Google API keys, see http://www.chromium.org/developers/how-tos/api-keys .
 # Note: these are for FreeBSD use ONLY. For your own distribution,
@@ -106,11 +104,12 @@ GN_ARGS+=	google_api_key="AIzaSyBsp9n41J
 SUB_FILES=	chromium-browser.desktop chrome
 SUB_LIST+=	COMMENT="${COMMENT}"
 
-OPTIONS_DEFINE=	ALSA CODECS GCONF PULSEAUDIO TEST KERBEROS DEBUG DRIVER
+OPTIONS_DEFINE=	ALSA CODECS DBUS GCONF PULSEAUDIO TEST KERBEROS DEBUG DRIVER
 CODECS_DESC=	Compile and enable patented codecs like H.264
+DBUS_DESC=	Hooks for sniffers (gnome dependencies)
 DRIVER_DESC=	Install chromedriver
 
-OPTIONS_DEFAULT=	ALSA CODECS GCONF KERBEROS
+OPTIONS_DEFAULT=	ALSA CODECS DBUS GCONF KERBEROS
 OPTIONS_SUB=	yes
 
 GCONF_USE=	GNOME=gconf2
@@ -137,6 +136,17 @@ GN_ARGS+=	ffmpeg_branding="Chromium"
 GN_ARGS+=	proprietary_codecs=false
 .endif
 
+.if ${PORT_OPTIONS:MDBUS}
+USE_GNOME+=	dconf
+LIB_DEPENDS+=	\
+		libdbus-1.so:devel/dbus \
+		libdbus-glib-1.so:devel/dbus-glib \
+		libgnome-keyring.so:security/libgnome-keyring
+.else
+GN_ARGS+=	use_dbus=false use_gconf=false use_gnome_keyring=false use_gio=false
+##GN_ARGS+=	use_glib=false use_atk=false # bug in x11_event_source_libevent.cc
+.endif
+
 .if ${PORT_OPTIONS:MGCONF}
 GN_ARGS+=	use_gconf=true
 .else
@@ -162,8 +172,8 @@ GN_BOOTSTRAP_FLAGS+=	--debug
 #GN_ARGS+=	is_component_build = true
 .else
 BUILDTYPE=	Release
-.if ${ARCH} == amd64
-GN_ARGS+=	use_lld=true # harder, better, faster, stronger
+.if ${OPSYS} == FreeBSD && ${ARCH} == amd64
+GN_ARGS+=	use_lld=true # softer, worser, slower, weaker
 .endif
 GN_ARGS+=	is_debug=false
 GN_ARGS+=	symbol_level=0
@@ -175,10 +185,7 @@ MAKE_ENV+=	BUILDTYPE=${BUILDTYPE} \
 
 .include <bsd.port.pre.mk>
 
-.if ${CHOSEN_COMPILER_TYPE} == gcc
-GN_ARGS+=	gcc_version=${CXX:S/g++//}
-EXTRA_PATCHES+=	${FILESDIR}/extra-patch-gcc
-.else
+.if ${OPSYS} == FreeBSD
 BUILD_DEPENDS+=	clang39:devel/llvm39
 CC=		clang39
 CXX=		clang++39
@@ -194,12 +201,14 @@ BUILD_DEPENDS+=	${LOCALBASE}/lib/c++/lib
 CXXFLAGS+=	-stdlib=libc++ -isystem ${LOCALBASE}/include/c++/v1
 .endif
 
-.if ${OSVERSION} < 1100000
+.if ${OPSYS} == FreeBSD && ${OSVERSION} < 1100000
 EXTRA_PATCHES+=	${FILESDIR}/extra-patch-10
 .endif
 
-.if ! ${PORT_OPTIONS:MKERBEROS}
+.if ${PORT_OPTIONS:MKERBEROS}
 GN_ARGS+=	use_kerberos=true
+.else
+GN_ARGS+=	use_kerberos=false
 .endif
 
 pre-everything::
