--- distinfo.orig	2013-01-05 21:50:00.966810000 +0100
+++ distinfo	2013-01-05 21:40:37.436022000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_mn.tar.gz) = 2fde647b115e280fffe1d3ba5f1879bc539c0dd8e49e9f2ffce21c8ddc60885c
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_mn.tar.gz) = 1688765
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_mn.tar.gz) = dbb22772c88e84936c8ed7e70ce03aacf85668834632a4d9607e12b833ffcfb3
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_mn.tar.gz) = 1641344
