--- Makefile.orig	2014-04-14 18:36:13.183972000 +0000
+++ Makefile
@@ -61,8 +61,6 @@ PGSQL_LIB_DEPENDS=	libpq.so:${PORTSDIR}/
 VIGRA_BUILD_DEPENDS=	${LOCALBASE}/lib/libvigraimpex.so:${PORTSDIR}/graphics/vigra
 VIGRA_RUN_DEPENDS=	${LOCALBASE}/lib/libvigraimpex.so:${PORTSDIR}/graphics/vigra
 
-.include <bsd.port.pre.mk>
-
 # CLANG is not able to handle neither Fortran nor OpenMP
 USE_GCC=	yes
 .if ${COMPILER_FEATURES:Mlibc++}
@@ -109,4 +107,4 @@ post-install:
 	@${STRIP_CMD} ${STAGEDIR}${PREFIX}/lib/libsaga_gdi-${PORTVERSION}.so
 	@${STRIP_CMD} ${STAGEDIR}${PREFIX}/lib/saga/*.so
 
-.include <bsd.port.post.mk>
+.include <bsd.port.mk>
