--- Makefile.orig	2019-02-28 12:45:40 UTC
+++ Makefile
@@ -12,7 +12,7 @@ COMMENT=	System for Automated Geoscienti
 LICENSE=	GPLv2 LGPL21
 LICENSE_COMB=	multi
 
-BUILD_DEPENDS=	clang70:devel/llvm70 \
+BUILD_DEPENDS=	\
 		swig3.0:devel/swig30
 LIB_DEPENDS=	libcurl.so:ftp/curl \
 		libfftw3.so:math/fftw3 \
@@ -56,8 +56,6 @@ VIGRA_DESC=	Enable 'Vision with Generic
 
 OPENMP_CONFIGURE_ENABLE=	openmp
 OPENMP_LDFLAGS=		-L${LOCALBASE}/lib -lm
-# Use OpenMP from lang/llvm70
-OPENMP_LDFLAGS+=	-L${LOCALBASE}/llvm70/lib -lomp
 PGSQL_CONFIGURE_OFF=	--with-postgresql=no
 PGSQL_CONFIGURE_ON=	--with-postgresql=${LOCALBASE}/bin/pg_config
 PGSQL_USES=		pgsql
@@ -68,11 +66,6 @@ INSTALL_TARGET=	install-strip
 
 .include <bsd.port.pre.mk>
 
-# make describe needs them here
-CPP=		clang-cpp70
-CC=		clang70
-CXX=		clang++70
-
 SUB_FILES=	pkg-message
 
 post-extract:
@@ -102,12 +95,6 @@ post-patch:
 	@${REINPLACE_CMD} -e 's|PORTVERSION|${PORTVERSION}|' \
 		${WRKSRC}/src/saga_core/saga_gui/dlg_about.cpp
 
-post-patch-OPENMP-on:
-	@${REINPLACE_CMD} -e 's|-lgomp|-lomp|' \
-		${WRKSRC}/configure.ac \
-		${WRKSRC}/configure \
-		${WRKSRC}/src/accessories/templates/template4saga-tools/my_tool_library/Makefile
-
 post-configure:
 	@${REINPLACE_CMD} -e 's|cd $$(pkgdatadir);|cd ${STAGEDIR}${DATADIR};|g' \
 		${WRKSRC}/src/saga_core/saga_gui/Makefile
