--- Makefile.orig	2023-04-11 19:15:48 UTC
+++ Makefile
@@ -79,8 +79,6 @@ post-patch:
 	@${REINPLACE_CMD} -e 's|GIT_TAGNAME|${GIT_TAGNAME}|g' \
 		${WRKSRC}/src/saga_core/saga_gui/CMakeLists.txt \
 		${WRKSRC}/src/saga_core/saga_gui/dlg_about.cpp
-	@${REINPLACE_CMD} -e 's|-lgomp|-lomp|' \
-		${WRKSRC}/src/accessories/templates/template4saga-tools/my_tool_library/Makefile
 
 post-install:
 	${MKDIR} ${STAGEDIR}${DATADIR}
