--- Makefile.orig	2020-07-28 22:56:06 UTC
+++ Makefile
@@ -13,7 +13,7 @@ COMMENT=	System for Automated Geoscienti
 LICENSE=	GPLv2 LGPL21
 LICENSE_COMB=	multi
 
-BUILD_DEPENDS=	clang${LLVM_DEFAULT}:devel/llvm${LLVM_DEFAULT} \
+BUILD_DEPENDS=	\
 		swig:devel/swig
 LIB_DEPENDS=	libcurl.so:ftp/curl \
 		libfftw3.so:math/fftw3 \
@@ -66,11 +66,6 @@ INSTALL_TARGET=	install-strip
 
 .include <bsd.port.pre.mk>
 
-# make describe needs them here
-CPP=		clang-cpp${LLVM_DEFAULT}
-CC=		clang${LLVM_DEFAULT}
-CXX=		clang++${LLVM_DEFAULT}
-
 SUB_FILES=	pkg-message
 
 post-extract:
@@ -101,12 +96,6 @@ post-patch:
 	@${REINPLACE_CMD} -e 's|PORTVERSION|${PORTVERSION}|' \
 		${WRKSRC}/src/saga_core/saga_gui/dlg_about.cpp
 
-post-patch-OPENMP-on:
-	@${REINPLACE_CMD} -e 's|-lgomp|-lomp|' \
-		${WRKSRC}/configure.ac \
-		${WRKSRC}/configure \
-		${WRKSRC}/src/accessories/templates/template4saga-tools/my_tool_library/Makefile
-
 post-configure:
 	@${REINPLACE_CMD} -e 's|^LDFLAGS = .*|& -lopencv_core|' \
 		${WRKSRC}/src/tools/imagery/imagery_opencv/Makefile
