--- Makefile.orig	2021-10-16 10:55:12 UTC
+++ Makefile
@@ -11,7 +11,7 @@ COMMENT=	System for Automated Geoscienti
 LICENSE=	GPLv2 LGPL21
 LICENSE_COMB=	multi
 
-BUILD_DEPENDS=	clang${LLVM_DEFAULT}:devel/llvm${LLVM_DEFAULT} \
+BUILD_DEPENDS=	\
 		swig:devel/swig
 LIB_DEPENDS=	libcurl.so:ftp/curl \
 		libfftw3.so:math/fftw3 \
@@ -65,11 +65,6 @@ VIGRA_RUN_DEPENDS=	${LOCALBASE}/lib/libv
 
 .include <bsd.port.pre.mk>
 
-# make describe needs them here
-CPP=		clang-cpp${LLVM_DEFAULT}
-CC=		clang${LLVM_DEFAULT}
-CXX=		clang++${LLVM_DEFAULT}
-
 SUB_FILES=	pkg-message
 
 post-patch:
@@ -79,8 +74,6 @@ post-patch:
 .endif
 	@${REINPLACE_CMD} -e 's|PORTVERSION|${PORTVERSION}|' \
 		${WRKSRC}/src/saga_core/saga_gui/dlg_about.cpp
-	@${REINPLACE_CMD} -e 's|-lgomp|-lomp|' \
-		${WRKSRC}/src/accessories/templates/template4saga-tools/my_tool_library/Makefile
 	@${REINPLACE_CMD} -e 's|COMMAND python -c|COMMAND python${PYTHON_VER} -c|' \
 		${WRKSRC}/src/saga_core/saga_api/saga_api_python/CMakeLists.txt
 
