--- Makefile.orig	Tue Apr  1 22:15:58 2025
+++ Makefile	Tue Apr
@@ -88,8 +88,6 @@ post-patch:
 	@${REINPLACE_CMD} -e 's|GIT_TAGNAME|${GIT_TAGNAME}|g' \
 		${WRKSRC}/src/saga_core/saga_gui/CMakeLists.txt \
 		${WRKSRC}/src/saga_core/saga_gui/dlg_about.cpp
-	@${REINPLACE_CMD} -e 's|-lgomp|-lomp|' \
-		${WRKSRC}/src/saga_core/saga_api/saga_api_to_python.py
 
 post-install:
 	${MKDIR} ${STAGEDIR}${DATADIR}
