--- pkg-plist.orig	2014-03-01 09:07:10.172570000 +0000
+++ pkg-plist
@@ -75,7 +75,6 @@ lib/kde4/kcm_kopete_accountconfig.so
 lib/kde4/kcm_kopete_addbookmarks.so
 lib/kde4/kcm_kopete_appearanceconfig.so
 lib/kde4/kcm_kopete_autoreplace.so
-lib/kde4/kcm_kopete_avdeviceconfig.so
 lib/kde4/kcm_kopete_behaviorconfig.so
 lib/kde4/kcm_kopete_chatwindowconfig.so
 lib/kde4/kcm_kopete_highlight.so
@@ -134,9 +133,6 @@ lib/libkopete_oscar.so.%%KDE4_GENERIC_LI
 lib/libkopete_otr_shared.so
 lib/libkopete_otr_shared.so.1
 lib/libkopete_otr_shared.so.1.0.0
-lib/libkopete_videodevice.so
-lib/libkopete_videodevice.so.4
-lib/libkopete_videodevice.so.%%KDE4_GENERIC_LIB_VERSION%%
 lib/libkopeteaddaccountwizard.so
 lib/libkopeteaddaccountwizard.so.1
 lib/libkopeteaddaccountwizard.so.1.0.0
@@ -252,7 +248,6 @@ share/apps/kopete/icons/oxygen/128x128/a
 share/apps/kopete/icons/oxygen/128x128/apps/bonjour_protocol.png
 share/apps/kopete/icons/oxygen/128x128/apps/icq_protocol.png
 share/apps/kopete/icons/oxygen/128x128/apps/jabber_protocol.png
-share/apps/kopete/icons/oxygen/128x128/apps/kopete_avdevice.png
 share/apps/kopete/icons/oxygen/128x128/apps/meanwhile_protocol.png
 share/apps/kopete/icons/oxygen/128x128/apps/qq_protocol.png
 share/apps/kopete/icons/oxygen/128x128/apps/skype_protocol.png
@@ -344,7 +339,6 @@ share/apps/kopete/icons/oxygen/22x22/app
 share/apps/kopete/icons/oxygen/22x22/apps/groupwise_protocol.png
 share/apps/kopete/icons/oxygen/22x22/apps/icq_protocol.png
 share/apps/kopete/icons/oxygen/22x22/apps/jabber_protocol.png
-share/apps/kopete/icons/oxygen/22x22/apps/kopete_avdevice.png
 share/apps/kopete/icons/oxygen/22x22/apps/meanwhile_protocol.png
 share/apps/kopete/icons/oxygen/22x22/apps/qq_protocol.png
 share/apps/kopete/icons/oxygen/22x22/apps/skype_protocol.png
@@ -356,7 +350,6 @@ share/apps/kopete/icons/oxygen/32x32/app
 share/apps/kopete/icons/oxygen/32x32/apps/groupwise_protocol.png
 share/apps/kopete/icons/oxygen/32x32/apps/icq_protocol.png
 share/apps/kopete/icons/oxygen/32x32/apps/jabber_protocol.png
-share/apps/kopete/icons/oxygen/32x32/apps/kopete_avdevice.png
 share/apps/kopete/icons/oxygen/32x32/apps/latex.png
 share/apps/kopete/icons/oxygen/32x32/apps/meanwhile_protocol.png
 share/apps/kopete/icons/oxygen/32x32/apps/preferences-plugin-text-effect-kopete.png
@@ -387,7 +380,6 @@ share/apps/kopete/icons/oxygen/64x64/app
 share/apps/kopete/icons/oxygen/64x64/apps/groupwise_protocol.png
 share/apps/kopete/icons/oxygen/64x64/apps/icq_protocol.png
 share/apps/kopete/icons/oxygen/64x64/apps/jabber_protocol.png
-share/apps/kopete/icons/oxygen/64x64/apps/kopete_avdevice.png
 share/apps/kopete/icons/oxygen/64x64/apps/meanwhile_protocol.png
 share/apps/kopete/icons/oxygen/64x64/apps/qq_protocol.png
 share/apps/kopete/icons/oxygen/64x64/apps/sms_protocol.png
@@ -762,7 +754,6 @@ share/kde4/services/kopete_addbookmarks.
 share/kde4/services/kopete_aim.desktop
 share/kde4/services/kopete_appearanceconfig.desktop
 share/kde4/services/kopete_autoreplace.desktop
-share/kde4/services/kopete_avdeviceconfig.desktop
 share/kde4/services/kopete_behaviorconfig.desktop
 share/kde4/services/kopete_bonjour.desktop
 share/kde4/services/kopete_chatwindowconfig.desktop
