--- Makefile.orig	2019-05-02 23:03:29 UTC
+++ Makefile
@@ -10,7 +10,9 @@ COMMENT=	Cross-platform application and
 
 # * phonon4 is not part of Qt itself.
 # * we don't want to pull in wayland just yet.
-USE_QT=		${_USE_QT_ALL:Nphonon4:Ndeclarative:Nwayland:S/$/_run/}
+# * do not include qdoc to avoid hard dep on devel/llvmXY
+# * do not include webengine (and webview that depends on it) to avoid distinfo out-of-sync
+USE_QT=		${_USE_QT_ALL:Nphonon4:Ndeclarative:Nwayland:Nqdoc:Nwebengine:Nwebview:S/$/_run/}
 USES=		metaport qt:5
 
 .include <bsd.port.mk>
