--- Makefile.orig	2017-06-23 20:18:00 UTC
+++ Makefile
@@ -8,7 +8,7 @@ CATEGORIES=	devel
 MAINTAINER=	kde@FreeBSD.org
 COMMENT=	Cross-platform application and UI framework (metaport)
 
-USE_QT5=	${_USE_QT_ALL:Nphonon4:Ndeclarative:S/$/_run/} # phonon4 is not part of Qt itself.
+USE_QT5=	${_USE_QT_ALL:Nphonon4:Ndeclarative:Ndoc:Nexamples:S/$/_run/} # phonon4 is not part of Qt itself.
 USES=		metaport
 
 .include <bsd.port.mk>
