--- pkg-plist.orig	2016-12-31 17:16:40 UTC
+++ pkg-plist
@@ -1,5 +1,5 @@
-%%MP%%%%WOLFDIR%%/iowolfded.%%ARCH%%
-%%MP%%%%WOLFDIR%%/iowolfmp.%%ARCH%%
+%%MP%%%%WOLFDIR%%/iowolfded.x86_64
+%%MP%%%%WOLFDIR%%/iowolfmp.x86_64
 %%MP%%%%WOLFDIR%%/main/cgame.mp.%%ARCH%%.so
 %%MP%%%%WOLFDIR%%/main/qagame.mp.%%ARCH%%.so
 %%MP%%%%WOLFDIR%%/main/ui.mp.%%ARCH%%.so
@@ -7,8 +7,8 @@
 %%MP%%%%WOLFDIR%%/renderer_mp_rend2_%%ARCH%%.so
 %%MP%%bin/iowolfded
 %%MP%%bin/iowolfmp
-%%SP%%%%WOLFDIR%%/iowolfsp.%%ARCH%%
-%%SP%%%%WOLFDIR%%/iowolfspded.%%ARCH%%
+%%SP%%%%WOLFDIR%%/iowolfsp.x86_64
+%%SP%%%%WOLFDIR%%/iowolfspded.x86_64
 %%SP%%%%WOLFDIR%%/main/cgame.sp.%%ARCH%%.so
 %%SP%%%%WOLFDIR%%/main/qagame.sp.%%ARCH%%.so
 %%SP%%%%WOLFDIR%%/main/ui.sp.%%ARCH%%.so
