--- distinfo.orig	2013-01-05 21:50:00.066808000 +0100
+++ distinfo	2013-01-05 21:38:17.745827000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_km.tar.gz) = fa7f3a799ca42d66fe5da386c67ea28c217e96e55d403a9fe064042137539148
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_km.tar.gz) = 1864966
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_km.tar.gz) = ccb11a356b6baccdee78288939feb7440b7ed6ee609247b524e1dfd560fab009
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_km.tar.gz) = 12185593
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_km.tar.gz) = f927547dad4320a9d53e0b8b83ad7720e28596c2d39bef018cc34ca7e4fe6d65
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_km.tar.gz) = 1808759
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_km.tar.gz) = 0103d194cb3dc292325bce6b9fa2eff63c34e946fbdf0ad3965a6b804780ffa1
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_km.tar.gz) = 13808670
