--- pkg-plist.orig	2013-10-20 10:58:36.907424000 +0000
+++ pkg-plist
@@ -383,16 +383,12 @@ GNUstep/System/Library/Headers/Interface
 GNUstep/System/Library/Headers/InterfaceBuilder/IBViewResourceDragging.h
 GNUstep/System/Library/Headers/InterfaceBuilder/InterfaceBuilder.h
 GNUstep/System/Library/Libraries/libGorm.so
-GNUstep/System/Library/Libraries/libGorm.so.1
 GNUstep/System/Library/Libraries/libGorm.so.1.1.0
 GNUstep/System/Library/Libraries/libGormCore.so
-GNUstep/System/Library/Libraries/libGormCore.so.1
 GNUstep/System/Library/Libraries/libGormCore.so.1.2.20
 GNUstep/System/Library/Libraries/libGormObjCHeaderParser.so
-GNUstep/System/Library/Libraries/libGormObjCHeaderParser.so.1
 GNUstep/System/Library/Libraries/libGormObjCHeaderParser.so.1.1.0
 GNUstep/System/Library/Libraries/libGormPrefs.so
-GNUstep/System/Library/Libraries/libGormPrefs.so.1
 GNUstep/System/Library/Libraries/libGormPrefs.so.1.2.20
 GNUstep/System/Tools/Gorm
 libdata/ldconfig/gorm
