--- pkg-plist.orig	2014-02-27 09:21:38.000000000 +0000
+++ pkg-plist
@@ -19,11 +19,6 @@ share/applications/roxterm.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lib/roxterm_logo.png
 %%PORTDOCS%%%%DOCSDIR%%/lib/sprites.png
 share/icons/hicolor/scalable/apps/roxterm.svg
-share/locale/es/LC_MESSAGES/roxterm.mo
-share/locale/fr/LC_MESSAGES/roxterm.mo
-share/locale/pt_BR/LC_MESSAGES/roxterm.mo
-share/locale/ru/LC_MESSAGES/roxterm.mo
-share/locale/uk/LC_MESSAGES/roxterm.mo
 %%DATADIR%%/Config/Colours/GTK
 %%DATADIR%%/Config/Colours/Tango
 %%DATADIR%%/Config/Shortcuts/Default
