--- pkg-plist.orig	2020-10-28 15:40:52 UTC
+++ pkg-plist
@@ -26,7 +26,6 @@ share/applications/gnome-region-panel.de
 share/applications/gnome-removable-media-panel.desktop
 share/applications/gnome-search-panel.desktop
 share/applications/gnome-sharing-panel.desktop
-share/applications/gnome-sound-panel.desktop
 share/applications/gnome-universal-access-panel.desktop
 share/applications/gnome-usage-panel.desktop
 share/applications/gnome-user-accounts-panel.desktop
@@ -323,7 +322,3 @@ share/polkit-1/actions/org.gnome.control
 share/polkit-1/actions/org.gnome.controlcenter.remote-login-helper.policy
 share/polkit-1/actions/org.gnome.controlcenter.user-accounts.policy
 share/polkit-1/rules.d/gnome-control-center.rules
-share/sounds/gnome/default/alerts/bark.ogg
-share/sounds/gnome/default/alerts/drip.ogg
-share/sounds/gnome/default/alerts/glass.ogg
-share/sounds/gnome/default/alerts/sonar.ogg
