--- pkg-plist.orig	2016-04-18 09:05:27 UTC
+++ pkg-plist
@@ -241,10 +241,6 @@ lib/ugene/plugins/libtest_runner.so
 lib/ugene/plugins/libtest_runner.so.1
 lib/ugene/plugins/libtest_runner.so.1.0
 lib/ugene/plugins/libtest_runner.so.1.0.0
-%%NOX64%%lib/ugene/plugins/libumuscle.so
-%%NOX64%%lib/ugene/plugins/libumuscle.so.1
-%%NOX64%%lib/ugene/plugins/libumuscle.so.1.0
-%%NOX64%%lib/ugene/plugins/libumuscle.so.1.0.0
 lib/ugene/plugins/libvariants.so
 lib/ugene/plugins/libvariants.so.1
 lib/ugene/plugins/libvariants.so.1.0
@@ -289,8 +285,6 @@ lib/ugene/plugins/smith_waterman.license
 lib/ugene/plugins/smith_waterman.plugin
 lib/ugene/plugins/test_runner.license
 lib/ugene/plugins/test_runner.plugin
-%%NOX64%%lib/ugene/plugins/umuscle.license
-%%NOX64%%lib/ugene/plugins/umuscle.plugin
 lib/ugene/plugins/variants.license
 lib/ugene/plugins/variants.plugin
 lib/ugene/plugins/weight_matrix.license
