--- pkg-plist.orig	2021-02-05 11:04:41 UTC
+++ pkg-plist
@@ -97,7 +97,6 @@ lib/ugene/plugins/librepeat_finder.so
 lib/ugene/plugins/libsitecon.so
 lib/ugene/plugins/libsmith_waterman.so
 lib/ugene/plugins/libtest_runner.so
-lib/ugene/plugins/libumuscle.so
 lib/ugene/plugins/libvariants.so
 lib/ugene/plugins/libweight_matrix.so
 lib/ugene/plugins/libwevote_support.so
@@ -136,8 +135,6 @@ lib/ugene/plugins/smith_waterman.license
 lib/ugene/plugins/smith_waterman.plugin
 lib/ugene/plugins/test_runner.license
 lib/ugene/plugins/test_runner.plugin
-lib/ugene/plugins/umuscle.license
-lib/ugene/plugins/umuscle.plugin
 lib/ugene/plugins/variants.license
 lib/ugene/plugins/variants.plugin
 lib/ugene/plugins/weight_matrix.license
