--- pkg-plist.orig	2014-09-13 13:57:50 UTC
+++ pkg-plist
@@ -253,10 +253,6 @@ lib/ugene/plugins/libtest_runner.so
 lib/ugene/plugins/libtest_runner.so.1
 lib/ugene/plugins/libtest_runner.so.1.0
 lib/ugene/plugins/libtest_runner.so.1.0.0
-%%NOX64%%lib/ugene/plugins/libumuscle.so
-%%NOX64%%lib/ugene/plugins/libumuscle.so.1
-%%NOX64%%lib/ugene/plugins/libumuscle.so.1.0
-%%NOX64%%lib/ugene/plugins/libumuscle.so.1.0.0
 lib/ugene/plugins/libvariants.so
 lib/ugene/plugins/libvariants.so.1
 lib/ugene/plugins/libvariants.so.1.0
@@ -301,8 +297,6 @@ lib/ugene/plugins/snp_effect.license
 lib/ugene/plugins/snp_effect.plugin
 lib/ugene/plugins/test_runner.license
 lib/ugene/plugins/test_runner.plugin
-%%NOX64%%lib/ugene/plugins/umuscle.license
-%%NOX64%%lib/ugene/plugins/umuscle.plugin
 lib/ugene/plugins/variants.license
 lib/ugene/plugins/variants.plugin
 lib/ugene/plugins/weight_matrix.license
