--- Makefile.ori	Sat Feb 22 22:04:43 2025
+++ Makefile	Sat Feb 22 22:05:42 2025
@@ -37,7 +37,7 @@ PORTDOCS=	javadoc platform-${PORTVERSION}-javadoc.jar
 
 OPTIONS_DEFINE=	DOCS
 
-ANT_ARCH=	${ARCH:S/amd64/x86-64/:S/i386/x86/:S/powerpc64/ppc64/}
+ANT_ARCH=	${ARCH:S/amd64/x86-64/:S/i386/x86/:S/x86_64/x86-64/:S/powerpc64/ppc64/}
 
 post-patch:
 	@${REINPLACE_CMD} -e 's|%%LOCALBASE%%|${LOCALBASE}|' \
@@ -47,7 +47,7 @@ post-build-DOCS-on:
 	cd ${WRKSRC} && ${SETENV} ${MAKE_ENV} ${ANT} javadoc
 
 do-install:
-	${INSTALL_DATA} ${WRKSRC}/build/freebsd-${ANT_ARCH}.jar \
+	${INSTALL_DATA} ${WRKSRC}/build/dragonflybsd-${ANT_ARCH}.jar \
 		${STAGEDIR}${JAVAJARDIR}/jna-platform.jar
 	${INSTALL_DATA} ${WRKSRC}/build/jna.jar \
 		${STAGEDIR}${JAVAJARDIR}/jna.jar
