--- distinfo.orig	2013-01-05 21:50:01.416810000 +0100
+++ distinfo	2013-01-05 21:42:13.586157000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_sq.tar.gz) = 3de282b806f1189076295a02d676ba673ade6b1a978391a8f97dcf8aa85b9a90
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_sq.tar.gz) = 1646808
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_sq.tar.gz) = 4b94aafcdbd53866918db73934e944e45abc43dd32b42ac626c3a27184b82656
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_sq.tar.gz) = 9821242
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_sq.tar.gz) = aa8877dce526d7dc7c769d48c0eded73734529bf6f4ff21fa9e25feffdd7b6f8
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_sq.tar.gz) = 1597181
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_sq.tar.gz) = b4908a5bff1ca89256b961ca0108bcacb671c1b879e58cc24ac502d300d40c77
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_sq.tar.gz) = 9828833
