--- GIDs.orig	2013-09-14 12:23:04.317462000 +0000
+++ GIDs
@@ -13,12 +13,10 @@ uucp:*:66:
 xten:*:67:
 dialer:*:68:
 network:*:69:
-pgsql:*:70:
 ircd:*:72:
 ircservices:*:73:
 simscan:*:74:
 hybserv:*:76:
-audit:*:77:
 virtuoso:*:79:
 www:*:80:
 qnofiles:*:81:
@@ -82,7 +80,8 @@ webcamd:*:145:
 ventrilo:*:146:
 kdm:*:147:
 ipv6mon:*:148:
-avenger:*:149:
+audit:*:149:
+pgsql:*:150:
 rbldns:*:153:
 trircd:*:154:
 sfs:*:171:
@@ -262,5 +261,6 @@ shibd:*:971:
 plex:*:972:
 boinc:*:973:
 radicale:*:974:
+avenger:*:60149:
 nogroup:*:65533:
 nobody:*:65534:
