--- GIDs.orig	2014-02-10 08:50:00.915245000 +0000
+++ GIDs
@@ -14,12 +14,10 @@ uucp:*:66:
 xten:*:67:
 dialer:*:68:
 network:*:69:
-pgsql:*:70:
 ircd:*:72:
 ircservices:*:73:
 simscan:*:74:
 hybserv:*:76:
-audit:*:77:
 virtuoso:*:79:
 www:*:80:
 qnofiles:*:81:
@@ -83,8 +81,8 @@ webcamd:*:145:
 ventrilo:*:146:
 kdm:*:147:
 ipv6mon:*:148:
-avenger:*:149:
-cbsd:*:150:
+audit:*:149:
+pgsql:*:150:
 rbldns:*:153:
 trircd:*:154:
 sfs:*:171:
@@ -270,5 +268,7 @@ boinc:*:973:
 radicale:*:974:
 unifi:*:975:
 minetest:*:976:
+avenger:*:60149:
+cbsd:*:60150:
 nogroup:*:65533:
 nobody:*:65534:
