--- GIDs.orig	2014-09-25 11:31:18.189747000 +0000
+++ GIDs
@@ -16,12 +16,10 @@ uucp:*:66:
 xten:*:67:
 dialer:*:68:
 network:*:69:
-pgsql:*:70:
 ircd:*:72:
 ircservices:*:73:
 simscan:*:74:
 hybserv:*:76:
-audit:*:77:
 virtuoso:*:79:
 www:*:80:
 qnofiles:*:81:
@@ -85,8 +83,8 @@ webcamd:*:145:
 ventrilo:*:146:
 kdm:*:147:
 ipv6mon:*:148:
-avenger:*:149:
-cbsd:*:150:
+audit:*:149:
+pgsql:*:150:
 rbldns:*:153:
 trircd:*:154:
 xorp:*:155:
@@ -297,5 +295,7 @@ syncthing:*:983:
 tvheadend:*:984:
 puppetdb:*:985:
 kamailio:*:986:
+avenger:*:60149:
+cbsd:*:60150:
 nogroup:*:65533:
 nobody:*:65534:
