--- pkg-plist.orig	2023-07-28 17:13:25 UTC
+++ pkg-plist
@@ -133,7 +133,6 @@ include/agar/agar/config/have_mysql.h
 include/agar/agar/config/have_nanosleep.h
 include/agar/agar/config/have_noreturn_attribute.h
 include/agar/agar/config/have_objc.h
-include/agar/agar/config/have_objc_warnings.h
 include/agar/agar/config/have_opengl.h
 include/agar/agar/config/have_packed_attribute.h
 include/agar/agar/config/have_png.h
