--- pkg-plist.orig	2020-08-13 10:38:34 UTC
+++ pkg-plist
@@ -30,13 +30,11 @@ lib/collectd/csv.so
 %%CURL_XML%%lib/collectd/curl_xml.so
 %%DBI%%lib/collectd/dbi.so
 lib/collectd/df.so
-lib/collectd/disk.so
 lib/collectd/dns.so
 lib/collectd/email.so
 lib/collectd/exec.so
 lib/collectd/filecount.so
 lib/collectd/interface.so
-lib/collectd/ipstats.so
 %%IPMI%%lib/collectd/ipmi.so
 %%JAVA%%lib/collectd/java.so
 lib/collectd/load.so
@@ -90,7 +88,6 @@ lib/collectd/target_replace.so
 lib/collectd/target_scale.so
 lib/collectd/target_set.so
 lib/collectd/target_v5upgrade.so
-lib/collectd/tcpconns.so
 lib/collectd/teamspeak2.so
 lib/collectd/ted.so
 lib/collectd/threshold.so
@@ -111,7 +108,6 @@ lib/collectd/write_log.so
 %%REDIS%%lib/collectd/write_redis.so
 %%RIEMANN%%lib/collectd/write_riemann.so
 %%TSDB%%lib/collectd/write_tsdb.so
-lib/collectd/zfs_arc.so
 %%JAVA%%%%DATADIR%%/java/collectd-api.jar
 %%JAVA%%%%DATADIR%%/java/generic-jmx.jar
 %%ZOOKEEPER%%lib/collectd/zookeeper.so
