--- a/deskutils/nextcloudclient/pkg-plist
+++ b/deskutils/nextcloudclient/pkg-plist
@@ -18,8 +18,8 @@ lib/libnextcloudsync.so
 lib/libnextcloudsync.so.0
 lib/libnextcloudsync.so.%%VERSION%%
 lib/nextcloudsync_vfs_suffix.so
-%%DOCS%%man/man1/nextcloud.1.gz
-%%DOCS%%man/man1/nextcloudcmd.1.gz
+share/man/man1/nextcloud.1.gz
+share/man/man1/nextcloudcmd.1.gz
 %%KDE%%%%QT_PLUGINDIR%%/kf5/kfileitemaction/nextclouddolphinactionplugin.so
 %%KDE%%%%QT_PLUGINDIR%%/kf5/overlayicon/nextclouddolphinoverlayplugin.so
 share/applications/com.nextcloud.desktopclient.nextcloud.desktop
