--- pkg-plist.orig	2024-02-10 00:54:01 UTC
+++ pkg-plist
@@ -214,9 +214,9 @@ lib/libpython%%PYTHON_VER%%%%ABI%%.so.1.
 %%PYTHON_LIBDIR%%/__pycache__/_strptime%%PYTHON_EXT_SUFFIX%%.opt-1.pyc
 %%PYTHON_LIBDIR%%/__pycache__/_strptime%%PYTHON_EXT_SUFFIX%%.opt-2.pyc
 %%PYTHON_LIBDIR%%/__pycache__/_strptime%%PYTHON_EXT_SUFFIX%%.pyc
-%%PYTHON_LIBDIR%%/__pycache__/_sysconfigdata_%%ABI%%_freebsd%%OSMAJOR%%_%%PYTHON_EXT_SUFFIX%%.opt-1.pyc
-%%PYTHON_LIBDIR%%/__pycache__/_sysconfigdata_%%ABI%%_freebsd%%OSMAJOR%%_%%PYTHON_EXT_SUFFIX%%.opt-2.pyc
-%%PYTHON_LIBDIR%%/__pycache__/_sysconfigdata_%%ABI%%_freebsd%%OSMAJOR%%_%%PYTHON_EXT_SUFFIX%%.pyc
+%%PYTHON_LIBDIR%%/__pycache__/_sysconfigdata_%%ABI%%_dragonfly%%OSMAJOR%%_%%PYTHON_EXT_SUFFIX%%.opt-1.pyc
+%%PYTHON_LIBDIR%%/__pycache__/_sysconfigdata_%%ABI%%_dragonfly%%OSMAJOR%%_%%PYTHON_EXT_SUFFIX%%.opt-2.pyc
+%%PYTHON_LIBDIR%%/__pycache__/_sysconfigdata_%%ABI%%_dragonfly%%OSMAJOR%%_%%PYTHON_EXT_SUFFIX%%.pyc
 %%PYTHON_LIBDIR%%/__pycache__/_threading_local%%PYTHON_EXT_SUFFIX%%.opt-1.pyc
 %%PYTHON_LIBDIR%%/__pycache__/_threading_local%%PYTHON_EXT_SUFFIX%%.opt-2.pyc
 %%PYTHON_LIBDIR%%/__pycache__/_threading_local%%PYTHON_EXT_SUFFIX%%.pyc
@@ -694,7 +694,7 @@ lib/libpython%%PYTHON_VER%%%%ABI%%.so.1.
 %%PYTHON_LIBDIR%%/_pyio.py
 %%PYTHON_LIBDIR%%/_sitebuiltins.py
 %%PYTHON_LIBDIR%%/_strptime.py
-%%PYTHON_LIBDIR%%/_sysconfigdata_%%ABI%%_freebsd%%OSMAJOR%%_.py
+%%PYTHON_LIBDIR%%/_sysconfigdata_%%ABI%%_dragonfly%%OSMAJOR%%_.py
 %%PYTHON_LIBDIR%%/_threading_local.py
 %%PYTHON_LIBDIR%%/_weakrefset.py
 %%PYTHON_LIBDIR%%/abc.py
