--- Makefile.orig	2014-08-20 09:54:49.173446000 +0000
+++ Makefile
@@ -16,8 +16,10 @@ BUILD_DEPENDS=	${LOCALBASE}/bin/python:$
 CONFLICTS_INSTALL=	v8-devel-[0-9]*
 
 ALL_TARGET=	native
-MAKE_ARGS=	library=shared
 MAKE_ENV=	CC.host=${CC} CXX.host=${CXX} LINK.host=${CXX} LINK.target=${CXX}
+MAKE_ENV+=	GYPFLAGS=-DOS=freebsd
+MAKE_ENV+=	library=shared
+CFLAGS+=	-D__FreeBSD__=1
 USE_LDCONFIG=	yes
 USES=		compiler cpe execinfo gmake python:2 tar:xz
 
