--- pkg-plist.orig	2014-08-02 13:59:22.301604000 +0000
+++ pkg-plist
@@ -2313,10 +2313,10 @@ lib/python%%XYDOT%%/pickle.py
 lib/python%%XYDOT%%/pickletools.py
 lib/python%%XYDOT%%/pipes.py
 lib/python%%XYDOT%%/pkgutil.py
-lib/python%%XYDOT%%/plat-freebsd%%OSMAJOR%%/IN.py
-lib/python%%XYDOT%%/plat-freebsd%%OSMAJOR%%/__pycache__/IN.cpython-%%XY%%.pyc
-lib/python%%XYDOT%%/plat-freebsd%%OSMAJOR%%/__pycache__/IN.cpython-%%XY%%.pyo
-lib/python%%XYDOT%%/plat-freebsd%%OSMAJOR%%/regen
+lib/python%%XYDOT%%/plat-dragonfly%%OSMAJOR%%/IN.py
+lib/python%%XYDOT%%/plat-dragonfly%%OSMAJOR%%/__pycache__/IN.cpython-%%XY%%.pyc
+lib/python%%XYDOT%%/plat-dragonfly%%OSMAJOR%%/__pycache__/IN.cpython-%%XY%%.pyo
+lib/python%%XYDOT%%/plat-dragonfly%%OSMAJOR%%/regen
 lib/python%%XYDOT%%/platform.py
 lib/python%%XYDOT%%/plistlib.py
 lib/python%%XYDOT%%/poplib.py
