--- pkg-plist.orig	2013-10-20 10:27:14.919728000 +0000
+++ pkg-plist
@@ -55,13 +55,11 @@ GNUstep/Local/Library/Frameworks/BioCoco
 GNUstep/Local/Library/Frameworks/BioCocoa.framework/Versions/2/Resources/aminoacids.plist
 GNUstep/Local/Library/Frameworks/BioCocoa.framework/Versions/2/Resources/nucleotides.plist
 GNUstep/Local/Library/Frameworks/BioCocoa.framework/Versions/2/libBioCocoa.so
-GNUstep/Local/Library/Frameworks/BioCocoa.framework/Versions/2/libBioCocoa.so.2
 GNUstep/Local/Library/Frameworks/BioCocoa.framework/Versions/2/libBioCocoa.so.2.2.2
 GNUstep/Local/Library/Frameworks/BioCocoa.framework/Versions/Current
 GNUstep/Local/Library/Frameworks/BioCocoa.framework/libBioCocoa.so
 GNUstep/Local/Library/Headers/BioCocoa
 GNUstep/Local/Library/Libraries/libBioCocoa.so
-GNUstep/Local/Library/Libraries/libBioCocoa.so.2
 GNUstep/Local/Library/Libraries/libBioCocoa.so.2.2.2
 @dirrmtry GNUstep/Local/Library/Frameworks/BioCocoa.framework/Versions/2/Resources
 @dirrmtry GNUstep/Local/Library/Frameworks/BioCocoa.framework/Versions/2/Headers
