--- Makefile.orig	2015-08-06 07:01:59 UTC
+++ Makefile
@@ -21,11 +21,11 @@ NO_WRKSUBDIR=	yes
 
 PLIST_FILES=	sbin/MegaCli
 
-MEGACLI_ARCH=	MegaCli${ARCH:S/i386//:S/amd64/64/}
+MEGACLI_ARCH=	MegaCli${ARCH:S/i386//:S/amd64/64/:S/x86_64/64/}
 
 post-extract:
 	@${MV} ${WRKSRC}/FreeBSD/MegaCLI.zip ${WRKSRC}/FreeBSD/MegaCli.zip
-	@${EXTRACT_CMD} ${EXTRACT_BEFORE_ARGS} ${WRKSRC}/FreeBSD/${MEGACLI_ARCH}.zip -d ${WRKSRC}/FreeBSD
+	@${TAR} ${EXTRACT_BEFORE_ARGS} ${WRKSRC}/FreeBSD/${MEGACLI_ARCH}.zip -d ${WRKSRC}/FreeBSD
 
 do-install:
 	${INSTALL_SCRIPT} ${WRKSRC}/FreeBSD/${MEGACLI_ARCH} ${STAGEDIR}${PREFIX}/sbin/MegaCli
