--- files/hald.in.orig	2013-01-05 13:30:20.714932000 +0100
+++ files/hald.in	2013-01-05 13:30:39.074958000 +0100
@@ -107,7 +107,7 @@
     echo "Starting ${name}."
 
     ( iter=0
-    while ! ps -axoargs | grep "^/usr/libexec/getty " | grep -qv grep >/dev/null 2>&1; do
+    while ! ps -axocommand | grep "^/usr/libexec/getty " | grep -qv grep >/dev/null 2>&1; do
 	if [ ${iter} -eq 60 ]; then
 	    break
 	fi
