--- Makefile.orig	2013-05-25 18:16:44.000000000 +0000
+++ Makefile
@@ -35,7 +35,7 @@ OPTIONS_DEFAULT+=	MANPAGES
 
 .if ${PORT_OPTIONS:MKMS}
 CONFIGURE_ARGS+=--enable-libkms
-EXTRA_PATCHES+=	${FILESDIR}/extra-xf86drmMode.c
+EXTRA_PATCHES+=	${DFLY_PATCHDIR}/extra-xf86drmMode.c
 PLIST_SUB+=	KMS=""
 .else
 PLIST_SUB+=	KMS="@comment "
@@ -83,8 +83,7 @@ PLIST_SUB+=	RADEON_DRIVERS="@comment "
 .endif
 
 pre-patch:
-	@${REINPLACE_CMD} 's|{libdir}/pkgconfig|{prefix}/libdata/pkgconfig|g; \
-			   s,i?86|x86_64),i?86|amd64|x86_64),g' \
+	@${REINPLACE_CMD} 's|{libdir}/pkgconfig|{prefix}/libdata/pkgconfig|g;' \
 		${WRKSRC}/configure
 
 .include <bsd.port.post.mk>
