--- pkg-plist.orig	2017-08-26 12:25:39.981078000 -0700
+++ pkg-plist	2017-08-26 12:26:00.440815000 -0700
@@ -1548,7 +1548,6 @@
 GNUstep/Local/Library/SOGo/WebServerResources/ckeditor/plugins/base64image/LICENSE.md
 GNUstep/Local/Library/SOGo/WebServerResources/ckeditor/plugins/base64image/README.md
 GNUstep/Local/Library/SOGo/WebServerResources/ckeditor/plugins/base64image/dialogs/base64image.js
-GNUstep/Local/Library/SOGo/WebServerResources/ckeditor/plugins/clipboard/dialogs/paste.js
 GNUstep/Local/Library/SOGo/WebServerResources/ckeditor/plugins/colordialog/dialogs/colordialog.css
 GNUstep/Local/Library/SOGo/WebServerResources/ckeditor/plugins/colordialog/dialogs/colordialog.js
 GNUstep/Local/Library/SOGo/WebServerResources/ckeditor/plugins/dialog/dialogDefinition.js
@@ -1932,3 +1931,44 @@
 @sample %%ETCDIR%%/SOGo-Apache.conf.sample
 @sample %%ETCDIR%%/SOGo-apple-ab.Apache.conf.sample
 @mode
+GNUstep/Local/Library/SOGo/AdministrationUI.SOGo/Resources/Hebrew.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/AdministrationUI.SOGo/Resources/Latvian.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/Appointments.SOGo/Resources/Hebrew.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/Appointments.SOGo/Resources/Latvian.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/CommonUI.SOGo/Resources/Hebrew.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/CommonUI.SOGo/Resources/Latvian.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/Contacts.SOGo/Resources/Hebrew.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/Contacts.SOGo/Resources/Latvian.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/ContactsUI.SOGo/Resources/Hebrew.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/ContactsUI.SOGo/Resources/Latvian.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/MailPartViewers.SOGo/Resources/Hebrew.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/MailPartViewers.SOGo/Resources/Latvian.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/Mailer.SOGo/Resources/Hebrew.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/Mailer.SOGo/Resources/Latvian.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/Mailer.SOGo/Resources/SOGoMailHebrewForward.wo/SOGoMailHebrewForward.html
+GNUstep/Local/Library/SOGo/Mailer.SOGo/Resources/SOGoMailHebrewForward.wo/SOGoMailHebrewForward.wod
+GNUstep/Local/Library/SOGo/Mailer.SOGo/Resources/SOGoMailHebrewReply.wo/SOGoMailHebrewReply.html
+GNUstep/Local/Library/SOGo/Mailer.SOGo/Resources/SOGoMailHebrewReply.wo/SOGoMailHebrewReply.wod
+GNUstep/Local/Library/SOGo/Mailer.SOGo/Resources/SOGoMailLatvianForward.wo/SOGoMailLatvianForward.html
+GNUstep/Local/Library/SOGo/Mailer.SOGo/Resources/SOGoMailLatvianForward.wo/SOGoMailLatvianForward.wod
+GNUstep/Local/Library/SOGo/Mailer.SOGo/Resources/SOGoMailLatvianReply.wo/SOGoMailLatvianReply.html
+GNUstep/Local/Library/SOGo/Mailer.SOGo/Resources/SOGoMailLatvianReply.wo/SOGoMailLatvianReply.wod
+GNUstep/Local/Library/SOGo/MailerUI.SOGo/Resources/Hebrew.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/MailerUI.SOGo/Resources/Latvian.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/MainUI.SOGo/Resources/Hebrew.lproj/Locale
+GNUstep/Local/Library/SOGo/MainUI.SOGo/Resources/Hebrew.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/MainUI.SOGo/Resources/Latvian.lproj/Locale
+GNUstep/Local/Library/SOGo/MainUI.SOGo/Resources/Latvian.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/PreferencesUI.SOGo/Resources/Hebrew.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/PreferencesUI.SOGo/Resources/Latvian.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/SchedulerUI.SOGo/Resources/Hebrew.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/SchedulerUI.SOGo/Resources/Latvian.lproj/Localizable.strings
+GNUstep/Local/Library/SOGo/Templates/SOGoACLHebrewAdditionAdvisory.wox
+GNUstep/Local/Library/SOGo/Templates/SOGoACLHebrewModificationAdvisory.wox
+GNUstep/Local/Library/SOGo/Templates/SOGoACLHebrewRemovalAdvisory.wox
+GNUstep/Local/Library/SOGo/Templates/SOGoFolderHebrewAdditionAdvisory.wox
+GNUstep/Local/Library/SOGo/Templates/SOGoFolderHebrewRemovalAdvisory.wox
+GNUstep/Local/Library/SOGo/WebServerResources/ckeditor/lang/he.js
+GNUstep/Local/Library/SOGo/WebServerResources/ckeditor/lang/lv.js
+GNUstep/Local/Library/SOGo/WebServerResources/ckeditor/plugins/scayt/dialogs/dialog.css
+GNUstep/Local/Library/SOGo/WebServerResources/ckeditor/plugins/widget/images/handle.png
