--- Makefile.orig	2020-06-28 18:44:21 UTC
+++ Makefile
@@ -97,7 +97,7 @@ SAFEPOINT_IMPLIES=	THREADS
 SBCL_VARS=		XC_HOST="${BOOT_WRKSRC}/src/runtime/sbcl --core ${BOOT_WRKSRC}/output/${CORE} --noinform --disable-debugger --no-sysinit --no-userinit"
 SBCL_DISTFILES=		${PORTNAME}-${SBCL_BOOT_LIST:M${ARCHOS_PATTERN}}-binary${EXTRACT_SUFX}:binaries
 
-THREADS_VARS=		MAKE_SH_ARGS+="--with-sb-thread --with-restore-fs-segment-register-from-tls"
+THREADS_VARS=		MAKE_SH_ARGS+="--with-sb-thread --without-restore-fs-segment-register-from-tls"
 THREADS_VARS_OFF=	MAKE_SH_ARGS+="--without-sb-thread --without-restore-fs-segment-register-from-tls"
 
 UNICODE_VARS=		MAKE_SH_ARGS+="--with-sb-unicode"
@@ -113,7 +113,7 @@ PORTDOCS=	*
 
 .include <bsd.port.options.mk>
 
-ARCHOS_PATTERN=	*-${ARCH:S/amd64/x86-64/:S/i386/x86/}-${OPSYS:tl}*
+ARCHOS_PATTERN=	*-${ARCH:S/amd64/x86-64/:S/i386/x86/:S/x86_64/x86-64/}-${OPSYS:tl}*
 BOOT_WRKSRC=	${WRKDIR}/${PORTNAME}-${SBCL_BOOT_LIST:M${ARCHOS_PATTERN}}
 
 # for port maintenance, invoke "make makesum PLUS_BOOTSTRAPS=1"
@@ -126,7 +126,7 @@ DISTFILES:=	${DISTFILES} ${PORTNAME}-${B
 .endif
 
 # Old FreeBSD bootstraps feature the older core name for SBCL bootstrap
-.if ${OPSYS} == FreeBSD
+.if ${OPSYS} == FreeBSD || ${OPSYS} == DragonFly
 CORE=	sbcl.core
 .else
 CORE=	sbcl_core
