--- Makefile.orig
+++ Makefile
@@ -110,12 +110,12 @@
 .if defined(DYNAMIC_SPACE_SIZE)
 MAKE_SH_ARGS+=	--dynamic-space-size=${DYNAMIC_SPACE_SIZE}
 .endif
-SBCL_BOOT_LIST=	1.2.7-x86-64-freebsd 1.2.7-x86-freebsd 1.2.7-x86-64-dragonfly \
+SBCL_BOOT_LIST=	1.2.7-x86-64-freebsd 1.2.7-x86-freebsd 1.2.9-x86-64-dragonfly \
 		2.5.7-powerpc64le-freebsd 2.6.5-powerpc64-freebsd
 
 .include <bsd.port.options.mk>
 
-ARCHOS_PATTERN=	*-${ARCH:S/amd64/x86-64/:S/i386/x86/}-${OPSYS:tl}*
+ARCHOS_PATTERN=	*-${ARCH:S/amd64/x86-64/:S/i386/x86/:S/x86_64/x86-64/}-${OPSYS:tl}*
 BOOT_WRKSRC=	${WRKDIR}/${PORTNAME}-${SBCL_BOOT_LIST:M${ARCHOS_PATTERN}}
 
 .if ${PORT_OPTIONS:MLINKABLE_RUNTIME}
