--- Makefile.common.orig	2020-10-31 09:34:23 UTC
+++ Makefile.common
@@ -46,12 +46,13 @@ LIB_DEPENDS+=	libdrm.so:graphics/libdrm
 .if ${COMPONENT} != libs
 LIB_DEPENDS+=	libglapi.so:graphics/mesa-libs
 .endif
+
+# all ports whine on WARNING: Gettext not found, all translation targets will be ignored.
+USES+=		gettext-tools
+
 .if ${ARCH} == aarch64 || ${ARCH} == amd64 || ${ARCH} == i386 || ${ARCH} == powerpc64 || ${ARCH} == powerpc64le
 LIB_DEPENDS+=	libunwind.so:devel/libunwind
 .endif
-.if ${OPSYS} == DragonFly
-LIB_DEPENDS+=	libelf.so:devel/libelf
-.endif
 
 USES+=		compiler:c++11-lib bison meson pathfix pkgconfig \
 		python:3.6+,build shebangfix tar:xz
@@ -111,8 +112,18 @@ BUILD_DEPENDS+=	llvm${LLVM_DEFAULT}>=10.
 .if ${COMPONENT} != libs
 RUN_DEPENDS+=	llvm${LLVM_DEFAULT}>=10.0.1_1:devel/llvm${LLVM_DEFAULT}
 .endif
+. if defined(DEVELOPER)
+# reduces binary differences somewhat while investigating
+LDFLAGS+=	-Wl,-rpath=${LOCALBASE}/llvm${LLVM_DEFAULT}/lib
+. endif
 .endif
 
+.if ${OPSYS} == DragonFly
+LIB_DEPENDS+=	libelf.so:devel/libelf
+MESON_ARGS+=	-Dllvm=disabled
+.else
 MESON_ARGS+=	--native-file="${WRKSRC}/llvm.ini"
 LDFLAGS+=	-Wl,-rpath=${LOCALBASE}/llvm${LLVM_DEFAULT}/lib
 MESON_ARGS+=	-Dllvm=enabled
+.endif
+
