--- distinfo.orig	2013-01-05 21:50:00.106808000 +0100
+++ distinfo	2013-01-05 21:38:19.725830000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_brx.tar.gz) = 5b2c617dcd93f6b433f4844e6770d7125af6abffdc23a9aa637259640ae500f3
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_brx.tar.gz) = 1675683
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_brx.tar.gz) = 6610bca6119672e97168b4628a7365166a54a9f05ba3066867af351805a7eb5d
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_brx.tar.gz) = 1628927
