--- distinfo.orig
+++ distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1779364278
-SHA256 (valgrind-3.27.1.tar.bz2) = 5d589152eb8071c02feab8ce6ab719e431a1fbc3e2b1700f5432632a8b9264dc
-SIZE (valgrind-3.27.1.tar.bz2) = 17372682
+TIMESTAMP = 1629109280
+SHA256 (1338-1-valgrind-dragonfly-3.15.0-dragonfly_GH0.tar.gz) = b8af76a743353e0df85b26b65c868cc73c63804d307adca7d6cd3114b73bc700
+SIZE (1338-1-valgrind-dragonfly-3.15.0-dragonfly_GH0.tar.gz) = 14593325
