--- distinfo.orig	2021-08-16 10:18:50 UTC
+++ distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1616493211
-SHA256 (paulfloyd-freebsd_valgrind-freebsd-3.17.0.g20210319-d333c54_GH0.tar.gz) = 2f631d7768fda08a62d6327cea0ea56062becad0083ba2751262215c2390387e
-SIZE (paulfloyd-freebsd_valgrind-freebsd-3.17.0.g20210319-d333c54_GH0.tar.gz) = 15796240
+TIMESTAMP = 1629109280
+SHA256 (1338-1-valgrind-dragonfly-3.15.0-dragonfly_GH0.tar.gz) = b8af76a743353e0df85b26b65c868cc73c63804d307adca7d6cd3114b73bc700
+SIZE (1338-1-valgrind-dragonfly-3.15.0-dragonfly_GH0.tar.gz) = 14593325
