--- distinfo.orig	2016-04-06 16:14:10 UTC
+++ distinfo
@@ -1,5 +1,5 @@
-SHA256 (rust-stage0-2016-02-17-4d3eebf-freebsd-x86_64-f38991fbb81c1cd8d0bbda396f98f13a55b42804.tar.bz2) = 6123aa870918555835623548e7edbf79480cd754c649fda844dc3c14e4e142f2
-SIZE (rust-stage0-2016-02-17-4d3eebf-freebsd-x86_64-f38991fbb81c1cd8d0bbda396f98f13a55b42804.tar.bz2) = 15922558
+SHA256 (rust-stage0-2015-06-12-50ab23d-dragonfly-x86_64-54171b3357723941a1025da60d090ca84634cdac.tar.bz2) = 54f21064bd793cd5e6f803fa1c66fbc8520b302d7cb29251f69f350ca303fbc2
+SIZE (rust-stage0-2015-06-12-50ab23d-dragonfly-x86_64-54171b3357723941a1025da60d090ca84634cdac.tar.bz2) = 14605293
 SHA256 (rust-lang-rust-1.9.0.20160318-235d77457d_GH0.tar.gz) = 6833cab6dc2c34a23cdabd565cb950b4e22f2606337c1d9f36c1a9e038eb4192
 SIZE (rust-lang-rust-1.9.0.20160318-235d77457d_GH0.tar.gz) = 5696553
 SHA256 (rust-lang-compiler-rt-57315f7_GH0.tar.gz) = d84fb1afd630c8dd9cca4fb37c4aa90b0b1ee95a7da04439ac109bcb2b63de3a
