--- distinfo.orig	2013-01-05 21:50:00.566809000 +0100
+++ distinfo	2013-01-05 21:39:46.785952000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_af.tar.gz) = a4d3e328696f5841e18a0e5c24707f1a9450dcc1f55535651242b1f9f2f056cc
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_af.tar.gz) = 2259675
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_af.tar.gz) = 07de965779ac145dc97e0176ecdc1dcdcbd8adbf8d0ebfabf6c06b83a5fc1b04
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_af.tar.gz) = 2208374
