--- pkg-plist.orig	2014-02-05 12:43:25.920921000 +0000
+++ pkg-plist
@@ -12,7 +12,6 @@ GNUstep/System/Applications/WrapperFacto
 GNUstep/System/Applications/WrapperFactory.app/WrapperFactory
 GNUstep/System/Applications/WrapperFactory.app/stamp.make
 GNUstep/System/Library/Libraries/libGSWrapper.so
-GNUstep/System/Library/Libraries/libGSWrapper.so.%%MAJORLIBVERSION%%
 GNUstep/System/Library/Libraries/libGSWrapper.so.%%LIBVERSION%%
 GNUstep/System/Tools/GSWrapper_Launcher
 GNUstep/System/Tools/WrapperFactory
