--- distinfo.orig	2013-01-05 21:50:00.896809000 +0100
+++ distinfo	2013-01-05 21:40:27.456008000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_tr.tar.gz) = b25e2bf3315d0212595e7108a6e34d31fa056498b3cb332b8ff50dfa222a2cf5
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_tr.tar.gz) = 1597609
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_tr.tar.gz) = 8102d800945dd8cbcc3e71aa095da1d4808f769b1d6fd79d85b6824289fdcf68
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_tr.tar.gz) = 10441730
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_tr.tar.gz) = 3846e37ad61fef09d7eb8dba4b1a416f23ffcf83ffcd24ff7d45ea2d06ef5b30
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_tr.tar.gz) = 1544327
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_tr.tar.gz) = 11b0a39a85f99e1c96d1a45f1a5769daf3e4bfaed80fba62507e19bfba4c7a8c
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_tr.tar.gz) = 10992060
