--- distinfo.orig	2016-11-13 20:31:27 UTC
+++ distinfo
@@ -7,3 +7,5 @@ SHA256 (rust-lang-cargo-0.13.0_GH0.tar.g
 SIZE (rust-lang-cargo-0.13.0_GH0.tar.gz) = 610125
 SHA256 (rust-lang-rust-installer-755bc3d_GH0.tar.gz) = 103a502bad7af80eef7ccd2dd2969a272a6fa3f237ebeaca083315a45878f07f
 SIZE (rust-lang-rust-installer-755bc3d_GH0.tar.gz) = 19193
+SHA256 (cargo-nightly-x86_64-unknown-dragonfly.tar.gz) = 78355e53e98bb7640d5481b8d291d076cc65c19bc83f90d54338a97d84ac58fe
+SIZE (cargo-nightly-x86_64-unknown-dragonfly.tar.gz) = 4429496
