--- Makefile.orig	2014-12-02 07:54:11 UTC
+++ Makefile
@@ -58,7 +58,8 @@ WEATHER_DESC=	Weather calendar backend
 
 .if ${PORT_OPTIONS:MKERBEROS} && exists(/usr/bin/krb5-config)
 CONFIGURE_ARGS+=	--with-krb5=/usr
-KRB5_LIB=		`/usr/bin/krb5-config gssapi --libs`
+KRB5_LIB=		`${LOCALBASE}/bin/krb5-config gssapi --libs`
+BUILD_DEPENDS+=		krb5-config:security/krb5
 .else
 CONFIGURE_ARGS+=	--without-krb5
 .endif
