--- Makefile.orig	2014-10-10 16:17:21.277507000 +0000
+++ Makefile
@@ -50,7 +50,8 @@ KERBEROS_CONFIGURE_OFF=	--without-krb5
 .include <bsd.port.options.mk>
 
 .if ${PORT_OPTIONS:MKERBEROS}
-KRB5_LIB=	`/usr/bin/krb5-config gssapi --libs`
+KRB5_LIB=	`${LOCALBASE}/bin/krb5-config gssapi --libs`
+BUILD_DEPENDS+= krb5-config:${PORTSDIR}/security/krb5
 .endif
 
 EDS_BDB_MSG=	"!!!WARNING!!!: Berkeley Database version is ${BDB_VER}.  If this is an upgrade, you may experience problems if you or any other Evolution-data-server users were previously using BDB_VER 41."
