--- Makefile.orig	2014-03-14 22:24:28.418124000 +0000
+++ Makefile
@@ -12,6 +12,7 @@ MAINTAINER=	gnome@FreeBSD.org
 COMMENT=	Framework for controlling access to system-wide components
 
 LIB_DEPENDS=	libexpat.so:${PORTSDIR}/textproc/expat2
+LIB_DEPENDS+=	libxml2.so:${PORTSDIR}/textproc/libxml2
 RUN_DEPENDS=	dbus-launch:${PORTSDIR}/devel/dbus
 
 USE_GNOME=	glib20 intlhack gnomehier introspection
@@ -21,7 +22,6 @@ USE_LDCONFIG=	yes
 CONFIGURE_ARGS=	--disable-gtk-doc \
 		--with-authfw=pam \
 		--with-pam-include=system \
-		--with-os-type=freebsd \
 		--disable-systemd \
 		--enable-introspection \
 		--localstatedir=/var
