--- Makefile.orig	2013-03-09 22:45:05.963085000 +0100
+++ Makefile
@@ -14,6 +14,7 @@
 
 BUILD_DEPENDS=	${LOCALBASE}/share/gir-1.0/GLib-2.0.gir:${PORTSDIR}/devel/gobject-introspection
 LIB_DEPENDS=	expat:${PORTSDIR}/textproc/expat2
+LIB_DEPENDS+=	libxml2.5:${PORTSDIR}/textproc/libxml2
 RUN_DEPENDS=	${LOCALBASE}/share/gir-1.0/GLib-2.0.gir:${PORTSDIR}/devel/gobject-introspection \
 		dbus-launch:${PORTSDIR}/devel/dbus
 
@@ -26,7 +27,6 @@
 CONFIGURE_ARGS=	--disable-gtk-doc \
 		--with-authfw=pam \
 		--with-pam-include=system \
-		--with-os-type=freebsd \
 		--disable-systemd \
 		--enable-introspection \
 		--localstatedir=/var
