--- distinfo.orig	2024-08-11 12:46:25 UTC
+++ distinfo
@@ -1,5 +1,5 @@
-TIMESTAMP = 1698027802
+TIMESTAMP = 1723391651
 SHA256 (x265-3.5.tar.gz) = 8a5fcfddb3e2db520b637e475a507fb00117d554bdb3204c5828626d888c306a
 SIZE (x265-3.5.tar.gz) = 1538610
-SHA256 (x265-asm-warnings.patch) = 0faa3278887a47ed8c968c994e4cc71a2921aac38f5d560dae898124f806871d
+SHA256 (x265-asm-warnings.patch) = b59df1758664d85347e8f04fdf0a2934b6e0248af62e03a8e0aba0840c6280f3
 SIZE (x265-asm-warnings.patch) = 4526
