--- distinfo.orig	2013-01-05 21:50:01.176810000 +0100
+++ distinfo	2013-01-05 21:41:18.276079000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_th.tar.gz) = 25e60bf5e2b3de7d1c512f95a68fee6190221c2fb8c0ba338e3d0a9e71ffd6f7
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_th.tar.gz) = 1833653
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_th.tar.gz) = 8e86c8e578b8bd97030bc67ccc4574a27fc9a8f8fb13538348110b64df0b8571
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_th.tar.gz) = 1785224
