--- distinfo.orig	2013-01-05 21:50:00.606809000 +0100
+++ distinfo	2013-01-05 21:39:52.445960000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_kk.tar.gz) = 9522689885e6c89c3c4c3a2fab21f71e4c85e52fd8c35fc4f76ec9271687403b
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_kk.tar.gz) = 1681735
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_kk.tar.gz) = a39aacc6b5888d23b6e09f1c9b32378a181b86c3529a1233420e5144c3f10996
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_kk.tar.gz) = 1635439
