--- pkg-plist.orig	2022-05-23 19:10:11 UTC
+++ pkg-plist
@@ -190,3 +190,35 @@ share/locale/xh/LC_MESSAGES/gtksourcevie
 share/locale/zh_CN/LC_MESSAGES/gtksourceview-2.0.mo
 share/locale/zh_HK/LC_MESSAGES/gtksourceview-2.0.mo
 share/locale/zh_TW/LC_MESSAGES/gtksourceview-2.0.mo
+share/doc/gtksourceview-2.0/GtkSourceBuffer.html
+share/doc/gtksourceview-2.0/GtkSourceCompletion.html                                    
+share/doc/gtksourceview-2.0/GtkSourceCompletionInfo.html
+share/doc/gtksourceview-2.0/GtkSourceCompletionItem.html                                
+share/doc/gtksourceview-2.0/GtkSourceCompletionProposal.html
+share/doc/gtksourceview-2.0/GtkSourceCompletionProvider.html
+share/doc/gtksourceview-2.0/GtkSourceGutter.html
+share/doc/gtksourceview-2.0/GtkSourceLanguage.html 
+share/doc/gtksourceview-2.0/GtkSourceLanguageManager.html
+share/doc/gtksourceview-2.0/GtkSourceMark.html
+share/doc/gtksourceview-2.0/GtkSourcePrintCompositor.html
+share/doc/gtksourceview-2.0/GtkSourceStyle.html
+share/doc/gtksourceview-2.0/GtkSourceStyleScheme.html
+share/doc/gtksourceview-2.0/GtkSourceStyleSchemeManager.html
+share/doc/gtksourceview-2.0/GtkSourceUndoManager.html
+share/doc/gtksourceview-2.0/GtkSourceView.html
+share/doc/gtksourceview-2.0/gtksourceview-2.0-Searching-in-a-GtkSourceBuffer.html
+share/doc/gtksourceview-2.0/gtksourceview-2.0-completioncontext.html
+share/doc/gtksourceview-2.0/gtksourceview-2.0.devhelp
+share/doc/gtksourceview-2.0/gtksourceview-2.0.devhelp2
+share/doc/gtksourceview-2.0/home.png
+share/doc/gtksourceview-2.0/index.html
+share/doc/gtksourceview-2.0/index.sgml
+share/doc/gtksourceview-2.0/lang-reference.html
+share/doc/gtksourceview-2.0/lang-tutorial.html
+share/doc/gtksourceview-2.0/left.png
+share/doc/gtksourceview-2.0/right.png
+share/doc/gtksourceview-2.0/rn01.html
+share/doc/gtksourceview-2.0/rn02.html
+share/doc/gtksourceview-2.0/style-reference.html
+share/doc/gtksourceview-2.0/style.css
+share/doc/gtksourceview-2.0/up.png
