--- distinfo.orig	2013-01-05 21:50:00.556809000 +0100
+++ distinfo	2013-01-05 21:39:45.125949000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_nb.tar.gz) = 63d25a9bca9e62b7173d98cbf7fe8ca220d6503a8f240e5d311b49b5908cf1d8
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_nb.tar.gz) = 4841366
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_nb.tar.gz) = 63526e36acbda4cd01c6a33a8a2c1e2fce196658098f9b98c770148cc238e964
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_nb.tar.gz) = 10624949
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_nb.tar.gz) = 45df90f4ecbb6c829acce745a0a656d3ec92eed95aeb94b24ef36c48616d7e40
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_nb.tar.gz) = 4788863
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_nb.tar.gz) = 2613030ebe77b5673ca9ab00936c0caa38009cb1f86ad7c1ba32c2dd4e94291b
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_nb.tar.gz) = 10636329
