--- distinfo.orig	2013-01-05 21:50:00.936809000 +0100
+++ distinfo	2013-01-05 21:40:34.806019000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_te.tar.gz) = 4054f20dcaa5a11c937ecf733eb617925659c2fdf5fb022c1dff7691ca165f80
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_te.tar.gz) = 2258692
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_te.tar.gz) = 6332f464c2c9fa6ad13ea4d5d048e15ff9fc8334b56cabe26604dbe868295920
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_te.tar.gz) = 2217093
