--- pkg-plist.orig	2015-06-29 21:47:36 UTC
+++ pkg-plist
@@ -93,9 +93,11 @@ lib/mysql/libmysqlclient_r.so.18
 lib/mysql/libperconaserverclient.a
 lib/mysql/libperconaserverclient.so
 lib/mysql/libperconaserverclient.so.18
+lib/mysql/libperconaserverclient.so.18.1.0
 lib/mysql/libperconaserverclient_r.a
 lib/mysql/libperconaserverclient_r.so
 lib/mysql/libperconaserverclient_r.so.18
+lib/mysql/libperconaserverclient_r.so.18.1.0
 lib/mysql/libmysqlservices.a
 man/man1/comp_err.1.gz
 man/man1/msql2mysql.1.gz
