--- distinfo.orig	2013-01-05 21:50:01.196810000 +0100
+++ distinfo	2013-01-05 21:41:27.876093000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_gu.tar.gz) = 77f170782ad4a9433ab255803869027fff2a1063912a16133b391710bfb3f9d1
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_gu.tar.gz) = 2334337
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_gu.tar.gz) = 578ae6fce4e0312a72878fe6a998f62fcf6ee2f56d15db6ed516afce56f8769a
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_gu.tar.gz) = 9499856
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_gu.tar.gz) = 9c0d2fdafe511196b69ab39c050cedaebebbc538fd4e8c2f9a187f41f5eddcf5
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_gu.tar.gz) = 2289638
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_gu.tar.gz) = e69028664637178c18fa118c49d3844811e90637ad989d7ba0e87a17822d2291
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_gu.tar.gz) = 9524366
