--- distinfo.orig	2013-01-05 21:50:01.366810000 +0100
+++ distinfo	2013-01-05 21:42:02.626141000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_tt.tar.gz) = 31dff5eeeb4db670f133a0d9738dfbcc2cc0be293c86d73e89ec664b727a83c7
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_tt.tar.gz) = 1625505
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_tt.tar.gz) = a8e269bbb70e0c14b4795d8c950c74ee048f7217e1bb611727aa31ab6d5d4e1a
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_tt.tar.gz) = 1592309
