--- Makefile.orig	2020-01-23 18:12:21 UTC
+++ Makefile
@@ -107,6 +107,6 @@ post-install:
 	${MKDIR} ${STAGEDIR}${ETCDIR}/snapshots
 	(cd ${STAGEDIR}${ETCDIR} && for f in *.config *.yaml; do ${MV} $$f $$f.sample; done)
 	${SED} -i '' -e 's|^${STAGEDIR}||g' \
-		${STAGEDIR}${PREFIX}/lib/perl5/${ARCH:S/powerpc64/powerpc/}-freebsd-thread-multi/auto/Apache/TS/.packlist
+		${STAGEDIR}${PREFIX}/lib/perl5/${ARCH:S/powerpc64/powerpc/}-dragonfly-thread-multi/auto/Apache/TS/.packlist
 
 .include <bsd.port.post.mk>
