--- pkg-plist.orig	2021-12-13 10:52:31 UTC
+++ pkg-plist
@@ -170,8 +170,8 @@ libexec/trafficserver/tslua.so
 libexec/trafficserver/tsmemcache.so
 libexec/trafficserver/url_sig.so
 libexec/trafficserver/xdebug.so
-lib/perl5/%%ARCH%%-freebsd-thread-multi/auto/Apache/TS/.packlist
-lib/perl5/%%ARCH%%-freebsd-thread-multi/perllocal.pod
+lib/perl5/%%ARCH%%-dragonfly-thread-multi/auto/Apache/TS/.packlist
+lib/perl5/%%ARCH%%-dragonfly-thread-multi/perllocal.pod
 lib/perl5/Apache/TS.pm
 lib/perl5/Apache/TS/AdminClient.pm
 lib/perl5/Apache/TS/Config.pm
