--- files/smart.in.orig	2023-10-15 16:23:26 UTC
+++ files/smart.in
@@ -13,7 +13,7 @@ smartctl=%%PREFIX%%/sbin/smartctl
 case "${daily_status_smart_devices}" in
 	# XXX AUTO mode selects only regular ad/da disks 
 	[Aa][Uu][Tt][Oo])
-		daily_status_smart_devices="$(sysctl -n kern.disks | sed -E 's/[[:<:]](cd|ar|vtbd)[0-9]+//g; s/n(da|vd)/nvme/g')"
+		daily_status_smart_devices="$(sysctl -n kern.disks | sed -E 's/[[:<:]](ar|cd|md|vn)[0-9]+//g; s@mapper/[^ ]*@@g')"
 		;;
 	*)	;;
 esac
