--- distinfo.orig	2017-07-02 19:01:30 UTC
+++ distinfo
@@ -21,3 +21,9 @@ SHA256 (rust/2017-04-24/cargo-0.18.0-aar
 SIZE (rust/2017-04-24/cargo-0.18.0-aarch64-unknown-freebsd.tar.gz) = 2935624
 SHA256 (rust/rust-registry-1.18.0.tar.xz) = 86c74c61aec5b94c9db9f569e6aa5cfeab2dde2a4ba73ea1da4c7320b31f5b36
 SIZE (rust/rust-registry-1.18.0.tar.xz) = 7714840
+SHA256 (rust/2017-04-27/rustc-1.17.0-x86_64-unknown-dragonfly.tar.gz) = a7f22cb60b2e440c4bdf4ef3988527b9b27ce95d978e7d792c162218ba70a51e
+SIZE (rust/2017-04-27/rustc-1.17.0-x86_64-unknown-dragonfly.tar.gz) = 43359305
+SHA256 (rust/2017-04-27/rust-std-1.17.0-x86_64-unknown-dragonfly.tar.gz) = 70685efdd1181be946919de8173f530f4202c2f2fcae8b71971ec872f8c1e226
+SIZE = (rust/2017-04-27/rust-std-1.17.0-x86_64-unknown-dragonfly.tar.gz) = 57572645
+SHA256 (rust/2017-04-27/cargo-0.18.0-x86_64-unknown-dragonfly.tar.gz) = 6a4d85ca24a791c24f8687ae4155551d0a1c9445f661e985d4645b3ae9e595e7
+SIZE (rust/2017-04-27/cargo-0.18.0-x86_64-unknown-dragonfly.tar.gz) = 4945570
