--- distinfo.orig	2018-04-23 13:46:37 UTC
+++ distinfo
@@ -18,4 +18,10 @@ SHA256 (rust/2018-02-15/rustc-1.24.0-x86
 SIZE (rust/2018-02-15/rustc-1.24.0-x86_64-unknown-freebsd.tar.gz) = 62428675
 SHA256 (rust/rustc-1.25.0-src.tar.xz) = 14fcb82d5959df758aaf422539359300917217fa8420e34bd596e3fb6ed2de87
 SIZE (rust/rustc-1.25.0-src.tar.xz) = 56108536
-TIMESTAMP = 1523914189
+SHA256 (rust/2018-02-15/rustc-1.24.0-x86_64-unknown-dragonfly.tar.gz) = 5b80eefd241164fea59d0e997e59960eae1227f31040af556198387172c47abe
+SIZE (rust/2018-02-15/rustc-1.24.0-x86_64-unknown-dragonfly.tar.gz) = 63804732
+SHA256 (rust/2018-02-15/rust-std-1.24.0-x86_64-unknown-dragonfly.tar.gz) = 11810c4c45ec0d832a6a33fa409ff10e9bf39ec84ec4f482aff196f6582125b7
+SIZE (rust/2018-02-15/rust-std-1.24.0-x86_64-unknown-dragonfly.tar.gz) = 81694555
+SHA256 (rust/2018-02-15/cargo-0.25.0-x86_64-unknown-dragonfly.tar.gz) = 95b0134c4de10f3cd7bdfe34421f604fd17afa80f787898054afb46d28cf4159
+SIZE (rust/2018-02-15/cargo-0.25.0-x86_64-unknown-dragonfly.tar.gz) = 5203798
+TIMESTAMP = 1524481855
