--- distinfo.orig	Sat Apr  5 16:55:16 2025
+++ distinfo	Sat Apr
@@ -49,3 +49,9 @@ SHA256 (rust/2025-01-09/rust-std-1.84.0-riscv64gc-unkn
 SIZE (rust/2025-01-09/rust-std-1.84.0-riscv64gc-unknown-freebsd.tar.xz) = 23039224
 SHA256 (rust/2025-01-09/cargo-1.84.0-riscv64gc-unknown-freebsd.tar.xz) = ee370bd3e89a947be859e6d482f815a73938496204deccd2cade4deb138fd996
 SIZE (rust/2025-01-09/cargo-1.84.0-riscv64gc-unknown-freebsd.tar.xz) = 10640860
+SHA256 (rust/2025-01-09/rustc-1.84.1-x86_64-unknown-dragonfly.tar.xz) = 6fdbc3619431e5d14d0eaae97aef1b486c13e128e3ad24effca1092d19f6df85
+SIZE (rust/2025-01-09/rustc-1.84.1-x86_64-unknown-dragonfly.tar.xz) = 85973184
+SHA256 (rust/2025-01-09/rust-std-1.84.1-x86_64-unknown-dragonfly.tar.xz) = 4c2eb3548bdba1216dfec3c57b02a61f873bffd7bf453f606de4ed907732bb83
+SIZE (rust/2025-01-09/rust-std-1.84.1-x86_64-unknown-dragonfly.tar.xz) = 25676108
+SHA256 (rust/2025-01-09/cargo-1.84.1-x86_64-unknown-dragonfly.tar.xz) = 521898bff2e342c801a913575d90718c690401d84b6575bef139a980427d5577
+SIZE (rust/2025-01-09/cargo-1.84.1-x86_64-unknown-dragonfly.tar.xz) = 9276024
