--- distinfo.orig	2021-07-10 00:18:11 UTC
+++ distinfo
@@ -55,3 +55,9 @@ SHA256 (rust/2021-05-10/rust-std-1.52.1-
 SIZE (rust/2021-05-10/rust-std-1.52.1-powerpc64-unknown-freebsd-elfv2.tar.xz) = 16924328
 SHA256 (rust/2021-05-10/cargo-1.52.1-powerpc64-unknown-freebsd-elfv2.tar.xz) = e96ed74c63003c64581f9d3265be18cae439f947c6008a13ea214245f957a1b9
 SIZE (rust/2021-05-10/cargo-1.52.1-powerpc64-unknown-freebsd-elfv2.tar.xz) = 4664628
+SHA256 (rust/2021-05-10/rustc-1.52.1-x86_64-unknown-dragonfly.tar.xz) = 0ce482a920ff9523aa1c96f6736f0aea55bc0e033f9c48939a8a1393ba1ab84d
+SIZE (rust/2021-05-10/rustc-1.52.1-x86_64-unknown-dragonfly.tar.xz) = 56005212
+SHA256 (rust/2021-05-10/rust-std-1.52.1-x86_64-unknown-dragonfly.tar.xz) = d5d8ed2d67cb2bfc6000a8ddba6bf2bfdc10a3e4b1cb6fbe7ee91e0c3cf154ea
+SIZE (rust/2021-05-10/rust-std-1.52.1-x86_64-unknown-dragonfly.tar.xz) = 21165704
+SHA256 (rust/2021-05-10/cargo-1.52.1-x86_64-unknown-dragonfly.tar.xz) = d91f449f152110a44d10a68c2ffa0109ee3fc91697d029328cafa1f76df7706c
+SIZE (rust/2021-05-10/cargo-1.52.1-x86_64-unknown-dragonfly.tar.xz) = 5248788
