--- distinfo.orig	2017-10-05 18:04:13 UTC
+++ distinfo
@@ -19,3 +19,9 @@ SHA256 (rust/2017-06-08/rust-std-1.18.0-
 SIZE (rust/2017-06-08/rust-std-1.18.0-x86_64-unknown-freebsd.tar.gz) = 66585539
 SHA256 (rust/2017-06-08/cargo-0.19.0-x86_64-unknown-freebsd.tar.gz) = a8d45ad508ecbe9ec9e19fddabda4476466486f97cfb1b59d0a43a1012acb788
 SIZE (rust/2017-06-08/cargo-0.19.0-x86_64-unknown-freebsd.tar.gz) = 5176510
+SHA256 (rust/2017-06-08/rustc-1.18.0-x86_64-unknown-dragonfly.tar.gz) = 96c0eb9f51977a739f7747cc61ead0d17014d53ea9eb9299e73136aa7c6058ba
+SIZE (rust/2017-06-08/rustc-1.18.0-x86_64-unknown-dragonfly.tar.gz) = 43791656
+SHA256 (rust/2017-06-08/rust-std-1.18.0-x86_64-unknown-dragonfly.tar.gz) = bcce68057a9b95608d9b0ad4920b67fc47b6ea501935a2c2ac83ea86a74b4adf
+SIZE (rust/2017-06-08/rust-std-1.18.0-x86_64-unknown-dragonfly.tar.gz) = 63025504
+SHA256 (rust/2017-06-08/cargo-0.19.0-x86_64-unknown-dragonfly.tar.gz) = c37b79a4441580a6ae2bef3fd0e0f699149e8d81425892f68b2160a606424eb8
+SIZE (rust/2017-06-08/cargo-0.19.0-x86_64-unknown-dragonfly.tar.gz) = 4933928
