--- distinfo.orig	2024-02-17 16:42:22 UTC
+++ distinfo
@@ -49,3 +49,9 @@ SHA256 (rust/2023-10-05/rust-std-1.73.0-
 SIZE (rust/2023-10-05/rust-std-1.73.0-riscv64gc-unknown-freebsd.tar.xz) = 24553484
 SHA256 (rust/2023-10-05/cargo-1.73.0-riscv64gc-unknown-freebsd.tar.xz) = 5d35edc0a333903844be52075020aedfe5de06ec1af96d35ff327536a25bc6c6
 SIZE (rust/2023-10-05/cargo-1.73.0-riscv64gc-unknown-freebsd.tar.xz) = 11004428
+SHA256 (rust/2023-10-05/rustc-1.73.0-x86_64-unknown-dragonfly.tar.xz) = 4256bed59cea4713195268d49ab954fa85da89f426e0f5f96c7e6e012380834b
+SIZE (rust/2023-10-05/rustc-1.73.0-x86_64-unknown-dragonfly.tar.xz) = 53601872
+SHA256 (rust/2023-10-05/rust-std-1.73.0-x86_64-unknown-dragonfly.tar.xz) = e415632e15dbd95c931d59bc269622efd49d49e134a7a992c0cc90b1da733c6b
+SIZE (rust/2023-10-05/rust-std-1.73.0-x86_64-unknown-dragonfly.tar.xz) = 23474632
+SHA256 (rust/2023-10-05/cargo-1.73.0-x86_64-unknown-dragonfly.tar.xz) = 0f046a242c3c4413bbc5a87a8dff541fc9fc99148304507326bf108e3f8eb680
+SIZE (rust/2023-10-05/cargo-1.73.0-x86_64-unknown-dragonfly.tar.xz) = 7549528
