--- distinfo.orig	2022-07-15 16:57:42 UTC
+++ distinfo
@@ -61,3 +61,9 @@ SHA256 (rust/2022-05-19/rust-std-1.60.0-
 SIZE (rust/2022-05-19/rust-std-1.60.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = 20979640
 SHA256 (rust/2022-05-19/cargo-1.60.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = cd104531239d5a854e1ce53e5c9f2239ed6d535b9e81b61ee880b8a55bc033a9
 SIZE (rust/2022-05-19/cargo-1.60.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = 4795344
+SHA256 (rust/2022-05-19/rustc-1.60.0-x86_64-unknown-dragonfly.tar.xz) = bd10a519b4e53e64020c4c900c2dc11591c725c029583b3eb7029f739bb196b1
+SIZE (rust/2022-05-19/rustc-1.60.0-x86_64-unknown-dragonfly.tar.xz) = 47523812
+SHA256 (rust/2022-05-19/rust-std-1.60.0-x86_64-unknown-dragonfly.tar.xz) = 42d56c7526b61de4d64156a8c520e3ed9c879b683f8fa9527207b1ea72fd16eb
+SIZE (rust/2022-05-19/rust-std-1.60.0-x86_64-unknown-dragonfly.tar.xz) = 23062464
+SHA256 (rust/2022-05-19/cargo-1.60.0-x86_64-unknown-dragonfly.tar.xz) = 2bba04fbfb41dafc58c711ac3e8a37f9edec532df2b58860bc3301753e1ca3fe
+SIZE (rust/2022-05-19/cargo-1.60.0-x86_64-unknown-dragonfly.tar.xz) = 5366480
