--- distinfo.orig	2017-11-17 10:22:46 UTC
+++ distinfo
@@ -19,3 +19,9 @@ SHA256 (rust/2017-08-31/rust-std-1.20.0-
 SIZE (rust/2017-08-31/rust-std-1.20.0-x86_64-unknown-freebsd.tar.gz) = 60215051
 SHA256 (rust/2017-08-31/cargo-0.21.0-x86_64-unknown-freebsd.tar.gz) = 522592cdee824d8d7b7c3472f29ded39d10f1fb3d07f3b3ea0673b8edf6d5b66
 SIZE (rust/2017-08-31/cargo-0.21.0-x86_64-unknown-freebsd.tar.gz) = 5183277
+SHA256 (rust/2017-08-31/rustc-1.20.0-x86_64-unknown-dragonfly.tar.gz) = ea000bb9947405861aceef37618ecb22e08fbbbe8c790f456366b7aadc2886f3
+SIZE (rust/2017-08-31/rustc-1.20.0-x86_64-unknown-dragonfly.tar.gz) = 48924184
+SHA256 (rust/2017-08-31/rust-std-1.20.0-x86_64-unknown-dragonfly.tar.gz) = e05c52e3eb98fe5737a0d29cb5a6a5230af5116451696138f53ba61559d2c5c5
+SIZE (rust/2017-08-31/rust-std-1.20.0-x86_64-unknown-dragonfly.tar.gz) = 67797251
+SHA256 (rust/2017-08-31/cargo-0.21.0-x86_64-unknown-dragonfly.tar.gz) = 1821aeeb790adc12401e73933380411201df3c6a5ec42d6cc52ee949b6684a1f
+SIZE (rust/2017-08-31/cargo-0.21.0-x86_64-unknown-dragonfly.tar.gz) = 6287819
