--- distinfo.orig	2021-01-17 23:30:47 UTC
+++ distinfo
@@ -49,3 +49,9 @@ SHA256 (rust/2020-11-19/rust-std-1.48.0-
 SIZE (rust/2020-11-19/rust-std-1.48.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = 16631184
 SHA256 (rust/2020-11-19/cargo-1.48.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = 7f6ab7edf18c87997fb0b6f563ab776a6af22dcbb655acc6d7c81b0a6fe0837c
 SIZE (rust/2020-11-19/cargo-1.48.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = 4412520
+SHA256 (rust/2020-11-19/rustc-1.48.0-x86_64-unknown-dragonfly.tar.xz) = 60306c65024dd56fc23ad23f3815bd2a405603eb9ca2f0695954127bc0c36308
+SIZE (rust/2020-11-19/rustc-1.48.0-x86_64-unknown-dragonfly.tar.xz) = 38980996
+SHA256 (rust/2020-11-19/rust-std-1.48.0-x86_64-unknown-dragonfly.tar.xz) = 2240679f9b4b94fa8dd028f215872d4255a67a6d8c119faeb73624e9a161ce60
+SIZE (rust/2020-11-19/rust-std-1.48.0-x86_64-unknown-dragonfly.tar.xz) = 19142612
+SHA256 (rust/2020-11-19/cargo-1.48.0-x86_64-unknown-dragonfly.tar.xz) = 284fc96854d9f189288a6b2e3039f0606ea64f937bae0ac63d55fd970bcc016a
+SIZE (rust/2020-11-19/cargo-1.48.0-x86_64-unknown-dragonfly.tar.xz) = 4822700
