--- distinfo.orig	2022-10-20 16:27:52 UTC
+++ distinfo
@@ -61,3 +61,9 @@ SHA256 (rust/2022-08-11/rust-std-1.62.0-
 SIZE (rust/2022-08-11/rust-std-1.62.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = 20893088
 SHA256 (rust/2022-08-11/cargo-1.62.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = a36cd48122c84601cf2610d0cb381e3482bc25ef3f37d6d790eaa6a1a5a1e500
 SIZE (rust/2022-08-11/cargo-1.62.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = 5048488
+SHA256 (rust/2022-08-11/rustc-1.62.1-x86_64-unknown-dragonfly.tar.xz) = 1fd4727a1f4bb913710e5b00f1afdcc9b57587073f1fd6c13f76bab8a0bdb7f6
+SIZE (rust/2022-08-11/rustc-1.62.1-x86_64-unknown-dragonfly.tar.xz) = 45282488
+SHA256 (rust/2022-08-11/rust-std-1.62.1-x86_64-unknown-dragonfly.tar.xz) = 5e5bf69bc9af174da63335e19716db6b589eed0348899541171dabbc47449afe
+SIZE (rust/2022-08-11/rust-std-1.62.1-x86_64-unknown-dragonfly.tar.xz) = 22618368
+SHA256 (rust/2022-08-11/cargo-1.62.1-x86_64-unknown-dragonfly.tar.xz) = cf9e86da19cd9b7bd718f14cec1ccde7920334459bf292e25c8cf330a85b6cb8
+SIZE (rust/2022-08-11/cargo-1.62.1-x86_64-unknown-dragonfly.tar.xz) = 5606596
