--- distinfo.orig	2020-02-17 19:27:57 UTC
+++ distinfo
@@ -19,6 +19,12 @@ SHA256 (rust/2020-01-11/rust-std-1.40.0-
 SIZE (rust/2020-01-11/rust-std-1.40.0-armv6-unknown-freebsd.tar.xz) = 13924280
 SHA256 (rust/2020-01-11/cargo-0.41.0-armv6-unknown-freebsd.tar.xz) = d04aba96a043fb798cee1f4b44a63ff83549c1c85ffaadbbda3edc28ebbb01fc
 SIZE (rust/2020-01-11/cargo-0.41.0-armv6-unknown-freebsd.tar.xz) = 3811020
+SHA256 (rust/2020-01-11/rustc-1.40.0-x86_64-unknown-dragonfly.tar.xz) = cb159091ebe6d709b050bb6495103f8cc04a233c3768bc6de47a8937ec7e1f11
+SIZE (rust/2020-01-11/rustc-1.40.0-x86_64-unknown-dragonfly.tar.xz) = 35221604
+SHA256 (rust/2020-01-11/rust-std-1.40.0-x86_64-unknown-dragonfly.tar.xz) = c7ed9e60314ebe69bbb6da1c4b0d76819decbc1d55ad2f9a2fd2ee4f4c72aeec
+SIZE (rust/2020-01-11/rust-std-1.40.0-x86_64-unknown-dragonfly.tar.xz) = 14524724
+SHA256 (rust/2020-01-11/cargo-0.41.0-x86_64-unknown-dragonfly.tar.xz) = c5f1fdd42aad458461eaaf75e4410cec436f513f23b35f57fc88829d2a9ed349
+SIZE (rust/2020-01-11/cargo-0.41.0-x86_64-unknown-dragonfly.tar.xz) = 4391112
 SHA256 (rust/2020-01-11/rustc-1.40.0-armv7-unknown-freebsd.tar.xz) = 9d89a89d0f441e1e074e2d0c9d255e07852971c60bd3f4e533e8d1e9fded60e0
 SIZE (rust/2020-01-11/rustc-1.40.0-armv7-unknown-freebsd.tar.xz) = 27393212
 SHA256 (rust/2020-01-11/rust-std-1.40.0-armv7-unknown-freebsd.tar.xz) = 5e4ef87f5403f273856ecfcb822d439628fc95dd7c77d2bd21ac7f33b7e4cba6
