--- distinfo.orig	2022-10-11 23:27:05 UTC
+++ distinfo
@@ -61,3 +61,9 @@ SHA256 (rust/2022-08-11/rust-std-1.62.0-
 SIZE (rust/2022-08-11/rust-std-1.62.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = 20893088
 SHA256 (rust/2022-08-11/cargo-1.62.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = a36cd48122c84601cf2610d0cb381e3482bc25ef3f37d6d790eaa6a1a5a1e500
 SIZE (rust/2022-08-11/cargo-1.62.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = 5048488
+SHA256 (rust/2022-08-11/rustc-1.62.0-x86_64-unknown-dragonfly.tar.xz) = 2de14a9ca4307453111a82d8733b0d986284d8e78b5bb2d0c5f6a483794502bc
+SIZE (rust/2022-08-11/rustc-1.62.0-x86_64-unknown-dragonfly.tar.xz) = 36186780
+SHA256 (rust/2022-08-11/rust-std-1.62.0-x86_64-unknown-dragonfly.tar.xz) = 0b2a78ec3636a90cab0923612c6ef4f68fe05e788306c33065425093de2846c7
+SIZE (rust/2022-08-11/rust-std-1.62.0-x86_64-unknown-dragonfly.tar.xz) = 22828324
+SHA256 (rust/2022-08-11/cargo-1.62.0-x86_64-unknown-dragonfly.tar.xz) = b8e36aa5a71d645a0c41eaf2ddeab96d244a6427a05c664a86090cc3d1dc93fd
+SIZE (rust/2022-08-11/cargo-1.62.0-x86_64-unknown-dragonfly.tar.xz) = 5581840
