--- distinfo.orig	2022-03-02 00:28:23 UTC
+++ distinfo
@@ -61,3 +61,9 @@ SHA256 (rust/2021-12-02/rust-std-1.57.0-
 SIZE (rust/2021-12-02/rust-std-1.57.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = 19848988
 SHA256 (rust/2021-12-02/cargo-1.57.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = 662772c902e8512ef9a0c2b6e3543de511835c591bb374218d43141ac3f77546
 SIZE (rust/2021-12-02/cargo-1.57.0-powerpc64-unknown-freebsd-elfv2.tar.xz) = 4833052
+SHA256 (rust/2021-12-02/rustc-1.57.0-x86_64-unknown-dragonfly.tar.xz) = 888f0afb4edf5976bf32513eeb5ad06fa1c0afe0af426a606fe012646bf48678
+SIZE (rust/2021-12-02/rustc-1.57.0-x86_64-unknown-dragonfly.tar.xz) = 61514736
+SHA256 (rust/2021-12-02/rust-std-1.57.0-x86_64-unknown-dragonfly.tar.xz) = aa924b58005f0af816f9197021261082803a8507b3f4d4b96ef6add9ea442640
+SIZE (rust/2021-12-02/rust-std-1.57.0-x86_64-unknown-dragonfly.tar.xz) = 21942336
+SHA256 (rust/2021-12-02/cargo-1.57.0-x86_64-unknown-dragonfly.tar.xz) = 50fbd3596ec090042ab521fbe3272badd9fa4ea18d110075c87215d4e676e123
+SIZE (rust/2021-12-02/cargo-1.57.0-x86_64-unknown-dragonfly.tar.xz) = 5375636
