--- Makefile.orig	2018-09-01 18:30:45 UTC
+++ Makefile
@@ -95,7 +95,7 @@ PLIST_FILES=		lib/rustlib/components \
 
 .include <bsd.port.pre.mk>
 
-X_PY_ENV=	HOME="${WRKDIR}" \
+X_PY_ENV+=	HOME="${WRKDIR}" \
 		OPENSSL_DIR="${OPENSSLBASE}"
 X_PY_CMD=	${PYTHON_CMD} ${WRKSRC}/x.py
 
@@ -130,7 +130,6 @@ RUST_STD_DIR=		${RUST_STD_BOOTSTRAP:T:R:
 
 post-extract:
 	@${MKDIR} \
-		${WRKSRC}/build/cache/${BOOTSTRAPS_DATE_${ARCH}:U${BOOTSTRAPS_DATE}} \
 		${WRKSRC}/build/cache/${BOOTSTRAPS_DATE_${ARCH}:U${BOOTSTRAPS_DATE}}
 	${LN} -sf ${DISTDIR}/${DIST_SUBDIR}/${RUSTC_BOOTSTRAP} \
 		${WRKSRC}/build/cache/${BOOTSTRAPS_DATE_${ARCH}:U${BOOTSTRAPS_DATE}}
@@ -184,14 +183,6 @@ post-patch:
 			${CP} "$$file" "$$file.orig"; \
 		done; \
 	done
-
-.if defined(WITH_CCACHE_BUILD) && !defined(NO_CCACHE)
-CCACHE_VALUE=	"${CCACHE_WRAPPER_PATH:C,/libexec/ccache$,,}/bin/ccache"
-.else
-CCACHE_VALUE=	false
-.endif
-
-do-configure:
 	${SED} -E \
 		-e 's,%PREFIX%,${PREFIX},' \
 		-e 's,%SYSCONFDIR%,${PREFIX}/etc,' \
@@ -202,6 +193,14 @@ do-configure:
 		-e 's,%CCACHE%,${CCACHE_VALUE},' \
 		< ${FILESDIR}/config.toml \
 		> ${WRKSRC}/config.toml
+
+.if defined(WITH_CCACHE_BUILD) && !defined(NO_CCACHE)
+CCACHE_VALUE=	"${CCACHE_WRAPPER_PATH:C,/libexec/ccache$,,}/bin/ccache"
+.else
+CCACHE_VALUE=	false
+.endif
+
+do-configure:
 # The FreeBSD 10 autotools fix may modify some files just before
 # `do-configure`. Like after `extra-path-ino64`, we need to update
 # `.cargo-checksum.json`.
