--- Makefile.orig	2018-10-18 14:39:54 UTC
+++ Makefile
@@ -97,7 +97,7 @@ PLIST_FILES=		lib/rustlib/components \
 
 .include <bsd.port.pre.mk>
 
-X_PY_ENV=	HOME="${WRKDIR}" \
+X_PY_ENV+=	HOME="${WRKDIR}" \
 		LIBGIT2_SYS_USE_PKG_CONFIG=1 \
 		LIBSSH2_SYS_USE_PKG_CONFIG=1 \
 		OPENSSL_DIR="${OPENSSLBASE}"
@@ -134,7 +134,6 @@ RUST_STD_DIR=		${RUST_STD_BOOTSTRAP:T:R:
 
 post-extract:
 	@${MKDIR} \
-		${WRKSRC}/build/cache/${BOOTSTRAPS_DATE_${ARCH}:U${BOOTSTRAPS_DATE}} \
 		${WRKSRC}/build/cache/${BOOTSTRAPS_DATE_${ARCH}:U${BOOTSTRAPS_DATE}}
 	${LN} -sf ${DISTDIR}/${DIST_SUBDIR}/${RUSTC_BOOTSTRAP} \
 		${WRKSRC}/build/cache/${BOOTSTRAPS_DATE_${ARCH}:U${BOOTSTRAPS_DATE}}
