--- Makefile.orig	2019-03-06 18:20:41 UTC
+++ Makefile
@@ -94,7 +94,7 @@ PLIST_FILES=		lib/rustlib/components \
 
 .include <bsd.port.pre.mk>
 
-X_PY_ENV=	HOME="${WRKDIR}" \
+X_PY_ENV+=	HOME="${WRKDIR}" \
 		LIBGIT2_SYS_USE_PKG_CONFIG=1 \
 		LIBSSH2_SYS_USE_PKG_CONFIG=1 \
 		OPENSSL_DIR="${OPENSSLBASE}"
@@ -104,7 +104,6 @@ RUST_STD_DIR=	${RUST_STD_BOOTSTRAP:T:R:R
 
 post-extract:
 	@${MKDIR} \
-		${WRKSRC}/build/cache/${BOOTSTRAPS_DATE_${ARCH}:U${BOOTSTRAPS_DATE}} \
 		${WRKSRC}/build/cache/${BOOTSTRAPS_DATE_${ARCH}:U${BOOTSTRAPS_DATE}}
 	${LN} -sf ${DISTDIR}/${DIST_SUBDIR}/${RUSTC_BOOTSTRAP} \
 		${WRKSRC}/build/cache/${BOOTSTRAPS_DATE_${ARCH}:U${BOOTSTRAPS_DATE}}
