--- Makefile.orig	2022-07-26 22:50:42 UTC
+++ Makefile
@@ -105,10 +105,6 @@ SHEBANG_FILES=		compiler/rustc_codegen_c
 
 .include <bsd.port.pre.mk>
 
-.if ${OPSYS} != FreeBSD
-IGNORE=		is only for FreeBSD
-.endif
-
 .if ${ARCH} == powerpc
 # bfd to link rustc_driver; lld currently can't
 BUILD_DEPENDS+=	ld.bfd:devel/binutils
@@ -229,6 +225,7 @@ do-configure:
 	@${ECHO_CMD} 'ccache=false' >> ${WRKSRC}/config.toml
 .endif
 	@${ECHO_CMD} 'ninja=true' >> ${WRKSRC}/config.toml
+	@${ECHO_CMD} 'static-libstdcpp=false' >> ${WRKSRC}/config.toml
 .if ${ARCH} == powerpc
 # Rust doesn't call the system compiler with the full version of the target.
 # This makes powerpc miscompile due to the secure-plt ABI change.
