--- Makefile.orig	2015-01-18 11:39:07 UTC
+++ Makefile
@@ -84,8 +84,6 @@ LIB_DEPENDS+=		libsasl2.so:${PORTSDIR}/s
 # GNUTLS
 LIB_DEPENDS+=		libgcrypt.so:${PORTSDIR}/security/libgcrypt
 LIB_DEPENDS+=		libgnutls.so:${PORTSDIR}/security/gnutls
-# NFS4 ACL glue
-LIB_DEPENDS+=		libsunacl.so:${PORTSDIR}/sysutils/libsunacl
 # External Samba dependencies
 # IDL compiler
 BUILD_DEPENDS+=		p5-Parse-Pidl>=4.0.0:${PORTSDIR}/devel/p5-Parse-Pidl
@@ -154,6 +152,7 @@ NSUPDATE_DESC=		Use samba NSUPDATE utili
 .include <bsd.port.options.mk>
 
 CONFIGURE_ARGS+=	\
+			--without-regedit \
 			--with-pam \
 			--with-iconv \
 			--with-winbind \
@@ -227,14 +226,18 @@ WANT_EXP_MODULES=	idmap_ad vfs_cacheprim
 .endif
 
 .if ${PORT_OPTIONS:MACL_SUPPORT}
+.  if ${OPSYS} == FreeBSD
 .	if ${OSVERSION} > 800000
 LIB_DEPENDS+=		libsunacl.so:${PORTSDIR}/sysutils/libsunacl
 WANT_EXP_MODULES+=	vfs_zfsacl
 SAMBA4_MODULES+=	vfs_zfsacl
-.	endif
 CONFIGURE_ARGS+=	--with-acl-support
-.else
+.       else
+CONFIGURE_ARGS+=	--without-acl-support
+.       endif
+.  else
 CONFIGURE_ARGS+=	--without-acl-support
+.  endif
 .endif
 
 .if ! ${PORT_OPTIONS:MAD_DC}
