--- pkg-plist.orig	2014-10-20 15:18:29 UTC
+++ pkg-plist
@@ -17,7 +17,6 @@ bin/regpatch
 bin/regshell
 bin/regtree
 bin/rpcclient
-bin/samba-regedit
 %%AD_DC%%bin/samba-tool
 bin/sharesec
 bin/smbcacls
@@ -181,6 +180,8 @@ lib/libndr.so
 lib/libndr.so.0
 lib/libnetapi.so
 lib/libnetapi.so.0
+lib/libnss_wins.so
+lib/libnss_wins.so.2
 lib/libpdb.so
 lib/libpdb.so.0
 lib/libregistry.so
@@ -210,8 +211,6 @@ lib/libtorture.so.0
 lib/libwbclient.so
 lib/libwbclient.so.0
 lib/mit_samba.so
-lib/nss_winbind.so.1
-lib/nss_wins.so.1
 %%SMBPASS%%lib/pam_smbpass.so
 lib/pam_winbind.so
 lib/winbind_krb5_locator.so
@@ -422,7 +421,6 @@ lib/shared-modules/vfs/syncops.so
 lib/shared-modules/vfs/time_audit.so
 lib/shared-modules/vfs/xattr_tdb.so
 %%DEVELOPER%%lib/shared-modules/vfs/nfs4acl_xattr.so
-lib/shared-modules/vfs/zfsacl.so
 %%LDAP%%lib/shared-modules/idmap/rfc2307.so
 %%MODULE_AUTH_SAMBA4%%lib/shared-modules/auth/samba4.so
 %%MODULE_AUTH_UNIX%%lib/shared-modules/auth/unix.so
