--- pkg-plist.orig	2022-10-15 18:11:44.965450000 +0200
+++ pkg-plist	2022-10-15 18:12:21.534641000 +0200
@@ -34,8 +34,6 @@
 lib/mysql/libmysqlclient_r.so
 lib/mysql/libmysqlclient_r.so.21
 %%SASLCLIENT%%lib/mysql/plugin/authentication_ldap_sasl_client.so
-lib/mysql/plugin/authentication_fido_client.so
-lib/mysql/plugin/authentication_oci_client.so
 libdata/pkgconfig/mysqlclient.pc
 share/man/man1/comp_err.1.gz
 share/man/man1/lz4_decompress.1.gz
