--- pkg-plist.orig	2013-10-20 17:24:11.278867000 +0000
+++ pkg-plist
@@ -164,7 +164,6 @@
 %%PORTDOCS%%%%DOCSDIR%%/overview-tree.html
 %%PORTDOCS%%%%DOCSDIR%%/package-list
 %%PORTDOCS%%%%DOCSDIR%%/resources/background.gif
-%%PORTDOCS%%%%DOCSDIR%%/resources/inherit.gif
 %%PORTDOCS%%%%DOCSDIR%%/resources/tab.gif
 %%PORTDOCS%%%%DOCSDIR%%/resources/titlebar.gif
 %%PORTDOCS%%%%DOCSDIR%%/resources/titlebar_end.gif
