--- Makefile.orig	2014-06-19 23:10:10.985545000 +0000
+++ Makefile
@@ -11,11 +11,14 @@ MAINTAINER=	ports@FreeBSD.org
 COMMENT=	Anti-aliased font support for gtk+
 
 LIB_DEPENDS=	libXft.so:${PORTSDIR}/x11-fonts/libXft
+LIB_DEPENDS+=	fontconfig:${PORTSDIR}/x11-fonts/fontconfig
+LIB_DEPENDS+=	freetype:${PORTSDIR}/print/freetype2
 
 USE_GNOME=	gtk12
-USES=		iconv pkgconfig libtool
+USES=		iconv pkgconfig libtool gettext
+USE_XORG+=	x11 xft
 GNU_CONFIGURE=	yes
-LDFLAGS+=	-L${LOCALBASE}/lib -lgnuregex ${ICONV_LIB}
+LDFLAGS+=	-L${LOCALBASE}/lib ${ICONV_LIB}
 CFLAGS+=	`pkg-config xft --cflags`
 CONFIGURE_ARGS=	--sbindir=${PREFIX}/bin \
 		--without-gnome
@@ -36,8 +39,8 @@ post-patch:
 		 s|DESTDIR)\$$(datadir|DESTDIR)\$$(pkgdatadir|g ; \
 		 s|la_LDFLAGS =.*$$|la_LDFLAGS = -module -avoid-version|g'
 .for file in gdkxft_sysinstall.in gdkxft.c generate.pl
-	@${REINPLACE_CMD} -e 's|<regex.h>|<gnuregex.h>|g ; \
-		 s|gtk-config |${GTK_CONFIG:T} |g' ${WRKSRC}/src/${file}
+	@${REINPLACE_CMD} -e 's|gtk-config |${GTK_CONFIG:T} |g' \
+	  ${WRKSRC}/src/${file}
 .endfor
 
 post-install:
