--- Makefile.orig	2016-04-06 16:43:35 UTC
+++ Makefile
@@ -14,11 +14,15 @@ LICENSE=	GPLv2+
 LICENSE_FILE=	${WRKSRC}/COPYING
 
 LIB_DEPENDS=	libXft.so:x11-fonts/libXft
+LIB_DEPENDS+=	libfontconfig.so:x11-fonts/fontconfig \
+		libfreetype.so:print/freetype2
 
 USE_GNOME=	gtk12
 USES=		iconv pkgconfig libtool
+USE_XORG=	x11 xft
+USES+=		gettext
 GNU_CONFIGURE=	yes
-LDFLAGS+=	-L${LOCALBASE}/lib -lgnuregex ${ICONV_LIB}
+LDFLAGS+=	-L${LOCALBASE}/lib ${ICONV_LIB}
 CFLAGS+=	`pkg-config xft --cflags`
 CONFIGURE_ARGS=	--sbindir=${PREFIX}/bin \
 		--without-gnome
@@ -32,7 +36,6 @@ post-patch:
 		's|src capplet|src @BUILD_CAPPLET_TRUE@capplet|g ; \
 		 s|DESTDIR)\$$(datadir|DESTDIR)\$$(pkgdatadir|g ; \
 		 s|la_LDFLAGS =.*$$|la_LDFLAGS = -module -avoid-version|g'
-	@${REINPLACE_CMD} 's/<regex\.h>/<gnuregex.h>/' ${WRKSRC}/src/gdkxft.c
 
 post-install:
 	@${STRIP_CMD} ${STAGEDIR}${PREFIX}/lib/libgdkxft.so
