--- distinfo.orig	2017-11-21 11:13:50 UTC
+++ distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1510447436
-SHA256 (kusumi-fileobj-v0.7.55_GH0.tar.gz) = 1c74475d87b5832efb507b0493092fe6bf366887cfd483741723d1eeedf7ade5
-SIZE (kusumi-fileobj-v0.7.55_GH0.tar.gz) = 128505
+TIMESTAMP = 1567429444
+SHA256 (kusumi-fileobj-v0.7.91_GH0.tar.gz) = b7a1d6799b967cc768215542c173aec2ea8b568feedad7658d1c6c133cd5bf8d
+SIZE (kusumi-fileobj-v0.7.91_GH0.tar.gz) = 163929
