--- distinfo.orig	2013-01-05 21:50:00.396809000 +0100
+++ distinfo	2013-01-05 21:39:07.835897000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_ss.tar.gz) = 27f93ed39e5e68c953ba58241871417cb8d8bedccd7dfd373968752c42091c7d
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_ss.tar.gz) = 1652814
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_ss.tar.gz) = 1975163f80ebe4586e81a57281c814df759c1663faa6510c184da4ffc6eec9ab
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_ss.tar.gz) = 1603628
