--- pkg-plist.orig
+++ pkg-plist
@@ -6,10 +6,12 @@
 share/icons/hicolor/scalable/apps/org.gnome.Calendar.svg
 share/icons/hicolor/symbolic/apps/org.gnome.Calendar-symbolic.svg
 share/icons/hicolor/symbolic/apps/org.gnome.Calendar.Devel-symbolic.svg
+share/locale/ab/LC_MESSAGES/gnome-calendar.mo
 share/locale/af/LC_MESSAGES/gnome-calendar.mo
 share/locale/an/LC_MESSAGES/gnome-calendar.mo
 share/locale/ar/LC_MESSAGES/gnome-calendar.mo
 share/locale/be/LC_MESSAGES/gnome-calendar.mo
+share/locale/be@latin/LC_MESSAGES/gnome-calendar.mo
 share/locale/bg/LC_MESSAGES/gnome-calendar.mo
 share/locale/bs/LC_MESSAGES/gnome-calendar.mo
 share/locale/ca/LC_MESSAGES/gnome-calendar.mo
@@ -33,20 +35,24 @@
 share/locale/gd/LC_MESSAGES/gnome-calendar.mo
 share/locale/gl/LC_MESSAGES/gnome-calendar.mo
 share/locale/he/LC_MESSAGES/gnome-calendar.mo
+share/locale/hi/LC_MESSAGES/gnome-calendar.mo
 share/locale/hr/LC_MESSAGES/gnome-calendar.mo
 share/locale/hu/LC_MESSAGES/gnome-calendar.mo
 share/locale/id/LC_MESSAGES/gnome-calendar.mo
 share/locale/is/LC_MESSAGES/gnome-calendar.mo
 share/locale/it/LC_MESSAGES/gnome-calendar.mo
 share/locale/ja/LC_MESSAGES/gnome-calendar.mo
+share/locale/ka/LC_MESSAGES/gnome-calendar.mo
 share/locale/kab/LC_MESSAGES/gnome-calendar.mo
 share/locale/kk/LC_MESSAGES/gnome-calendar.mo
 share/locale/ko/LC_MESSAGES/gnome-calendar.mo
+share/locale/kw/LC_MESSAGES/gnome-calendar.mo
 share/locale/lt/LC_MESSAGES/gnome-calendar.mo
 share/locale/lv/LC_MESSAGES/gnome-calendar.mo
 share/locale/mjw/LC_MESSAGES/gnome-calendar.mo
 share/locale/ml/LC_MESSAGES/gnome-calendar.mo
 share/locale/ms/LC_MESSAGES/gnome-calendar.mo
+share/locale/my/LC_MESSAGES/gnome-calendar.mo
 share/locale/nb/LC_MESSAGES/gnome-calendar.mo
 share/locale/ne/LC_MESSAGES/gnome-calendar.mo
 share/locale/nl/LC_MESSAGES/gnome-calendar.mo
@@ -66,13 +72,11 @@
 share/locale/tg/LC_MESSAGES/gnome-calendar.mo
 share/locale/th/LC_MESSAGES/gnome-calendar.mo
 share/locale/tr/LC_MESSAGES/gnome-calendar.mo
+share/locale/ug/LC_MESSAGES/gnome-calendar.mo
 share/locale/uk/LC_MESSAGES/gnome-calendar.mo
+share/locale/uz/LC_MESSAGES/gnome-calendar.mo
 share/locale/vi/LC_MESSAGES/gnome-calendar.mo
 share/locale/zh_CN/LC_MESSAGES/gnome-calendar.mo
 share/locale/zh_HK/LC_MESSAGES/gnome-calendar.mo
 share/locale/zh_TW/LC_MESSAGES/gnome-calendar.mo
-share/locale/ab/LC_MESSAGES/gnome-calendar.mo
-share/locale/hi/LC_MESSAGES/gnome-calendar.mo
-share/locale/ka/LC_MESSAGES/gnome-calendar.mo
-share/locale/my/LC_MESSAGES/gnome-calendar.mo
 share/metainfo/org.gnome.Calendar.metainfo.xml
