--- Makefile.orig	2020-03-14 01:32:49 UTC
+++ Makefile
@@ -69,6 +69,12 @@ INSTALL_TARGET=	install-strip
 .if ${SLAVE_PORT} == "no" || ${PORTNAME} == "xephyr" || ${PORTNAME} == "xwayland"
 LIB_DEPENDS+=	libdrm.so:graphics/libdrm \
 		libepoxy.so:graphics/libepoxy
+# Recommended by M.Dillon for sane security defaults
+CONFIGURE_ARGS+=	--enable-xcsecurity
+# zrj: no glamor for xwayland (needs DRI3)
+. if ${PORTNAME} == "xwayland"
+CONFIGURE_ARGS+= --disable-dri3 --disable-glamor
+. endif
 .else
 BUILD_DEPENDS+=	libepoxy>0:graphics/libepoxy	# only for configure
 .endif
@@ -95,9 +101,9 @@ PLIST=		${.CURDIR}/pkg-plist
 CONFIGURE_ENV=	SHA1_LIB="-L/usr/lib -lcrypto" SHA1_CFLAGS="-I/usr/include"
 .endif
 
-.if ${ARCH} == "i386" || ${ARCH} == "amd64"
-LIB_DEPENDS+=	libunwind.so:devel/libunwind
-.endif
+#.if ${ARCH} == "i386" || ${ARCH} == "amd64"
+#LIB_DEPENDS+=	libunwind.so:devel/libunwind
+#.endif
 
 .if ${ARCH} == "sparc64"
 PLIST_SUB+=	SPARC64=""
