--- Makefile.orig	2021-02-22 19:06:34 UTC
+++ Makefile
@@ -76,6 +76,12 @@ CPE_VENDOR=	x.org
 .if ${SLAVE_PORT} == "no" || ${PORTNAME} == "xephyr" || ${PORTNAME} == "xwayland"
 LIB_DEPENDS+=	libdrm.so:graphics/libdrm \
 		libepoxy.so:graphics/libepoxy
+# Recommended by M.Dillon for sane security defaults
+CONFIGURE_ARGS+=	--enable-xcsecurity
+# zrj: no glamor for xwayland (needs DRI3)
+. if ${PORTNAME} == "xwayland"
+CONFIGURE_ARGS+= --disable-dri3 --disable-glamor
+. endif
 .else
 BUILD_DEPENDS+=	libepoxy>0:graphics/libepoxy	# only for configure
 .endif
