--- Makefile.orig	2020-02-14 00:14:45 UTC
+++ Makefile
@@ -54,12 +54,18 @@ CONFIGURE_ARGS+=--without-doxygen --with
 		--with-default-font-path="$$(${DEFAULT_FONTPATH_CMD})" \
 		--localstatedir=/var --with-shared-memory-dir=/tmp \
 		--disable-config-udev-kms --disable-systemd-logind \
-		--without-dtrace --enable-glamor
+		--without-dtrace --enable-glamor --disable-dri3
 INSTALL_TARGET=	install-strip
 
 .if ${SLAVE_PORT} == "no" || ${PORTNAME} == "xephyr" || ${PORTNAME} == "xwayland"
 LIB_DEPENDS+=	libdrm.so:graphics/libdrm \
 		libepoxy.so:graphics/libepoxy
+# Recommended by M.Dillon for sane security defaults
+CONFIGURE_ARGS+=	--enable-xcsecurity
+# zrj: no glamor for xwayland (needs DRI3)
+. if ${PORTNAME} == "xwayland"
+CONFIGURE_ARGS+= --disable-glamor
+. endif
 .else
 BUILD_DEPENDS+=	libepoxy>0:graphics/libepoxy	# only for configure
 .endif
@@ -103,9 +109,9 @@ CONFIGURE_ARGS+= --disable-config-udev
 # We handle Xorg setuid in the plist. This allows to build xorg-server as a user.
 CONFIGURE_ARGS+=--disable-install-setuid
 
-.if ${ARCH} == "i386" || ${ARCH} == "amd64"
-LIB_DEPENDS+=	libunwind.so:devel/libunwind
-.endif
+#.if ${ARCH} == "i386" || ${ARCH} == "amd64"
+#LIB_DEPENDS+=	libunwind.so:devel/libunwind
+#.endif
 
 .if ${ARCH} == "sparc64"
 PLIST_SUB+=	SPARC64=""
