--- Makefile.orig	2022-10-12 11:12:47 UTC
+++ Makefile
@@ -82,6 +82,12 @@ MESON_ARGS+=	-Dudev=false
 MESON_ARGS+=	-Ddrm=true
 LIB_DEPENDS+=	libdrm.so:graphics/libdrm \
 		libepoxy.so:graphics/libepoxy
+# Recommended by M.Dillon for sane security defaults
+CONFIGURE_ARGS+=	--enable-xcsecurity
+# zrj: no glamor for xwayland (needs DRI3)
+. if ${PORTNAME} == "xwayland"
+CONFIGURE_ARGS+= --disable-dri3 --disable-glamor
+. endif
 .endif
 
 .if ${FLAVOR} == xephyr
