--- distinfo.orig	2013-01-05 21:50:00.416809000 +0100
+++ distinfo	2013-01-05 21:39:15.035907000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_et.tar.gz) = ce5f60583418ce876f41a2ef0edac826bd7af592d362b1badb496a6d6d6c9832
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_et.tar.gz) = 2533919
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_et.tar.gz) = be9834d6c4c37d4dbbcb0d49c9bea47fd9f9b70145c8fccc010b3d66e9748405
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_et.tar.gz) = 10424853
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_et.tar.gz) = 490f3060f956513b771da307a9f6c288eb70c2736b9b548e6accbb3a37b8b638
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_et.tar.gz) = 2480386
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_et.tar.gz) = 07073788ca0d8b4e32e22f2e4d8847bad7c062680eef139f7821b21f5c8ae9cc
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_et.tar.gz) = 10429385
