--- Makefile.orig	2013-04-22 18:45:59.000000000 +0000
+++ Makefile
@@ -32,7 +32,7 @@ CMAKE_ARGS+=	-DINSTALL_DOCDIR="share/doc
 		-DINSTALL_SQLBENCHDIR="share/mysql" \
 		-DINSTALL_SUPPORTFILESDIR="share/mysql" \
 		-DWITH_LIBEDIT=0 \
-		-DWITH_LIBWRAP=1
+		-DWITH_READLINE=1
 
 .ifdef USE_MYSQL
 .error You have `USE_MYSQL' variable defined either in environment or in make(1) arguments. Please undefine and try again.
