--- distinfo.orig	2020-03-14 00:47:35 UTC
+++ distinfo
@@ -3,3 +3,5 @@ SHA256 (apache-ant-1.9.14-bin.tar.gz) =
 SIZE (apache-ant-1.9.14-bin.tar.gz) = 5837539
 SHA256 (battleblow-jdk7u-7.251.02.1-jdk7u251-b02.1_GH0.tar.gz) = b709d9e344c4ebbc5bb71646a7e78137ad47f2e17fb61739c7be0420ececc707
 SIZE (battleblow-jdk7u-7.251.02.1-jdk7u251-b02.1_GH0.tar.gz) = 69828420
+SHA256 (bootstrap-openjdk6-b41.txz) = b25715684f710544148bf1b246a6e0982d0601755fe9466823c6d8fd1b8bb8a5
+SIZE (bootstrap-openjdk6-b41.txz) = 26562215
