--- files/pkg-message.in.orig	2020-04-16 14:36:54 UTC
+++ files/pkg-message.in
@@ -18,17 +18,7 @@ To enable spamd you need:
    spamd.conf.sample file.  Copy then to spamd.conf file and
    edit to suit your needs.
 
-4) mount fdescfs to /dev/fd with the following line in /etc/fstab
-	fdescfs  	/dev/fd  	fdescfs rw 	0 	0
-
-   Note for XEN users:
-   The XEN kernel ships without modules, therefore you have to add
-   the following lines to the kernel config and build a new kernel.
-	options FDESCFS
-	device	pf
-	device	pflog
-
-5) Add following lines to the pf.conf(5)
+4) Add following lines to the pf.conf(5)
 
   table <spamd-white> persist
   no rdr inet proto tcp from <spamd-white> to any \
