--- pkg-plist.orig	2013-05-11 19:36:40.000000000 +0000
+++ pkg-plist
@@ -73,7 +73,7 @@ lib/libpyglib-gi-2.0-python%%PYTHON_VER%
 %%PYTHON_SITELIBDIR%%/gi/types.py
 %%PYTHON_SITELIBDIR%%/gi/types.pyc
 %%PYTHON_SITELIBDIR%%/gi/types.pyo
-%%PYTHON_SITELIBDIR%%/pygobject-%%VERSION%%-pypython%%PYTHON_VER%%-freebsd-%%UNAME_R%%-%%ARCH%%.egg-info
+%%PYTHON_SITELIBDIR%%/pygobject-%%VERSION%%-pypython%%PYTHON_VER%%-dragonfly-%%UNAME_R%%-%%ARCH%%.egg-info
 %%PYTHON_SITELIBDIR%%/pygtkcompat/__init__.py
 %%PYTHON_SITELIBDIR%%/pygtkcompat/__init__.pyc
 %%PYTHON_SITELIBDIR%%/pygtkcompat/__init__.pyo
