--- distinfo.orig	2022-01-13 04:21:05 UTC
+++ distinfo
@@ -5,6 +5,8 @@
 SIZE (ghc-8.10.7-boot-amd64-freebsd.tar.xz) = 70607916
 SHA256 (ghc-8.10.7-boot-i386-freebsd.tar.xz) = d841a6abf2b97e34feb9c71a85520a78dd667bfb6d6a7f401fa8a92b558928dc
 SIZE (ghc-8.10.7-boot-i386-freebsd.tar.xz) = 70040192
+SHA256 (ghc-8.8.4-boot-x86_64-dragonfly.tar.xz) = 84e197185ca7d7150c2e5019814dc4b7a6894f8008164a256f4d8acb27ced3fe
+SIZE (ghc-8.8.4-boot-x86_64-dragonfly.tar.xz) = 73121164
 SHA256 (ghc-8.10.4-boot-armv6-freebsd.tar.xz) = 04b207b787bdff71028b8d4c311a47105d7b098809e8fd21b788b1ab0dccdf22
 SIZE (ghc-8.10.4-boot-armv6-freebsd.tar.xz) = 148679080
 SHA256 (ghc-8.10.4-boot-armv7-freebsd.tar.xz) = db94e69b83a93fba9c136e890ccb0bf76b6d163ac3979dadb1568e81c3267744
