--- pkg-plist.orig	2013-01-20 19:09:00.193402000 +0000
+++ pkg-plist
@@ -9,7 +9,6 @@
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/__init__.py
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/__init__.pyc
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/__init__.pyo
-%%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/CHANGELOG.txt
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/LICENSE.txt
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/README.txt
@@ -17,18 +16,15 @@
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/__init__.py
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/__init__.pyc
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/__init__.pyo
-%%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/lib
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/lib/libtaup.py
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/lib/libtaup.pyc
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/lib/libtaup.pyo
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/lib/libtaup.so
-%%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/src
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/src/emdlv.f
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/src/iaspei-tau.tgz
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/src/libtau.f
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/src/ttimes_subrout.f
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/src/ttlim.inc
-%%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/tables
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/tables/ak135.hed
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/tables/ak135.tbl
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/tables/iasp91.hed
@@ -36,11 +32,9 @@
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/taup.py
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/taup.pyc
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/taup.pyo
-%%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/tests
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/tests/__init__.py
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/tests/__init__.pyc
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/tests/__init__.pyo
-%%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/tests/data
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/tests/data/sample_ttimes_ak135.lst
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/tests/data/sample_ttimes_iasp91.lst
 %%PYTHON_SITELIBDIR%%/%%PYEASYINSTALL_EGG%%/obspy/taup/tests/test_taup.py
