--- distinfo.orig	2013-01-05 21:50:01.386810000 +0100
+++ distinfo	2013-01-05 21:42:08.426149000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_bs.tar.gz) = f09060f9aa4b914cf14197f229e0209d5d0e3c47063c47acd738fb952eb7471f
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_bs.tar.gz) = 1642739
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_bs.tar.gz) = ed2535e62b55bba7bacba329d32429cdd52ff7af52f250cee14f10e8e5522d5a
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_bs.tar.gz) = 10063413
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_bs.tar.gz) = 4d3b715b0491099c5528076c5e807806c5c3c9d6d36c5522a87c369ee4a59b9c
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_bs.tar.gz) = 1592756
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_bs.tar.gz) = 95370d0d98714fa2eb528fd3ac794e216499cc536245ee0a3e6dc4e3381e172f
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_bs.tar.gz) = 10071117
