--- files/patch-cmake_external_glib_cppgir_tools_genutils.hpp.orig	2024-09-08 19:29:03 UTC
+++ files/patch-cmake_external_glib_cppgir_tools_genutils.hpp
@@ -0,0 +1,10 @@
+--- cmake/external/glib/cppgir/tools/genutils.hpp.orig	2023-12-20 18:54:36 UTC
++++ cmake/external/glib/cppgir/tools/genutils.hpp
+@@ -4,6 +4,7 @@
+ #include "common.hpp"
+ 
+ #include <fstream>
++#include <sstream>
+ #include <regex>
+ #include <set>
+ #include <string>
