--- Makefile.orig	2024-09-23 14:55:30 UTC
+++ Makefile
@@ -29,7 +29,6 @@ LIB_DEPENDS=	libabsl_base.so:devel/absei
 		libavformat.so:multimedia/ffmpeg \
 		libcrc32c.so:devel/crc32c \
 		libdbus-1.so:devel/dbus \
-		libdispatch.so:devel/libdispatch \
 		libfmt.so:devel/libfmt \
 		libhunspell-1.7.so:textproc/hunspell \
 		liblz4.so:archivers/liblz4 \
@@ -84,9 +83,6 @@ CMAKE_ON=	DESKTOP_APP_USE_PACKAGED DESKT
 # Since cmake 3.25.0, BSDs are no longer recognised as Linux (in this port).
 CMAKE_ARGS+=	-DLINUX=true
 
-# libdispatch components
-LDFLAGS+=	-lBlocksRuntime
-
 # Telegram uses OpenAL for its audio, but libtgvoip (for voice calls) can use PulseAudio or ALSA.
 # It dynamically loads PulseAudio, and if this fails, it loads ALSA.
 # If both of them are not installed, then voice calls do not work, but other functionalities still work.
@@ -128,14 +124,11 @@ post-extract:
 	# can be safely removed
 	${RM} -rf \
 		${WRKSRC}/Telegram/ThirdParty/QR \
-		${WRKSRC}/Telegram/ThirdParty/dispatch \
 		${WRKSRC}/Telegram/ThirdParty/expected \
 		${WRKSRC}/Telegram/ThirdParty/fcitx5-qt \
 		${WRKSRC}/Telegram/ThirdParty/hime \
 		${WRKSRC}/Telegram/ThirdParty/hunspell \
-		${WRKSRC}/Telegram/ThirdParty/jemalloc \
 		${WRKSRC}/Telegram/ThirdParty/jimf \
-		${WRKSRC}/Telegram/ThirdParty/kcoreaddons \
 		${WRKSRC}/Telegram/ThirdParty/kimageformats \
 		${WRKSRC}/Telegram/ThirdParty/lz4 \
 		${WRKSRC}/Telegram/ThirdParty/minizip \
