--- Makefile.orig	2023-11-17 19:54:39 UTC
+++ Makefile
@@ -28,7 +28,6 @@ BUILD_DEPENDS=	boost-libs>0:devel/boost-
 LIB_DEPENDS=	libabsl_base.so:devel/abseil \
 		libavformat.so:multimedia/ffmpeg \
 		libdbus-1.so:devel/dbus \
-		libdispatch.so:devel/libdispatch \
 		libfmt.so:devel/libfmt \
 		libhunspell-1.7.so:textproc/hunspell \
 		liblz4.so:archivers/liblz4 \
@@ -78,9 +77,6 @@ CMAKE_ON=	DESKTOP_APP_USE_PACKAGED DESKT
 # Since cmake 3.25.0, BSDs are no longer recognised as Linux (in this port).
 CMAKE_ARGS+=	-DLINUX=true
 
-# libdispatch components
-LDFLAGS+=	-lBlocksRuntime
-
 # Telegram uses OpenAL for its audio, but libtgvoip (for voice calls) can use PulseAudio or ALSA.
 # It dynamically loads PulseAudio, and if this fails, it loads ALSA.
 # If both of them are not installed, then voice calls do not work, but other functionalities still work.
