--- pkg-plisit.orig	2021-12-13 10:52:31 UTC
+++ pkg-plist
@@ -2139,8 +2139,6 @@ GNUstep/Local/Library/WOxElemBuilders-4.
 GNUstep/Local/Library/WOxElemBuilders-4.9/SOGoElements.wox/SOGoElements
 GNUstep/Local/Library/WOxElemBuilders-4.9/SOGoElements.wox/bundle-info.plist
 GNUstep/Local/Library/WOxElemBuilders-4.9/SOGoElements.wox/stamp.make
-GNUstep/Local/Tools/Admin/cron-ealarms-notify.sample
-GNUstep/Local/Tools/Admin/cron-expire-autoreply.sample
 GNUstep/Local/Tools/Admin/sogo-ealarms-notify
 GNUstep/Local/Tools/Admin/sogo-slapd-sockd
 GNUstep/Local/Tools/Admin/sogo-tool
