--- Makefile.orig	2016-03-17 11:55:25 UTC
+++ Makefile
@@ -39,7 +39,7 @@ ALL_TARGET=	pypy-c
 BUILD_WRKSRC=	${WRKDIR}/build/usession-release-5.0-0/testing_1
 MAKE_ENV+=	PYPY_LOCALBASE=${LOCALBASE}
 USE_LDCONFIG=	${PREFIX}/${PYPY_DIR}/bin
-USES=		gettext-runtime gmake tar:bzip2
+USES=		gettext-runtime gmake ncurses tar:bzip2
 
 PYPY_VARS=	PYTHON_CMD=${LOCALBASE}/bin/pypy
 PYPY_MINMEM_VARS=	PYTHON_CMD="${SETENV} PYPY_GC_MAX_DELTA=200MB ${LOCALBASE}/bin/pypy --jit loop_longevity=300"
@@ -68,10 +68,10 @@ post-build:
 	${CP} ${BUILD_WRKSRC}/pypy-c ${BUILD_WRKSRC}/libpypy-c.so ${WRKSRC}/pypy/goal/
 
 do-install:
-	${SETENV} TMPDIR=${WRKDIR}/build \
+	${SETENV} ${MAKE_ENV} TMPDIR=${WRKDIR}/build \
 		${PYTHON_CMD} ${WRKSRC}/pypy/tool/release/package.py --builddir ${WRKDIR}/build --archive-name ${PYPY_DIR} \
 		--without-gdbm --without-sqlite3 --without-tk
-	${EXTRACT_CMD} -C ${STAGEDIR}${PREFIX} -xf ${WRKDIR}/build/${PYPY_DIR}.tar.bz2
+	${TAR} -C ${STAGEDIR}${PREFIX} -xf ${WRKDIR}/build/${PYPY_DIR}.tar.bz2
 	${LN} -fs ../${PYPY_DIR}/bin/pypy ${STAGEDIR}${PREFIX}/bin/pypy
 
 do-test:
