--- Makefile.orig	2017-04-12 22:47:37 UTC
+++ Makefile
@@ -46,7 +46,7 @@ BUILD_WRKSRC?=	${WRKDIR}/build/usession-
 MAKE_ENV+=	PYPY_LOCALBASE=${LOCALBASE} LDSHARED="${CC} -shared"
 PACKAGE_ARGS?=	--without-gdbm --without-sqlite3 --without-tk
 USE_LDCONFIG=	${PREFIX}/${PYPY_DIR}/bin
-USES=		gettext-runtime gmake ssl tar:bzip2
+USES=		gettext-runtime gmake ncurses ssl tar:bzip2
 
 .if ${PORTNAME} != pypy
 PYPY_BUILD_DEPENDS=	pypy:lang/pypy
