--- Makefile.orig	2017-05-03 14:05:32 UTC
+++ Makefile
@@ -47,7 +47,7 @@ BUILD_WRKSRC?=	${WRKDIR}/build/usession-
 MAKE_ENV+=	PYPY_LOCALBASE=${LOCALBASE} LDSHARED="${CC} -shared"
 PACKAGE_ARGS?=	--without-gdbm --without-sqlite3 --without-tk
 USE_LDCONFIG=	${PREFIX}/${PYPY_DIR}/bin
-USES=		gettext-runtime gmake shebangfix ssl tar:bzip2
+USES=		gettext-runtime gmake ncurses shebangfix ssl tar:bzip2
 SHEBANG_FILES?=	lib-python/2.7/plat-mac/appletrunner.py \
 		lib-python/2.7/plat-mac/bundlebuilder.py \
 		lib-python/2.7/keyword.py \
