--- Makefile.orig	2020-04-16 13:55:58 UTC
+++ Makefile
@@ -25,7 +25,7 @@ TEST_DEPENDS=	${LOCALBASE}/${PYPY_DIR}/l
 		${LOCALBASE}/${PYPY_DIR}/lib_pypy/_sqlite3_cffi.${PYPY_CFFI_VER}.so:databases/pypy-sqlite3 \
 		${LOCALBASE}/${PYPY_DIR}/lib_pypy/_tkinter/tklib_cffi.${PYPY_CFFI_VER}.so:x11-toolkits/pypy-tkinter
 
-USES=		gettext-runtime gmake shebangfix ssl tar:bzip2
+USES=		gettext-runtime gmake shebangfix ssl tar:bzip2 ncurses
 USE_LDCONFIG=	${PREFIX}/${PYPY_DIR}/bin
 
 OPTIONS_SINGLE=	TRANS
@@ -106,11 +106,7 @@ PLIST=		${.CURDIR}/pkg-plist
 .include "${MASTERDIR}/bsd.pypy.mk"
 .include <bsd.port.pre.mk>
 
-.if ${OPSYS} == FreeBSD && ${ARCH} != i386 && ${ARCH} != amd64
 PYPY_ARGS=	--withoutmod-_vmprof
-.else
-LIB_DEPENDS+=	libunwind.so:devel/libunwind
-.endif
 
 post-extract:
 	${FIND} ${WRKSRC} -name '*.swn' -delete
