--- Makefile.orig	2018-08-22 16:59:11 UTC
+++ Makefile
@@ -50,7 +50,7 @@ MAKE_ENV+=	PYPY_LOCALBASE=${LOCALBASE} L
 PACKAGE_ARGS?=	--without-gdbm --without-sqlite3 --without-tk
 USE_LDCONFIG=	${PREFIX}/${PYPY_DIR}/bin
 LLD_UNSAFE=	yes
-USES=		gettext-runtime gmake shebangfix ssl tar:bzip2
+USES=		gettext-runtime gmake shebangfix ssl tar:bzip2 ncurses
 SHEBANG_FILES?=	lib-python/2.7/plat-mac/appletrunner.py \
 		lib-python/2.7/plat-mac/bundlebuilder.py \
 		lib-python/2.7/keyword.py \
@@ -105,11 +105,7 @@ PLIST=		${.CURDIR}/pkg-plist
 .include "${MASTERDIR}/bsd.pypy.mk"
 .include <bsd.port.pre.mk>
 
-.if ${OPSYS} == FreeBSD && ${ARCH} != i386 && ${ARCH} != amd64
 PYPY_ARGS=	--withoutmod-_vmprof
-.else
-LIB_DEPENDS+=	libunwind.so:devel/libunwind
-.endif
 
 post-extract:
 	${FIND} ${WRKSRC} -name '*.swn' -delete
