--- Makefile.orig	2016-06-30 16:18:21 UTC
+++ Makefile
@@ -38,7 +38,7 @@ ONLY_FOR_ARCHS=	i386 amd64 powerpc power
 
 OPTIONS_DEFINE=	DEBUG EXPAT GDB_LINK GUILE KGDB PYTHON TUI
 
-OPTIONS_DEFAULT=	GDB_LINK KGDB PYTHON TUI PORT_READLINE
+OPTIONS_DEFAULT=	GDB_LINK PYTHON TUI PORT_READLINE
 
 OPTIONS_SINGLE=	READLINE
 OPTIONS_SINGLE_READLINE=	BASE_READLINE BUNDLED_READLINE PORT_READLINE
@@ -74,12 +74,8 @@ TUI_CONFIGURE_ENABLE=	tui
 EXCLUDE+=	readline
 .endif
 
-.if ${ARCH} == "amd64"
-CONFIGURE_TARGET=	x86_64-portbld-freebsd${OSREL}
-.endif
-
 post-patch:
-	@${REINPLACE_CMD} -e 's|$$| [GDB v${PORTVERSION} for FreeBSD]|' \
+	@${REINPLACE_CMD} -e 's|$$| [GDB v${PORTVERSION} for ${OPSYS}]|' \
 		${WRKSRC}/gdb/version.in
 
 post-patch-KGDB-on:
@@ -105,6 +101,12 @@ do-install-GDB_LINK-on:
 	${LN} -sf kgdb${VER} ${STAGEDIR}${PREFIX}/bin/kgdb
 .endif
 
+dfly-patch:
+	@${CP} ${DFLY_FILESDIR}/dfly64.mh ${WRKSRC}/gdb/config/i386/
+	@${CP} ${DFLY_FILESDIR}/dfly-nat.* ${WRKSRC}/gdb/
+	@${CP} ${DFLY_FILESDIR}/i386* ${WRKSRC}/gdb/
+	@${CP} ${DFLY_FILESDIR}/amd64* ${WRKSRC}/gdb/
+
 do-install-PYTHON-on:
 	(cd ${WRKSRC}/gdb; ${SETENV} ${MAKE_ENV} ${MAKE_CMD} ${MAKE_ARGS} install-python )
 	(cd ${WRKSRC}/gdb/data-directory ; \
