--- Makefile.orig	2018-08-22 16:49:11 UTC
+++ Makefile
@@ -57,7 +57,7 @@ PLIST_SUB=	VER=${VER}
 
 OPTIONS_DEFINE=	DEBUG GDB_LINK GUILE KGDB PYTHON TUI
 
-OPTIONS_DEFAULT=	GDB_LINK KGDB PYTHON TUI PORT_READLINE PORT_ICONV SYSTEM_ZLIB
+OPTIONS_DEFAULT=	GDB_LINK PYTHON TUI PORT_READLINE PORT_ICONV SYSTEM_ZLIB
 
 OPTIONS_SINGLE=	READLINE ICONV ZLIB
 OPTIONS_SINGLE_READLINE=	BUNDLED_READLINE PORT_READLINE
@@ -100,16 +100,12 @@ EXCLUDE+=	readline
 EXCLUDE+=	zlib
 .endif
 
-.if ${ARCH} == "amd64"
-CONFIGURE_TARGET=	x86_64-portbld-freebsd${OSREL}
-.endif
-
-.if ${COMPILER_TYPE} == "clang"
-CFLAGS+=	-Wno-extended-offsetof
-.endif
+# Wow c++ smth.c too on gdb, gcc wasn't enough? throw helpers for c-- and try to kill it
+CFLAGS+=	-D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS
+CONFIGURE_ARGS+=	--disable-build-with-cxx
 
 post-patch:
-	@${REINPLACE_CMD} -e 's|$$| [GDB v${PORTVERSION} for FreeBSD]|' \
+	@${REINPLACE_CMD} -e 's|$$| [GDB v${PORTVERSION} for ${OPSYS}]|' \
 		${WRKSRC}/gdb/version.in
 
 post-patch-KGDB-on:
@@ -139,6 +135,15 @@ do-install-GDB_LINK-on:
 	${LN} -sf kgdb${VER} ${STAGEDIR}${PREFIX}/bin/kgdb
 .endif
 
+dfly-patch:
+	${CP} ${DFLY_FILESDIR}/dfly64.mh ${WRKSRC}/gdb/config/i386/
+	${CP} ${DFLY_FILESDIR}/dfly-nat.* ${WRKSRC}/gdb/
+	${CP} ${DFLY_FILESDIR}/i386* ${WRKSRC}/gdb/
+	${CP} ${DFLY_FILESDIR}/amd64* ${WRKSRC}/gdb/
+.if ${PORT_OPTIONS:MKGDB}
+	${PATCH} ${PATCH_ARGS} < ${DFLY_FILESDIR}/extrapatch-kgdb
+.endif
+
 do-install-PYTHON-on:
 	(cd ${WRKSRC}/gdb; ${SETENV} ${MAKE_ENV} ${MAKE_CMD} ${MAKE_ARGS} install-python )
 	(cd ${WRKSRC}/gdb/data-directory ; \
