--- Makefile.orig	2013-03-25 02:33:52.414740000 +0000
+++ Makefile
@@ -14,6 +14,7 @@ COMMENT?=	Multithreaded SQL database (se
 
 SLAVEDIRS=	databases/mysql55-client
 USES=		cmake
+USE_READLINE=	yes
 MAKE_JOBS_SAFE=	yes
 NO_OPTIONS_SORT=yes
 
@@ -38,7 +39,7 @@ CMAKE_ARGS+=	-DINSTALL_DOCDIR="share/doc
 		-DINSTALL_SQLBENCHDIR="share/mysql" \
 		-DINSTALL_SUPPORTFILESDIR="share/mysql" \
 		-DWITH_LIBEDIT=0 \
-		-DWITH_LIBWRAP=1
+		-DWITH_READLINE=1
 
 .ifdef USE_MYSQL
 .error You have `USE_MYSQL' variable defined either in environment or in make(1) arguments. Please undefine and try again.
