--- Makefile.orig	Fri Feb 28 12:01:40 2025
+++ Makefile	Fri Feb
@@ -2,6 +2,7 @@ PORTNAME=	fpc
 PORTVERSION=	3.2.3
 CATEGORIES=	lang
 MASTER_SITES=	http://leaf.dragonflybsd.org/~marino/dports-src/:DragonFlybootstrap \
+		http://avalon.dragonflybsd.org/misc/distfiles/freepascal/:DragonFlybootstrap \
 		LOCAL/acm/freepascal/:bootstrap \
 		LOCAL/acm/freepascal/:man
 DIST_SUBDIR=	freepascal
@@ -12,7 +13,7 @@ WWW=		https://www.freepascal.org/
 
 CONFLICTS=	fpc-devel
 
-RUN_DEPENDS=	${LOCALBASE}/bin/as:devel/binutils
+RUN_DEPENDS=	${LOCALBASE}/bin/as:devel/binutils@native
 
 ONLY_FOR_ARCHS=	amd64 i386
 
@@ -37,7 +38,7 @@ FPC_ARCH=	i386
 #OPT=		"-dFPC_USE_LIBC"
 PLIST_SUB+=	FPC_I386="" \
 		FPC_AMD64="@comment "
-.elif ${ARCH} == "amd64"
+.elif ${ARCH} == "x86_64"
 PPNAME=		ppcx64
 FPC_ARCH=	x86_64
 #OPT=		"-dFPC_USE_LIBC"
@@ -111,52 +112,86 @@ post-patch:
 		${WRKDIR}/${FPCSRCDIR}/packages/fpmkunit/src/fpmkunit.pp
 
 # enable units
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+# NOTE: make sure that the value is NOT repeated in the target subs or
+# you will get:
+# Error: range check error in set constructor or duplicate set element
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/a52/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/dbus/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/dts/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|freebsd,||g' \
+	@${REINPLACE_CMD} -i "" -e 's|dragonfly,||g' \
 		${WRKSRC}/packages/fpgtk/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|iphonesim|freebsd,iphonesim|g' \
+	@${REINPLACE_CMD} -i "" -e 's|iphonesim|freebsd,dragonfly,iphonesim|g' \
 		${WRKSRC}/packages/gnome1/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|iphonesim|freebsd,iphonesim|g' \
+	@${REINPLACE_CMD} -i "" -e 's|iphonesim|freebsd,dragonfly,iphonesim|g' \
 		${WRKSRC}/packages/gtk1/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|iphonesim|freebsd,iphonesim|g' \
+	@${REINPLACE_CMD} -i "" -e 's|iphonesim|freebsd,dragonfly,iphonesim|g' \
 		${WRKSRC}/packages/imlib/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/ldap/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/libsee/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/lua/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/mad/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/modplug/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|Linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|Linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/newt/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/oggvorbis/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/openal/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/opencl/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/proj4/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/ptc/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|os2|freebsd,os2|g' \
+	@${REINPLACE_CMD} -i "" -e 's|os2|freebsd,dragonfly,os2|g' \
 		${WRKSRC}/packages/rexx/fpmake.pp
-	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,linux|g' \
+	@${REINPLACE_CMD} -i "" -e 's|linux|freebsd,dragonfly,linux|g' \
 		${WRKSRC}/packages/zorba/fpmake.pp
+	@${REINPLACE_CMD} -i "" -e 's|linux,freebsd|linux,freebsd,dragonfly|g' \
+		${WRKSRC}/packages/rtl-extra/fpmake.pp
 
+# dragonfly specifics
+	@${REINPLACE_CMD} -i "" -e 's|,dragonfly||g' \
+		${WRKSRC}/packages/fpgtk/fpmake.pp
+	@${REINPLACE_CMD} -i "" -e 's|freebsd,|freebsd,dragonfly,|g' \
+		${WRKSRC}/packages/users/fpmake.pp
+	@${REINPLACE_CMD} -i "" -e 's|freebsd,|freebsd,dragonfly,|g' \
+		${WRKSRC}/utils/instantfpc/fpmake.pp \
+		${WRKSRC}/utils/fpcres/fpmake.pp
+	@${REINPLACE_CMD} -i "" -e 's|freebsd|freebsd, dragonfly|g' \
+		${WRKSRC}/packages/fcl-report/fpmake.pp
+	@${REINPLACE_CMD} -i "" -e 's|openbsd, |openbsd, dragonfly, |g' \
+		${WRKSRC}/packages/ide/fpmake.pp
+	@${REINPLACE_CMD} -i "" -e 's|freebsd,|freebsd,dragonfly,|g' \
+		${WRKSRC}/packages/libxml/fpmake.pp
+	@${REINPLACE_CMD} -i "" -e 's|freebsd,|freebsd,dragonfly,|g' \
+		${WRKSRC}/packages/gmp/fpmake.pp
+	@${REINPLACE_CMD} -i "" -e 's/freebsd,/freebsd,dragonfly,/g' \
+		${WRKSRC}/packages/fcl-xml/fpmake.pp
+	@${REINPLACE_CMD} -i "" -E -e '/(ptc|fcl-base)/ s|,freebsd|,freebsd,dragonfly|g' \
+		-e '/(ptc|fcl-base)/! s|freebsd,|freebsd,dragonfly,|g' \
+		${WRKSRC}/packages/graph/fpmake.pp
+	@${REINPLACE_CMD} -i "" -e 's/freebsd,/freebsd,dragonfly,/g' \
+		${WRKSRC}/packages/iconvenc/fpmake.pp
+	@${REINPLACE_CMD} -i "" -e 's/freebsd,/freebsd,dragonfly,/g' \
+		${WRKSRC}/packages/libfontconfig/fpmake.pp
+	@${REINPLACE_CMD} -i ""	-e 's/freebsd,/freebsd,dragonfly,/g' \
+		-e '/P.OSes/ s/freebsd/freebsd, dragonfly/g' \
+		${WRKSRC}/packages/libvlc/fpmake.pp
+
 do-build:
 # build fpc compiler
 	@${ECHO_MSG} "##### STARTING COMPILER #####"
 	(cd ${WRKDIR}/${FPCSRCDIR}/compiler && ${SETENV} ${MAKE_ENV} \
-		${MAKE_CMD} cycle ${MAKE_ARGS} ${BOOTPPC})
+		LDVER=ld.bfd ${MAKE_CMD} cycle ${MAKE_ARGS} ${BOOTPPC})
 	@${ECHO_MSG} "##### COMPLETE COMPILER #####"
 
 # build units
