--- Makefile.units.orig	2015-01-17 23:40:32 UTC
+++ Makefile.units
@@ -41,7 +41,7 @@ post-extract:
 
 post-patch:
 .if ${PKGNAMESUFFIX} == "-fpmkunit"
-	@${REINPLACE_CMD} -e 's|\[Linux\]|\[Linux,FreeBSD,NetBSD,OpenBSD\]|g' \
+	@${REINPLACE_CMD} -e 's|\[Linux\]|\[Linux,FreeBSD,DragonFly\]|g' \
 		${WRKDIR}/${WRKUNITDIR}/src/${PKGNAMESUFFIX:S/-//}.pp
 .endif
 .if ${PKGNAMESUFFIX} == "-fppkg"
@@ -54,3 +54,5 @@ post-patch:
 	@${REINPLACE_CMD} -e 's|$$(INSTALL) Package.fpc $$(INSTALL_UNITDIR)||g' \
 		${WRKDIR}/${WRKUNITDIR}/fppkg/Makefile
 .endif
+	${FIND} ${WRKSRC} -type f -name Makefile | \
+		${XARGS} ${REINPLACE_CMD} -e 's|freebsd|dragonfly|g'
