--- Makefile.orig	2023-12-02 13:07:07 UTC
+++ Makefile
@@ -5,6 +5,7 @@ CATEGORIES=	lang
 MASTER_SITES=	SF/freepascal/Source/${PORTVERSION}:source \
 		https://downloads.freepascal.org/${PORTNAME}/dist/${PORTVERSION}/source/:source \
 		http://leaf.dragonflybsd.org/~marino/dports-src/:DragonFlybootstrap \
+		http://avalon.dragonflybsd.org/misc/distfiles/freepascal/:DragonFlybootstrap \
 		LOCAL/acm/freepascal/:bootstrap \
 		LOCAL/acm/freepascal/:man
 DISTFILES=	${DISTNAME:S/$/.source/}${EXTRACT_SUFX}:source
@@ -16,7 +17,7 @@ WWW=		https://www.freepascal.org/
 
 CONFLICTS=	fpc-devel
 
-RUN_DEPENDS=	${LOCALBASE}/bin/as:devel/binutils
+RUN_DEPENDS=	${LOCALBASE}/bin/as:devel/binutils@native
 
 ONLY_FOR_ARCHS=	amd64 i386
 
@@ -25,7 +26,7 @@ USE_BINUTILS=	yes
 BOOTVER=	${PORTVERSION}
 MANVER=		${PORTVERSION}
 
-FPC_LDPATH?=	"${LOCALBASE}/bin/ld.bfd"
+FPC_LDPATH?=	"/usr/bin/ld.bfd"
 
 FPCSRCDIR=	${PORTNAME}-${PORTVERSION}
 
@@ -149,7 +150,7 @@ do-build:
 # build fpc compiler
 	@${ECHO_MSG} "##### STARTING COMPILER #####"
 	(cd ${WRKDIR}/${FPCSRCDIR}/compiler && ${SETENV} ${MAKE_ENV} \
-		${MAKE_CMD} cycle ${MAKE_ARGS} ${BOOTPPC})
+		LDVER=ld.bfd ${MAKE_CMD} cycle ${MAKE_ARGS} ${BOOTPPC})
 	@${ECHO_MSG} "##### COMPLETE COMPILER #####"
 
 # build units
