--- a/x11/terminology/pkg-plist
+++ b/x11/terminology/pkg-plist
@@ -6,15 +6,15 @@ bin/tyls
 bin/typop
 bin/tyq
 bin/tysend
-man/man1/terminology-helpers.1.gz
-man/man1/terminology.1.gz
-man/man1/tyalpha.1.gz
-man/man1/tybg.1.gz
-man/man1/tycat.1.gz
-man/man1/tyls.1.gz
-man/man1/typop.1.gz
-man/man1/tyq.1.gz
-man/man1/tysend.1.gz
+share/man/man1/terminology-helpers.1.gz
+share/man/man1/terminology.1.gz
+share/man/man1/tyalpha.1.gz
+share/man/man1/tybg.1.gz
+share/man/man1/tycat.1.gz
+share/man/man1/tyls.1.gz
+share/man/man1/typop.1.gz
+share/man/man1/tyq.1.gz
+share/man/man1/tysend.1.gz
 share/applications/terminology.desktop
 share/icons/hicolor/128x128/apps/terminology.png
 %%NLS%%share/locale/ca/LC_MESSAGES/terminology.mo
