--- pkg-plist.orig	2020-08-13 10:37:52 UTC
+++ pkg-plist
@@ -11,3 +11,7 @@ lib/fpc/%%PORTVERSION%%/units/%%BUILDNAM
 lib/fpc/%%PORTVERSION%%/units/%%BUILDNAME%%/fcl-pdf/fpttfencodings.ppu
 lib/fpc/%%PORTVERSION%%/units/%%BUILDNAME%%/fcl-pdf/fpttfsubsetter.o
 lib/fpc/%%PORTVERSION%%/units/%%BUILDNAME%%/fcl-pdf/fpttfsubsetter.ppu
+lib/fpc/%%PORTVERSION%%/units/%%BUILDNAME%%/fcl-pdf/fpparsettf.rsj
+lib/fpc/%%PORTVERSION%%/units/%%BUILDNAME%%/fcl-pdf/fppdf.rsj
+lib/fpc/%%PORTVERSION%%/units/%%BUILDNAME%%/fcl-pdf/fpttf.rsj
+lib/fpc/%%PORTVERSION%%/units/%%BUILDNAME%%/fcl-pdf/fpttfsubsetter.rsj
