--- pkg-plist.orig	2021-07-04 09:48:46 UTC
+++ pkg-plist
@@ -12022,6 +12022,7 @@ lib/zig/std/meta/trait.zig
 lib/zig/std/multi_array_list.zig
 lib/zig/std/net.zig
 lib/zig/std/net/test.zig
+@dir lib/zig/std/net
 lib/zig/std/once.zig
 lib/zig/std/os.zig
 lib/zig/std/os/bits.zig
@@ -12302,6 +12303,7 @@ lib/zig/std/time.zig
 lib/zig/std/time/epoch.zig
 lib/zig/std/unicode.zig
 lib/zig/std/unicode/throughput_test.zig
+@dir lib/zig/std/unicode
 lib/zig/std/valgrind.zig
 lib/zig/std/valgrind/callgrind.zig
 lib/zig/std/valgrind/memcheck.zig
