--- pkg-plist.orig	2026-10-05 00:00:00 UTC
+++ pkg-plist
@@ -1094,5 +1094,7 @@ bin/vagrant
 %%GEMS_DIR%%/extensions/%%BUILD_TARGET%%/%%RUBY_VER%%/vagrant-%%PORTVERSION%%/vagrant/vagrant_ssl.so
 %%GEMS_DIR%%/specifications/vagrant-%%PORTVERSION%%.gemspec
 %%DATADIR%%/embedded/rgloader/loader.rb
+%%DATADIR%%/plugins.json
+@dir %%DATADIR%%/plugins.d
 @dir %%GEMS_DIR%%/extensions
 @dir %%GEMS_DIR%%/plugins
