--- /dev/null	2013-02-11 22:47:52.035547064 +0100
+++ distinfo	2013-02-10 14:10:10.000000000 +0100
@@ -0,0 +1,18 @@
+SHA256 (libreoffice/libreoffice-core-3.6.5.2.tar.xz) = e6a97b77dc5a8ba4f57028fd2fab3e5b008e8776ac549f71ba3ec154550606e7
+SIZE (libreoffice/libreoffice-core-3.6.5.2.tar.xz) = 109108684
+SHA256 (libreoffice/libreoffice-dictionaries-3.6.5.2.tar.xz) = 62e79064ddacd9f039c71e36f439cd2694d45799c594c1945e04ebb9f3e4a43d
+SIZE (libreoffice/libreoffice-dictionaries-3.6.5.2.tar.xz) = 35255572
+SHA256 (libreoffice/libreoffice-help-3.6.5.2.tar.xz) = e7fe01ba83fc33d39f57aed6cc6d383095cb3c669db7d1b56b37b72eac3f4f8c
+SIZE (libreoffice/libreoffice-help-3.6.5.2.tar.xz) = 1912108
+SHA256 (libreoffice/48d647fbd8ef8889e5a7f422c1bfda94-clucene-core-2.3.3.4.tar.gz) = ddfdc433dd8ad31b5c5819cc4404a8d2127472a3b720d3e744e8c51d79732eab
+SIZE (libreoffice/48d647fbd8ef8889e5a7f422c1bfda94-clucene-core-2.3.3.4.tar.gz) = 2241498
+SHA256 (libreoffice/86261f06c097d3e425a2f6d0b0635380-hyphen-2.8.3.tar.gz) = 21c461c558c61a51f24d35a48e2bf3b3bebf27ffc361f99bec45496903938c42
+SIZE (libreoffice/86261f06c097d3e425a2f6d0b0635380-hyphen-2.8.3.tar.gz) = 614264
+SHA256 (libreoffice/0d2dcdfbf28d6208751b33057f5361f0-libcmis-0.2.3.tar.gz) = a4ace7c0280e5ee8d4941b33449ee10edf1515437483d55ae387cc4de2c2bea0
+SIZE (libreoffice/0d2dcdfbf28d6208751b33057f5361f0-libcmis-0.2.3.tar.gz) = 432064
+SHA256 (libreoffice/9f9e15966b5624834157fe3d748312bc-mdds_0.6.1.tar.bz2) = 4381b38db6ee807090c6f352d62575ea8ce9768701ad95f01afbcda497e69960
+SIZE (libreoffice/9f9e15966b5624834157fe3d748312bc-mdds_0.6.1.tar.bz2) = 128115
+SHA256 (libreoffice/e1e255dc43dbcbb34cb19e8a0eba90ae-mythes-1.2.2.tar.gz) = 1a1bec56d05b52a9fcc94ce7c53eb3c85defcb6b7770448d79b8be0a62e85677
+SIZE (libreoffice/e1e255dc43dbcbb34cb19e8a0eba90ae-mythes-1.2.2.tar.gz) = 4906938
+SHA256 (libreoffice/1f24ab1d39f4a51faf22244c94a6203f-xmlsec1-1.2.14.tar.gz) = 390a5085651828b8fe12aa978b200f59b9155eedbb91a4be89bf7cf39eefdd4a
+SIZE (libreoffice/1f24ab1d39f4a51faf22244c94a6203f-xmlsec1-1.2.14.tar.gz) = 1652670
