--- distinfo.orig	2013-02-08 09:08:41.000000000 +0100
+++ distinfo	2013-02-16 11:15:49.255333000 +0100
@@ -52,3 +52,13 @@
 SIZE (libreoffice/185d60944ea767075d27247c3162b3bc-unowinreg.dll) = 12288
 SHA256 (libreoffice/bd30e9cf5523cdfc019b94f5e1d7fd19-cppunit-1.12.1.tar.gz) = ac28a04c8e6c9217d910b0ae7122832d28d9917fa668bcc9e0b8b09acb4ea44a
 SIZE (libreoffice/bd30e9cf5523cdfc019b94f5e1d7fd19-cppunit-1.12.1.tar.gz) = 762803
+SHA256 (libreoffice/86261f06c097d3e425a2f6d0b0635380-hyphen-2.8.3.tar.gz) = 21c461c558c61a51f24d35a48e2bf3b3bebf27ffc361f99bec45496903938c42
+SIZE (libreoffice/86261f06c097d3e425a2f6d0b0635380-hyphen-2.8.3.tar.gz) = 614264
+SHA256 (libreoffice/0d2dcdfbf28d6208751b33057f5361f0-libcmis-0.2.3.tar.gz) = a4ace7c0280e5ee8d4941b33449ee10edf1515437483d55ae387cc4de2c2bea0
+SIZE (libreoffice/0d2dcdfbf28d6208751b33057f5361f0-libcmis-0.2.3.tar.gz) = 432064
+SHA256 (libreoffice/9f9e15966b5624834157fe3d748312bc-mdds_0.6.1.tar.bz2) = 4381b38db6ee807090c6f352d62575ea8ce9768701ad95f01afbcda497e69960
+SIZE (libreoffice/9f9e15966b5624834157fe3d748312bc-mdds_0.6.1.tar.bz2) = 128115
+SHA256 (libreoffice/e1e255dc43dbcbb34cb19e8a0eba90ae-mythes-1.2.2.tar.gz) = 1a1bec56d05b52a9fcc94ce7c53eb3c85defcb6b7770448d79b8be0a62e85677
+SIZE (libreoffice/e1e255dc43dbcbb34cb19e8a0eba90ae-mythes-1.2.2.tar.gz) = 4906938
+SHA256 (libreoffice/1f24ab1d39f4a51faf22244c94a6203f-xmlsec1-1.2.14.tar.gz) = 390a5085651828b8fe12aa978b200f59b9155eedbb91a4be89bf7cf39eefdd4a
+SIZE (libreoffice/1f24ab1d39f4a51faf22244c94a6203f-xmlsec1-1.2.14.tar.gz) = 1652670
