--- Makefile.orig	2015-09-28 22:44:12 UTC
+++ Makefile
@@ -284,7 +284,14 @@ CXXFLAGS_WARN=	-Woverloaded-virtual -Wno
 CXXFLAGS_WARN+=	-Wno-unused-local-typedefs
 .endif
 .else
-CXXFLAGS_WARN=	-Wshadow -Woverloaded-virtual
+CXXFLAGS_WARN=	-Wno-deprecated-declarations \
+		-Wno-undef \
+		-Wno-unused-parameter \
+		-Wno-unused-function \
+		-Wno-unused-local-typedefs \
+		-Wno-unused-variable \
+		-Wno-ignored-qualifiers \
+		-Woverloaded-virtual
 .endif
 
 pre-configure:
