--- Makefile.orig	2014-10-23 14:23:57.394624000 +0000
+++ Makefile
@@ -302,7 +302,8 @@ EXTRA_PATCHES+=	${FILESDIR}/extra-libc++
 EXTRA_PATCHES+=	${FILESDIR}/extra-libc++.i386:-p0
 .endif
 
-.if ${COMPILER_FEATURES:Mlibstdc++} && ${CHOSEN_COMPILER_TYPE} == clang
+#.if ${COMPILER_FEATURES:Mlibstdc++} && ${CHOSEN_COMPILER_TYPE} == clang
+.if ${OPSYS} == DragonFly
 DISTFILES+=	bae83fa5dc7f081768daace6e199adc3-glm-0.9.4.6-libreoffice.zip:src
 # XXX fdo#84008 (fixed in 4.3.3)
 CONFIGURE_ARGS+=	--disable-collada \
