--- distinfo.orig	2013-01-05 21:50:01.056810000 +0100
+++ distinfo	2013-01-05 21:40:58.526052000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_ts.tar.gz) = c7b3db7407b88b6071403cbdfdb9944df2c0a8bd435b9a2421f670847a40bb97
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_ts.tar.gz) = 1653464
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_ts.tar.gz) = 9720e9256eea9b8fff6b8f47396a7fd927fbcaf39b14346dcf7c0f06ae121b26
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_ts.tar.gz) = 1602863
