--- pkg-plist.orig	2014-01-30 18:49:08.609957000 +0000
+++ pkg-plist
@@ -2135,6 +2135,10 @@ man/man1/python3.3.1.gz
 %%PYTHON_LIBDIR%%/pickletools.py
 %%PYTHON_LIBDIR%%/pipes.py
 %%PYTHON_LIBDIR%%/pkgutil.py
+%%PYTHON_LIBDIR%%/plat-dragonfly3/IN.py
+%%PYTHON_LIBDIR%%/plat-dragonfly3/__pycache__/IN.cpython-33.pyc
+%%PYTHON_LIBDIR%%/plat-dragonfly3/__pycache__/IN.cpython-33.pyo
+%%PYTHON_LIBDIR%%/plat-dragonfly3/regen
 %%PYTHON_LIBDIR%%/plat-freebsd10/IN.py
 %%PYTHON_LIBDIR%%/plat-freebsd10/__pycache__/IN.cpython-33.pyc
 %%PYTHON_LIBDIR%%/plat-freebsd10/__pycache__/IN.cpython-33.pyo
@@ -4593,6 +4597,8 @@ man/man1/python3.3.1.gz
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd10
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd11/__pycache__
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd11
+@dirrm %%PYTHON_LIBDIR%%/plat-dragonfly3/__pycache__
+@dirrm %%PYTHON_LIBDIR%%/plat-dragonfly3
 @dirrm %%PYTHON_LIBDIR%%/pydoc_data/__pycache__
 @dirrm %%PYTHON_LIBDIR%%/pydoc_data
 @dirrm %%PYTHON_LIBDIR%%/site-packages
