--- pkg-plist.orig	2014-08-02 13:57:09.773801000 +0000
+++ pkg-plist
@@ -2180,10 +2180,10 @@ lib/python3.3/pickle.py
 lib/python3.3/pickletools.py
 lib/python3.3/pipes.py
 lib/python3.3/pkgutil.py
-lib/python3.3/plat-freebsd%%OSMAJOR%%/IN.py
-lib/python3.3/plat-freebsd%%OSMAJOR%%/__pycache__/IN.cpython-33.pyc
-lib/python3.3/plat-freebsd%%OSMAJOR%%/__pycache__/IN.cpython-33.pyo
-lib/python3.3/plat-freebsd%%OSMAJOR%%/regen
+lib/python3.3/plat-dragonfly%%OSMAJOR%%/IN.py
+lib/python3.3/plat-dragonfly%%OSMAJOR%%/__pycache__/IN.cpython-33.pyc
+lib/python3.3/plat-dragonfly%%OSMAJOR%%/__pycache__/IN.cpython-33.pyo
+lib/python3.3/plat-dragonfly%%OSMAJOR%%/regen
 lib/python3.3/platform.py
 lib/python3.3/plistlib.py
 lib/python3.3/poplib.py
