--- pkg-plist.orig	2014-01-14 23:29:25.000000000 +0000
+++ pkg-plist
@@ -2134,6 +2134,10 @@ libdata/pkgconfig/python-%%PYVER%%.pc
 %%PYTHON_LIBDIR%%/pickletools.py
 %%PYTHON_LIBDIR%%/pipes.py
 %%PYTHON_LIBDIR%%/pkgutil.py
+%%PYTHON_LIBDIR%%/plat-dragonfly3/IN.py
+%%PYTHON_LIBDIR%%/plat-dragonfly3/__pycache__/IN.cpython-33.pyc
+%%PYTHON_LIBDIR%%/plat-dragonfly3/__pycache__/IN.cpython-33.pyo
+%%PYTHON_LIBDIR%%/plat-dragonfly3/regen
 %%PYTHON_LIBDIR%%/plat-freebsd10/IN.py
 %%PYTHON_LIBDIR%%/plat-freebsd10/__pycache__/IN.cpython-33.pyc
 %%PYTHON_LIBDIR%%/plat-freebsd10/__pycache__/IN.cpython-33.pyo
@@ -4592,6 +4596,8 @@ libdata/pkgconfig/python-%%PYVER%%.pc
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd10
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd11/__pycache__
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd11
+@dirrm %%PYTHON_LIBDIR%%/plat-dragonfly3/__pycache__
+@dirrm %%PYTHON_LIBDIR%%/plat-dragonfly3
 @dirrm %%PYTHON_LIBDIR%%/pydoc_data/__pycache__
 @dirrm %%PYTHON_LIBDIR%%/pydoc_data
 @dirrm %%PYTHON_LIBDIR%%/site-packages
