--- Makefile.orig	2021-01-17 19:45:33 UTC
+++ Makefile
@@ -16,9 +16,9 @@ BROKEN_powerpc=		fails to compile: invok
 BROKEN_riscv64=		fails to compile: common/dynamips_common.h:71:2: Please define your architecture!
 BROKEN_sparc64=		fails to compile: invokes x86 assembler
 
-LIB_DEPENDS=	libelf.so:devel/libelf
+LIB_DEPENDS=	libelf.so.0:devel/libelf
 
-CFLAGS+=	-I${PREFIX}/include/libelf
+CFLAGS+=	-I${PREFIX}/include -I${PREFIX}/include/libelf
 
 CONFLICTS=	dynamips-[0-9]* dynamips-devel-[0-9]*
 
