--- pkg-plist.orig	Tue Apr  1 22:09:13 2025
+++ pkg-plist	Tue Apr
@@ -1929,11 +1929,6 @@ gnat12/include/c++/%%VER%%/%%CONFTRGT%%/ext/opt_random
 %%LIB32%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/32/adainclude/s-memcop.ads
 %%LIB32%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/32/adainclude/s-memory.adb
 %%LIB32%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/32/adainclude/s-memory.ads
-%%LIB32%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/32/adainclude/s-mmap.adb
-%%LIB32%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/32/adainclude/s-mmap.ads
-%%LIB32%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/32/adainclude/s-mmauni.ads
-%%LIB32%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/32/adainclude/s-mmosin.adb
-%%LIB32%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/32/adainclude/s-mmosin.ads
 %%LIB32%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/32/adainclude/s-mudido.adb
 %%LIB32%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/32/adainclude/s-mudido.ads
 %%LIB32%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/32/adainclude/s-multip.adb
@@ -3245,6 +3240,9 @@ gnat12/include/c++/%%VER%%/%%CONFTRGT%%/ext/opt_random
 %%AARCH64%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/plugin/include/config/aarch64/aarch64.h
 %%AARCH64%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/plugin/include/config/aarch64/biarchlp64.h
 %%AARCH64%%gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/plugin/include/config/arm/aarch-common-protos.h
+gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/plugin/include/config/dragonfly-stdint.h
+gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/plugin/include/config/dragonfly.h
+gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/plugin/include/config/i386/dragonfly.h
 gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/ada_target_properties
 gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/adainclude/a-assert.adb
 gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/adainclude/a-assert.ads
@@ -4803,6 +4801,8 @@ gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/adainclude/s-wwden
 gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/adainclude/s-wwdwch.adb
 gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/adainclude/s-wwdwch.ads
 gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/adainclude/sequenio.ads
+gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/adainclude/s-mmap.adb
+gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/adainclude/s-mmap.ads
 gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/adainclude/standard.ads.h
 gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/adainclude/system.ads
 gnat12/lib/gcc/%%CONFTRGT%%/%%VER%%/adainclude/text_io.ads
