--- distinfo.orig	2013-01-05 21:50:01.016810000 +0100
+++ distinfo	2013-01-05 21:40:50.776041000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_id.tar.gz) = a2ef3ba5d6a3f29671508d79c70901720a4e3cb339cf91226a06973717029aef
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_id.tar.gz) = 1620248
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_id.tar.gz) = c8cdcd39cb93e75ebf577775e0bc07fc4e59e50068311574ae6060f5759d9e83
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_id.tar.gz) = 9629742
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_id.tar.gz) = 45639f2b98ea327bf41d4a749047b99fdbdaff74e34084a55cc80917823778f1
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_id.tar.gz) = 1567774
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_id.tar.gz) = 5db874d715175049ffc70507c4667dd936444cf66fd4cdc087ed6b71e6b21c81
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_id.tar.gz) = 9668290
