--- distinfo.orig	2013-01-05 21:50:00.986810000 +0100
+++ distinfo	2013-01-05 21:40:43.036030000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_bn.tar.gz) = b518d0525d27002bf9a78312b352fc043633e4b0fadcdf05fdd4aa704e7205e4
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_bn.tar.gz) = 2068344
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_bn.tar.gz) = a9e995a5fd062905146a2382b35fc326bbee4fd67755ba0b42fbc34b158293ba
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_bn.tar.gz) = 11806641
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_bn.tar.gz) = 4742eede947c88b70bad6eb6ed5024ad62d2edf379b39e4ce04045b51840af88
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_bn.tar.gz) = 2021615
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_bn.tar.gz) = 04fa5e27589e6f76e97a0dd8dd1347b2fb6fbc9006ef19259a9352e17551b7c2
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_bn.tar.gz) = 13303178
