--- pkg-plist.orig	2014-01-14 23:29:25.000000000 +0000
+++ pkg-plist
@@ -880,6 +880,8 @@ libdata/pkgconfig/python-3.1.pc
 %%PYTHON_LIBDIR%%/pickletools.py
 %%PYTHON_LIBDIR%%/pipes.py
 %%PYTHON_LIBDIR%%/pkgutil.py
+%%PYTHON_LIBDIR%%/plat-dragonfly3/IN.py
+%%PYTHON_LIBDIR%%/plat-dragonfly3/regen
 %%PYTHON_LIBDIR%%/plat-freebsd4/IN.py
 %%PYTHON_LIBDIR%%/plat-freebsd4/regen
 %%PYTHON_LIBDIR%%/plat-freebsd5/IN.py
@@ -2220,6 +2222,7 @@ libdata/pkgconfig/python-3.1.pc
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd6
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd5
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd4
+@dirrm %%PYTHON_LIBDIR%%/plat-dragonfly3
 @dirrm %%PYTHON_LIBDIR%%/multiprocessing/dummy
 @dirrm %%PYTHON_LIBDIR%%/multiprocessing
 @dirrm %%PYTHON_LIBDIR%%/logging
