--- pkg-plist.orig	2013-10-20 11:29:18.435807000 +0000
+++ pkg-plist
@@ -108,13 +108,11 @@ GNUstep/System/Library/Frameworks/MDKit.
 GNUstep/System/Library/Frameworks/MDKit.framework/Versions/0/Resources/whiteArrowDown.tiff
 GNUstep/System/Library/Frameworks/MDKit.framework/Versions/0/Resources/whiteArrowRight.tiff
 GNUstep/System/Library/Frameworks/MDKit.framework/Versions/0/libMDKit.so
-GNUstep/System/Library/Frameworks/MDKit.framework/Versions/0/libMDKit.so.0
 GNUstep/System/Library/Frameworks/MDKit.framework/Versions/0/libMDKit.so.0.1.0
 GNUstep/System/Library/Frameworks/MDKit.framework/Versions/Current
 GNUstep/System/Library/Frameworks/MDKit.framework/libMDKit.so
 GNUstep/System/Library/Headers/MDKit
 GNUstep/System/Library/Libraries/libMDKit.so
-GNUstep/System/Library/Libraries/libMDKit.so.0
 GNUstep/System/Library/Libraries/libMDKit.so.0.1.0
 GNUstep/System/Tools/MDFinder
 GNUstep/System/Tools/gmds
