--- pkg-plist.orig	2023-05-01 21:48:14 UTC
+++ pkg-plist
@@ -8,13 +8,6 @@ etc/logrotate.d/consolekit
 include/ConsoleKit/ck-connector/ck-connector.h
 include/ConsoleKit/libconsolekit.h
 include/ConsoleKit/sd-login.h
-lib/ConsoleKit/scripts/ck-get-x11-display-device
-lib/ConsoleKit/scripts/ck-get-x11-server-pid
-@sample lib/ConsoleKit/scripts/ck-system-hibernate.sample
-@sample lib/ConsoleKit/scripts/ck-system-hybridsleep.sample
-@sample lib/ConsoleKit/scripts/ck-system-restart.sample
-@sample lib/ConsoleKit/scripts/ck-system-stop.sample
-@sample lib/ConsoleKit/scripts/ck-system-suspend.sample
 lib/girepository-1.0/libconsolekit-1.0.typelib
 lib/libck-connector.so
 lib/libck-connector.so.0
