--- pkg-plist.orig	Thu Apr 10 21:31:57 2025
+++ pkg-plist	Thu Apr
@@ -158,7 +158,6 @@ libdata/pkgconfig/gstreamer-net-%%VERSION%%.pc
 libexec/gstreamer-%%VERSION%%/gst-hotdoc-plugins-scanner
 libexec/gstreamer-%%VERSION%%/gst-plugin-scanner
 libexec/gstreamer-%%VERSION%%/gst-plugins-doc-cache-generator
-%%PTP%%@(root,wheel,4755) libexec/gstreamer-%%VERSION%%/gst-ptp-helper
 share/aclocal/gst-element-check-%%VERSION%%.m4
 %%BASH%%share/bash-completion/completions/gst-inspect-%%VERSION%%
 %%BASH%%share/bash-completion/completions/gst-launch-%%VERSION%%
