--- pkg-plist.orig	2023-01-23 17:34:29 UTC
+++ pkg-plist
@@ -20,14 +20,6 @@ include/gstreamer-%%VERSION%%/gst/base/g
 include/gstreamer-%%VERSION%%/gst/base/gstpushsrc.h
 include/gstreamer-%%VERSION%%/gst/base/gstqueuearray.h
 include/gstreamer-%%VERSION%%/gst/base/gsttypefindhelper.h
-include/gstreamer-%%VERSION%%/gst/check/check-prelude.h
-include/gstreamer-%%VERSION%%/gst/check/check.h
-include/gstreamer-%%VERSION%%/gst/check/gstbufferstraw.h
-include/gstreamer-%%VERSION%%/gst/check/gstcheck.h
-include/gstreamer-%%VERSION%%/gst/check/gstconsistencychecker.h
-include/gstreamer-%%VERSION%%/gst/check/gstharness.h
-include/gstreamer-%%VERSION%%/gst/check/gsttestclock.h
-include/gstreamer-%%VERSION%%/gst/check/internal-check.h
 include/gstreamer-%%VERSION%%/gst/controller/controller-enumtypes.h
 include/gstreamer-%%VERSION%%/gst/controller/controller-prelude.h
 include/gstreamer-%%VERSION%%/gst/controller/controller.h
@@ -126,7 +118,6 @@ include/gstreamer-%%VERSION%%/gst/net/ne
 include/gstreamer-%%VERSION%%/gst/net/net.h
 lib/girepository-%%VERSION%%/Gst-%%VERSION%%.typelib
 lib/girepository-%%VERSION%%/GstBase-%%VERSION%%.typelib
-lib/girepository-%%VERSION%%/GstCheck-%%VERSION%%.typelib
 lib/girepository-%%VERSION%%/GstController-%%VERSION%%.typelib
 lib/girepository-%%VERSION%%/GstNet-%%VERSION%%.typelib
 lib/gstreamer-%%VERSION%%/libgstcoreelements.so
@@ -134,9 +125,6 @@ lib/gstreamer-%%VERSION%%/libgstcoretrac
 lib/libgstbase-%%VERSION%%.so
 lib/libgstbase-%%VERSION%%.so.0
 lib/libgstbase-%%VERSION%%.so.%%SOVERSION%%
-lib/libgstcheck-%%VERSION%%.so
-lib/libgstcheck-%%VERSION%%.so.0
-lib/libgstcheck-%%VERSION%%.so.%%SOVERSION%%
 lib/libgstcontroller-%%VERSION%%.so
 lib/libgstcontroller-%%VERSION%%.so.0
 lib/libgstcontroller-%%VERSION%%.so.%%SOVERSION%%
@@ -148,7 +136,6 @@ lib/libgstreamer-%%VERSION%%.so.0
 lib/libgstreamer-%%VERSION%%.so.%%SOVERSION%%
 libdata/pkgconfig/gstreamer-%%VERSION%%.pc
 libdata/pkgconfig/gstreamer-base-%%VERSION%%.pc
-libdata/pkgconfig/gstreamer-check-%%VERSION%%.pc
 libdata/pkgconfig/gstreamer-controller-%%VERSION%%.pc
 libdata/pkgconfig/gstreamer-net-%%VERSION%%.pc
 libexec/gstreamer-%%VERSION%%/gst-hotdoc-plugins-scanner
@@ -165,7 +152,6 @@ share/gdb/auto-load/usr/local/lib/libgst
 %%DATADIR%%-%%VERSION%%/gdb/gst_gdb.py
 share/gir-%%VERSION%%/Gst-%%VERSION%%.gir
 share/gir-%%VERSION%%/GstBase-%%VERSION%%.gir
-share/gir-%%VERSION%%/GstCheck-%%VERSION%%.gir
 share/gir-%%VERSION%%/GstController-%%VERSION%%.gir
 share/gir-%%VERSION%%/GstNet-%%VERSION%%.gir
 %%NLS%%share/locale/af/LC_MESSAGES/gstreamer-%%VERSION%%.mo
