--- compiled.mk.orig	2013-11-18 00:32:22.398871000 +0000
+++ compiled.mk
@@ -20,23 +20,7 @@ BJAM_ARGS+=	cxxflags=-fPIC
 # of gcc/g++):
 BJAM_ARGS+=    cxxflags="${CXXFLAGS:N-O*}" cflags="${CFLAGS:N-O*}"
 
-.if ${CC:T:Mgcc*}
 BOOST_TOOLSET:=	gcc
-.elif ${CC:T:Mclang}
-BOOST_TOOLSET:=	clang
-.else
-_COMPILER_VERSION!=	${CC} --version
-. if ${_COMPILER_VERSION:Mgcc}
-BOOST_TOOLSET:=	gcc
-. elif ${_COMPILER_VERSION:M\(GCC\)}
-BOOST_TOOLSET:=	gcc
-. elif ${_COMPILER_VERSION:Mclang}
-BOOST_TOOLSET:=	clang
-. else
-.error Unable to determine compiler type for ${CC}
-. endif
-. undef _COMPILER_VERSION
-.endif
 
 BJAM_ARGS+=	--toolset=${BOOST_TOOLSET} \
 		${_MAKE_JOBS}
