--- compiled.mk.orig	2013-06-02 23:06:49.152775000 +0000
+++ compiled.mk
@@ -12,7 +12,7 @@ PKG_MESSAGE_FILE_PYTHON=	${PORTSDIR}/dev
 BJAM_ARGS=	--layout=system \
 		--prefix=${PREFIX}
 
-.if ${ARCH} == amd64
+.if ${ARCH} == x86_64
 BJAM_ARGS+=	cxxflags=-fPIC
 .endif
 
@@ -22,23 +22,7 @@ BJAM_ARGS+=	cxxflags=-fPIC
 BJAM_ARGS+=    cxxflags="${CXXFLAGS:N-O*}" cflags="${CFLAGS:N-O*}"
 
 NO_CCACHE=	yes
-.if ${CC:T:Mgcc*}
 BOOST_TOOLSET:=	gcc
-.elif ${CC:T:Mclang}
-BOOST_TOOLSET:=	clang
-.else
-_COMPILER_VERSION!=	${CC} --version
-. if ${_COMPILER_VERSION:Mgcc}
-BOOST_TOOLSET:=	gcc
-. elif ${_COMPILER_VERSION:M\(GCC\)}
-BOOST_TOOLSET:=	gcc
-. elif ${_COMPILER_VERSION:Mclang}
-BOOST_TOOLSET:=	clang
-. else
-.error Unable to determine compiler type for ${CC}
-. endif
-. undef _COMPILER_VERSION
-.endif
 
 BJAM_ARGS+=	--toolset=${BOOST_TOOLSET} \
 		${_MAKE_JOBS}
