--- compiled.mk.orig	2016-11-23 18:28:19.000000000 +0200
+++ compiled.mk
@@ -18,7 +18,7 @@ MAKE_ARGS=	--layout=system \
 # Our compiler-flags will be added AFTER those set by bjam. We remove
 # the optimization level, because Boost sets it itself (to -O3 in case
 # of gcc/g++):
-MAKE_ARGS+=    cxxflags="${CXXFLAGS:N-O*}" cflags="${CFLAGS:N-O*}"
+MAKE_ARGS+=    cxxflags="${CXXFLAGS:N-O*} -std=c++11" cflags="${CFLAGS:N-O*}"
 
 MAKE_ARGS+=	--toolset=${CHOSEN_COMPILER_TYPE} \
 		${_MAKE_JOBS}
