--- files/Makefile.bsd.orig	2013-01-02 16:53:41.969919000 +0100
+++ files/Makefile.bsd	2013-01-02 18:21:23.547268000 +0100
@@ -10,7 +10,7 @@
 		-DQT_SQL_LIB -DQT_CORE_LIB -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -DQT_SHARED
 LDADD=		-L${QT_LIBDIR} -L${LOCALBASE}/lib -L${LOCALBASE}/lib/mysql ${PTHREAD_LIBS} -lQtSql -lmysqlclient_r 
 MOC?=		${LOCALBASE}/bin/moc-qt4
-SRCS=		main.cpp qsql_${DB}.cpp moc_qsql_${DB}.cpp qsql_${DB}.moc
+SRCS=		main.cpp qsql_${DB}.cpp moc_qsql_${DB}.cpp 
 
 LIBDIR=		${PREFIX}/lib/qt4/plugins/sqldrivers
 
@@ -18,10 +18,13 @@
 	mkdir -p ${LIBDIR}
 
 moc_qsql_${DB}.cpp: qsql_${DB}.h
-	$(MOC) $(.ALLSRC) -o $(.TARGET)
+	${MOC} $(.ALLSRC) -o $(.TARGET)
 
-qsql_${DB}.moc: qsql_${DB}.cpp
-	$(MOC) $(.ALLSRC) -o ${DRIVER}/$(.TARGET)
+
+qsql_${DB}.moc:
+	${MOC} ${DRIVER}/qsql_${DB}.cpp -o ${DRIVER}/$(.TARGET)
+
+qsql_${DB}.cpp: qsql_${DB}.moc
 
 beforeinstall:	${LIBDIR}
 
