--- Makefile.orig	2021-10-16 11:20:41 UTC
+++ Makefile
@@ -50,7 +50,7 @@ CONFIGURE_ARGS?=--disable-fsck \
 		--includedir='${PREFIX}/include/e2fsprogs/' \
 		--with-root-prefix='${PREFIX}' \
 		LDFLAGS='${LDFLAGS} -L${LOCALBASE}/lib -Wl,--rpath -Wl,${LOCALBASE}/lib/e2fsprogs'
-CONFIGURE_ENV?=	LIBS='-Wl,--as-needed ${LIBS} -lexecinfo -lelf'
+CONFIGURE_ENV?=	LIBS='-Wl,--as-needed ${LIBS} -lexecinfo'
 # apparently HAVE_PTHREAD_H checks go missing if the FUSEFS option is unset, force it:
 CPPFLAGS+=	-I${WRKSRC}/lib -I${LOCALBASE}/include # -D_EXT2_USE_C_VERSIONS
 MAKE_ARGS+=	pkgconfigdir='${PREFIX}/libdata/pkgconfig'
@@ -258,7 +258,7 @@ _check_timeout=180
 .endif
 
 .if ${PORT_OPTIONS:MLIBUNWIND}
-_staticlibs+=${LOCALBASE}/lib/libunwind.a /usr/lib/liblzma.a /usr/lib/libmd.a
+_staticlibs+=${LOCALBASE}/lib/libunwind.a /usr/lib/liblzma.a
 .endif
 
 post-build:
@@ -272,7 +272,7 @@ post-build:
 	cd ${WRKSRC}/e2fsck && ${RM} -f e2fsck \
 		&& ${MAKE_CMD} e2fsck V=1 \
 		LIBS="-static-libgcc -lc -Bstatic ../lib/libsupport.a ../lib/libext2fs.a ../lib/libcom_err.a \
-		${_staticlibs} /usr/lib/libexecinfo.a /usr/lib/libelf.a \
+		${_staticlibs} /usr/lib/libexecinfo.a \
 		${LOCALBASE}/lib/libblkid.a ${LOCALBASE}/lib/libuuid.a ${libintl} ../lib/libe2p.a " \
 		&& ${STRIP_CMD} e2fsck
 # Regression check: avoid a port (not upstream!) regression from 1.40.5,
