--- pkg-plist.orig	2021-03-01 14:52:23 UTC
+++ pkg-plist
@@ -1498,15 +1498,15 @@ lib/libsaturne.so
-%%PYTHON_SITELIBDIR%%/ple/Coupling.py
-%%PYTHON_SITELIBDIR%%/ple/Init.py
-%%PYTHON_SITELIBDIR%%/ple/__init__.py
-%%PYTHON_SITELIBDIR%%/ple/__pycache__/Coupling.cpython-%%PYTHON_SUFFIX%%.opt-1.pyc
-%%PYTHON_SITELIBDIR%%/ple/__pycache__/Coupling.cpython-%%PYTHON_SUFFIX%%.pyc
-%%PYTHON_SITELIBDIR%%/ple/__pycache__/Init.cpython-%%PYTHON_SUFFIX%%.opt-1.pyc
-%%PYTHON_SITELIBDIR%%/ple/__pycache__/Init.cpython-%%PYTHON_SUFFIX%%.pyc
-%%PYTHON_SITELIBDIR%%/ple/__pycache__/__init__.cpython-%%PYTHON_SUFFIX%%.opt-1.pyc
-%%PYTHON_SITELIBDIR%%/ple/__pycache__/__init__.cpython-%%PYTHON_SUFFIX%%.pyc
-%%PYTHON_SITELIBDIR%%/ple/__pycache__/pyple_coupler.cpython-%%PYTHON_SUFFIX%%.opt-1.pyc
-%%PYTHON_SITELIBDIR%%/ple/__pycache__/pyple_coupler.cpython-%%PYTHON_SUFFIX%%.pyc
-%%PYTHON_SITELIBDIR%%/ple/libpyplecoupling.so
-%%PYTHON_SITELIBDIR%%/ple/libpyplecoupling.so.0
-%%PYTHON_SITELIBDIR%%/ple/libpyplecoupling.so.0.0.0
-%%PYTHON_SITELIBDIR%%/ple/pyple_coupler.py
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/Coupling.py
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/Init.py
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/__init__.py
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/__pycache__/Coupling.cpython-%%PYTHON_SUFFIX%%.opt-1.pyc
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/__pycache__/Coupling.cpython-%%PYTHON_SUFFIX%%.pyc
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/__pycache__/Init.cpython-%%PYTHON_SUFFIX%%.opt-1.pyc
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/__pycache__/Init.cpython-%%PYTHON_SUFFIX%%.pyc
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/__pycache__/__init__.cpython-%%PYTHON_SUFFIX%%.opt-1.pyc
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/__pycache__/__init__.cpython-%%PYTHON_SUFFIX%%.pyc
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/__pycache__/pyple_coupler.cpython-%%PYTHON_SUFFIX%%.opt-1.pyc
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/__pycache__/pyple_coupler.cpython-%%PYTHON_SUFFIX%%.pyc
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/libpyplecoupling.so
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/libpyplecoupling.so.0
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/libpyplecoupling.so.0.0.0
+%%MPI%%%%PYTHON_SITELIBDIR%%/ple/pyple_coupler.py
