--- distinfo.orig	2013-01-05 21:50:00.156808000 +0100
+++ distinfo	2013-01-05 21:38:30.995846000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_da.tar.gz) = 5800e2b106b79750b8882f59046e425e8476ef806e994da395133a76fc7fa9a0
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_da.tar.gz) = 3365992
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_da.tar.gz) = 5a8ab0f049006fa9a051cca8f97c8bee827c8d24c020df45099c11aab8c07e15
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_da.tar.gz) = 10517872
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_da.tar.gz) = bb357b3a64692aa18fa79ecbe6d586d4bd924e3fca19cbd04ed7847317e1964b
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_da.tar.gz) = 3394757
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_da.tar.gz) = 29e87d21a98c3ee91b5162615ab7c40ec20657b7cf3ad437ba022f527d5127dc
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_da.tar.gz) = 10526971
