--- distinfo.orig	2013-01-05 22:07:27.108271000 +0100
+++ distinfo	2013-01-05 22:06:52.638223000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_pl.tar.gz) = 6d9759184727a0dd59666915a1392f9b17e785213af1c227eabb5c6ce30db998
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_pl.tar.gz) = 4349175
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_pl.tar.gz) = 40f178621fb2542c42f97c04428ebd7b8ca6bda771710ff9adf25074e7b9b0b4
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_pl.tar.gz) = 11044624
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_pl.tar.gz) = 2a854251c390958be3c38b3127157b8b7c4c3820d9b4717a6ddb29c2189ec85e
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_pl.tar.gz) = 4297586
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_pl.tar.gz) = 01605df193b879117e67001248a0cace309b8782e1f57e7a386d3b29eefca140
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_pl.tar.gz) = 11064681
