--- pkg-plist.orig	2024-02-09 23:46:10 UTC
+++ pkg-plist
@@ -14,15 +14,6 @@ lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/c
 lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/crtend.o
 lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/crti.o
 lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/crtn.o
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/README
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/limits.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/stddef.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/stdio.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/stdlib.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/sys/types.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/syslimits.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/unistd.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/wchar.h
 lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include/float.h
 lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include/gcov.h
 lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include/iso646.h
