--- Makefile.orig	2013-07-14 15:48:12.057303000 +0000
+++ Makefile
@@ -25,7 +25,7 @@ PATCH_SITES=	${MASTER_SITES:S|unix|patch
 PATCHFILES!=	/usr/bin/jot -s " " -w ${PORTVERSION:C/\.[0-9]*$//}.%03d  \
 			${PATCHLEVEL} 1 ${PATCHLEVEL}
 #	bits to remove
-BADPATCHES=	527 780 796 873 890 1104 1111
+BADPATCHES=	527 640 780 796 873 890 1104 1111
 .for p in ${BADPATCHES}
 PATCHFILES:=	${PATCHFILES:N7.3.${p}}
 .endfor
@@ -148,7 +148,7 @@ USE_XORG+=	xaw
 . if ${PORT_OPTIONS:MGTK2}
 USE_GNOME=	gtk20
 CONF_OPT_GUI="--enable-gui=gtk2 --with-gtk-prefix=${LOCALBASE}"
-MAKE_ARGS+=	X_LIBS="$(X_LIBS) -lXt"
+MAKE_ARGS+=	X_LIBS="$(X_LIBS) -lXt -lX11"
 USE_XORG+=	xt
 . endif
 
@@ -156,7 +156,7 @@ USE_XORG+=	xt
 USE_GNOME=	libgnomeui
 PKGNAMESUFFIX=	-gnome2
 CONF_OPT_GUI="--enable-gui=gnome2"
-MAKE_ARGS+=	X_LIBS="$(X_LIBS) -lXt"
+MAKE_ARGS+=	X_LIBS="$(X_LIBS) -lXt -lX11"
 USE_XORG+=	xt glproto
 . endif
 
