--- Makefile.orig	2013-03-25 02:38:05.068473000 +0000
+++ Makefile
@@ -143,7 +143,7 @@ CONF_OPT_GUI="--enable-gui=athena"
 . if ${PORT_OPTIONS:MGTK2}
 USE_GNOME=	gtk20
 CONF_OPT_GUI="--enable-gui=gtk2 --with-gtk-prefix=${LOCALBASE}"
-MAKE_ARGS+=	X_LIBS="$(X_LIBS) -lXt"
+MAKE_ARGS+=	X_LIBS="$(X_LIBS) -lXt -lX11"
 USE_XORG+=	xt
 . endif
 
@@ -151,7 +151,7 @@ USE_XORG+=	xt
 USE_GNOME=	libgnomeui
 PKGNAMESUFFIX=	-gnome2
 CONF_OPT_GUI="--enable-gui=gnome2"
-MAKE_ARGS+=	X_LIBS="$(X_LIBS) -lXt"
+MAKE_ARGS+=	X_LIBS="$(X_LIBS) -lXt -lX11"
 USE_XORG+=	xt glproto
 . endif
 
@@ -224,7 +224,7 @@ post-install:
 	${ECHO_CMD} "r ${WRKDIR}/PLIST.share-vim"	>> ${WRKDIR}/ex.script
 	${ECHO_CMD} "x!"				>> ${WRKDIR}/ex.script
 	${CP} -p ${TMPPLIST} ${TMPPLIST}.pre-share-vim
-	cd ${WRKDIR} ; ex < ex.script
+	cd ${WRKDIR} ; env TERM=cons25 ex < ex.script
 	@${ECHO_CMD}
 	@${CAT} ${PKGMESSAGE}
 	@${ECHO_CMD}
