--- Makefile.orig	2012-11-17 06:57:19.000000000 +0100
+++ Makefile	2013-01-12 22:09:09.363213000 +0100
@@ -236,7 +236,7 @@
 	${ECHO_CMD} "r ${WRKDIR}/PLIST.share-vim"	>> ${WRKDIR}/ex.script
 	${ECHO_CMD} "x!"				>> ${WRKDIR}/ex.script
 	${CP} -p ${TMPPLIST} ${TMPPLIST}.pre-share-vim
-	cd ${WRKDIR} ; ex < ex.script
+	cd ${WRKDIR} ; env TERM=cons25 ex < ex.script
 	@${ECHO_CMD}
 	@${CAT} ${PKGMESSAGE}
 	@${ECHO_CMD}
