--- Makefile.orig	2013-01-15 16:13:52.852463000 +0100
+++ Makefile	2013-01-15 16:14:57.782553000 +0100
@@ -148,13 +148,13 @@
 .  elif defined(WITH_GTK2)
 USE_GNOME=	gtk20
 MAKE_ARGS+=	CONF_OPT_GUI="--enable-gui=gtk2 --with-gtk-prefix=${LOCALBASE}" ${I18N}
-MAKE_ARGS+=	X_LIBS="$(X_LIBS) -lXt"
+MAKE_ARGS+=	X_LIBS="$(X_LIBS) -lXt -lX11"
 USE_XORG+=	xt
 .  elif defined(WITH_GNOME2)
 USE_GNOME=	libgnomeui
 PKGNAMESUFFIX=	-gnome2
 MAKE_ARGS+=	CONF_OPT_GUI="--enable-gui=gnome2" ${I18N}
-MAKE_ARGS+=	X_LIBS="$(X_LIBS) -lXt"
+MAKE_ARGS+=	X_LIBS="$(X_LIBS) -lXt -X11"
 USE_XORG+=	xt glproto
 .  elif defined(WITH_MOTIF)
 USE_MOTIF=	yes
@@ -236,7 +236,7 @@
 	${ECHO_CMD} "r ${WRKDIR}/PLIST.share-vim"	>> ${WRKDIR}/ex.script
 	${ECHO_CMD} "x!"				>> ${WRKDIR}/ex.script
 	${CP} -p ${TMPPLIST} ${TMPPLIST}.pre-share-vim
-	cd ${WRKDIR} ; ex < ex.script
+	cd ${WRKDIR} ; env TERM=cons25 ex < ex.script
 	@${ECHO_CMD}
 	@${CAT} ${PKGMESSAGE}
 	@${ECHO_CMD}
