--- Makefile.orig	2016-01-30 22:25:27 UTC
+++ Makefile
@@ -24,10 +24,10 @@ XTERM_SAVE_DESC=	Restore xterm screen af
 PORTSCOUT=	site:https://github.com/vim/vim/releases
 
 ALL_TARGET=	# empty
-MAKE_ARGS+=	CC="${CC}" CONF_ARGS="--prefix=${PREFIX} ${CSCOPE_ARG}" STRIP="${STRIP_CMD}" CONF_OPT_MULTIBYTE=--enable-multibyte CONF_TERM_LIB=--with-tlib=termlib
+MAKE_ARGS+=	CC="${CC}" CONF_ARGS="--prefix=${PREFIX} ${CSCOPE_ARG}" STRIP="${STRIP_CMD}" CONF_OPT_MULTIBYTE=--enable-multibyte CONF_TERM_LIB=--with-tlib=ncurses
 MAKE_ARGS+=	VIMRUNTIMEDIR=${PREFIX}/share/vim/${VIM_VER} VIMRCLOC=${ETCDIR}
 REINPLACE_ARGS=	-i ''
-USES=		cpe iconv pkgconfig shebangfix tar:bzip2
+USES=		cpe iconv ncurses pkgconfig shebangfix tar:bzip2
 WANT_GNOME=	yes
 
 PLIST_SUB=	VIM_VER=${VIM_VER}
