--- bsd.java.mk.orig	2014-04-16 18:39:18.578142000 +0000
+++ bsd.java.mk
@@ -273,7 +273,7 @@ JAVA_RUN=	jre
 .		undef _JAVA_PORTS_INSTALLED
 .		undef _JAVA_PORTS_POSSIBLE
 .		if defined(JAVA_VERSION)
-_JAVA_VERSION=	${JAVA_VERSION:S/1.6+/1.6 1.7+/:S/1.7+/1.7 1.8+/:S/1.8+/1.8/}
+_JAVA_VERSION=	${JAVA_VERSION:S/1.6+/1.7/:S/1.7+/1.7/:S/1.8+/1.8/}
 .		else
 _JAVA_VERSION=	${__JAVA_VERSION_LIST}
 .		endif
