--- bsd.commands.mk.orig	2014-06-19 22:32:52.928650000 +0000
+++ bsd.commands.mk
@@ -62,7 +62,7 @@ MOUNT_DEVFS?=		${MOUNT} -t devfs devfs
 # XXX: this is a work-around for an obscure bug where
 # mount -t nullfs returns zero status on errors within
 # a make target
-MOUNT_NULLFS?=		/sbin/mount_nullfs
+MOUNT_NULLFS?=		/sbin/mount_null
 MV?=			/bin/mv
 OBJCOPY?=		/usr/bin/objcopy
 OBJDUMP?=		/usr/bin/objdump
