--- bsd.default-versions.mk.orig	2021-11-13 19:10:58 UTC
+++ bsd.default-versions.mk
@@ -31,6 +31,9 @@ _l=		${lang:C/=.*//g}
 ${_l:tu}_DEFAULT=	${lang:C/.*=//g}
 .endfor
 
+# tuxillo: since 2021Q4 we default to openssl again
+SSL_DEFAULT?=  openssl
+
 # Possible values: 2.4
 APACHE_DEFAULT?=	2.4
 # Possible values: 5, 18
@@ -80,7 +83,7 @@ LINUX_DEFAULT?=		c7
 .if ${ARCH} == powerpc
 LLVM_DEFAULT?=		10
 .else
-LLVM_DEFAULT?=		90
+LLVM_DEFAULT?=		10
 .endif
 # Possible values: 5.1, 5.2, 5.3, 5.4
 LUA_DEFAULT?=		5.2
