--- bsd.java.mk.orig	2012-12-11 06:15:41.000000000 +0100
+++ bsd.java.mk	2013-01-04 19:27:38.994192000 +0100
@@ -157,9 +157,12 @@
 
 # The complete list of Java versions, os and vendors supported.
 __JAVA_VERSION_LIST=	1.5 1.6 1.7
-_JAVA_VERSION_LIST=		${__JAVA_VERSION_LIST} ${__JAVA_VERSION_LIST:S/$/+/}
-_JAVA_OS_LIST=			native linux
-_JAVA_VENDOR_LIST=		sun openjdk
+_JAVA_VERSION_LIST=	${__JAVA_VERSION_LIST} ${__JAVA_VERSION_LIST:S/$/+/}
+#_JAVA_OS_LIST=		native linux
+#_JAVA_VENDOR_LIST=	sun openjdk
+
+_JAVA_OS_LIST=		native
+_JAVA_VENDOR_LIST=	openjdk
 
 # Set all meta-information about JDK ports:
 # port location, corresponding JAVA_HOME, JDK version, OS, vendor
@@ -167,9 +170,9 @@
 											VERSION=1.7.0	OS=native	VENDOR=openjdk
 _JAVA_PORT_NATIVE_OPENJDK_JDK_1_6_INFO=		PORT=java/openjdk6			HOME=${LOCALBASE}/openjdk6 \
 											VERSION=1.6.0	OS=native	VENDOR=openjdk
-_JAVA_PORT_LINUX_SUN_JDK_1_6_INFO=			PORT=java/linux-sun-jdk16		HOME=${LOCALBASE}/linux-sun-jdk1.6.0 \
+_JAVA_PORT_LINUX_SUN_JDK_1_6_INFO=		PORT=java/linux-sun-jdk16		HOME=${LOCALBASE}/linux-sun-jdk1.6.0 \
 											VERSION=1.6.0	OS=linux	VENDOR=sun
-_JAVA_PORT_LINUX_SUN_JDK_1_7_INFO=			PORT=java/linux-sun-jdk17		HOME=${LOCALBASE}/linux-sun-jdk1.7.0 \
+_JAVA_PORT_LINUX_SUN_JDK_1_7_INFO=		PORT=java/linux-sun-jdk17		HOME=${LOCALBASE}/linux-sun-jdk1.7.0 \
 											VERSION=1.7.0	OS=linux	VENDOR=sun
 
 # Verbose description for each VENDOR
@@ -181,16 +184,21 @@
 _JAVA_OS_linux=		Linux
 
 # Enforce preferred Java ports according to OS
-_JAVA_PREFERRED_PORTS+=	JAVA_PORT_NATIVE_OPENJDK_JDK_1_6
+_JAVA_PREFERRED_PORTS+=	JAVA_PORT_NATIVE_OPENJDK_JDK_1_7
 
 # List all JDK ports
-__JAVA_PORTS_ALL=	JAVA_PORT_NATIVE_OPENJDK_JDK_1_7 \
-					JAVA_PORT_NATIVE_OPENJDK_JDK_1_6 \
-					JAVA_PORT_LINUX_SUN_JDK_1_7 \
-					JAVA_PORT_LINUX_SUN_JDK_1_6
-_JAVA_PORTS_ALL=	${JAVA_PREFERRED_PORTS} \
-					${_JAVA_PREFERRED_PORTS} \
-					${__JAVA_PORTS_ALL}
+#__JAVA_PORTS_ALL=	JAVA_PORT_NATIVE_OPENJDK_JDK_1_7 \
+#					JAVA_PORT_NATIVE_OPENJDK_JDK_1_6 \
+#					JAVA_PORT_LINUX_SUN_JDK_1_7 \
+#					JAVA_PORT_LINUX_SUN_JDK_1_6
+
+__JAVA_PORTS_ALL=	JAVA_PORT_NATIVE_OPENJDK_JDK_1_7
+
+#_JAVA_PORTS_ALL=	${JAVA_PREFERRED_PORTS} \
+#					${_JAVA_PREFERRED_PORTS} \
+#					${__JAVA_PORTS_ALL}
+
+_JAVA_PORTS_ALL=	${__JAVA_PORTS_ALL}
 
 # Set the name of the file that indicates that a JDK is indeed installed, as a
 # relative path within the JAVA_HOME directory.
@@ -252,7 +260,8 @@
 .		undef _JAVA_PORTS_INSTALLED
 .		undef _JAVA_PORTS_POSSIBLE
 .		if defined(JAVA_VERSION)
-_JAVA_VERSION=	${JAVA_VERSION:S/1.5/1.6/:S/1.5+/1.6+/:S/1.6+/1.6 1.7+/:S/1.7+/1.7/}
+#_JAVA_VERSION=	${JAVA_VERSION:S/1.5/1.6/:S/1.5+/1.6+/:S/1.6+/1.6 1.7+/:S/1.7+/1.7/}
+_JAVA_VERSION=	${JAVA_VERSION:S/1.5/1.7/:S/1.6/1.7/:S/1.5+/1.7/:S/1.6+/1.7/:S/1.7+/1.7/}
 .		else
 _JAVA_VERSION=	${__JAVA_VERSION_LIST}
 .		endif
