--- bsd.perl.mk.orig	2012-10-29 13:37:14.000000000 +0100
+++ bsd.perl.mk	2012-10-29 13:46:02.000000000 +0100
@@ -99,7 +99,7 @@
 USE_PERL5_STRING=	yes  # currently unreachable
 .endif
 
-.if ${USE_PERL5_STRING:L} != "yes"
+.if ${USE_PERL5_STRING:tl} != "yes"
 want_perl_sign=		${USE_PERL5_STRING:C|^[0-9.]+||}
 want_perl_ver=		${USE_PERL5_STRING:S|${want_perl_sign}$||}
 want_perl_major=	${want_perl_ver:C|\..*||}
@@ -136,7 +136,7 @@
 .else # wrong suffix
 IGNORE=	improper use of USE_PERL5
 .endif
-.endif #${USE_PERL5_STRING:L} != "yes"
+.endif #${USE_PERL5_STRING:tl} != "yes"
 
 .endif # !defined(_POSTMKINCLUDED) && !defined(Perl_Pre_Include)
 
