--- bsd.default-versions.mk.orig	2021-05-09 12:00:52 UTC
+++ bsd.default-versions.mk
@@ -31,6 +31,10 @@ _l=		${lang:C/=.*//g}
 ${_l:tu}_DEFAULT=	${lang:C/.*=//g}
 .endfor
 
+.if exists(/usr/lib/priv/libprivate_ssl.so)
+SSL_DEFAULT?=  libressl
+.endif
+
 # Possible values: 2.4
 APACHE_DEFAULT?=	2.4
 # Possible values: 5, 6, 18
@@ -73,7 +77,7 @@ LINUX_DEFAULT?=		c7
 .if ${ARCH} == powerpc
 LLVM_DEFAULT?=		10
 .else
-LLVM_DEFAULT?=		90
+LLVM_DEFAULT?=		10
 .endif
 # Possible values: 5.1, 5.2, 5.3, 5.4
 LUA_DEFAULT?=		5.2
