--- bsd.commands.mk.orig	2013-03-19 09:27:52.000000000 +0000
+++ bsd.commands.mk
@@ -65,7 +65,7 @@ MOUNT_DEVFS?=	${MOUNT} -t devfs devfs
 # XXX: this is a work-around for an obscure bug where
 # mount -t nullfs returns zero status on errors within
 # a make target
-MOUNT_NULLFS?=	/sbin/mount_nullfs
+MOUNT_NULLFS?=	/sbin/mount_null
 MV?=		/bin/mv
 OBJCOPY?=	/usr/bin/objcopy
 OBJDUMP?=	/usr/bin/objdump
