--- bsd.java.mk.orig	2013-11-17 14:26:36.005094000 +0000
+++ bsd.java.mk
@@ -158,8 +158,8 @@ SUB_LIST+=		JAVA_OS="${JAVA_OS}"
 # The complete list of Java versions, os and vendors supported.
 __JAVA_VERSION_LIST=	1.5 1.6 1.7
 _JAVA_VERSION_LIST=		${__JAVA_VERSION_LIST} ${__JAVA_VERSION_LIST:S/$/+/}
-_JAVA_OS_LIST=			native linux
-_JAVA_VENDOR_LIST=		sun openjdk
+_JAVA_OS_LIST=		native
+_JAVA_VENDOR_LIST=	openjdk
 
 # Set all meta-information about JDK ports:
 # port location, corresponding JAVA_HOME, JDK version, OS, vendor
@@ -184,13 +184,8 @@ _JAVA_OS_linux=		Linux
 _JAVA_PREFERRED_PORTS+=	JAVA_PORT_NATIVE_OPENJDK_JDK_1_6
 
 # List all JDK ports
-__JAVA_PORTS_ALL=	JAVA_PORT_NATIVE_OPENJDK_JDK_1_7 \
-					JAVA_PORT_NATIVE_OPENJDK_JDK_1_6 \
-					JAVA_PORT_LINUX_SUN_JDK_1_7 \
-					JAVA_PORT_LINUX_SUN_JDK_1_6
-_JAVA_PORTS_ALL=	${JAVA_PREFERRED_PORTS} \
-					${_JAVA_PREFERRED_PORTS} \
-					${__JAVA_PORTS_ALL}
+_JAVA_PORTS_ALL=	JAVA_PORT_NATIVE_OPENJDK_JDK_1_7 \
+					JAVA_PORT_NATIVE_OPENJDK_JDK_1_6
 
 # Set the name of the file that indicates that a JDK is indeed installed, as a
 # relative path within the JAVA_HOME directory.
@@ -269,7 +264,8 @@ JAVA_RUN=	jre
 .		undef _JAVA_PORTS_INSTALLED
 .		undef _JAVA_PORTS_POSSIBLE
 .		if defined(JAVA_VERSION)
-_JAVA_VERSION=	${JAVA_VERSION:S/1.5/1.6/:S/1.5+/1.6+/:S/1.6+/1.6 1.7+/:S/1.7+/1.7/}
+#_JAVA_VERSION= ${JAVA_VERSION:S/1.5/1.6/:S/1.5+/1.6+/:S/1.6+/1.6 1.7+/:S/1.7+/1.7/}
+_JAVA_VERSION=	${JAVA_VERSION:S/1.5/1.6/:S/1.5+/1.6+/:S/1.6+/1.7/:S/1.7+/1.7/}
 .		else
 _JAVA_VERSION=	${__JAVA_VERSION_LIST}
 .		endif
