--- bsd.gecko.mk.orig	2018-10-18 13:27:52 UTC
+++ bsd.gecko.mk
@@ -102,7 +102,7 @@ USE_XORG+=	xcb
 .endif
 
 .if ${MOZILLA_VER:R:R} >= 56
-LLVM_DEFAULT?=	70
+LLVM_DEFAULT?=	60
 BUILD_DEPENDS+=	llvm${LLVM_DEFAULT}>0:devel/llvm${LLVM_DEFAULT}
 MOZ_EXPORT+=	LLVM_CONFIG=llvm-config${LLVM_DEFAULT}
 # Require newer Clang than what's in base system unless user opted out
@@ -120,11 +120,6 @@ BUILD_DEPENDS+=	${LOCALBASE}/bin/python$
 MOZ_EXPORT+=	PYTHON3="${LOCALBASE}/bin/python${PYTHON3_DEFAULT}"
 .endif
 
-.if ${MOZILLA_VER:R:R} >= 63
-BUILD_DEPENDS+=	rust-cbindgen>=0.6.2:devel/rust-cbindgen \
-				node:www/node
-.endif
-
 MOZILLA_SUFX?=	none
 MOZSRC?=	${WRKSRC}
 PLISTF?=	${WRKDIR}/plist_files
@@ -388,7 +383,7 @@ post-patch-SNDIO-on:
 .endif
 
 .if ${PORT_OPTIONS:MRUST} || ${MOZILLA_VER:R:R} >= 54
-BUILD_DEPENDS+=	${RUST_PORT:T}>=1.28:${RUST_PORT}
+BUILD_DEPENDS+=	${RUST_PORT:T}>=1.24:${RUST_PORT}
 RUST_PORT?=		lang/rust
 . if ${MOZILLA_VER:R:R} < 54
 MOZ_OPTIONS+=	--enable-rust
