--- bsd.lua.mk.orig	2012-10-29 14:19:30.000000000 +0100
+++ bsd.lua.mk	2012-11-30 21:07:11.000000000 +0100
@@ -264,7 +264,7 @@
 .		endif
 _HAVE_LUA=				#
 # Check for all versions.
-.		if ${_WANT_LUA:L} == "yes"
+.		if ${_WANT_LUA:tl} == "yes"
 .			for comp in ${_LUA_COMPS_ALL}
 .				for ver in ${_LUA_VERS_ALL}
 _LUA_COMP=				_LUA_FILE_${comp}_${ver}
@@ -555,7 +555,7 @@
 RUN_DEPENDS+=			${_LUA_FILE_${comp_part}_${_LUA_VER}}:${PORTSDIR}/${_LUA_PORT_${comp_part}_${_LUA_VER}}
 .			endif
 .		else
-${_LUA_DEP_TYPE:U}_DEPENDS+=	${_LUA_FILE_${comp_part}_${_LUA_VER}}:${PORTSDIR}/${_LUA_PORT_${comp_part}_${_LUA_VER}}
+${_LUA_DEP_TYPE:tu}_DEPENDS+=	${_LUA_FILE_${comp_part}_${_LUA_VER}}:${PORTSDIR}/${_LUA_PORT_${comp_part}_${_LUA_VER}}
 .		endif
 .	endfor
 .endfor
