--- bsd.port.subdir.mk.orig	2012-11-17 06:54:18.000000000 +0100
+++ bsd.port.subdir.mk	2012-12-15 14:53:49.521551000 +0100
@@ -68,20 +68,16 @@
 .endif
 
 .if !defined(OSVERSION)
-.if exists(/usr/include/sys/param.h)
-OSVERSION!=	${AWK} '/^\#define[[:blank:]]__FreeBSD_version/ {print $$3}' < /usr/include/sys/param.h
-.elif exists(${SRC_BASE}/sys/sys/param.h)
-OSVERSION!=	${AWK} '/^\#define[[:blank:]]__FreeBSD_version/ {print $$3}' < ${SRC_BASE}/sys/sys/param.h
-.else
-OSVERSION!=	${SYSCTL} -n kern.osreldate
-.endif
+OSVERSION=	9999999
 .endif
 
-.if ${OSVERSION} >= 1000017
-.if !defined(WITHOUT_PKGNG)
-WITH_PKGNG=	yes
+if !defined(DFLYVERSION)
+.if exists(/usr/include/sys/param.h)
+DFLYVERSION!=	${AWK} '/^\#define[[:blank:]]__DragonFly_version/ {print $$3}' < /usr/include/sys/param.h
+.elif exists(${SRC_BASE}/sys/sys/param.h)
+DFLYVERSION!=	${AWK} '/^\#define[[:blank:]]__DragonFly_version/ {print $$3}' < ${SRC_BASE}/sys/sys/param.h
 .else
-.undef	WITH_PKGNG
+DFLYVERSION!=	${SYSCTL} -n kern.osreldate
 .endif
 .endif
 
@@ -140,19 +136,10 @@
 UID!=	${ID} -u
 .endif
 
-.if exists(${LOCALBASE}/sbin/pkg_info)
-PKG_INFO?=	${LOCALBASE}/sbin/pkg_info
-.else
-PKG_INFO?=	/usr/sbin/pkg_info
-.endif
-.if !defined(PKGINSTALLVER)
-PKGINSTALLVER!=	${PKG_INFO} -P 2>/dev/null | ${SED} -e 's/.*: //'
-.endif
-
-.endif
+PKGINSTALLVER?= 20100403
 
 INDEXDIR?=	${PORTSDIR}
-INDEXFILE?=	INDEX-${OSVERSION:C/([0-9]*)[0-9]{5}/\1/}
+INDEXFILE?=	INDEX-${DFLYVERSION:C/([0-9]*)[0-9]{5}/\1/}
 
 # local customization of the ports tree
 .if exists(${.CURDIR}/Makefile.local)
@@ -373,6 +360,7 @@
 	OPSYS="${OPSYS:S/"/"'"'"/g:S/\$/\$\$/g:S/\\/\\\\/g}" \
 	OSREL="${OSREL:S/"/"'"'"/g:S/\$/\$\$/g:S/\\/\\\\/g}" \
 	OSVERSION="${OSVERSION:S/"/"'"'"/g:S/\$/\$\$/g:S/\\/\\\\/g}" \
+	DFLYVERSION="${DFLYVERSION:Q}" \
 	UID="${UID:S/"/"'"'"/g:S/\$/\$\$/g:S/\\/\\\\/g}" \
 	PKGINSTALLVER="${PKGINSTALLVER:S/"/"'"'"/g:S/\$/\$\$/g:S/\\/\\\\/g}" \
 	HAVE_COMPAT_IA32_KERN="${HAVE_COMPAT_IA32_KERN}" \
