--- bsd.tcl.mk.orig	2013-02-06 19:11:29.000000000 +0100
+++ bsd.tcl.mk	2013-02-09 01:29:33.695408000 +0100
@@ -129,11 +129,11 @@
 
 # Override the global WITH_TCL_VER or WITH_TK_VER with the
 # port specific <UNIQUENAME>_WITH_TCL_VER or <UNIQUENAME>_WITH_TK_VER
-. if defined(${UNIQUENAME:U:S,-,_,}_WITH_TCL_VER)
-WITH_TCL_VER:=	${${UNIQUENAME:U:S,-,_,}_WITH_TCL_VER}
+. if defined(${UNIQUENAME:tu:S,-,_,}_WITH_TCL_VER)
+WITH_TCL_VER:=	${${UNIQUENAME:tu:S,-,_,}_WITH_TCL_VER}
 . endif
-. if defined(${UNIQUENAME:U:S,-,_,}_WITH_TK_VER)
-WITH_TK_VER:=	${${UNIQUENAME:U:S,-,_,}_WITH_TK_VER}
+. if defined(${UNIQUENAME:tu:S,-,_,}_WITH_TK_VER)
+WITH_TK_VER:=	${${UNIQUENAME:tu:S,-,_,}_WITH_TK_VER}
 . endif
 
 # If USE_TK, WITH_TK_VER, INVALID_TK_VER is defined,
