--- bsd.database.mk.orig	2012-11-17 06:54:17.000000000 +0100
+++ bsd.database.mk	2012-12-30 11:57:27.399405000 +0100
@@ -57,7 +57,7 @@
 #				  If no version is given (by the maintainer via the port or
 #				  by the user via defined variable), try to find the
 #				  currently installed version.  Fall back to default if
-#				  necessary (db41+).
+#				  necessary (db48).
 # INVALID_BDB_VER
 #				- This variable can be defined when the port does not
 #				  support one or more versions of Berkeley DB.
@@ -73,14 +73,14 @@
 #				- Use the highest installed version of Berkeley DB.
 # BDB_LIB_NAME
 #				- This variable is automatically set to the name of the
-#				  Berkeley DB library (default: db41).
+#				  Berkeley DB library (default: db48).
 # BDB_LIB_CXX_NAME
 #				- This variable is automatically set to the name of the
-#				  Berkeley DB C++ library (default: db41_cxx).
+#				  Berkeley DB C++ library (default: db48_cxx).
 # BDB_INCLUDE_DIR
 #				- This variable is automatically set to the location of
 #				  the Berkeley DB include directory (default:
-#				  ${LOCALBASE}/include/db41).
+#				  ${LOCALBASE}/include/db48).
 # BDB_LIB_DIR
 #				- This variable is automatically set to the location of
 #				  the Berkeley DB library directory.
@@ -312,9 +312,9 @@
 .endif
 _WANT_BDB_VER=	${USE_BDB}
 
-# Assume the default bdb version as 41
-.if ${USE_BDB:tl} == "yes"
-_WANT_BDB_VER=	41+
+# Assume the default bdb version as 48
+.if ${USE_BDB:tl} == "yes"
+_WANT_BDB_VER=	48
 .endif
 
 # Detect bdb version
@@ -362,8 +362,8 @@
 .   endif
 .  endfor
 .  if ${_BDB_VER} == "no"
-# No existing db4 version is detected in system
-_BDB_VER=	${_MATCHED_DB_VER}
+# No existing db4 version is detected in system, use the default
+_BDB_VER=	48
 .  endif
 . endif
 .endif
