--- bsd.default-versions.mk.orig	2018-11-23 04:30:39 UTC
+++ bsd.default-versions.mk
@@ -28,6 +28,10 @@ _l=		${lang:C/=.*//g}
 ${_l:tu}_DEFAULT=	${lang:C/.*=//g}
 .endfor
 
+.if exists(/usr/lib/priv/libprivate_ssl.so)
+SSL_DEFAULT?=  libressl
+.endif
+
 # Possible values: 2.2, 2.4
 APACHE_DEFAULT?=	2.4
 # Possible values: 48, 5, 6
@@ -43,7 +47,7 @@ FORTRAN_DEFAULT?=	gfortran
 # Possible values: 3.0.4
 FPC_DEFAULT?=		3.0.4
 # Possible values: 50, 60, 70 (to be used when non-base compiler is required)
-LLVM_DEFAULT?=		70
+LLVM_DEFAULT?=		60
 # Possible values: 5, 6, 7, 8
 GCC_DEFAULT?=		7
 # Possible values: 7, 8, 9, agpl
