--- bsd.gnustep.mk.orig	2014-11-03 17:34:54.681017000 +0000
+++ bsd.gnustep.mk
@@ -192,7 +192,6 @@ MAKE_ENV+=	"CC=${CC} CXX=${CXX}"
 #
 .if defined(USE_GNUSTEP_BASE)
 BUILD_DEPENDS+=	${GNUSTEP_SYSTEM_LIBRARIES}/libgnustep-base.so:${PORTSDIR}/${GNUSTEP_BASE_PORT}
-LIB_DEPENDS+=	libobjc.so.4.6:${PORTSDIR}/lang/libobjc2
 RUN_DEPENDS+=	${GNUSTEP_SYSTEM_LIBRARIES}/libgnustep-base.so:${PORTSDIR}/${GNUSTEP_BASE_PORT}
 .endif
 
