--- /usr/ports/Mk/bsd.ldap.mk	2012-11-17 06:54:17.000000000 +0100
+++ /usr/gen-dports/merged-ports/Mk/bsd.ldap.mk	2012-12-10 17:10:25.746219000 +0100
@@ -64,7 +64,7 @@
 
 CFLAGS+=	-DLDAP_DEPRECATED
 
-_OPENLDAP_CLIENT_PKG!=	${PKG_INFO} -Ex openldap.\*-client 2>/dev/null || ${TRUE}
+_OPENLDAP_CLIENT_PKG!=	${PKG_INFO} -Ex openldap.\*-client 2>/dev/null || echo
 _OPENLDAP_FLAVOUR=	${_OPENLDAP_CLIENT_PKG:C/openldap//:C/-client-.*//}
 
 .if defined(WANT_OPENLDAP_SASL)
