--- bsd.fpc.mk.orig	2012-10-29 14:19:30.000000000 +0100
+++ bsd.fpc.mk	2012-11-30 21:05:52.000000000 +0100
@@ -69,7 +69,7 @@
 _FPC_CFG_UNITS=	fastcgi fcl-web
 
 .if defined(WANT_FPC_BASE)
-.       if ${WANT_FPC_BASE:L} == "yes"
+.       if ${WANT_FPC_BASE:tl} == "yes"
 USE_FPC=	gdbint graph hash httpd22 ibase mysql odbc oracle pasjpeg paszlib \
 		postgres pthreads regexpr sqlite
 .       else
@@ -78,14 +78,14 @@
 .endif
 
 .if defined(WANT_FPC_ALL)
-.	if ${WANT_FPC_ALL:L} == "yes"
+.	if ${WANT_FPC_ALL:tl} == "yes"
 USE_FPC=	${_FPC_ALL_UNITS}
 .	else
 IGNORE=	unknown value, please use "yes" instead of
 .	endif
 .endif
 
-.if ${USE_FPC:L} != "yes"
+.if ${USE_FPC:tl} != "yes"
 .	for UNITS in ${USE_FPC}
 .		if ${_FPC_ALL_UNITS:M${UNITS}}==""
 IGNORE= cannot install: unknown FPC unit ${UNITS}
