--- bsd.mail.mk.orig	2012-10-29 14:19:30.000000000 +0100
+++ bsd.mail.mk	2012-11-30 21:07:41.000000000 +0100
@@ -40,7 +40,7 @@
 
 .if defined(QMAIL_SLAVEPORT)
 .	for slave in ${_QMAIL_VALID_SLAVEPORTS}
-.		if ${QMAIL_SLAVEPORT:L} == ${slave}
+.		if ${QMAIL_SLAVEPORT:tl} == ${slave}
 _QMAIL_SLAVEPORT_OKAY=	true
 .		endif
 .	endfor
@@ -51,7 +51,7 @@
 .endif
 
 .if defined(QMAIL_SLAVEPORT)
-QMAIL_DEPENDS=	${QMAIL_PREFIX}/bin/qmail-send:${PORTSDIR}/mail/qmail-${QMAIL_SLAVEPORT:L}
+QMAIL_DEPENDS=	${QMAIL_PREFIX}/bin/qmail-send:${PORTSDIR}/mail/qmail-${QMAIL_SLAVEPORT:tl}
 .else
 QMAIL_DEPENDS=	${QMAIL_PREFIX}/bin/qmail-send:${PORTSDIR}/mail/qmail
 .endif
