--- bsd.gecko.mk.orig	2018-10-18 12:22:12 UTC
+++ bsd.gecko.mk
@@ -101,30 +101,11 @@ USES+=		compiler:c++11-lang
 USE_XORG+=	xcb
 .endif
 
-.if ${MOZILLA_VER:R:R} >= 56
-LLVM_DEFAULT?=	70
-BUILD_DEPENDS+=	llvm${LLVM_DEFAULT}>0:devel/llvm${LLVM_DEFAULT}
-MOZ_EXPORT+=	LLVM_CONFIG=llvm-config${LLVM_DEFAULT}
-# Require newer Clang than what's in base system unless user opted out
-. if ${CC} == cc && ${CXX} == c++ && exists(/usr/lib/libc++.so)
-BUILD_DEPENDS+=	${LOCALBASE}/bin/clang${LLVM_DEFAULT}:devel/llvm${LLVM_DEFAULT}
-CPP=			${LOCALBASE}/bin/clang-cpp${LLVM_DEFAULT}
-CC=				${LOCALBASE}/bin/clang${LLVM_DEFAULT}
-CXX=			${LOCALBASE}/bin/clang++${LLVM_DEFAULT}
-USES:=			${USES:Ncompiler\:*} # XXX avoid warnings
-. endif
-.endif
-
 .if ${MOZILLA_VER:R:R} >= 61
 BUILD_DEPENDS+=	${LOCALBASE}/bin/python${PYTHON3_DEFAULT}:lang/python${PYTHON3_DEFAULT:S/.//g}
 MOZ_EXPORT+=	PYTHON3="${LOCALBASE}/bin/python${PYTHON3_DEFAULT}"
 .endif
 
-.if ${MOZILLA_VER:R:R} >= 63
-BUILD_DEPENDS+=	rust-cbindgen>=0.6.2:devel/rust-cbindgen \
-				node:www/node
-.endif
-
 MOZILLA_SUFX?=	none
 MOZSRC?=	${WRKSRC}
 PLISTF?=	${WRKDIR}/plist_files
@@ -388,7 +369,7 @@ post-patch-SNDIO-on:
 .endif
 
 .if ${PORT_OPTIONS:MRUST} || ${MOZILLA_VER:R:R} >= 54
-BUILD_DEPENDS+=	${RUST_PORT:T}>=1.28:${RUST_PORT}
+BUILD_DEPENDS+=	${RUST_PORT:T}>=1.24:${RUST_PORT}
 RUST_PORT?=		lang/rust
 . if ${MOZILLA_VER:R:R} < 54
 MOZ_OPTIONS+=	--enable-rust
