--- bsd.default-versions.mk.orig	2020-10-27 22:44:07 UTC
+++ bsd.default-versions.mk
@@ -33,6 +33,10 @@ _l=		${lang:C/=.*//g}
 ${_l:tu}_DEFAULT=	${lang:C/.*=//g}
 .endfor
 
+.if exists(/usr/lib/priv/libprivate_ssl.so)
+SSL_DEFAULT?=  libressl
+.endif
+
 # Possible values: 2.4
 APACHE_DEFAULT?=	2.4
 # Possible values: 48, 5, 6
@@ -66,7 +70,7 @@ LINUX_DEFAULT?=		c7
 # Possible values: 60, 70, 80, 90, -devel (to be used when non-base compiler is required)
 # Please give notice to the Graphics Team (x11@FreeBSD.org) in advance before 
 # bumping the LLVM version.
-LLVM_DEFAULT?=		90
+LLVM_DEFAULT?=		80
 # Possible values: 5.1, 5.2, 5.3, 5.4
 LUA_DEFAULT?=		5.2
 # Possible values: 5.5, 5.6, 5.7, 8.0, 10.3m, 10.4m, 10.5m, 5.5p, 5.6p, 5.7p, 5.6w, 5.7w
