--- bsd.xorg.mk.orig	2015-10-08 16:26:22 UTC
+++ bsd.xorg.mk
@@ -60,7 +60,7 @@ USE_XORG+=      xorg-macros
 . if ${XORG_CAT} == "driver"
 USE_XORG+=	xorg-server xproto randrproto xi renderproto xextproto \
 			inputproto kbproto fontsproto videoproto dri2proto xf86driproto \
-			glproto xineramaproto resourceproto scrnsaverproto
+			glproto xineramaproto resourceproto scrnsaverproto presentproto
 # work around a llvm bug on i386, llvm bug #15806 
 # reproduced with clang 3.2 (current release) and 3.1
 .  if ${ARCH} == i386
