--- bsd.options.mk.orig	2012-12-10 10:20:14.000000000 +0000
+++ bsd.options.mk
@@ -46,6 +46,9 @@ OPTIONSMKINCLUDED=	bsd.options.mk
 
 OPTIONSFILE?=	${PORT_DBDIR}/${UNIQUENAME}/options
 
+OPTIONS_EXCLUDE_i386+=		ALSA
+OPTIONS_EXCLUDE_x86_64+=	ALSA
+
 #ALL_OPTIONS=	DOCS \
 #		NLS
 
@@ -68,6 +71,12 @@ PORT_OPTIONS+=	IPV6
 
 # Exclude per arch options
 .for opt in ${OPTIONS_EXCLUDE_${ARCH}}
+OPTIONS_DEFINE_${ARCH}:=	${OPTIONS_DEFINE_${ARCH}:N${opt}}
+OPTIONS_DEFAULT_${ARCH}:=	${OPTIONS_DEFAULT_${ARCH}:N${opt}}
+.endfor
+
+# Exclude global options
+.for opt in ${OPTIONS_EXCLUDE}
 OPTIONS_DEFINE:=	${OPTIONS_DEFINE:N${opt}}
 OPTIONS_DEFAULT:=	${OPTIONS_DEFAULT:N${opt}}
 .endfor
@@ -81,7 +90,7 @@ OPTIONS_DEFINE+=	${opt}
 
 # Add per arch defaults
 .for opt in ${OPTIONS_DEFAULT_${ARCH}}
-.if empty(OPTIONS_DEFAULT:M${opt}}
+.if empty(OPTIONS_DEFAULT:M${opt})
 OPTIONS_DEFAULT+=	${opt}
 .endif
 .endfor
@@ -108,14 +117,16 @@ opt:=	${O}
 optname:=	${O}
 ALL_OPTIONS+=	${O}
 .if !defined(OPTIONS_DEFINE) || empty(OPTIONS_DEFINE:M${O})
+.  if empty(OPTIONS_EXCLUDE:M${O}) && empty(OPTIONS_EXCLUDE_${ARCH}:M${O})
 OPTIONS_DEFINE+=	${O}
+.  endif
 .endif
 PORT_OPTIONS+=	${O}
 .    elif !defined(optdesc)
 optdesc:=	${opt}
 ${optname}_DESC:=	${opt:S|"||g}
 .    else
-.      if ${opt:L} == off
+.      if ${opt:tl} == off
 .        if defined(PORT_OPTIONS) && defined(optname)
 NO_OPTIONS+=	${optname}
 NO_OPTIONS:=	${NO_OPTIONS:O:u}
@@ -137,6 +148,22 @@ PORT_OPTIONS:=	 ${PORT_OPTIONS:N${O}}
 
 ALL_OPTIONS:=	${ALL_OPTIONS:O:u}
 
+# exclude illegal options
+.for opt in ${OPTIONS_EXCLUDE_${ARCH}}
+.  for single in ${OPTIONS_SINGLE}
+OPTIONS_SINGLE_${single}:= ${OPTIONS_SINGLE_${single}:N${opt}}
+.  endfor
+.  for radio in ${OPTIONS_RADIO}
+OPTIONS_RADIO_${radio}:= ${OPTIONS_RADIO_${radio}:N${opt}}
+.  endfor
+.  for multi in ${OPTIONS_MULTI}
+OPTIONS_MULTI_${multi}:= ${OPTIONS_MULTI_${multi}:N${opt}}
+.  endfor
+.  for group in ${OPTIONS_GROUP}
+OPTIONS_GROUP_${group}:= ${OPTIONS_GROUP_${group}:N${opt}}
+.  endfor
+.endfor
+
 # complete list
 COMPLETE_OPTIONS_LIST=	${ALL_OPTIONS}
 .for single in ${OPTIONS_SINGLE}
