--- bsd.gcc.mk.orig	2015-04-26 13:43:46 UTC
+++ bsd.gcc.mk
@@ -37,40 +37,48 @@ GCC_Include_MAINTAINER=		gerald@FreeBSD.
 # ascending order and in sync with the table below. 
 GCCVERSIONS=	040200 040600 040700 040800 040900 050000
 
-# The first field is the OSVERSION in which it disappeared from the base.
-# The second field is the version as USE_GCC would use.
-GCCVERSION_040200=	9999999 4.2
-GCCVERSION_040600=	      0 4.6
-GCCVERSION_040700=	      0 4.7
-GCCVERSION_040800=	      0 4.8
-GCCVERSION_040900=	      0 4.9
-GCCVERSION_050000=	      0 5
+# The first field if the OSVERSION in which it appeared in the base.
+# The second field is the OSVERSION in which it disappeared from the base.
+# The third field is the version as USE_GCC would use.
+GCCVERSION_040200=	     0       0 4.2 42
+GCCVERSION_040600=	     0       0 4.6 43
+GCCVERSION_040800=	     0       0 4.8 48
+GCCVERSION_040900=	     0       0 4.9 49
+GCCVERSION_040700=	300400  400102 4.7 47
+GCCVERSION_050000=	400103 9999999 5   50
 
 # No configurable parts below this. ####################################
 #
 
-.if defined(USE_GCC) && ${USE_GCC} == yes
-USE_GCC=	${GCC_DEFAULT}+
-.endif
-
 # Extract the fields from GCCVERSION_...
 .for v in ${GCCVERSIONS}
 . for j in ${GCCVERSION_${v}}
-.  if !defined(_GCCVERSION_${v}_R)
+.  if !defined(_GCCVERSION_${v}_L)
+_GCCVERSION_${v}_L=	${j}
+.  elif !defined(_GCCVERSION_${v}_R)
 _GCCVERSION_${v}_R=	${j}
 .  elif !defined(_GCCVERSION_${v}_V)
 _GCCVERSION_${v}_V=	${j}
+.  elif !defined(_GCCVERSION_${v}_VV)
+_GCCVERSION_${v}_VV=	${j}
 .  endif
 . endfor
 .endfor
 
+LASTBASE=		5
+DFLY_DEFAULT_VERSION=	50
+
+.if ${USE_GCC:Myes}
+USE_GCC=	${LASTBASE}
+.endif
+
 .if defined(USE_GCC) && !defined(FORCE_BASE_CC_FOR_TESTING)
 
 . if ${USE_GCC} == any
 
 # Enable the clang-is-cc workaround.  Default to the last GCC imported
 # into base.
-_USE_GCC:=	4.2
+_USE_GCC:=	${LASTBASE}
 _GCC_ORLATER:=	true
 
 . else # ${USE_GCC} == any
@@ -86,12 +94,10 @@ _GCC_ORLATER:=	true
 # Initialize _GCC_FOUND${v}.  In parallel, check if USE_GCC points to a
 # valid version to begin with.
 .for v in ${GCCVERSIONS}
-. if exists(${LOCALBASE}/bin/gcc${_GCCVERSION_${v}_V:S/.//})
-_GCC_FOUND${v}=	port
-. elif ${OSVERSION} < ${_GCCVERSION_${v}_R}
-.  if exists(/usr/bin/gcc)
+. if ${DFLY_DEFAULT_VERSION} >= ${_GCCVERSION_${v}_VV}
 _GCC_FOUND${v}=	base
-.  endif
+. elif exists(${LOCALBASE}/bin/gcc${_GCCVERSION_${v}_V:S/.//})
+_GCC_FOUND${v}=	port
 . endif
 . if ${_USE_GCC}==${_GCCVERSION_${v}_V}
 _GCCVERSION_OKAY=	true
@@ -108,6 +114,11 @@ IGNORE=	Unknown version of GCC specified
 # get the first available version.
 #
 .if defined(_GCC_ORLATER)
+. if ${_USE_GCC:M5} && ${DFLY_DEFAULT_VERSION:M50}
+#  _USE_GCC is already correct
+. elif ${_USE_GCC:N5} && (${_USE_GCC:S/.//} <= ${DFLY_DEFAULT_VERSION})
+_USE_GCC= ${LASTBASE}
+. else # HEAD ORIG - run on 4.0 and below
 . for v in ${GCCVERSIONS}
 .  if ${_USE_GCC} == ${_GCCVERSION_${v}_V}
 _GCC_MIN1:=	true
@@ -130,6 +141,12 @@ _USE_GCC:=	${_GCC_FOUND}
 . elif ${_USE_GCC} < ${GCC_DEFAULT}
 _USE_GCC:=	${GCC_DEFAULT}
 . endif
+. endif   # TAIL ORIG
+.else  # defined(_GCC_ORLATER)
+. if ${_USE_GCC:M4.[67]}
+_USE_GCC=		4.7
+DFLY_DEFAULT_VERSION=	47
+. endif
 .endif # defined(_GCC_ORLATER)
 
 .endif # defined(USE_GCC)
@@ -141,7 +158,8 @@ _USE_GCC:=	${GCC_DEFAULT}
 # dependencies, CC, CXX, CPP, and flags.
 .for v in ${GCCVERSIONS}
 . if ${_USE_GCC} == ${_GCCVERSION_${v}_V}
-.  if ${OSVERSION} > ${_GCCVERSION_${v}_R} || !exists(/usr/bin/gcc)
+.  if (${_GCCVERSION_${v}_L} == 0 && ${_GCCVERSION_${v}_R} == 0) || \
+      ${DFLYVERSION} < ${_GCCVERSION_${v}_L}
 V:=			${_GCCVERSION_${v}_V:S/.//}
 _GCC_PORT_DEPENDS:=	gcc${V}
 .   if ${_USE_GCC} == ${LANG_GCC_IS}
@@ -159,11 +177,9 @@ LDFLAGS+=		-Wl,-rpath=${_GCC_RUNTIME} -L
 .  else # Use GCC in base.
 CC:=			gcc
 CXX:=			g++
-.   if exists(/usr/bin/gcpp)
-CPP:=			gcpp
-.   else
 CPP:=			cpp
-.   endif
+CONFIGURE_ENV+=		CCVER=gcc${DFLY_DEFAULT_VERSION}
+MAKE_ENV+=		CCVER=gcc${DFLY_DEFAULT_VERSION}
 .  endif # Use GCC in base.
 . endif # ${_USE_GCC} == ${_GCCVERSION_${v}_V}
 .endfor
@@ -195,13 +211,15 @@ test-gcc:
 .if defined(_GCC_FOUND${v})
 	@echo -n "(${_GCC_FOUND${v}}) "
 .endif
-	@echo "- OSVERSION up to ${_GCCVERSION_${v}_R}"
-#	@echo ${v} - ${_GCC_FOUND${v}} - up to ${_GCCVERSION_${v}_R} - ${_GCCVERSION_${v}_V}
+	@echo "- OSVERSION from ${_GCCVERSION_${v}_L} to ${_GCCVERSION_${v}_R}"
+#	@echo ${v} - ${_GCC_FOUND${v}} - ${_GCCVERSION_${v}_L} to ${_GCCVERSION_${v}_R} - ${_GCCVERSION_${v}_V}
 .endfor
 	@echo Using GCC version ${_USE_GCC}
 .endif
 	@echo CC=${CC} - CXX=${CXX} - CPP=${CPP} - CFLAGS=\"${CFLAGS}\"
 	@echo LDFLAGS=\"${LDFLAGS}\"
+	@echo CONFIGURE_ENV=${CONFIGURE_ENV}
+	@echo MAKE_ENV=${MAKE_ENV}
 	@echo "BUILD_DEPENDS=${BUILD_DEPENDS}"
 	@echo "RUN_DEPENDS=${RUN_DEPENDS}"
 .endif
