--- bsd.ruby.mk.orig	2014-02-02 23:24:19.238133000 +0000
+++ bsd.ruby.mk
@@ -231,9 +231,9 @@ IGNORE=	Only ruby 1.9 and 2.0 are suppor
 . endif
 .endif # defined(RUBY_VER)
 
-CONFIGURE_TARGET?=	${ARCH}-portbld-freebsd${OSREL:C/\..*//}
+CONFIGURE_TARGET?=	${ARCH}-portbld-${OPSYS:tl}${OSREL:C/\..*//}
 
-RUBY_ARCH?=		${ARCH}-freebsd${OSREL:C/\..*//}
+RUBY_ARCH?=		${ARCH}-${OPSYS:tl}${OSREL:C/\..*//}
 RUBY_NAME?=		ruby${RUBY_SUFFIX}
 
 _RUBY_SYSLIBDIR?=	${PREFIX}/lib
