--- bsd.php.mk.orig	2012-10-29 14:19:30.000000000 +0100
+++ bsd.php.mk	2012-11-30 21:08:05.000000000 +0100
@@ -235,7 +235,7 @@
 .endif
 
 # Extensions
-.if defined(_POSTMKINCLUDED) && ${USE_PHP:L} != "yes"
+.if defined(_POSTMKINCLUDED) && ${USE_PHP:tl} != "yes"
 # non-version specific components
 _USE_PHP_ALL=	apc bcmath bitset bz2 calendar ctype curl dba dom \
 		exif fileinfo filter ftp gd gettext gmp \
@@ -346,7 +346,7 @@
 BUILD_DEPENDS+=	${PHPBASE}/lib/php/${PHP_EXT_DIR}/hash.so:${PORTSDIR}/${hash_DEPENDS}
 .				endif
 RUN_DEPENDS+=	${PHPBASE}/lib/php/${PHP_EXT_DIR}/hash.so:${PORTSDIR}/${hash_DEPENDS}
-.			elif ${ext:L} != "yes"
+.			elif ${ext:tl} != "yes"
 check-makevars::
 			@${ECHO_CMD} "Unknown extension ${extension} for PHP ${PHP_VER}."
 			@${FALSE}
