--- bsd.port.subdir.mk.orig	2015-10-13 18:18:55 UTC
+++ bsd.port.subdir.mk
@@ -66,15 +66,19 @@ ARCH!=	${UNAME} -p
 _EXPORTED_VARS+=	ARCH
 
 .if !defined(OSVERSION)
-.if exists(/usr/include/sys/param.h)
-OSVERSION!=	${AWK} '/^\#define[[:blank:]]__FreeBSD_version/ {print $$3}' < /usr/include/sys/param.h
-.elif exists(${SRC_BASE}/sys/sys/param.h)
-OSVERSION!=	${AWK} '/^\#define[[:blank:]]__FreeBSD_version/ {print $$3}' < ${SRC_BASE}/sys/sys/param.h
-.else
-OSVERSION!=	${SYSCTL} -n kern.osreldate
+OSVERSION=	9999999
 .endif
+
+.if !defined(DFLYVERSION)
+.  if exists(/usr/include/sys/param.h)
+DFLYVERSION!=	${AWK} '/^\#define[[:blank:]]__DragonFly_version/ {print $$3}' < /usr/include/sys/param.h
+.  elif exists(${SRC_BASE}/sys/sys/param.h)
+DFLYVERSION!=	${AWK} '/^\#define[[:blank:]]__DragonFly_version/ {print $$3}' < ${SRC_BASE}/sys/sys/param.h
+.  else
+DFLYVERSION!=	${SYSCTL} -n kern.osreldate
+.  endif
 .endif
-_EXPORTED_VARS+=	OSVERSION
+_EXPORTED_VARS+=	OSVERSION DFLYVERSION
 
 WITH_PKG=	yes
 WITH_PKGNG=	yes
@@ -136,7 +140,7 @@ _EXPORTED_VARS+=	UID
 .endif
 
 INDEXDIR?=	${PORTSDIR}
-INDEXFILE?=	INDEX-${OSVERSION:C/([0-9]*)[0-9]{5}/\1/}
+INDEXFILE?=	INDEX-${DFLYVERSION:C/([0-9]*)[0-9]{5}/\1/}
 
 # local customization of the ports tree
 .sinclude "${.CURDIR}/Makefile.local"
