--- bsd.default-versions.mk.orig	2019-04-25 18:41:30 UTC
+++ bsd.default-versions.mk
@@ -29,6 +29,10 @@ _l=		${lang:C/=.*//g}
 ${_l:tu}_DEFAULT=	${lang:C/.*=//g}
 .endfor
 
+.if exists(/usr/lib/priv/libprivate_ssl.so)
+SSL_DEFAULT?=  libressl
+.endif
+
 # Possible values: 2.4
 APACHE_DEFAULT?=	2.4
 # Possible values: 48, 5, 6
@@ -60,7 +64,7 @@ LINUX_DEFAULT?=		c7
 # Possible values: 60, 70, 80, 90, -devel (to be used when non-base compiler is required)
 # Please give notice to the Graphics Team (x11@FreeBSD.org) in advance before 
 # bumping the LLVM version.
-LLVM_DEFAULT?=		90
+LLVM_DEFAULT?=		80
 # Possible values: 5.1, 5.2, 5.3
 LUA_DEFAULT?=		5.2
 # Possible values: 5.5, 5.6, 5.7, 8.0, 5.5m, 10.0m, 10.1m, 10.2m, 10.3m, 5.5p, 5.6p, 5.7p, 5.6w
