--- bsd.qt.mk.orig	2013-05-02 14:12:58.000000000 +0000
+++ bsd.qt.mk
@@ -310,10 +310,7 @@ Qt_Post_Include= bsd.qt.mk
 #
 # Translate `c++` to its real name and select the appropriate mkspec.
 #
-QMAKE_BASE_COMPILER!=	(cc --version 2> /dev/null | ${AWK} 'NR == 1 { gsub(/[()]/, "", $$2); print $$2 }') || ${ECHO_CMD} "gcc"
-.if ${QMAKE_BASE_COMPILER:tl} == "gcc"
 QMAKE_BASE_COMPILER=	g++
-.endif
 .if ${CXX} == "c++"
 # Why CXX instead of CXX:T? Because if you're setting the full path of
 # `c++` you probably want to define QMAKESPEC by hand too.
