--- Scripts/do-fetch.sh.orig	2016-06-21 09:15:07 UTC
+++ Scripts/do-fetch.sh
@@ -131,7 +131,8 @@ for _file in "${@}"; do
 		case ${dp_TARGET} in
 			do-fetch|makesum)
 				${dp_ECHO_MSG} "=> Attempting to fetch ${site}${file}"
-				if env -S "${dp_FETCH_ENV}" ${_fetch_cmd}; then
+# not				if env -S "${dp_FETCH_ENV}" ${_fetch_cmd}; then
+				if /bin/sh "${dp_SCRIPTSDIR}/not_posix_env.sh" "${dp_FETCH_ENV}" ${_fetch_cmd}; then
 					actual_size=$(stat -f %z "${file}")
 					if [ -n "${dp_DISABLE_SIZE}" ] || [ -z "${CKSIZE}" ] || [ "${actual_size}" -eq "${CKSIZE}" ]; then
 						continue 2
