--- Scripts/do-fetch.sh.orig	2019-07-31 12:02:51 UTC
+++ Scripts/do-fetch.sh
@@ -26,6 +26,16 @@ case ${dp_TARGET} in
 		;;
 esac
 
+if [ ! -n "${DO_FETCH_ISLOCKED:=}" ]; then
+	for _file in "${@}"; do
+		mkdir -p .locks
+		file=${_file%%:*}
+		lkfile="`echo "$file" | md5`"
+		DO_FETCH_ISLOCKED=YES lockf -k -s .locks/${lkfile}.lk sh $0 "${_file}" || exit 1
+	done
+	exit 0
+fi
+
 for _file in "${@}"; do
 	file=${_file%%:*}
 
@@ -148,7 +158,8 @@ for _file in "${@}"; do
 		case ${dp_TARGET} in
 			do-fetch|makesum)
 				${dp_ECHO_MSG} "=> Attempting to fetch ${site}${file}"
-				if env -S "${dp_FETCH_ENV}" ${_fetch_cmd}; then
+# not				if env -S "${dp_FETCH_ENV}" ${_fetch_cmd}; then
+				if /bin/sh "${dp_SCRIPTSDIR}/not_posix_env.sh" "${dp_FETCH_ENV}" ${_fetch_cmd}; then
 					actual_size=$(stat -f %z "${file}")
 					if [ -n "${dp_DISABLE_SIZE}" ] || [ -z "${CKSIZE}" ] || [ "${actual_size}" -eq "${CKSIZE}" ]; then
 						continue 2
