--- Uses/fortran.mk.orig	2017-10-30 17:09:57 UTC
+++ Uses/fortran.mk
@@ -26,7 +26,12 @@ LDFLAGS+=	-L${LOCALBASE}/flang/lib -lfla
 IGNORE=		USES=fortran: flang argument only available for amd64
 .endif
 .elif ${fortran_ARGS} == gcc
-_GCC_VER=	${GCC_DEFAULT:S/.//}
+_GCC_VER=	5
+. if defined(USE_GCC)
+.  if ${USE_GCC} == 4.8 || ${USE_GCC} == 4.9 || ${USE_GCC} == 6
+_GCC_VER=	${USE_GCC:S/.//}
+.  endif
+. endif
 BUILD_DEPENDS+=	gfortran${_GCC_VER}:lang/gcc${_GCC_VER}
 RUN_DEPENDS+=	gfortran${_GCC_VER}:lang/gcc${_GCC_VER}
 F77=		gfortran${_GCC_VER}
