--- Uses/ncurses.mk.orig	2016-02-04 09:00:08 UTC
+++ Uses/ncurses.mk
@@ -64,18 +64,28 @@ NCURSES_INSTALLED!=	${PKG_BIN} ${PKGARGS
 .  endif
 NCURSES_INSTALLED?=
 
-.if ${NCURSES_INSTALLED} != ""
+.  if ${NCURSES_INSTALLED} != ""
 NCURSES_PORT=	${NCURSES_INSTALLED}
 NCURSES_SHLIBFILE!=	${PKG_INFO} -ql ${NCURSES_INSTALLED} | grep -m 1 "^`pkg query "%p" ${NCURSES_INSTALLED}`/lib/libncurses.so."
 NCURSES_SHLIBVER?=	${NCURSES_SHLIBFILE:E}
-.endif
+.  endif
 
 NCURSES_PORT?=		devel/ncurses
+.  if exists (/usr/lib/priv/libprivate_ncursesw.so)
 NCURSES_SHLIBVER?=	6
+.  else
+NCURSES_SHLIBVER?=	6P
+.  endif
 
 BUILD_DEPENDS+=		${LOCALBASE}/lib/libncurses.so.${NCURSES_SHLIBVER}:${PORTSDIR}/${NCURSES_PORT}
 RUN_DEPENDS+=		${LOCALBASE}/lib/libncurses.so.${NCURSES_SHLIBVER}:${PORTSDIR}/${NCURSES_PORT}
 NCURSESRPATH=		${NCURSESBASE}/lib
+LDFLAGS+=		-L${NCURSESRPATH} -Wl,-rpath=${NCURSESRPATH}
+CFLAGS+=		-I${NCURSESBASE}/include -I${NCURSESINC}
+
+.  if defined(NCURSES_RPATH)
+CFLAGS+=		-Wl,-rpath,${NCURSESRPATH}
+.  endif
 
 .else
 .error		USES=ncurses only accept 'port' and 'base' as arguments, got ${ncurses_ARGS}
@@ -83,9 +93,4 @@ NCURSESRPATH=		${NCURSESBASE}/lib
 
 NCURSESLIB=	${NCURSESBASE}/lib
 
-.if defined(NCURSES_RPATH)
-CFLAGS+=	-Wl,-rpath,${NCURSESRPATH}
-.endif
-LDFLAGS+=	-Wl,-rpath=${NCURSESRPATH}
-
 .endif
