--- Uses/gem.mk.orig	2018-01-15 07:25:58 UTC
+++ Uses/gem.mk
@@ -73,11 +73,13 @@ GEMFILES=	${DISTNAME}${EXTRACT_SUFX}
 
 RUBYGEM_ARGS=-l --no-update-sources --install-dir ${STAGEDIR}${PREFIX}/lib/ruby/gems/${RUBY_VER} --ignore-dependencies --bindir=${STAGEDIR}${PREFIX}/bin
 
-.if ${PORT_OPTIONS:MDOCS}
-RUBYGEM_ARGS+=	--rdoc --ri
-.else
+# below condition ALWAYS true (somebody brainfarted here)
+# Just disable rdoc unconditionally.  it freezes with 100% CPU sometimes
+# .if ${PORT_OPTIONS:MDOCS}
+# RUBYGEM_ARGS+=	--rdoc --ri
+# .else
 RUBYGEM_ARGS+=	--no-rdoc --no-ri
-.endif
+# .endif
 
 .if !target(do-extract)
 do-extract:
@@ -111,9 +113,9 @@ do-install:
 	${RM} -r ${STAGEDIR}${PREFIX}/${GEM_LIB_DIR}/ext \
 		${STAGEDIR}${PREFIX}/${CACHE_DIR} 2> /dev/null || ${TRUE}
 	${RMDIR} ${STAGEDIR}${PREFIX}/${EXT_DIR} 2> /dev/null || ${TRUE}
-.if !${PORT_OPTIONS:MDOCS}
-	-@${RMDIR} ${STAGEDIR}${PREFIX}/${DOC_DIR}
-.endif
+# .if !${PORT_OPTIONS:MDOCS}
+	-${RMDIR} ${STAGEDIR}${PREFIX}/${DOC_DIR}
+# .endif
 .endif
 
 .if empty(gem_ARGS:Mnoautoplist)
