--- Uses/fortran.mk.orig	2019-04-25 18:39:33 UTC
+++ Uses/fortran.mk
@@ -26,7 +26,12 @@ LDFLAGS+=	-L${LOCALBASE}/flang/lib -Wl,-
 IGNORE=		USES=fortran: flang argument only available for amd64
 .endif
 .elif ${fortran_ARGS} == gfortran
-_GCC_VER=	${GCC_DEFAULT:S/.//}
+_GCC_VER=	8
+. if defined(USE_GCC)
+.  if ${USE_GCC} == 6 || ${USE_GCC} == 7 || ${USE_GCC} == 9
+_GCC_VER=	${USE_GCC:S/.//}
+.  endif
+. endif
 BUILD_DEPENDS+=	gfortran${_GCC_VER}:lang/gcc${_GCC_VER}
 RUN_DEPENDS+=	gfortran${_GCC_VER}:lang/gcc${_GCC_VER}
 F77=		gfortran${_GCC_VER}
