--- Uses/zip.mk.orig	2014-09-29 11:50:17 UTC
+++ Uses/zip.mk
@@ -12,14 +12,29 @@ _INCLUDE_USES_ZIP_MK=	yes
 
 EXTRACT_SUFX?=	.zip
 
-EXTRACT_BEFORE_ARGS?=	-qo
-EXTRACT_AFTER_ARGS?=	-d ${WRKDIR}
+ZIP_BEFORE_ARGS?=	-qo
+ZIP_AFTER_ARGS?=	-d ${EXTRACT_WRKDIR}
+
+.if defined(EXTRACT_BEFORE_ARGS)
+ZIP_BEFORE_ARGS:= ${EXTRACT_BEFORE_ARGS}
+.undef EXTRACT_BEFORE_ARGS
+.endif
+
+.if defined(EXTRACT_AFTER_ARGS)
+ZIP_AFTER_ARGS:= ${EXTRACT_AFTER_ARGS}
+.undef EXTRACT_AFTER_ARGS
+.endif
 
 .if empty(zip_ARGS)
-EXTRACT_CMD?=		${UNZIP_NATIVE_CMD}
+.  if ${OPSYS} == FreeBSD
+ZIP_EXTRACT_CMD?=	${UNZIP_NATIVE_CMD}
+.  else
+EXTRACT_DEPENDS+=	${UNZIP_CMD}:${PORTSDIR}/archivers/unzip
+ZIP_EXTRACT_CMD?=	${UNZIP_CMD}
+.  endif
 .elif ${zip_ARGS} == "infozip"
 EXTRACT_DEPENDS+=	${UNZIP_CMD}:${PORTSDIR}/archivers/unzip
-EXTRACT_CMD?=		${UNZIP_CMD}
+ZIP_EXTRACT_CMD?=	${UNZIP_CMD}
 .else
 IGNORE=	Incorrect 'USES+=zip:${zip_ARGS}' expecting 'USES+=zip[:infozip]'
 .endif
