--- Uses/qt.mk.orig	2024-09-10 23:37:33 UTC
+++ Uses/qt.mk
@@ -108,7 +108,7 @@ QMAKESPEC?=		${QT_MKSPECDIR}/${QMAKESPEC
 # better to support custom settings. Dereferencing the detection allows to
 # avoid forking a shell on each inclusion of this file, and to catch any CXX
 # customization (via USE_GCC, etc.).
-QMAKE_COMPILER=	$$(ccver="$$(${CXX} --version)"; case "$$ccver" in *clang*) echo clang ;; *) echo g++ ;; esac)
+QMAKE_COMPILER=	g++
 
 .  for dir in BIN INC LIB ARCH PLUGIN LIBEXEC IMPORT \
 	QML DATA DOC L10N ETC EXAMPLE TEST MKSPEC \
