--- Uses/objc.mk.orig	2016-01-17 10:32:56 UTC
+++ Uses/objc.mk
@@ -15,17 +15,7 @@ objc_ARGS?=
 IGNORE=	USES=objc only accepts no arguments or 'compiler'
 .endif
 
-_CCVERSION!=	${CC} --version
-.if defined(.PARSEDIR)
-COMPILER_VERSION=	${_CCVERSION:M[0-9].[0-9]*:tW:C/([0-9]).([0-9]).*/\1\2/g}
-.else
-COMPILER_VERSION=	${_CCVERSION:M[0-9].[0-9]*:C/([0-9]).([0-9]).*/\1\2/g:u}
-.endif
-.if ${_CCVERSION:Mclang}
-COMPILER_TYPE=	clang
-.else
 COMPILER_TYPE=	gcc
-.endif
 
 ALT_COMPILER_VERSION=	0
 ALT_COMPILER_TYPE=	none
