--- Uses/qt.mk.orig	2022-07-12 00:15:36 UTC
+++ Uses/qt.mk
@@ -94,7 +94,7 @@ QMAKESPEC?=		${QT_MKSPECDIR}/${QMAKESPEC
 # better to support custom settings. Dereferencing the detection allows to
 # avoid forking a shell on each inclusion of this file, and to catch any CXX
 # customization (via USE_GCC, etc.).
-QMAKE_COMPILER=	$$(ccver="$$(${CXX} --version)"; case "$$ccver" in *clang*) echo clang ;; *) echo g++ ;; esac)
+QMAKE_COMPILER=	g++
 
 .  for dir in BIN INC LIB ARCH PLUGIN LIBEXEC IMPORT \
 	QML DATA DOC L10N ETC EXAMPLE TEST MKSPEC \
@@ -133,7 +133,7 @@ _USE_QT_ALL=		assistant dbus declarative
 			scripttools sql sql-mysql sql-odbc sql-pgsql \
 			sql-sqlite2 sql-sqlite3 svg testlib webkit \
 			xml xmlpatterns
-.  if ${ARCH} == amd64 || ${ARCH} == i386
+.  if ${ARCH} == x86_64 || ${ARCH} == i386
 _USE_QT_ALL+=	sql-ibase
 .  endif
 
