--- Uses/iconv.mk.orig	2015-07-04 07:40:51 UTC
+++ Uses/iconv.mk
@@ -13,7 +13,7 @@
 .if !defined(_INCLUDE_USES_ICONV_MK)
 _INCLUDE_USES_ICONV_MK=	yes
 
-.if !exists(/usr/include/iconv.h) || ${iconv_ARGS:Mwchar_t} || ${iconv_ARGS:Mtranslit}
+.if ${iconv_ARGS:Mwchar_t} || ${iconv_ARGS:Mtranslit}
 
 ICONV_CMD=	${LOCALBASE}/bin/iconv
 ICONV_LIB=	-liconv
@@ -37,7 +37,8 @@ ICONV_PREFIX=	/usr
 ICONV_CONFIGURE_ARG=
 ICONV_CONFIGURE_BASE=
 
-.if ${OPSYS} == DragonFly || (${OPSYS} == FreeBSD && (${OSVERSION} < 1001514 \
+.if (${OPSYS} == DragonFly && ${DFLYVERSION} < 400301) \
+ || (${OPSYS} == FreeBSD && (${OSVERSION} < 1001514 \
  || (${OSVERSION} >= 1100000 && ${OSVERSION} < 1100069))) \
  || exists(${LOCALBASE}/include/iconv.h)
 BUILD_DEPENDS+=	libiconv>=1.14_8:${PORTSDIR}/converters/libiconv
