--- Uses/qmail.mk.orig	2013-03-15 22:03:42.000000000 +0000
+++ Uses/qmail.mk
@@ -43,7 +43,7 @@ _QMAIL_VALID_SLAVEPORTS=	ldap mysql spam
 
 .  if defined(QMAIL_SLAVEPORT)
 .    for slave in ${_QMAIL_VALID_SLAVEPORTS}
-.      if ${QMAIL_SLAVEPORT:L} == ${slave}
+.      if ${QMAIL_SLAVEPORT:tl} == ${slave}
 _QMAIL_SLAVEPORT_OKAY=	true
 .      endif
 .    endfor
@@ -54,7 +54,7 @@ IGNORE=	Invalid QMAIL_SLAVEPORT value. O
 .  endif
 
 .  if defined(QMAIL_SLAVEPORT)
-_QMAIL_DEPENDS=	${QMAIL_PREFIX}/bin/qmail-send:${PORTSDIR}/mail/qmail-${QMAIL_SLAVEPORT:L}
+_QMAIL_DEPENDS=	${QMAIL_PREFIX}/bin/qmail-send:${PORTSDIR}/mail/qmail-${QMAIL_SLAVEPORT:tl}
 .  else
 _QMAIL_DEPENDS=	${QMAIL_PREFIX}/bin/qmail-send:${PORTSDIR}/mail/qmail
 .  endif
