--- Uses/libtool.mk.orig	2014-02-17 14:45:50.530918000 +0000
+++ Uses/libtool.mk
@@ -20,6 +20,7 @@ patch-libtool:
 	@${FIND} ${WRKDIR} \( -name configure -or -name ltconfig \)	\
 		-type f | ${XARGS} ${REINPLACE_CMD}			\
 		-e '/link_all_deplibs[0-9A-Z_]*=/s/=unknown/=no/'	\
+		-e 's,freebsd\*),freebsd\*|dragonfly\*),g'		\
 		-e '/objformat=/s/echo aout/echo elf/'			\
 		-e "/freebsd-elf\\*)/,/;;/ {				\
 		    /deplibs_check_method=/s/=.*/=pass_all/; }"
