--- Uses/compiler.mk.orig	2018-03-03 03:40:05 UTC
+++ Uses/compiler.mk
@@ -69,6 +69,11 @@ _COMPILER_ARGS=	#
 _COMPILER_ARGS+=	features
 .endif
 
+.if ${CC} == cc 
+# This is the DragonFly base compiler, we know it's gcc
+COMPILER_TYPE=		gcc
+COMPILER_VERSION=	53
+.else
 .if defined(_CCVERSION_${_CC_hash})
 _CCVERSION=	${_CCVERSION_${_CC_hash}}
 .else
@@ -82,6 +87,7 @@ COMPILER_TYPE=	clang
 .else
 COMPILER_TYPE=	gcc
 .endif
+.endif
 
 ALT_COMPILER_VERSION=	0
 ALT_COMPILER_TYPE=	none
@@ -124,6 +130,12 @@ CHOSEN_COMPILER_TYPE=	gcc
 .endif
 
 .if ${_COMPILER_ARGS:Mfeatures}
+.  if ${CC} == cc && ${CXX} == c++
+   # This is DragonFly's base gcc50 compiler
+   # Use a cache for DF rather than testing each feature.
+COMPILER_FEATURES=	libstdc++ c89 c99 c11 gnu89 gnu99 gnu11 c++98 \
+			c++0x c++11 c++14 gnu++98 gnu++11 dragonfly
+.  else
 .if defined(_CXXINTERNAL_${_CXX_hash})
 _CXXINTERNAL=	${_CXXINTERNAL_${_CXX_hash}}
 .else
@@ -156,6 +168,7 @@ PORTS_ENV_VARS+=		OUTPUT_${std:hash}_${_
 COMPILER_FEATURES+=	${std}
 .endif
 .endfor
+.  endif	# DragonFly base compiler
 .endif
 
 .if ${_COMPILER_ARGS:Mc++11-lib}
