--- Uses/qt.mk.orig	2022-10-11 20:47:05 UTC
+++ Uses/qt.mk
@@ -98,7 +98,7 @@ QMAKESPEC?=		${QT_MKSPECDIR}/${QMAKESPEC
 # better to support custom settings. Dereferencing the detection allows to
 # avoid forking a shell on each inclusion of this file, and to catch any CXX
 # customization (via USE_GCC, etc.).
-QMAKE_COMPILER=	$$(ccver="$$(${CXX} --version)"; case "$$ccver" in *clang*) echo clang ;; *) echo g++ ;; esac)
+QMAKE_COMPILER=	g++
 
 .  for dir in BIN INC LIB ARCH PLUGIN LIBEXEC IMPORT \
 	QML DATA DOC L10N ETC EXAMPLE TEST MKSPEC \
@@ -144,7 +144,7 @@ _USE_QT5_ONLY=		assistant buildtools con
 			sql-pgsql sql-sqlite2 sql-sqlite3 sql-tds testlib uiplugin \
 			uitools webengine webglplugin webkit websockets-qml webview \
 			widgets x11extras xml xmlpatterns
-.  if ${ARCH} == amd64 || ${ARCH} == i386
+.  if ${ARCH} == x86_64 || ${ARCH} == i386
 _USE_QT5_ONLY+=		sql-ibase
 .  endif
 
