--- Uses/fortran.mk.orig	2022-07-11 00:15:14 UTC
+++ Uses/fortran.mk
@@ -14,17 +14,22 @@ fortran_ARGS=	${FORTRAN_DEFAULT}
 .  endif
 
 .  if ${fortran_ARGS} == flang
-.    if ${ARCH} == amd64
+.    if ${ARCH} == x86_64
 BUILD_DEPENDS+=	flang>0:devel/flang
 RUN_DEPENDS+=	flang>0:devel/flang
 F77=		flang
 FC=		flang
 LDFLAGS+=	-L${LOCALBASE}/flang/lib -Wl,--as-needed -lflang -lexecinfo -Wl,--no-as-needed
 .    else
-IGNORE=		USES=fortran: flang argument only available for amd64
+IGNORE=		USES=fortran: flang argument only available for x86_64
 .    endif
 .  elif ${fortran_ARGS} == gfortran
-_GCC_VER=	${GCC_DEFAULT:S/.//}
+_GCC_VER=	8
+. if defined(USE_GCC)
+.  if ${USE_GCC} == 6 || ${USE_GCC} == 7 || ${USE_GCC} == 9
+_GCC_VER=	${USE_GCC:S/.//}
+.  endif
+. endif
 BUILD_DEPENDS+=	gfortran${_GCC_VER}:lang/gcc${_GCC_VER}
 RUN_DEPENDS+=	gfortran${_GCC_VER}:lang/gcc${_GCC_VER}
 F77=		gfortran${_GCC_VER}
