--- Uses/guessfix.mk.orig	2013-05-24 11:48:31.193829000 +0000
+++ Uses/guessfix.mk
@@ -0,0 +1,21 @@
+# $FreeBSD$
+#
+# Replace config.guess and config.sub files in specified directories
+#
+# MAINTAINER: draco@marino.st
+#
+# Feature:	guessfix
+# Usage:	USES=guessfix
+#
+
+.if !defined(_INCLUDE_USES_GUESSFIX_Mk)
+_INCLUDE_USES_GUESSFIX_MK=	yes
+
+pre-patch: fix-subguess
+
+fix-subguess:
+	@cd ${WRKSRC}; ${FIND} * -type f -name config.sub -exec \
+		${CP} ${PORTSDIR}/Templates/config.sub {} \;
+	@cd ${WRKSRC}; ${FIND} * -type f -name config.guess -exec \
+		${CP} ${PORTSDIR}/Templates/config.guess {} \;
+.endif
