--- Uses/qt.mk.orig	2025-10-06 20:17:31 UTC
+++ Uses/qt.mk
@@ -109,7 +109,7 @@ QMAKESPEC?=		${QT_MKSPECDIR}/${QMAKESPECNAME}
 # better to support custom settings. Dereferencing the detection allows to
 # avoid forking a shell on each inclusion of this file, and to catch any CXX
 # customization (via USE_GCC, etc.).
-QMAKE_COMPILER=	$$(ccver="$$(${CXX} --version)"; case "$$ccver" in *clang*) echo clang ;; *) echo g++ ;; esac)
+QMAKE_COMPILER=	g++
 
 .  for dir in BIN INC LIB ARCH PLUGIN LIBEXEC IMPORT \
 	QML SBOM DATA DOC L10N ETC EXAMPLE TEST MKSPEC \
@@ -125,6 +125,8 @@ PLIST_SUB+=		QT_${dir}DIR="${QT_${dir}DIR_REL}"
 # Suppress warnings from rcc about not using a UTF-8 locale.
 .  if ${_QT_VER:M6}
 USE_LOCALE?=		C.UTF-8
+# Avoid having the compiler specified per port
+USE_GCC_VERSION=	${GCC_DEFAULT}
 .  endif
 
 CONFIGURE_ENV+=		QT_SELECT=${_QT_RELNAME}
