--- Uses/fortran.mk.orig	2016-04-06 14:58:53 UTC
+++ Uses/fortran.mk
@@ -16,15 +16,9 @@ fortran_ARGS=	gcc
 .endif
 
 .if ${fortran_ARGS} == gcc
-.include "${PORTSDIR}/Mk/bsd.default-versions.mk"
-_GCC_VER=	${GCC_DEFAULT:S/.//}
-.if ${GCC_DEFAULT} == ${LANG_GCC_IS}
-BUILD_DEPENDS+=	gfortran${_GCC_VER}:lang/gcc
-RUN_DEPENDS+=	gfortran${_GCC_VER}:lang/gcc
-.else
+_GCC_VER=	5
 BUILD_DEPENDS+=	gfortran${_GCC_VER}:lang/gcc${_GCC_VER}
 RUN_DEPENDS+=	gfortran${_GCC_VER}:lang/gcc${_GCC_VER}
-.endif
 USE_BINUTILS=	yes
 F77=		gfortran${_GCC_VER}
 FC=		gfortran${_GCC_VER}
