--- Uses/iconv.mk.orig	2015-07-05 10:07:01 UTC
+++ Uses/iconv.mk
@@ -37,9 +37,9 @@ ICONV_PREFIX=	/usr
 ICONV_CONFIGURE_ARG=
 ICONV_CONFIGURE_BASE=
 
-.if ${OPSYS} == DragonFly || (${OPSYS} == FreeBSD && (${OSVERSION} < 1001514 \
- || (${OSVERSION} >= 1100000 && ${OSVERSION} < 1100069))) \
- || exists(${LOCALBASE}/include/iconv.h)
+.if (${OPSYS} == DragonFly && ${DFLYVERSION} < 400301) \
+ || (${OPSYS} == FreeBSD && (${OSVERSION} < 1001514 \
+ || (${OSVERSION} >= 1100000 && ${OSVERSION} < 1100069)))
 BUILD_DEPENDS+=	libiconv>=1.14_8:${PORTSDIR}/converters/libiconv
 CPPFLAGS+=	-DLIBICONV_PLUG
 CFLAGS+=	-DLIBICONV_PLUG
