--- Uses/libtool.mk.orig	2014-09-13 19:48:38 UTC
+++ Uses/libtool.mk
@@ -77,7 +77,7 @@ patch-lafiles:
 		${XARGS} ${SED} -i '' -e "/dependency_libs=/s/=.*/=''/"
 .else
 	@${FIND} ${STAGEDIR} -type l -exec ${SH} -c			\
-		'case `${READLINK_CMD} -f "{}"` in			\
+		'case `${REALPATH} "{}"` in			\
 			*.la) ${ECHO_CMD} "{}" ;; esac' \; |		\
 		${XARGS} ${GREP} -l 'libtool library' | ${XARGS} ${RM}
 	@${FIND} ${STAGEDIR} -type f -name '*.la' |			\
