--- Uses/kde.mk.orig	2017-09-13 12:56:26 UTC
+++ Uses/kde.mk
@@ -217,9 +217,10 @@ _USE_KDE4_ALL+=		akonadi attica automoc4
 _USE_FRAMEWORKS_TIER1=	apidox archive attica5 breeze-icons codecs config \
 			coreaddons dbusaddons dnssd i18n idletime itemmodels \
 			itemviews kirigami2 oxygen-icons5 plotting prison solid \
-			sonnet syntaxhighlighting threadweaver wayland \
+			sonnet syntaxhighlighting threadweaver \
 			widgetsaddons windowsystem
 # NOT LISTED TIER1: modemmanagerqt networkmanagerqt (not applicable)
+# NOT AVAILABLE DF: wayland
 
 _USE_FRAMEWORKS_TIER2=	auth completion crash doctools filemetadata5 \
 			kimageformats jobwidgets notifications package \
