--- Uses/cmake.mk.orig	2017-07-03 13:28:04 UTC
+++ Uses/cmake.mk
@@ -87,6 +87,7 @@ CMAKE_ARGS+=		-DCMAKE_C_COMPILER:STRING=
 			-DCMAKE_BUILD_TYPE:STRING="${CMAKE_BUILD_TYPE}" \
 			-DTHREADS_HAVE_PTHREAD_ARG:BOOL=YES \
 			-DCMAKE_INSTALL_RPATH_USE_LINK_PATH=YES \
+			-DCMAKE_NO_BUILTIN_CHRPATH:BOOL=ON \
 			-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON
 
 CMAKE_INSTALL_PREFIX?=	${PREFIX}
@@ -117,10 +118,11 @@ TEST_WRKSRC?=		${CONFIGURE_WRKSRC}
 #             as the build.ninja file won't be where ninja expects it.
 .if empty(cmake_ARGS:Mnoninja) && empty(cmake_ARGS:Mrun) && empty(USES:Mfortran)
 .  if "${CONFIGURE_WRKSRC}" == "${BUILD_WRKSRC}" && "${CONFIGURE_WRKSRC}" == "${INSTALL_WRKSRC}"
-.    if ! empty(USES:Mgmake)
-BROKEN=		USES=gmake is incompatible with cmake's ninja-generator
-.    endif
-.      include "${USESDIR}/ninja.mk"
+# disable ninja by default on DPORTS
+# .    if ! empty(USES:Mgmake)
+# BROKEN=		USES=gmake is incompatible with cmake's ninja-generator
+# .    endif
+# .      include "${USESDIR}/ninja.mk"
 .  endif
 .endif
 
