--- Makefile.orig	2021-10-16 10:59:17 UTC
+++ Makefile
@@ -29,13 +29,16 @@ PLIST_SUB+=	X8664=""
 PLIST_SUB+=	X8664="@comment "
 .endif
 
+.if ${OPSYS} == FreeBSD
 CWARNFLAGS+=	-Wno-address-of-packed-member
+.endif
+
 .if ${COMPILER_TYPE} == clang && ${COMPILER_VERSION} >= 130
 CWARNFLAGS+=	-Wno-unused-but-set-variable
 .endif
 
 do-build:
-	cd ${WRKSRC}; ${SETENV} CC=${CC} AR=${AR} NM=${NM} OBJCOPY=${OBJCOPY} ./buildrump.sh -F CFLAGS="${CFLAGS}" -F CWARNFLAGS="${CWARNFLAGS}" -r fullbuild
+	cd ${WRKSRC}; ${SETENV} CC=${CC} AR=${AR} NM=${NM} OBJCOPY=${OBJCOPY} NOGCCERROR=yes ./buildrump.sh -F CFLAGS="${CFLAGS}" -F CWARNFLAGS="${CWARNFLAGS}" -r fullbuild
 
 do-install:
 	cd ${WRKSRC}/rump; ${PAX} -wr lib include ${STAGEDIR}${PREFIX}/
