--- pkg-plist.orig	2019-01-22 15:25:34 UTC
+++ pkg-plist
@@ -47,7 +47,7 @@ lib/libgnuefi.a
 %%ARCH_amd64%%include/efi/x86_64/efisetjmp_arch.h
 %%ARCH_amd64%%include/efi/x86_64/pe.h
 %%ARCH_amd64%%lib/crt0-efi-x86_64.o
-%%ARCH_amd64%%lib/elf_x86_64_fbsd_efi.lds
+%%ARCH_amd64%%lib/elf_x86_64_efi.lds
 @comment Architecture Specific Packages List, i386
 %%ARCH_i386%%include/efi/ia32/efibind.h
 %%ARCH_i386%%include/efi/ia32/efilibplat.h
