--- pkg-plist.orig	2022-09-01 20:31:34 UTC
+++ pkg-plist
@@ -22,15 +22,6 @@ lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/i
 lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/ilp32/crtn.o
 lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/ilp32/libgcc.a
 lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/ilp32/libgcov.a
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/README
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/limits.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/stddef.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/stdio.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/stdlib.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/sys/types.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/syslimits.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/unistd.h
-lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include-fixed/wchar.h
 lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include/arm_acle.h
 lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include/arm_bf16.h
 lib/gcc/%%GCC_TARGET%%/%%GCC_VERSION%%/include/arm_fp16.h
