--- files/Makefile.bsd.orig	2012-12-19 16:17:33.422123000 +0100
+++ files/Makefile.bsd	2012-12-19 16:36:49.458885000 +0100
@@ -31,12 +31,12 @@
 lib${LIB}.a ${SHLIB_NAME}: ${SRCS}
 	${MAKE} LIB=${LIB} SRCS="${SRCS}" \
 		SHLIB_MAJOR=${SHLIB_MAJOR} SHLIB_MINOR=${SHLIB_MINOR} \
-		CFLAGS="${CFLAGS}" -ECFLAGS LDADD="${LDADD}" \
+		CFLAGS="${CFLAGS}" LDADD="${LDADD}" \
 		-f ${MAKEDIR}/bsd.lib.mk ${.TARGET}
 
 ${PROGS} ${TESTS}: ${SHLIB_NAME} lib${LIB}.a
 	${MAKE} PROG=${.TARGET} NO_MAN=1 NOMAN=1 LDADD="-L${.CURDIR} -lgd" \
-		CFLAGS="${CFLAGS}" -ECFLAGS -f ${MAKEDIR}/bsd.prog.mk
+		CFLAGS="${CFLAGS}" -f ${MAKEDIR}/bsd.prog.mk
 
 install:
 	mkdir -p ${PREFIX}/include
