--- Uses/qt.mk
+++ Uses/qt.mk
@@ -109,7 +109,12 @@
 # better to support custom settings. Dereferencing the detection allows to
 # avoid forking a shell on each inclusion of this file, and to catch any CXX
 # customization (via USE_GCC, etc.).
-QMAKE_COMPILER=	$$(ccver="$$(${CXX} --version)"; case "$$ccver" in *clang*) echo clang ;; *) echo g++ ;; esac)
+QMAKE_COMPILER=	g++
+
+# Avoid having the compiler specified per port
+.  if ${_QT_VER:M6}
+USE_GCC_VERSION=	${GCC_DEFAULT}
+.  endif
 
 .  for dir in BIN INC LIB ARCH PLUGIN LIBEXEC IMPORT \
 	QML SBOM DATA DOC L10N ETC EXAMPLE TEST MKSPEC \
