--- pkg-plist.orig	2023-04-11 18:53:19 UTC
+++ pkg-plist
@@ -100,12 +100,12 @@ lib/critcl%%VER%%/critcl_c/tkstubs_nocon
 lib/critcl%%VER%%/license.terms
 lib/critcl%%VER%%/pkgIndex.tcl
 lib/critcl_callback1.1/critcl-rt.tcl
-lib/critcl_callback1.1/freebsd-%%ARCH%%/callback.so
+lib/critcl_callback1.1/dragonfly-%%ARCH%%/callback.so
 lib/critcl_callback1.1/license.terms
 lib/critcl_callback1.1/pkgIndex.tcl
 lib/critcl_callback1.1/teapot.txt
 lib/critcl_md5c0.12/critcl-rt.tcl
-lib/critcl_md5c0.12/freebsd-%%ARCH%%/md5c.so
+lib/critcl_md5c0.12/dragonfly-%%ARCH%%/md5c.so
 lib/critcl_md5c0.12/license.terms
 lib/critcl_md5c0.12/pkgIndex.tcl
 lib/critcl_md5c0.12/teapot.txt
