--- pkg-plist.orig	Tue Apr  1 18:57:59 2025
+++ pkg-plist	Tue Apr
@@ -78,12 +78,12 @@ lib/critcl-platform-1.1.1/platform.tcl
 lib/critcl-util-1.2.1/pkgIndex.tcl
 lib/critcl-util-1.2.1/util.tcl
 lib/critcl_callback%%TCL_SUFFIX%%-1.1/critcl-rt.tcl
-lib/critcl_callback%%TCL_SUFFIX%%-1.1/freebsd-%%ARCH%%/callback.so
+lib/critcl_callback%%TCL_SUFFIX%%-1.1/dragonfly-%%ARCH%%/callback.so
 lib/critcl_callback%%TCL_SUFFIX%%-1.1/license.terms
 lib/critcl_callback%%TCL_SUFFIX%%-1.1/pkgIndex.tcl
 lib/critcl_callback%%TCL_SUFFIX%%-1.1/teapot.txt
 lib/critcl_md5c%%TCL_SUFFIX%%-0.12/critcl-rt.tcl
-lib/critcl_md5c%%TCL_SUFFIX%%-0.12/freebsd-%%ARCH%%/md5c.so
+lib/critcl_md5c%%TCL_SUFFIX%%-0.12/dragonfly-%%ARCH%%/md5c.so
 lib/critcl_md5c%%TCL_SUFFIX%%-0.12/license.terms
 lib/critcl_md5c%%TCL_SUFFIX%%-0.12/pkgIndex.tcl
 lib/critcl_md5c%%TCL_SUFFIX%%-0.12/teapot.txt
