--- pkg-plist.orig	2023-10-12 19:24:32 UTC
+++ pkg-plist
@@ -70,6 +70,7 @@ libdata/pkgconfig/qtermwidget5.pc
 %%DATADIR%%5/translations/qtermwidget_lt.qm
 %%DATADIR%%5/translations/qtermwidget_nb_NO.qm
 %%DATADIR%%5/translations/qtermwidget_nl.qm
+%%DATADIR%%5/translations/qtermwidget_oc.qm
 %%DATADIR%%5/translations/qtermwidget_pl.qm
 %%DATADIR%%5/translations/qtermwidget_pt.qm
 %%DATADIR%%5/translations/qtermwidget_pt_BR.qm
