--- Makefile.orig	2024-09-23 15:09:47 UTC
+++ Makefile
@@ -17,7 +17,6 @@ INPUT_DRIVERS=		egalax \
 			evdev \
 			joystick \
 			keyboard \
-			libinput \
 			mouse \
 			synaptics \
 			void \
@@ -41,11 +40,11 @@ ${a:tu}_DESC=		Install ${a} ${type} driv
 . endfor
 .endfor
 
-OPTIONS_DEFINE_amd64=	AMDGPU ATI INTEL VMMOUSE VMWARE
-OPTIONS_DEFAULT_amd64=	VESA
+OPTIONS_DEFINE_x86_64=	AMDGPU ATI INTEL VMMOUSE VMWARE
+OPTIONS_DEFAULT_x86_64=	ATI INTEL VESA
 
-OPTIONS_DEFINE_i386:=	${OPTIONS_DEFINE_amd64} GEODE
-OPTIONS_DEFAULT_i386:=	${OPTIONS_DEFAULT_amd64}
+OPTIONS_DEFINE_i386:=	${OPTIONS_DEFINE_x86_64} GEODE
+OPTIONS_DEFAULT_i386:=	${OPTIONS_DEFAULT_x86_64}
 
 AMDGPU_DESC=		Install amdgpu video driver
 ATI_DESC=		Install ati (radeon) video driver
@@ -60,7 +59,7 @@ QUIRKS=		keyboard:kbd
 .include <bsd.port.options.mk>
 
 # Manual add arch specific drivers so they be added to depend lines.
-.if ${ARCH}==i386 || ${ARCH}==amd64
+.if ${ARCH}==i386 || ${ARCH}==x86_64
 INPUT_DRIVERS+= vmmouse
 VIDEO_DRIVERS+=	amdgpu ati intel vmware
 .endif
