--- Makefile.orig	2015-10-08 16:06:50 UTC
+++ Makefile
@@ -27,13 +27,10 @@ INPUT_DRIVERS=		mouse \
 			void \
 			wacom
 
-VIDEO_DRIVERS=		mach64 \
-			nv \
-			r128 \
+VIDEO_DRIVERS=		nv \
 			vesa \
 			apm \
 			ark \
-			chips \
 			cyrix \
 			dummy \
 			fbdev \
@@ -41,24 +38,26 @@ VIDEO_DRIVERS=		mach64 \
 			i128 \
 			i740 \
 			mga \
+			s3 \
+			tdfx \
+			trident \
+			tseng \
+			vmware
+
+VIDEO_DRIVERS_REMOVED=	mach64 \
+			r128 \
+			chips \
 			neomagic \
 			rendition \
-			s3 \
 			s3virge \
 			savage \
 			siliconmotion \
 			sis \
-			tdfx \
-			trident \
-			tseng \
-			vmware \
 			voodoo
 
 OPTIONS_DEFAULT=	MOUSE \
 			KEYBOARD \
-			MACH64 \
 			NV \
-			R128 \
 			VESA
 
 .for type in input video
@@ -71,8 +70,8 @@ ${a:tu}_DESC=	Install ${a} ${type} drive
 OPTIONS_DEFINE_i386=	ATI INTEL OPENCHROME
 OPTIONS_DEFAULT_i386=	ATI INTEL OPENCHROME
 
-OPTIONS_DEFINE_amd64:=	${OPTIONS_DEFINE_i386}
-OPTIONS_DEFAULT_amd64:=	${OPTIONS_DEFAULT_i386}
+OPTIONS_DEFINE_amd64:=	ATI INTEL29 OPENCHROME
+OPTIONS_DEFAULT_amd64:=	ATI INTEL29 OPENCHROME
 
 OPTIONS_DEFINE_sparc64=	SUNFFB
 OPTIONS_DEFAULT_sparc64=SUNFFB
@@ -81,20 +80,24 @@ OPTIONS_DEFINE_ia64=	OPENCHROME
 OPTIONS_DEFAULT_ia64=	OPENCHROME
 
 ATI_DESC=		Install ati video driver
-INTEL_DESC=		Install intel video driver
+INTEL29_DESC=		Install intel video driver
 OPENCHROME_DESC=	Install openchrome video driver
 SUNFFB_DESC=		Install sunffb video driver
 
 # these drivers have a different module name compared to the plugin they install
-QUIRKS=		keyboard:kbd
+QUIRKS=		keyboard:kbd intel29:intel
 
 .include <bsd.port.options.mk>
 
 # Manual add arch specific drivers so they be added to depend lines.
-.if ${ARCH}==i386 || ${ARCH}==amd64
+.if ${ARCH}==i386
 VIDEO_DRIVERS+=	ati intel openchrome
 .endif
 
+.if ${ARCH}==x86_64
+VIDEO_DRIVERS+=	ati intel29 openchrome
+.endif
+
 .if ${ARCH}==sparc64
 VIDEO_DRIVERS+=	sunffb
 .endif
