--- pkg-message.orig	2025-06-08 12:14:25 UTC
+++ pkg-message
@@ -1,17 +1,15 @@
 [
 { type: install
   message: <<EOM
-This OpenJDK implementation may require fdescfs(5) mounted on /dev/fd
-and procfs(5) mounted on /proc for some applications.
 
+This OpenJDK implementation requires procfs(5) mounted on /proc.
+
 If you have not done it yet, please do the following:
 
-	mount -t fdescfs fdesc /dev/fd
 	mount -t procfs proc /proc
 
 To make it permanent, you need the following lines in /etc/fstab:
 
-	fdesc	/dev/fd		fdescfs		rw	0	0
 	proc	/proc		procfs		rw	0	0
 EOM
 }
