--- pkg-plist.orig	2024-09-15 20:19:17 UTC
+++ pkg-plist
@@ -5,7 +5,6 @@ bin/%%GNU_HOST%%-gcc-%%GCC_VERSION%%
 bin/%%GNU_HOST%%-gcc-ar%%SUFFIX%%
 bin/%%GNU_HOST%%-gcc-nm%%SUFFIX%%
 bin/%%GNU_HOST%%-gcc-ranlib%%SUFFIX%%
-bin/%%GNU_HOST%%-gdc%%SUFFIX%%
 bin/%%GNU_HOST%%-gfortran%%SUFFIX%%
 bin/c++%%SUFFIX%%
 bin/cpp%%SUFFIX%%
@@ -17,7 +16,6 @@ bin/gcc-ranlib%%SUFFIX%%
 bin/gcov%%SUFFIX%%
 bin/gcov-dump%%SUFFIX%%
 bin/gcov-tool%%SUFFIX%%
-bin/gdc%%SUFFIX%%
 bin/gfortran%%SUFFIX%%
 bin/lto-dump%%SUFFIX%%
 share/man/man1/cpp%%SUFFIX%%.1.gz
@@ -26,7 +24,6 @@ share/man/man1/gcc%%SUFFIX%%.1.gz
 share/man/man1/gcov%%SUFFIX%%.1.gz
 share/man/man1/gcov-dump%%SUFFIX%%.1.gz
 share/man/man1/gcov-tool%%SUFFIX%%.1.gz
-share/man/man1/gdc%%SUFFIX%%.1.gz
 share/man/man1/gfortran%%SUFFIX%%.1.gz
 share/man/man1/lto-dump%%SUFFIX%%.1.gz
 share/gcc-%%GCC_VERSION%%/python/libstdcxx/__init__.py
