--- Makefile.orig	2018-11-22 03:31:42 UTC
+++ Makefile
@@ -21,16 +21,6 @@ USES=		desktop-file-utils qmake qt:5
 USE_QT=		qmake_build buildtools_build core declarative gui widgets \
 		graphicaleffects_run quickcontrols_run
 
-.include <bsd.port.pre.mk>
-
-.if ${ARCH:Mmips*} || ${ARCH:Mpowerpc*} || ${ARCH} == sparc64
-USE_GCC=	yes
-.else
-CC=		clang
-CXX=		clang++
-CPP=		clang-cpp
-.endif
-
 post-extract:
 	@${RMDIR} ${WRKSRC}/qmltermwidget
 	@${LN} -s ../qmltermwidget-0.1.0 ${WRKSRC}/qmltermwidget
