--- distinfo.orig	2013-01-05 21:49:59.856808000 +0100
+++ distinfo	2013-01-05 21:37:45.955783000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_nso.tar.gz) = 53f9d588ecaab357daf758bfe9497d7c1125184d40ea010bdd8f4d848c5efa37
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_nso.tar.gz) = 1647766
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_nso.tar.gz) = 9b2952dec7d7479948f79385355b698cd0abaf25ca2c3d8fd9c3f991b56a3d41
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_nso.tar.gz) = 1598169
