--- distinfo.orig	2013-01-05 21:50:00.686809000 +0100
+++ distinfo	2013-01-05 21:40:01.245972000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_gd.tar.gz) = edbcfa9d78bbdbcfc6dce5e285314821705c28e4fcb55bfa4903b7017683d031
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_gd.tar.gz) = 1964088
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_gd.tar.gz) = 9d64cb044dcca976d3118bd218570951b703c74a77b1ff366eeaa37f8ca7c896
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_gd.tar.gz) = 1972472
