--- pkg-plist.orig	2015-12-28 10:45:34 UTC
+++ pkg-plist
@@ -2256,10 +2256,10 @@ lib/python3.2/pickle.py
 lib/python3.2/pickletools.py
 lib/python3.2/pipes.py
 lib/python3.2/pkgutil.py
-lib/python3.2/plat-freebsd%%OSMAJOR%%/IN.py
-lib/python3.2/plat-freebsd%%OSMAJOR%%/__pycache__/IN.cpython-32.pyc
-lib/python3.2/plat-freebsd%%OSMAJOR%%/__pycache__/IN.cpython-32.pyo
-lib/python3.2/plat-freebsd%%OSMAJOR%%/regen
+lib/python3.2/plat-dragonfly%%OSMAJOR%%/IN.py
+lib/python3.2/plat-dragonfly%%OSMAJOR%%/__pycache__/IN.cpython-32.pyc
+lib/python3.2/plat-dragonfly%%OSMAJOR%%/__pycache__/IN.cpython-32.pyo
+lib/python3.2/plat-dragonfly%%OSMAJOR%%/regen
 lib/python3.2/platform.py
 lib/python3.2/plistlib.py
 lib/python3.2/poplib.py
