--- pkg-plist.orig	2014-08-02 13:54:18.816634000 +0000
+++ pkg-plist
@@ -2258,10 +2258,10 @@ lib/python3.2/pickle.py
 lib/python3.2/pickletools.py
 lib/python3.2/pipes.py
 lib/python3.2/pkgutil.py
-lib/python3.2/plat-freebsd%%OSMAJOR%%/__pycache__/IN.cpython-32.pyc
-lib/python3.2/plat-freebsd%%OSMAJOR%%/__pycache__/IN.cpython-32.pyo
-lib/python3.2/plat-freebsd%%OSMAJOR%%/IN.py
-lib/python3.2/plat-freebsd%%OSMAJOR%%/regen
+lib/python3.2/plat-dragonfly%%OSMAJOR%%/__pycache__/IN.cpython-32.pyc
+lib/python3.2/plat-dragonfly%%OSMAJOR%%/__pycache__/IN.cpython-32.pyo
+lib/python3.2/plat-dragonfly%%OSMAJOR%%/IN.py
+lib/python3.2/plat-dragonfly%%OSMAJOR%%/regen
 lib/python3.2/platform.py
 lib/python3.2/plistlib.py
 lib/python3.2/poplib.py
