--- pkg-plist.orig	2014-01-30 18:20:44.974982000 +0000
+++ pkg-plist
@@ -2259,6 +2259,10 @@ man/man1/python3.2.1.gz
 %%PYTHON_LIBDIR%%/pickletools.py
 %%PYTHON_LIBDIR%%/pipes.py
 %%PYTHON_LIBDIR%%/pkgutil.py
+%%PYTHON_LIBDIR%%/plat-dragonfly3/IN.py
+%%PYTHON_LIBDIR%%/plat-dragonfly3/__pycache__/IN.cpython-32.pyc
+%%PYTHON_LIBDIR%%/plat-dragonfly3/__pycache__/IN.cpython-32.pyo
+%%PYTHON_LIBDIR%%/plat-dragonfly3/regen
 %%PYTHON_LIBDIR%%/plat-freebsd4/IN.py
 %%PYTHON_LIBDIR%%/plat-freebsd4/__pycache__/IN.cpython-32.pyc
 %%PYTHON_LIBDIR%%/plat-freebsd4/__pycache__/IN.cpython-32.pyo
@@ -4364,6 +4368,8 @@ man/man1/python3.2.1.gz
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd10
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd11/__pycache__
 @dirrm %%PYTHON_LIBDIR%%/plat-freebsd11
+@dirrm %%PYTHON_LIBDIR%%/plat-dragonfly3/__pycache__
+@dirrm %%PYTHON_LIBDIR%%/plat-dragonfly3
 @dirrm %%PYTHON_LIBDIR%%/pydoc_data/__pycache__
 @dirrm %%PYTHON_LIBDIR%%/pydoc_data
 @dirrm %%PYTHON_LIBDIR%%/site-packages
