--- pkg-plist.orig	2022-07-12 22:20:05 UTC
+++ pkg-plist
@@ -15163,10 +15163,6 @@ lib/cmake/boost_math_c99f-%%BOOST_SHARED
 lib/cmake/boost_math_c99f-%%BOOST_SHARED_LIB_VER%%/boost_math_c99f-config.cmake
 lib/cmake/boost_math_c99f-%%BOOST_SHARED_LIB_VER%%/libboost_math_c99f-variant-shared.cmake
 lib/cmake/boost_math_c99f-%%BOOST_SHARED_LIB_VER%%/libboost_math_c99f-variant-static.cmake
-lib/cmake/boost_math_c99l-%%BOOST_SHARED_LIB_VER%%/boost_math_c99l-config-version.cmake
-lib/cmake/boost_math_c99l-%%BOOST_SHARED_LIB_VER%%/boost_math_c99l-config.cmake
-lib/cmake/boost_math_c99l-%%BOOST_SHARED_LIB_VER%%/libboost_math_c99l-variant-shared.cmake
-lib/cmake/boost_math_c99l-%%BOOST_SHARED_LIB_VER%%/libboost_math_c99l-variant-static.cmake
 lib/cmake/boost_math_tr1-%%BOOST_SHARED_LIB_VER%%/boost_math_tr1-config-version.cmake
 lib/cmake/boost_math_tr1-%%BOOST_SHARED_LIB_VER%%/boost_math_tr1-config.cmake
 lib/cmake/boost_math_tr1-%%BOOST_SHARED_LIB_VER%%/libboost_math_tr1-variant-shared.cmake
@@ -15175,10 +15171,6 @@ lib/cmake/boost_math_tr1f-%%BOOST_SHARED
 lib/cmake/boost_math_tr1f-%%BOOST_SHARED_LIB_VER%%/boost_math_tr1f-config.cmake
 lib/cmake/boost_math_tr1f-%%BOOST_SHARED_LIB_VER%%/libboost_math_tr1f-variant-shared.cmake
 lib/cmake/boost_math_tr1f-%%BOOST_SHARED_LIB_VER%%/libboost_math_tr1f-variant-static.cmake
-lib/cmake/boost_math_tr1l-%%BOOST_SHARED_LIB_VER%%/boost_math_tr1l-config-version.cmake
-lib/cmake/boost_math_tr1l-%%BOOST_SHARED_LIB_VER%%/boost_math_tr1l-config.cmake
-lib/cmake/boost_math_tr1l-%%BOOST_SHARED_LIB_VER%%/libboost_math_tr1l-variant-shared.cmake
-lib/cmake/boost_math_tr1l-%%BOOST_SHARED_LIB_VER%%/libboost_math_tr1l-variant-static.cmake
 lib/cmake/boost_mpi-%%BOOST_SHARED_LIB_VER%%/boost_mpi-config-version.cmake
 lib/cmake/boost_mpi-%%BOOST_SHARED_LIB_VER%%/boost_mpi-config.cmake
 lib/cmake/boost_nowide-%%BOOST_SHARED_LIB_VER%%/boost_nowide-config-version.cmake
@@ -15335,11 +15327,6 @@ lib/libboost_math_c99f.so
 lib/libboost_math_c99f.so.%%BOOST_MAJOR_VER%%
 lib/libboost_math_c99f.so.%%BOOST_MAJOR_VER%%.%%BOOST_MINOR_VER%%
 lib/libboost_math_c99f.so.%%BOOST_SHARED_LIB_VER%%
-lib/libboost_math_c99l.a
-lib/libboost_math_c99l.so
-lib/libboost_math_c99l.so.%%BOOST_MAJOR_VER%%
-lib/libboost_math_c99l.so.%%BOOST_MAJOR_VER%%.%%BOOST_MINOR_VER%%
-lib/libboost_math_c99l.so.%%BOOST_SHARED_LIB_VER%%
 lib/libboost_math_tr1.a
 lib/libboost_math_tr1.so
 lib/libboost_math_tr1.so.%%BOOST_MAJOR_VER%%
@@ -15350,11 +15337,6 @@ lib/libboost_math_tr1f.so
 lib/libboost_math_tr1f.so.%%BOOST_MAJOR_VER%%
 lib/libboost_math_tr1f.so.%%BOOST_MAJOR_VER%%.%%BOOST_MINOR_VER%%
 lib/libboost_math_tr1f.so.%%BOOST_SHARED_LIB_VER%%
-lib/libboost_math_tr1l.a
-lib/libboost_math_tr1l.so
-lib/libboost_math_tr1l.so.%%BOOST_MAJOR_VER%%
-lib/libboost_math_tr1l.so.%%BOOST_MAJOR_VER%%.%%BOOST_MINOR_VER%%
-lib/libboost_math_tr1l.so.%%BOOST_SHARED_LIB_VER%%
 lib/libboost_nowide.a
 lib/libboost_nowide.so
 lib/libboost_nowide.so.%%BOOST_MAJOR_VER%%
