--- pkg-plist.orig	Tue Apr  1 18:56:35 2025
+++ pkg-plist	Tue Apr
@@ -15960,10 +15960,6 @@ lib/cmake/boost_math_c99f-%%BOOST_SHARED_LIB_VER%%/boo
 lib/cmake/boost_math_c99f-%%BOOST_SHARED_LIB_VER%%/boost_math_c99f-config.cmake
 lib/cmake/boost_math_c99f-%%BOOST_SHARED_LIB_VER%%/libboost_math_c99f-variant-shared.cmake
 lib/cmake/boost_math_c99f-%%BOOST_SHARED_LIB_VER%%/libboost_math_c99f-variant-static.cmake
-lib/cmake/boost_math_c99l-%%BOOST_SHARED_LIB_VER%%/boost_math_c99l-config-version.cmake
-lib/cmake/boost_math_c99l-%%BOOST_SHARED_LIB_VER%%/boost_math_c99l-config.cmake
-lib/cmake/boost_math_c99l-%%BOOST_SHARED_LIB_VER%%/libboost_math_c99l-variant-shared.cmake
-lib/cmake/boost_math_c99l-%%BOOST_SHARED_LIB_VER%%/libboost_math_c99l-variant-static.cmake
 lib/cmake/boost_math_tr1-%%BOOST_SHARED_LIB_VER%%/boost_math_tr1-config-version.cmake
 lib/cmake/boost_math_tr1-%%BOOST_SHARED_LIB_VER%%/boost_math_tr1-config.cmake
 lib/cmake/boost_math_tr1-%%BOOST_SHARED_LIB_VER%%/libboost_math_tr1-variant-shared.cmake
@@ -15972,10 +15968,6 @@ lib/cmake/boost_math_tr1f-%%BOOST_SHARED_LIB_VER%%/boo
 lib/cmake/boost_math_tr1f-%%BOOST_SHARED_LIB_VER%%/boost_math_tr1f-config.cmake
 lib/cmake/boost_math_tr1f-%%BOOST_SHARED_LIB_VER%%/libboost_math_tr1f-variant-shared.cmake
 lib/cmake/boost_math_tr1f-%%BOOST_SHARED_LIB_VER%%/libboost_math_tr1f-variant-static.cmake
-lib/cmake/boost_math_tr1l-%%BOOST_SHARED_LIB_VER%%/boost_math_tr1l-config-version.cmake
-lib/cmake/boost_math_tr1l-%%BOOST_SHARED_LIB_VER%%/boost_math_tr1l-config.cmake
-lib/cmake/boost_math_tr1l-%%BOOST_SHARED_LIB_VER%%/libboost_math_tr1l-variant-shared.cmake
-lib/cmake/boost_math_tr1l-%%BOOST_SHARED_LIB_VER%%/libboost_math_tr1l-variant-static.cmake
 lib/cmake/boost_math-%%BOOST_SHARED_LIB_VER%%/boost_math-config-version.cmake
 lib/cmake/boost_math-%%BOOST_SHARED_LIB_VER%%/boost_math-config.cmake
 lib/cmake/boost_nowide-%%BOOST_SHARED_LIB_VER%%/boost_nowide-config-version.cmake
@@ -16149,11 +16141,6 @@ lib/libboost_math_c99f.so
 lib/libboost_math_c99f.so.%%BOOST_MAJOR_VER%%
 lib/libboost_math_c99f.so.%%BOOST_MAJOR_VER%%.%%BOOST_MINOR_VER%%
 lib/libboost_math_c99f.so.%%BOOST_SHARED_LIB_VER%%
-lib/libboost_math_c99l.a
-lib/libboost_math_c99l.so
-lib/libboost_math_c99l.so.%%BOOST_MAJOR_VER%%
-lib/libboost_math_c99l.so.%%BOOST_MAJOR_VER%%.%%BOOST_MINOR_VER%%
-lib/libboost_math_c99l.so.%%BOOST_SHARED_LIB_VER%%
 lib/libboost_math_tr1.a
 lib/libboost_math_tr1.so
 lib/libboost_math_tr1.so.%%BOOST_MAJOR_VER%%
@@ -16164,11 +16151,6 @@ lib/libboost_math_tr1f.so
 lib/libboost_math_tr1f.so.%%BOOST_MAJOR_VER%%
 lib/libboost_math_tr1f.so.%%BOOST_MAJOR_VER%%.%%BOOST_MINOR_VER%%
 lib/libboost_math_tr1f.so.%%BOOST_SHARED_LIB_VER%%
-lib/libboost_math_tr1l.a
-lib/libboost_math_tr1l.so
-lib/libboost_math_tr1l.so.%%BOOST_MAJOR_VER%%
-lib/libboost_math_tr1l.so.%%BOOST_MAJOR_VER%%.%%BOOST_MINOR_VER%%
-lib/libboost_math_tr1l.so.%%BOOST_SHARED_LIB_VER%%
 lib/libboost_nowide.a
 lib/libboost_nowide.so
 lib/libboost_nowide.so.%%BOOST_MAJOR_VER%%
