--- pkg-plist.orig	2020-07-28 22:39:56 UTC
+++ pkg-plist
@@ -3,19 +3,21 @@ bin/virt-admin
 bin/virt-host-validate
 bin/virt-pki-validate
 bin/virt-xml-validate
+%%QEMU%%bin/virt-qemu-run
 %%BHYVE%%sbin/virtbhyved
 sbin/virtnetworkd
 sbin/virtproxyd
 %%QEMU%%sbin/virtqemud
 sbin/virtsecretd
 sbin/virtstoraged
-sbin/virtvboxd
+%%VBOX%%sbin/virtvboxd
 %%XEN%%sbin/virtxend
 man/man1/virsh.1.gz
 man/man1/virt-admin.1.gz
 man/man1/virt-host-validate.1.gz
 man/man1/virt-pki-validate.1.gz
 man/man1/virt-xml-validate.1.gz
+%%QEMU%%man/man1/virt-qemu-run.1.gz
 man/man7/virkeycode-atset1.7.gz
 man/man7/virkeycode-atset2.7.gz
 man/man7/virkeycode-atset3.7.gz
@@ -42,7 +44,7 @@ man/man8/virtlogd.8.gz
 %%QEMU%%@sample %%ETCDIR%%/virtqemud.conf.sample
 @sample %%ETCDIR%%/virtsecretd.conf.sample
 @sample %%ETCDIR%%/virtstoraged.conf.sample
-@sample %%ETCDIR%%/virtvboxd.conf.sample
+%%VBOX%%@sample %%ETCDIR%%/virtvboxd.conf.sample
 %%XEN%%@sample %%ETCDIR%%/virtxend.conf.sample
 %%EXAMPLESDIR%%/networks/default.xml
 %%BHYVE%%@sample %%ETCDIR%%/bhyve.conf.sample
@@ -116,14 +118,14 @@ share/augeas/lenses/tests/test_virtproxy
 %%QEMU%%share/augeas/lenses/tests/test_virtqemud.aug
 share/augeas/lenses/tests/test_virtsecretd.aug
 share/augeas/lenses/tests/test_virtstoraged.aug
-share/augeas/lenses/tests/test_virtvboxd.aug
+%%VBOX%%share/augeas/lenses/tests/test_virtvboxd.aug
 %%BHYVE%%share/augeas/lenses/virtbhyved.aug
 share/augeas/lenses/virtnetworkd.aug
 share/augeas/lenses/virtproxyd.aug
 %%QEMU%%share/augeas/lenses/virtqemud.aug
 share/augeas/lenses/virtsecretd.aug
 share/augeas/lenses/virtstoraged.aug
-share/augeas/lenses/virtvboxd.aug
+%%VBOX%%share/augeas/lenses/virtvboxd.aug
 %%BHYVE%%share/augeas/lenses/tests/test_libvirtd_bhyve.aug
 %%QEMU%%share/augeas/lenses/tests/test_libvirtd_qemu.aug
 %%XEN%%share/augeas/lenses/libvirtd_libxl.aug
@@ -277,9 +279,10 @@ share/augeas/lenses/virtvboxd.aug
 lib/libvirt/connection-driver/libvirt_driver_network.so
 lib/libvirt/connection-driver/libvirt_driver_secret.so
 lib/libvirt/connection-driver/libvirt_driver_storage.so
-lib/libvirt/connection-driver/libvirt_driver_vbox.so
+%%VBOX%%lib/libvirt/connection-driver/libvirt_driver_vbox.so
 %%XEN%%lib/libvirt/connection-driver/libvirt_driver_libxl.so
 lib/libvirt/storage-backend/libvirt_storage_backend_fs.so
+lib/libvirt/storage-backend/libvirt_storage_backend_logical.so
 %%ZFS%%lib/libvirt/storage-backend/libvirt_storage_backend_zfs.so
 lib/libvirt/storage-file/libvirt_storage_file_fs.so
 %%QEMU%%@dir /var/cache/libvirt/qemu
