--- pkg-plist.orig	2016-12-31 16:23:50 UTC
+++ pkg-plist
@@ -148,9 +148,9 @@ lib/python%%XYDOT%%/__pycache__/_sitebui
 lib/python%%XYDOT%%/__pycache__/_strptime.cpython-%%XY%%.opt-1.pyc
 lib/python%%XYDOT%%/__pycache__/_strptime.cpython-%%XY%%.opt-2.pyc
 lib/python%%XYDOT%%/__pycache__/_strptime.cpython-%%XY%%.pyc
-lib/python%%XYDOT%%/__pycache__/_sysconfigdata_%%ABI%%_freebsd%%OSMAJOR%%_.cpython-%%XY%%.opt-1.pyc
-lib/python%%XYDOT%%/__pycache__/_sysconfigdata_%%ABI%%_freebsd%%OSMAJOR%%_.cpython-%%XY%%.opt-2.pyc
-lib/python%%XYDOT%%/__pycache__/_sysconfigdata_%%ABI%%_freebsd%%OSMAJOR%%_.cpython-%%XY%%.pyc
+lib/python%%XYDOT%%/__pycache__/_sysconfigdata_%%ABI%%_dragonfly%%OSMAJOR%%_.cpython-%%XY%%.opt-1.pyc
+lib/python%%XYDOT%%/__pycache__/_sysconfigdata_%%ABI%%_dragonfly%%OSMAJOR%%_.cpython-%%XY%%.opt-2.pyc
+lib/python%%XYDOT%%/__pycache__/_sysconfigdata_%%ABI%%_dragonfly%%OSMAJOR%%_.cpython-%%XY%%.pyc
 lib/python%%XYDOT%%/__pycache__/_threading_local.cpython-%%XY%%.opt-1.pyc
 lib/python%%XYDOT%%/__pycache__/_threading_local.cpython-%%XY%%.opt-2.pyc
 lib/python%%XYDOT%%/__pycache__/_threading_local.cpython-%%XY%%.pyc
@@ -630,7 +630,7 @@ lib/python%%XYDOT%%/_pydecimal.py
 lib/python%%XYDOT%%/_pyio.py
 lib/python%%XYDOT%%/_sitebuiltins.py
 lib/python%%XYDOT%%/_strptime.py
-lib/python%%XYDOT%%/_sysconfigdata_%%ABI%%_freebsd%%OSMAJOR%%_.py
+lib/python%%XYDOT%%/_sysconfigdata_%%ABI%%_dragonfly%%OSMAJOR%%_.py
 lib/python%%XYDOT%%/_threading_local.py
 lib/python%%XYDOT%%/_weakrefset.py
 lib/python%%XYDOT%%/abc.py
