--- distinfo.orig	2013-01-05 21:50:01.276810000 +0100
+++ distinfo	2013-01-05 21:41:46.096118000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_ml.tar.gz) = 4a3551a589b8e2c8f3d3196d45df48e56897b12fd09c7a6c34903b942e2f4ff3
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_ml.tar.gz) = 1715235
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_ml.tar.gz) = f237d888c54d1d17718f16b127d3bf9ddda0df979176644b8d8b98d67c75d6a8
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_ml.tar.gz) = 1673109
