--- pkg-plist.orig	2015-11-21 21:27:42 UTC
+++ pkg-plist
@@ -2175,7 +2175,7 @@ llvm-devel/lib/libLTO.so
 %%LLDB%%llvm-devel/lib/python2.7/site-packages/lldb/runtime/__init__.py
 %%LLDB%%llvm-devel/lib/python2.7/site-packages/lldb/utils/__init__.py
 %%LLDB%%llvm-devel/lib/python2.7/site-packages/lldb/utils/symbolication.py
-%%CLANG%%llvm-devel/lib/python2.7/site-packages/six.py
+%%LLDB%%llvm-devel/lib/python2.7/site-packages/six.py
 %%CLANG%%llvm-devel/libexec/c++-analyzer
 %%CLANG%%llvm-devel/libexec/ccc-analyzer
 %%CLANG%%llvm-devel/share/clang/clang-format-bbedit.applescript
