--- distinfo.orig	2013-01-05 21:50:00.616809000 +0100
+++ distinfo	2013-01-05 21:39:53.955962000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_or.tar.gz) = a47af1506c2bb9ad20a4dd915a668d2252e011911775ab311d642f4b4e085877
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_or.tar.gz) = 1694530
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_or.tar.gz) = 00c41cc8d56a1000beed63ffd55eace56f8230b4bad7c6cc5a08c1fa521ef2fe
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_or.tar.gz) = 1650582
