--- Makefile.orig	2012-12-09 23:09:14.000000000 +0100
+++ Makefile	2012-12-21 23:35:10.881658000 +0100
@@ -23,6 +23,7 @@
 LDFLAGS+=	-L${LOCALBASE}/lib
 DSOFLAGS=	-Wl,-rpath,${PREFIX}/lib -L${PREFIX}/lib ${LDFLAGS}
 CONFIGURE_ENV=	DSOFLAGS="${DSOFLAGS}"
+CONFIGURE_ENV+=	ac_cv_search_iconv_open=-liconv
 CONFIGURE_ARGS+=	--localstatedir=/var			\
 			--disable-slp				\
 			--disable-gssapi			\
@@ -225,7 +226,9 @@
 .else
 CONFIGURE_ARGS+=	--disable-libusb
 .endif
-.if ${OSVERSION} >= 800000
+
+DFLY_GCC!=	gcc -dumpversion
+.if ${DFLY_GCC} != "4.4.7"
 CONFIGURE_ARGS+=	LIBS="-lssp_nonshared"
 .endif
 
@@ -383,6 +386,7 @@
 	${INSTALL_DATA} ${WRKSRC}/cups/raster.h ${PREFIX}/include/cups/
 .else
 .if  ${PORT_OPTIONS:MGHOSTSCRIPT} || ${PORT_OPTIONS:MXPDF}
+	${INSTALL} -d ${PREFIX}/libexec/cups/filter
 	${INSTALL_PROGRAM} ${WRKSRC}/filter/pdftops ${PREFIX}/libexec/cups/filter/
 .endif
 	${LN} -s ${PREFIX}/bin/lpr ${PREFIX}/bin/lpr-cups
