--- /usr/ports/print/cups-base/Makefile	2013-03-07 04:25:10.000000000 +0100
+++ Makefile	2013-03-31 08:40:18.004987000 +0200
@@ -23,6 +23,7 @@
 LDFLAGS+=	-L${LOCALBASE}/lib
 DSOFLAGS=	-Wl,-rpath,${PREFIX}/lib -L${PREFIX}/lib ${LDFLAGS}
 CONFIGURE_ENV=	DSOFLAGS="${DSOFLAGS}"
+CONFIGURE_ENV+=	ac_cv_search_iconv_open=-liconv
 CONFIGURE_ARGS+=	--localstatedir=/var			\
 			--disable-slp				\
 			--disable-gssapi			\
@@ -41,7 +42,6 @@
 CUPS_CACHEDIR?=	/var/db/cups
 CUPS_SPOOLDIR=	/var/spool/cups
 CUPS_SOCKET?=	/var/run/cups.sock
-CUPS_PDFTOPS?=	${LOCALBASE}/libexec/xpdf/pdftops
 WRKSRC=		${WRKDIR}/${PORTNAME}-${DISTVERSION}
 
 # file, dir ownership
@@ -73,8 +73,6 @@
 .endif
 
 LIBUSB_DESC=		USB support
-GHOSTSCRIPT_DESC=	Build pdftops with GHOSTSCRIPT
-XPDF_DESC=		Build pdftops with XPDF
 XDG_OPEN=		Build with XDG_OPEN as browser
 
 .include <bsd.port.options.mk>
@@ -95,7 +93,6 @@
 # force build if old cups is installed.
 BUILD_DEPENDS+=	cups-client${PKGNAMESUFFIX2}>=${PORTVERSION}:${PORTSDIR}/${PKGCATEGORY}/cups-client
 COMMENT2=	Library cupsimage
-CONFIGURE_ARGS+=	--disable-pdftops
 INSTALL_WRKSRC=	${WRKSRC}/filter
 PLIST=		${MASTERDIR}/pkg-plist.image
 USE_LDCONFIG=	yes
@@ -144,22 +141,6 @@
 PLIST_SUB+=		PHP="@comment "
 .endif
 
-.if !defined(CUPS_CLIENT) && !defined(CUPS_IMAGE) && ${PORT_OPTIONS:MGHOSTSCRIPT}
-.if ${PORT_OPTIONS:MXPDF}
-IGNORE=			You must select one and only one option to build pdftops
-.endif
-USE_GHOSTSCRIPT=	yes
-CONFIGURE_ARGS+=	--enable-pdftops --with-pdftops=${LOCALBASE}/bin/gs
-PLIST_SUB+=		WITH_PDFTOPS=""
-.elif !defined(CUPS_CLIENT) && !defined(CUPS_IMAGE) && ${PORT_OPTIONS:MXPDF}
-BUILD_DEPENDS+=		${CUPS_PDFTOPS}:${PORTSDIR}/graphics/xpdf
-RUN_DEPENDS+=		${CUPS_PDFTOPS}:${PORTSDIR}/graphics/xpdf
-CONFIGURE_ARGS+=	--enable-pdftops --with-pdftops=${CUPS_PDFTOPS}
-PLIST_SUB+=		WITH_PDFTOPS=""
-.else
-PLIST_SUB+=		WITH_PDFTOPS="@comment "
-.endif
-
 .if !defined(CUPS_CLIENT) && !defined(CUPS_IMAGE) && ${PORT_OPTIONS:MPYTHON}
 CONFIGURE_ARGS+=	--with-python
 USE_PYTHON=		yes
@@ -225,9 +206,6 @@
 .else
 CONFIGURE_ARGS+=	--disable-libusb
 .endif
-.if ${OSVERSION} >= 800000
-CONFIGURE_ARGS+=	LIBS="-lssp_nonshared"
-.endif
 
 .if defined(CUPS_CLIENT)
 MAN1=	cups-config.1
@@ -287,7 +265,7 @@
 .endif
 
 post-patch:
-.if ${ARCH} == "amd64"
+.if ${ARCH} == "x86_64"
 	@${REINPLACE_CMD} -e 's|@PIEFLAGS@||' ${WRKSRC}/Makedefs.in
 .endif
 .if ${PORT_OPTIONS:MMDNSRESPONDER}
@@ -386,9 +364,8 @@
 .elif defined(CUPS_IMAGE)
 	${INSTALL_DATA} ${WRKSRC}/cups/raster.h ${PREFIX}/include/cups/
 .else
-.if  ${PORT_OPTIONS:MGHOSTSCRIPT} || ${PORT_OPTIONS:MXPDF}
+	${INSTALL} -d ${PREFIX}/libexec/cups/filter
 	${INSTALL_PROGRAM} ${WRKSRC}/filter/pdftops ${PREFIX}/libexec/cups/filter/
-.endif
 	${LN} -s ${PREFIX}/bin/lpr ${PREFIX}/bin/lpr-cups
 	${INSTALL} -d ${CUPS_ETCDIR}/
 	${CP} -p ${WRKSRC}/conf/snmp.conf ${WRKSRC}/conf/snmp.conf.N
