--- Makefile.orig	2013-02-05 17:54:25.000000000 +0100
+++ Makefile	2013-02-16 11:52:21.741910000 +0100
@@ -23,6 +23,7 @@
 LDFLAGS+=	-L${LOCALBASE}/lib
 DSOFLAGS=	-Wl,-rpath,${PREFIX}/lib -L${PREFIX}/lib ${LDFLAGS}
 CONFIGURE_ENV=	DSOFLAGS="${DSOFLAGS}"
+CONFIGURE_ENV+=	ac_cv_search_iconv_open=-liconv
 CONFIGURE_ARGS+=	--localstatedir=/var			\
 			--disable-slp				\
 			--disable-gssapi			\
@@ -225,9 +226,6 @@
 .else
 CONFIGURE_ARGS+=	--disable-libusb
 .endif
-.if ${OSVERSION} >= 800000
-CONFIGURE_ARGS+=	LIBS="-lssp_nonshared"
-.endif
 
 .if defined(CUPS_CLIENT)
 MAN1=	cups-config.1
@@ -387,6 +385,7 @@
 	${INSTALL_DATA} ${WRKSRC}/cups/raster.h ${PREFIX}/include/cups/
 .else
 .if  ${PORT_OPTIONS:MGHOSTSCRIPT} || ${PORT_OPTIONS:MXPDF}
+	${INSTALL} -d ${PREFIX}/libexec/cups/filter
 	${INSTALL_PROGRAM} ${WRKSRC}/filter/pdftops ${PREFIX}/libexec/cups/filter/
 .endif
 	${LN} -s ${PREFIX}/bin/lpr ${PREFIX}/bin/lpr-cups
