--- Makefile.orig	2014-06-19 23:22:53.910567000 +0000
+++ Makefile
@@ -19,11 +19,11 @@ CONFLICTS=	LPRng-[0-9]*
 
 UNIQUENAME=	${PORTNAME}${PKGNAMESUFFIX}
 
-USES+=		gmake pkgconfig tar:bzip2
+USES+=		gmake pkgconfig iconv tar:bzip2
 DESTDIRNAME=	DSTROOT
 GNU_CONFIGURE=	yes
 CPPFLAGS+=	-I${LOCALBASE}/include
-LDFLAGS+=	-L${LOCALBASE}/lib
+LDFLAGS+=	-L${LOCALBASE}/lib ${ICONV_LIB}
 DSOFLAGS=	-Wl,-rpath,${PREFIX}/lib ${LDFLAGS} ${LIBS}
 CONFIGURE_ENV=	DSOFLAGS="${DSOFLAGS}"
 CONFIGURE_ARGS+=	--localstatedir=/var			\
@@ -41,7 +41,6 @@ CONFIGURE_ARGS+=	--localstatedir=/var
 CUPS_CACHEDIR?=	/var/db/cups
 CUPS_SPOOLDIR=	/var/spool/cups
 CUPS_SOCKET?=	/var/run/cups.sock
-CUPS_PDFTOPS?=	${LOCALBASE}/libexec/xpdf/pdftops
 WRKSRC=		${WRKDIR}/${PORTNAME}-${DISTVERSION}
 
 CUPSSYSGRP=	wheel
@@ -74,12 +73,14 @@ XDG_OPEN_DESC=		Build with XDG_OPEN as b
 
 .include <bsd.port.options.mk>
 
+.if !empty(ICONV_LIB)
+CONFIGURE_ENV+=	ac_cv_search_iconv_open=-liconv
+.endif
+
 .if defined(CUPS_CLIENT)
 COMMENT2=	Library cups
 INSTALL_WRKSRC=	${WRKSRC}/cups
 PLIST=		${MASTERDIR}/pkg-plist.client
-USES+=		iconv
-LIBS+=		${ICONV_LIB}
 USE_LDCONFIG=	yes
 PKGMESSAGE=	${NONEXISTENT}
 DESCR=		${MASTERDIR}/pkg-descr.client
@@ -188,6 +189,7 @@ RUN_DEPENDS+=		xdg-open:${PORTSDIR}/deve
 .endif
 
 .if !defined(CUPS_CLIENT) && !defined(CUPS_IMAGE) && ${PORT_OPTIONS:MLIBUSB}
+USES+=			usb
 CONFIGURE_ARGS+=	--enable-libusb
 .else
 CONFIGURE_ARGS+=	--disable-libusb
@@ -214,7 +216,7 @@ post-patch:
 	@${REINPLACE_CMD} -e 's|@PIEFLAGS@||' ${WRKSRC}/Makedefs.in
 .endif
 .if ${PORT_OPTIONS:MMDNSRESPONDER}
-	@${REINPLACE_CMD} -e 's|Darwin|FreeBSD|' \
+	@${REINPLACE_CMD} -e 's|Darwin|${OPSYS}|' \
 		${WRKSRC}/backend/Makefile
 .endif
 .if  ${PORT_OPTIONS:MXDG_OPEN}
@@ -293,6 +295,7 @@ post-install:
 	${STRIP_CMD} ${STAGEDIR}${PREFIX}/lib/libcupsppdc.so.1
 	${STRIP_CMD} ${STAGEDIR}${PREFIX}/lib/libcupsmime.so.1
 .if  ${PORT_OPTIONS:MGHOSTSCRIPT} || ${PORT_OPTIONS:MXPDF}
+	${INSTALL} -d ${PREFIX}/libexec/cups/filter
 	${INSTALL_PROGRAM} ${WRKSRC}/filter/pdftops  ${STAGEDIR}${PREFIX}/libexec/cups/filter/
 .endif
 	${LN} -sf lpr ${STAGEDIR}${PREFIX}/bin/lpr-cups
