--- Makefile.orig	2013-10-14 09:25:08.597404000 +0000
+++ Makefile
@@ -44,7 +44,6 @@ CUPS_ETCDIR=	${PREFIX}/etc/cups
 CUPS_CACHEDIR?=	/var/db/cups
 CUPS_SPOOLDIR=	/var/spool/cups
 CUPS_SOCKET?=	/var/run/cups.sock
-CUPS_PDFTOPS?=	${LOCALBASE}/libexec/xpdf/pdftops
 WRKSRC=		${WRKDIR}/${PORTNAME}-${DISTVERSION}
 
 # file, dir ownership
@@ -72,6 +71,7 @@ PORTREVISION=	1
 CUPS_SUFFIX=	-base
 OPTIONS_DEFINE=	GNUTLS LIBPAPER PHP PYTHON PAM LDAP DBUS LIBUSB XDG_OPEN GHOSTSCRIPT XPDF AVAHI MDNSRESPONDER
 OPTIONS_DEFAULT=	LIBPAPER GHOSTSCRIPT
+OPTIONS_EXCLUDE+=	GHOSTSCRIPT XPDF
 NO_OPTIONS_SORT=	yes
 .endif
 
@@ -92,6 +92,9 @@ LDFLAGS+=	${ICONV_LIB}
 USE_LDCONFIG=	yes
 PKGMESSAGE=	${NONEXISTENT}
 DESCR=		${MASTERDIR}/pkg-descr.client
+.if !empty(ICONV_LIB)
+CONFIGURE_ENV+=	ac_cv_search_iconv_open=-liconv
+.endif
 .elif defined(CUPS_IMAGE)
 LIB_DEPENDS+=	cups:${PORTSDIR}/${PKGCATEGORY}/cups-client \
 		jpeg:${PORTSDIR}/graphics/jpeg \
@@ -100,7 +103,8 @@ LIB_DEPENDS+=	cups:${PORTSDIR}/${PKGCATE
 # force build if old cups is installed.
 BUILD_DEPENDS+=	cups-client${PKGNAMESUFFIX2}>=${PORTVERSION}:${PORTSDIR}/${PKGCATEGORY}/cups-client
 COMMENT2=	Library cupsimage
-CONFIGURE_ARGS+=	--disable-pdftops
+#CONFIGURE_ARGS+=	--disable-pdftops
+PLIST_SUB+=	WITH_PDFTOPS=""
 INSTALL_WRKSRC=	${WRKSRC}/filter
 PLIST=		${MASTERDIR}/pkg-plist.image
 USE_LDCONFIG=	yes
@@ -223,12 +227,11 @@ RUN_DEPENDS+=		xdg-open:${PORTSDIR}/deve
 .endif
 
 .if !defined(CUPS_CLIENT) && !defined(CUPS_IMAGE) && ${PORT_OPTIONS:MLIBUSB}
-CPPFLAGS+=		-I/usr/include
-LDFLAGS+=		-L/usr/lib
+USES+=			usb
 .else
 CONFIGURE_ARGS+=	--disable-libusb
 .endif
-CONFIGURE_ARGS+=	LIBS="-lssp_nonshared"
+#CONFIGURE_ARGS+=	LIBS="-lssp_nonshared"
 
 .if defined(CUPS_CLIENT)
 MAN1=	cups-config.1
@@ -387,9 +390,8 @@ post-install:
 .elif defined(CUPS_IMAGE)
 	${INSTALL_DATA} ${WRKSRC}/cups/raster.h ${PREFIX}/include/cups/
 .else
-.if  ${PORT_OPTIONS:MGHOSTSCRIPT} || ${PORT_OPTIONS:MXPDF}
+	${INSTALL} -d ${PREFIX}/libexec/cups/filter
 	${INSTALL_PROGRAM} ${WRKSRC}/filter/pdftops ${PREFIX}/libexec/cups/filter/
-.endif
 	${LN} -sf ${PREFIX}/bin/lpr ${PREFIX}/bin/lpr-cups
 	${INSTALL} -d ${CUPS_ETCDIR}/
 	${CP} -p ${WRKSRC}/conf/snmp.conf ${WRKSRC}/conf/snmp.conf.N
