--- Makefile.orig	2019-12-06 18:57:01 UTC
+++ Makefile
@@ -97,6 +97,7 @@ INFO=		as \
 .endif
 
 CONFIGURE_ARGS+=	--disable-werror \
+			--with-lib-path=/lib:/usr/lib \
 			--enable-deterministic-archives \
 			--with-sysroot=/ \
 			--enable-x86-relax-relocations=no
@@ -115,14 +116,6 @@ PLIST_SUB+=	GOLD=""
 PLIST_SUB+=	GOLD="@comment "
 .endif
 
-.if ${ARCH} == amd64
-CONFIGURE_TARGET=	x86_64-portbld-freebsd${OSREL}
-.elif ${ARCH} == powerpcspe
-CONFIGURE_TARGET=	powerpc-portbld-freebsd${OSREL}
-.else
-CONFIGURE_TARGET=	${ARCH}-portbld-freebsd${OSREL}
-.endif
-
 post-patch:
 	${REINPLACE_CMD} -e "s/-ldl//g" ${WRKSRC}/gold/testsuite/Makefile.in
 
