--- Makefile.orig	2026-08-31 00:00:00 UTC
+++ Makefile
@@ -31,6 +31,7 @@
 GNU_CONFIGURE=	yes
 
 CONFIGURE_ARGS+=	--disable-werror \
+			--with-lib-path=/lib:/usr/lib \
 			--enable-deterministic-archives \
 			--with-sysroot=/ \
 			--enable-x86-relax-relocations=no \
@@ -143,14 +144,6 @@
 CONFIGURE_ARGS+=	--enable-new-dtags
 .endif
 
-.if ${ARCH} == amd64
-CONFIGURE_TARGET=	x86_64-portbld-freebsd${OSREL}
-.elif ${ARCH} == powerpcspe
-CONFIGURE_TARGET=	powerpc-portbld-freebsd${OSREL}
-.else
-CONFIGURE_TARGET=	${ARCH}-portbld-freebsd${OSREL}
-.endif
-
 post-patch:
 	${REINPLACE_CMD} -e "s/-ldl//g" ${WRKSRC}/gold/testsuite/Makefile.in
 # Force the generation of man pages
