--- pkg-plist.orig	2013-10-20 10:53:06.703093000 +0000
+++ pkg-plist
@@ -26,7 +26,6 @@ GNUstep/Local/Library/Headers/Performanc
 GNUstep/Local/Library/Headers/Performance/GSThroughput.h
 GNUstep/Local/Library/Headers/Performance/GSTicker.h
 GNUstep/Local/Library/Libraries/libPerformance.so
-GNUstep/Local/Library/Libraries/libPerformance.so.0.4
 GNUstep/Local/Library/Libraries/libPerformance.so.0.4.0
 libdata/ldconfig/performance
 @dirrmtry GNUstep/Local/Library/Headers/Performance
