--- Makefile.orig	2014-02-10 09:16:47.961637000 +0000
+++ Makefile
@@ -11,17 +11,11 @@ COMMENT=	A multiwindow mouse-based text
 
 USE_XORG=	x11
 GNU_CONFIGURE=	yes
-CONFIGURE_ENV=	LIBS="-lgnuregex"
 CONFIGURE_ARGS=	--libdir=${PREFIX}/share
 MAKE_JOBS_UNSAFE=	yes
 
 PLIST_SUB=	VERSION="${PORTVERSION}"
 
-post-patch:
-	@${REINPLACE_CMD} -e 's| regex\.c | |g ; \
-		 s| -g | |g' ${WRKSRC}/Makefile.in
-	@${REINPLACE_CMD} -e \
-		's|"regex\.h"|<gnu/regex.h>|g' ${WRKSRC}/bm_search.c
 
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/xcoral ${STAGEDIR}${PREFIX}/bin
