--- Makefile.orig	2021-01-17 19:43:01 UTC
+++ Makefile
@@ -19,7 +19,6 @@ USES=		bison compiler:c11 xorg
 USE_XORG=	x11
 GNU_CONFIGURE=	yes
 CONFIGURE_ARGS=	--libdir=${PREFIX}/share
-LIBS+=		-lgnuregex
 LDFLAGS+=	-L${LOCALBASE}/lib
 CFLAGS+=	-isystem ${LOCALBASE}/include
 EXTRACT_AFTER_ARGS=	--exclude BR_Parser.[ch]
@@ -34,10 +33,6 @@ OPTIONS_DEFINE=	DOCS
 LIB_DEPENDS+=	libgnuregex.so.[6-9]:devel/libgnuregex
 .endif
 
-post-patch:
-	@${REINPLACE_CMD} -e 's| regex\.[co] | |g ; \
-		 s| -g | |g' ${WRKSRC}/Makefile.in
-
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/xcoral ${STAGEDIR}${PREFIX}/bin
 	${MKDIR} ${STAGEDIR}${DATADIR}
