--- Makefile.orig	2024-02-09 23:51:44 UTC
+++ Makefile
@@ -19,17 +19,12 @@ USES=		bison compiler:c11 localbase:ldfl
 USE_XORG=	x11
 GNU_CONFIGURE=	yes
 CONFIGURE_ARGS=	--libdir=${PREFIX}/share
-LIBS+=		-lgnuregex
 EXTRACT_AFTER_ARGS=	--exclude BR_Parser.[ch]
 
 PLIST_SUB=	VERSION="${PORTVERSION}"
 
 OPTIONS_DEFINE=	DOCS
 
-post-patch:
-	@${REINPLACE_CMD} -e 's| regex\.[co] | |g ; \
-		 s| -g | |g' ${WRKSRC}/Makefile.in
-
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/xcoral ${STAGEDIR}${PREFIX}/bin
 	${MKDIR} ${STAGEDIR}${DATADIR}
