--- distinfo.orig	2019-08-28 19:53:49 UTC
+++ distinfo
@@ -73,3 +73,7 @@ SHA256 (rust/crates/winapi-i686-pc-windo
 SIZE (rust/crates/winapi-i686-pc-windows-gnu-0.4.0.tar.gz) = 2918815
 SHA256 (rust/crates/winapi-x86_64-pc-windows-gnu-0.4.0.tar.gz) = 712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f
 SIZE (rust/crates/winapi-x86_64-pc-windows-gnu-0.4.0.tar.gz) = 2947998
+SHA256 (rust/crates/gcc-0.3.55.tar.gz) = 8f5f3913fa0bfe7ee1fd8248b6b9f42a5af4b9d65ec2dd2c3c26132b950ecfc2
+SIZE (rust/crates/gcc-0.3.55.tar.gz) = 37262
+SHA256 (rust/crates/errno-dragonfly-0.1.1.tar.gz) = 14ca354e36190500e1e1fb267c647932382b54053c50b14970856c0b00a35067
+SIZE (rust/crates/errno-dragonfly-0.1.1.tar.gz) = 1370
