--- pkg-plist.orig	2012-12-18 12:45:44.000000000 +0100
+++ pkg-plist	2013-01-05 12:52:41.451777000 +0100
@@ -2,8 +2,6 @@
 bin/cdr2xhtml
 bin/cmx2raw
 bin/cmx2xhtml
-include/libcdr-0.0
-include/libcdr-0.0/libcdr
 include/libcdr-0.0/libcdr/CDRDocument.h
 include/libcdr-0.0/libcdr/CDRStringVector.h
 include/libcdr-0.0/libcdr/CMXDocument.h
@@ -13,3 +11,5 @@
 lib/libcdr-0.0.so
 lib/libcdr-0.0.so.0
 libdata/pkgconfig/libcdr-0.0.pc
+@dirrm include/libcdr-0.0/libcdr
+@dirrm include/libcdr-0.0
