--- pkg-plist.orig	2023-04-11 19:17:31 UTC
+++ pkg-plist
@@ -267,12 +267,20 @@ libexec/singular/MOD/machinelearning.a
 libexec/singular/MOD/machinelearning.so
 libexec/singular/MOD/p_Procs_FieldGeneral.a
 libexec/singular/MOD/p_Procs_FieldGeneral.so
+libexec/singular/MOD/p_Procs_FieldGeneral.so.0
+libexec/singular/MOD/p_Procs_FieldGeneral.so.0.0.0
 libexec/singular/MOD/p_Procs_FieldIndep.a
 libexec/singular/MOD/p_Procs_FieldIndep.so
+libexec/singular/MOD/p_Procs_FieldIndep.so.0
+libexec/singular/MOD/p_Procs_FieldIndep.so.0.0.0
 libexec/singular/MOD/p_Procs_FieldQ.a
 libexec/singular/MOD/p_Procs_FieldQ.so
+libexec/singular/MOD/p_Procs_FieldQ.so.0
+libexec/singular/MOD/p_Procs_FieldQ.so.0.0.0
 libexec/singular/MOD/p_Procs_FieldZp.a
 libexec/singular/MOD/p_Procs_FieldZp.so
+libexec/singular/MOD/p_Procs_FieldZp.so.0
+libexec/singular/MOD/p_Procs_FieldZp.so.0.0.0
 libexec/singular/MOD/partialgb.a
 libexec/singular/MOD/partialgb.so
 libexec/singular/MOD/pyobject.a
