--- Makefile.orig	2018-09-05 03:04:18 UTC
+++ Makefile
@@ -20,6 +20,10 @@ GNU_CONFIGURE=	yes
 INSTALL_TARGET=	install-strip
 USE_LDCONFIG=	yes
 USES=		autoreconf libtool
+USES+=		ssl
+LDFLAGS+=	-L${OPENSSLLIB} -lcrypto
+
+FILESDIR=	${.CURDIR}/does-not-exist
 
 GH_ACCOUNT=	ArtifexSoftware
 USE_GITHUB=	yes
@@ -34,13 +38,6 @@ TESTS_DISTFILES=	jb2streams${EXTRACT_SUF
 TESTS_TEST_TARGET=	check-TESTS
 TESTS_USES=		python:build shebangfix
 
-post-patch:
-	@${RM} ${WRKSRC}/getopt* ${WRKSRC}/sha1.h
-	@${REINPLACE_CMD} -E 's|SHA1_Final\( *([^,]+), *([^\)]+)\)|SHA1_Final(\2, \1)|' ${WRKSRC}/jbig2dec.c ${WRKSRC}/sha1.c
-
-post-patch-TESTS-off:
-	@${REINPLACE_CMD} -e '/^TESTS = / s| test_jbig2dec.py||' ${WRKSRC}/Makefile.am
-
 pre-configure:
 	@cd ${WRKSRC}/ && ${SH} autogen.sh
 
