--- pkg-plist.orig	2015-09-18 14:47:59 UTC
+++ pkg-plist
@@ -28,6 +28,7 @@ bin/ksysguard
 bin/ksystraycmd
 bin/kwin
 %%GLES%%bin/kwin_gles
+bin/krited
 bin/oxygen-demo
 bin/oxygen-settings
 bin/oxygen-shadow-demo
@@ -190,7 +191,6 @@ lib/kde4/kded_kephal.so
 lib/kde4/kded_keyboard.so
 lib/kde4/kded_khotkeys.so
 lib/kde4/kded_ktouchpadenabler.so
-lib/kde4/kded_kwrited.so
 lib/kde4/kded_powerdevil.so
 lib/kde4/kded_randrmonitor.so
 lib/kde4/kded_statusnotifierwatcher.so
@@ -429,6 +429,7 @@ lib/libweather_ion.so
 lib/libweather_ion.so.6
 lib/libweather_ion.so.6.0.0
 lib/strigi/strigita_font.so
+share/autostart/kwrited-autostart.desktop
 share/applications/kde4/kdesystemsettings.desktop
 share/applications/kde4/kfontview.desktop
 share/applications/kde4/kinfocenter.desktop
@@ -1810,7 +1811,6 @@ share/kde4/services/kded/kephal.desktop
 share/kde4/services/kded/keyboard.desktop
 share/kde4/services/kded/khotkeys.desktop
 share/kde4/services/kded/ktouchpadenabler.desktop
-share/kde4/services/kded/kwrited.desktop
 share/kde4/services/kded/powerdevil.desktop
 share/kde4/services/kded/randrmonitor.desktop
 share/kde4/services/kded/statusnotifierwatcher.desktop
