--- distinfo.orig	2013-01-05 21:50:00.036808000 +0100
+++ distinfo	2013-01-05 21:38:12.755820000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_en-ZA.tar.gz) = 664942b9acb699f48c168db6f0828592625079aebd10a9024340ede82c40791c
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_en-ZA.tar.gz) = 8153320
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_en-ZA.tar.gz) = f5115903cdc063bf81ef6c5d674de5d3b6d4aef32f3a8ad4ab49281bae699943
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_en-ZA.tar.gz) = 9334546
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_en-ZA.tar.gz) = a9a38476984b0a90695834150920073b50085c8bd2b8f5561cef5715aad3ceb5
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_en-ZA.tar.gz) = 8108362
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_en-ZA.tar.gz) = e67c4bfbc3d682750cf9baf377c44ed25e1db3a71e2aa57e43b2dea2c81cce83
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_en-ZA.tar.gz) = 9339042
