--- distinfo.orig	2013-01-05 21:50:00.446809000 +0100
+++ distinfo	2013-01-05 21:39:18.525912000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_sd.tar.gz) = 9b9ec512a5d29a952a14c12f5f41f015a98735d1229846bcf3b8d0605766dfad
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_sd.tar.gz) = 1670543
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_sd.tar.gz) = 1a11240b629416139e607766d260641cb2a2386ad2a0574217c26f2299be353e
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_sd.tar.gz) = 1624195
