--- distinfo.orig	2013-01-05 21:50:00.866809000 +0100
+++ distinfo	2013-01-05 21:40:21.606000000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_lv.tar.gz) = 06f2079b6b650740da84ed147ce1ab349ee15dc2115f71a6c6fe770fe9ee146c
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_lv.tar.gz) = 2393177
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_lv.tar.gz) = 81107fe3f806bb8e58a6141ba01522167da4f32b16e93cb98765795e1ac2e1c4
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_lv.tar.gz) = 2344503
