--- distinfo.orig	2013-01-05 22:07:27.158271000 +0100
+++ distinfo	2013-01-05 22:06:55.958227000 +0100
@@ -1,4 +1,4 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_pt.tar.gz) = 3e902d22b45002d3a16d44ff61d87a87317699030c2cac5514cb0b163b1ef173
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_pt.tar.gz) = 2205648
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_pt.tar.gz) = 21e4d07fe15c172a27c9ebcacb45e8d14df425bf48b8154dcd6d280e0b8e4e4a
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_helppack-deb_pt.tar.gz) = 10756380
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_pt.tar.gz) = 363fefc972dc350370f8324f7ba2c1aa381f3048b2393ada4722ff436cf6c82b
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_pt.tar.gz) = 2159680
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_pt.tar.gz) = b033f108cabcd2998e5a3355e33b117158703e13033128edb7f49d95b2dad808
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_helppack-deb_pt.tar.gz) = 10763221
