--- pkg-plist.orig	2016-08-19 15:40:06 UTC
+++ pkg-plist
@@ -30,8 +30,6 @@ lib/cinnamon-settings-daemon-3.0/libhous
 lib/cinnamon-settings-daemon-3.0/libhousekeeping.so
 lib/cinnamon-settings-daemon-3.0/libkeyboard.a
 lib/cinnamon-settings-daemon-3.0/libkeyboard.so
-lib/cinnamon-settings-daemon-3.0/libmedia-keys.a
-lib/cinnamon-settings-daemon-3.0/libmedia-keys.so
 lib/cinnamon-settings-daemon-3.0/libmouse.a
 lib/cinnamon-settings-daemon-3.0/libmouse.so
 lib/cinnamon-settings-daemon-3.0/libpower.a
@@ -42,19 +40,15 @@ lib/cinnamon-settings-daemon-3.0/libscre
 lib/cinnamon-settings-daemon-3.0/libscreensaver-proxy.so
 %%SMARTCARD%%lib/cinnamon-settings-daemon-3.0/libsmartcard.a
 %%SMARTCARD%%lib/cinnamon-settings-daemon-3.0/libsmartcard.so
-lib/cinnamon-settings-daemon-3.0/libsound.a
-lib/cinnamon-settings-daemon-3.0/libsound.so
 lib/cinnamon-settings-daemon-3.0/libxrandr.a
 lib/cinnamon-settings-daemon-3.0/libxrandr.so
 lib/cinnamon-settings-daemon-3.0/libxsettings.a
 lib/cinnamon-settings-daemon-3.0/libxsettings.so
-lib/cinnamon-settings-daemon-3.0/media-keys.cinnamon-settings-plugin
 lib/cinnamon-settings-daemon-3.0/mouse.cinnamon-settings-plugin
 lib/cinnamon-settings-daemon-3.0/power.cinnamon-settings-plugin
 %%CUPS%%lib/cinnamon-settings-daemon-3.0/print-notifications.cinnamon-settings-plugin
 lib/cinnamon-settings-daemon-3.0/screensaver-proxy.cinnamon-settings-plugin
 %%SMARTCARD%%lib/cinnamon-settings-daemon-3.0/smartcard.cinnamon-settings-plugin
-lib/cinnamon-settings-daemon-3.0/sound.cinnamon-settings-plugin
 lib/cinnamon-settings-daemon-3.0/xrandr.cinnamon-settings-plugin
 lib/cinnamon-settings-daemon-3.0/xsettings.cinnamon-settings-plugin
 libdata/pkgconfig/cinnamon-settings-daemon.pc
@@ -67,13 +61,11 @@ libexec/csd-test-a11y-keyboard
 libexec/csd-test-a11y-settings
 libexec/csd-test-background
 libexec/csd-test-input-helper
-libexec/csd-test-media-keys
 libexec/csd-test-mouse
 libexec/csd-test-power
 %%CUPS%%libexec/csd-test-print-notifications
 libexec/csd-test-screensaver-proxy
 %%SMARTCARD%%libexec/csd-test-smartcard
-libexec/csd-test-sound
 libexec/csd-test-xsettings
 man/man1/cinnamon-settings-daemon.1.gz
 share/applications/cinnamon-settings-daemon.desktop
