--- distinfo.orig	2013-01-05 22:07:26.608270000 +0100
+++ distinfo	2013-01-05 22:06:10.318164000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_ar.tar.gz) = e4235aae79f06eebbbfb5fd595465dd811b1b6c8173793e1e9ad19d03fbceb9a
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_ar.tar.gz) = 3398329
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_ar.tar.gz) = 3fa6dd1382f8a5462b0924c3ab6098feb5cdd9beb1ce32db631df5ad0ef01e7b
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_ar.tar.gz) = 3350015
