--- distinfo.orig	2013-01-05 21:50:01.396810000 +0100
+++ distinfo	2013-01-05 21:42:10.046152000 +0100
@@ -1,2 +1,2 @@
-SHA256 (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_lt.tar.gz) = 94dee4fc1fae6fede9ac48974fd3f19665c2b63003bd15e46054579d805b5e05
-SIZE (libreoffice/i18n/LibO_3.5.7_Linux_x86_langpack-deb_lt.tar.gz) = 2012485
+SHA256 (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_lt.tar.gz) = ebb407617f3057b51ea403f3b1cc59c8dfab6709dcde593ae3315da0bc3ad4da
+SIZE (libreoffice/i18n/LibO_3.6.4_Linux_x86_langpack-deb_lt.tar.gz) = 1961462
